{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Untitled0.ipynb",
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "2de69b90e7f243809da7b5acd13e45bf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_90b76275fa9840dc97b91904f763b057",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_26091ac6b4be4a21851474bd3138a567",
              "IPY_MODEL_f00eaec31d0145ab88572031f2f415e3",
              "IPY_MODEL_967fa4f219884e1a9f22291285c4ab48"
            ]
          }
        },
        "90b76275fa9840dc97b91904f763b057": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "26091ac6b4be4a21851474bd3138a567": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_72ba7e2b27ab45ce9cd86c301ebbcd16",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": "Downloading: 100%",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_364b2883f6504ddfafe1e97fe727c532"
          }
        },
        "f00eaec31d0145ab88572031f2f415e3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_8f791efaf30144bbab36d69793a385d1",
            "_dom_classes": [],
            "description": "",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 1197,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 1197,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_363863b9cf7341728155364bc60fa548"
          }
        },
        "967fa4f219884e1a9f22291285c4ab48": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_72cf8f1169f1482795a34c54aa54b714",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 1.20k/1.20k [00:00&lt;00:00, 35.4kB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_1f2b253bb7774cb6b83cb284295bb71a"
          }
        },
        "72ba7e2b27ab45ce9cd86c301ebbcd16": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "364b2883f6504ddfafe1e97fe727c532": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "8f791efaf30144bbab36d69793a385d1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "363863b9cf7341728155364bc60fa548": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "72cf8f1169f1482795a34c54aa54b714": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "1f2b253bb7774cb6b83cb284295bb71a": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "b6773000e0624414be62c5142e155a6e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_56700317de2e4dfd973a85cfaaa2a65f",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_462d770a48694fa9b7eaa706df4e7653",
              "IPY_MODEL_0a02e7475bb04caca20b8fb86b8d8a69",
              "IPY_MODEL_c0867c825231497bafefc4c15a9a7352"
            ]
          }
        },
        "56700317de2e4dfd973a85cfaaa2a65f": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "462d770a48694fa9b7eaa706df4e7653": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_e772a734fa6340daa0d2d66f1de3d80d",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": "Downloading:  43%",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_ce33bdc7f68f45a9a1145733830e7a40"
          }
        },
        "0a02e7475bb04caca20b8fb86b8d8a69": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_504ab56e5f074b38b444b39dab1265af",
            "_dom_classes": [],
            "description": "",
            "_model_name": "FloatProgressModel",
            "bar_style": "",
            "max": 242065649,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 104516608,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_a289d7713a6c4fa8aed49c6b5f57acb9"
          }
        },
        "c0867c825231497bafefc4c15a9a7352": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_996172a784f141cfa025ed4a8782153f",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 105M/242M [00:19&lt;00:05, 23.3MB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_dc1878fd97ce471cb7af600d7c296674"
          }
        },
        "e772a734fa6340daa0d2d66f1de3d80d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "ce33bdc7f68f45a9a1145733830e7a40": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "504ab56e5f074b38b444b39dab1265af": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "a289d7713a6c4fa8aed49c6b5f57acb9": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "996172a784f141cfa025ed4a8782153f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "dc1878fd97ce471cb7af600d7c296674": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "d32080a6580c4ea398d1bc1d96b1afc0": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_969ecbd87b22438a9e4c7caba7cce2e9",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_312c0e164ed44fc68796c2c42c00739b",
              "IPY_MODEL_5f64044bc9784ee5ac8f89a1e55d66c2",
              "IPY_MODEL_ef300e4fbffb472388351be84b864ca3"
            ]
          }
        },
        "969ecbd87b22438a9e4c7caba7cce2e9": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "312c0e164ed44fc68796c2c42c00739b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_3e7b56e425ac46a8a90665d470ba560c",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": "Downloading: 100%",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_fda549c0c69041a08a55b7159ae1bcc2"
          }
        },
        "5f64044bc9784ee5ac8f89a1e55d66c2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_58a9cd2ed3bb458985c9272d06a1a34f",
            "_dom_classes": [],
            "description": "",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 791656,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 791656,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_4833b72c2d794421abff6a70dbb1db00"
          }
        },
        "ef300e4fbffb472388351be84b864ca3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_45c446fe7b2a4fd89c4a28054d3885db",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 792k/792k [00:00&lt;00:00, 2.12MB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_7f3430a9f4764594b634817825d73a0e"
          }
        },
        "3e7b56e425ac46a8a90665d470ba560c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "fda549c0c69041a08a55b7159ae1bcc2": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "58a9cd2ed3bb458985c9272d06a1a34f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "4833b72c2d794421abff6a70dbb1db00": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "45c446fe7b2a4fd89c4a28054d3885db": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "7f3430a9f4764594b634817825d73a0e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "7b4995c7b6fc46f0be8b94160d6c05ec": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_76bd40872c984fb2ba76167f3bc0d2e4",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_847f4d8e817f4271b38ce5e0ef651b38",
              "IPY_MODEL_328012e7262440efa79e3995c68aa007",
              "IPY_MODEL_084f670d569a4fefb3efe23195103ccf"
            ]
          }
        },
        "76bd40872c984fb2ba76167f3bc0d2e4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "847f4d8e817f4271b38ce5e0ef651b38": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_9f4b247f4a304400a5de85a50a76f8cc",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": "Downloading: 100%",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_59fbf2f35f7849e6a7d2ca0ddf006da5"
          }
        },
        "328012e7262440efa79e3995c68aa007": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_459971060eca4aec95697a8aa4770ec3",
            "_dom_classes": [],
            "description": "",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 1208,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 1208,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_bce914497da740fc8e2aa187f3055bfd"
          }
        },
        "084f670d569a4fefb3efe23195103ccf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_523bec92145c4c6aaba2c8a459285eb8",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 1.21k/1.21k [00:00&lt;00:00, 21.8kB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_8b780a15640d41dfb74cde32118eab68"
          }
        },
        "9f4b247f4a304400a5de85a50a76f8cc": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "59fbf2f35f7849e6a7d2ca0ddf006da5": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "459971060eca4aec95697a8aa4770ec3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "bce914497da740fc8e2aa187f3055bfd": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "523bec92145c4c6aaba2c8a459285eb8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "8b780a15640d41dfb74cde32118eab68": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "a2a372f5d68f472e85961cffaa43871a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HBoxView",
            "_dom_classes": [],
            "_model_name": "HBoxModel",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "box_style": "",
            "layout": "IPY_MODEL_956186e628b94fc4b284eb0fb5b77266",
            "_model_module": "@jupyter-widgets/controls",
            "children": [
              "IPY_MODEL_68fa6f6936684a328fbbd94971364912",
              "IPY_MODEL_7218a364fbc04955b9a9c9bb9d228638",
              "IPY_MODEL_ce64ca612f9a40c8a6e174b3e616a36b"
            ]
          }
        },
        "956186e628b94fc4b284eb0fb5b77266": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "68fa6f6936684a328fbbd94971364912": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_86ffc8df091a49e997d6468023e2fd46",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": "Downloading: 100%",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_c0a9c0b4e29a4bc8857f67a80a7f833e"
          }
        },
        "7218a364fbc04955b9a9c9bb9d228638": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "ProgressView",
            "style": "IPY_MODEL_86c0919aa9254fc2b2f785cb3ee9312d",
            "_dom_classes": [],
            "description": "",
            "_model_name": "FloatProgressModel",
            "bar_style": "success",
            "max": 891691413,
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": 891691413,
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "orientation": "horizontal",
            "min": 0,
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_7a20de4590634881b19778b40b39b483"
          }
        },
        "ce64ca612f9a40c8a6e174b3e616a36b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "HTMLView",
            "style": "IPY_MODEL_f8bef635edd44941ad7f3578e0991b4a",
            "_dom_classes": [],
            "description": "",
            "_model_name": "HTMLModel",
            "placeholder": "​",
            "_view_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "value": " 892M/892M [00:59&lt;00:00, 14.9MB/s]",
            "_view_count": null,
            "_view_module_version": "1.5.0",
            "description_tooltip": null,
            "_model_module": "@jupyter-widgets/controls",
            "layout": "IPY_MODEL_a2dc661c88bd484fa624cc763f8a2c11"
          }
        },
        "86ffc8df091a49e997d6468023e2fd46": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "c0a9c0b4e29a4bc8857f67a80a7f833e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "86c0919aa9254fc2b2f785cb3ee9312d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "ProgressStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "bar_color": null,
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "7a20de4590634881b19778b40b39b483": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        },
        "f8bef635edd44941ad7f3578e0991b4a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_view_name": "StyleView",
            "_model_name": "DescriptionStyleModel",
            "description_width": "",
            "_view_module": "@jupyter-widgets/base",
            "_model_module_version": "1.5.0",
            "_view_count": null,
            "_view_module_version": "1.2.0",
            "_model_module": "@jupyter-widgets/controls"
          }
        },
        "a2dc661c88bd484fa624cc763f8a2c11": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_view_name": "LayoutView",
            "grid_template_rows": null,
            "right": null,
            "justify_content": null,
            "_view_module": "@jupyter-widgets/base",
            "overflow": null,
            "_model_module_version": "1.2.0",
            "_view_count": null,
            "flex_flow": null,
            "width": null,
            "min_width": null,
            "border": null,
            "align_items": null,
            "bottom": null,
            "_model_module": "@jupyter-widgets/base",
            "top": null,
            "grid_column": null,
            "overflow_y": null,
            "overflow_x": null,
            "grid_auto_flow": null,
            "grid_area": null,
            "grid_template_columns": null,
            "flex": null,
            "_model_name": "LayoutModel",
            "justify_items": null,
            "grid_row": null,
            "max_height": null,
            "align_content": null,
            "visibility": null,
            "align_self": null,
            "height": null,
            "min_height": null,
            "padding": null,
            "grid_auto_rows": null,
            "grid_gap": null,
            "max_width": null,
            "order": null,
            "_view_module_version": "1.2.0",
            "grid_template_areas": null,
            "object_position": null,
            "object_fit": null,
            "grid_auto_columns": null,
            "margin": null,
            "display": null,
            "left": null
          }
        }
      }
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "YNxVdkEPU1_v",
        "outputId": "8b13de9d-eb4e-4a47-808c-e17b413cf251"
      },
      "source": [
        "!pip install pdfplumber --quiet\n",
        "!pip install earthy\n",
        "!pip install git+https://github.com/ramsrigouthamg/Questgen.ai\n",
        "!pip install sense2vec==1.0.2\n",
        "!pip install git+https://github.com/boudinfl/pke.git\n",
        "!python -m nltk.downloader universal_tagset\n",
        "!python -m spacy download en\n",
        "!wget https://github.com/explosion/sense2vec/releases/download/v1.0.0/s2v_reddit_2015_md.tar.gz\n",
        "!tar -xvf  s2v_reddit_2015_md.tar.gz\n",
        "!ls s2v_old"
      ],
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[K     |████████████████████████████████| 45 kB 1.4 MB/s \n",
            "\u001b[K     |████████████████████████████████| 5.6 MB 9.7 MB/s \n",
            "\u001b[K     |████████████████████████████████| 139 kB 43.7 MB/s \n",
            "\u001b[K     |████████████████████████████████| 1.9 MB 19.1 MB/s \n",
            "\u001b[?25h  Building wheel for pdfplumber (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting earthy\n",
            "  Downloading earthy-0.0.12.tar.gz (46 kB)\n",
            "\u001b[K     |████████████████████████████████| 46 kB 2.0 MB/s \n",
            "\u001b[?25hRequirement already satisfied: gensim in /usr/local/lib/python3.7/dist-packages (from earthy) (3.6.0)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.7/dist-packages (from earthy) (0.22.2.post1)\n",
            "Requirement already satisfied: nltk in /usr/local/lib/python3.7/dist-packages (from earthy) (3.2.5)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from earthy) (1.15.0)\n",
            "Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.7/dist-packages (from earthy) (4.6.3)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from earthy) (2.23.0)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from earthy) (1.19.5)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.7/dist-packages (from earthy) (1.1.5)\n",
            "Collecting charguana\n",
            "  Downloading charguana-0.1.15.tar.gz (157 kB)\n",
            "\u001b[K     |████████████████████████████████| 157 kB 8.7 MB/s \n",
            "\u001b[?25hRequirement already satisfied: scipy>=0.18.1 in /usr/local/lib/python3.7/dist-packages (from gensim->earthy) (1.4.1)\n",
            "Requirement already satisfied: smart-open>=1.2.1 in /usr/local/lib/python3.7/dist-packages (from gensim->earthy) (5.2.1)\n",
            "Requirement already satisfied: pytz>=2017.2 in /usr/local/lib/python3.7/dist-packages (from pandas->earthy) (2018.9)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.7/dist-packages (from pandas->earthy) (2.8.2)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->earthy) (1.24.3)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->earthy) (2021.5.30)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->earthy) (3.0.4)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->earthy) (2.10)\n",
            "Requirement already satisfied: joblib>=0.11 in /usr/local/lib/python3.7/dist-packages (from scikit-learn->earthy) (1.0.1)\n",
            "Building wheels for collected packages: earthy, charguana\n",
            "  Building wheel for earthy (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for earthy: filename=earthy-0.0.12-py3-none-any.whl size=46975 sha256=fd14379c750a02f3861212f521384576d89ab421fc951f5db5f32528658954d6\n",
            "  Stored in directory: /root/.cache/pip/wheels/e8/57/c3/c7aca92949eedb50f3e5c97bae137920c69057ecb5318295bd\n",
            "  Building wheel for charguana (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for charguana: filename=charguana-0.1.15-py3-none-any.whl size=187890 sha256=82a1897652c825cd878da3c5340f7fdc68f70ba1cb15f38b46cffe0b2b4aabf3\n",
            "  Stored in directory: /root/.cache/pip/wheels/31/4b/79/803d79c08c25911181fdfa969fc61d06375dfba663a839bfa6\n",
            "Successfully built earthy charguana\n",
            "Installing collected packages: charguana, earthy\n",
            "Successfully installed charguana-0.1.15 earthy-0.0.12\n",
            "Collecting git+https://github.com/ramsrigouthamg/Questgen.ai\n",
            "  Cloning https://github.com/ramsrigouthamg/Questgen.ai to /tmp/pip-req-build-sx2f4k_f\n",
            "  Running command git clone -q https://github.com/ramsrigouthamg/Questgen.ai /tmp/pip-req-build-sx2f4k_f\n",
            "Requirement already satisfied: torch==1.9.0 in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (1.9.0+cu111)\n",
            "Collecting transformers==3.0.2\n",
            "  Downloading transformers-3.0.2-py3-none-any.whl (769 kB)\n",
            "\u001b[K     |████████████████████████████████| 769 kB 5.1 MB/s \n",
            "\u001b[?25hCollecting pytorch_lightning==0.8.1\n",
            "  Downloading pytorch_lightning-0.8.1-py3-none-any.whl (293 kB)\n",
            "\u001b[K     |████████████████████████████████| 293 kB 57.7 MB/s \n",
            "\u001b[?25hCollecting sense2vec==1.0.3\n",
            "  Downloading sense2vec-1.0.3-py2.py3-none-any.whl (35 kB)\n",
            "Collecting strsim==0.0.3\n",
            "  Downloading strsim-0.0.3-py3-none-any.whl (42 kB)\n",
            "\u001b[K     |████████████████████████████████| 42 kB 1.1 MB/s \n",
            "\u001b[?25hRequirement already satisfied: six==1.15.0 in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (1.15.0)\n",
            "Collecting networkx==2.4.0\n",
            "  Downloading networkx-2.4-py3-none-any.whl (1.6 MB)\n",
            "\u001b[K     |████████████████████████████████| 1.6 MB 43.7 MB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (1.19.5)\n",
            "Requirement already satisfied: scipy in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (1.4.1)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (0.22.2.post1)\n",
            "Collecting unidecode==1.1.1\n",
            "  Downloading Unidecode-1.1.1-py2.py3-none-any.whl (238 kB)\n",
            "\u001b[K     |████████████████████████████████| 238 kB 62.5 MB/s \n",
            "\u001b[?25hCollecting future==0.18.2\n",
            "  Downloading future-0.18.2.tar.gz (829 kB)\n",
            "\u001b[K     |████████████████████████████████| 829 kB 60.6 MB/s \n",
            "\u001b[?25hCollecting joblib==0.14.1\n",
            "  Downloading joblib-0.14.1-py2.py3-none-any.whl (294 kB)\n",
            "\u001b[K     |████████████████████████████████| 294 kB 54.0 MB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy==2.2.4 in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (2.2.4)\n",
            "Collecting pytz==2020.1\n",
            "  Downloading pytz-2020.1-py2.py3-none-any.whl (510 kB)\n",
            "\u001b[K     |████████████████████████████████| 510 kB 60.4 MB/s \n",
            "\u001b[?25hCollecting python-dateutil==2.8.1\n",
            "  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)\n",
            "\u001b[K     |████████████████████████████████| 227 kB 63.0 MB/s \n",
            "\u001b[?25hCollecting boto3==1.14.40\n",
            "  Downloading boto3-1.14.40-py2.py3-none-any.whl (129 kB)\n",
            "\u001b[K     |████████████████████████████████| 129 kB 60.1 MB/s \n",
            "\u001b[?25hCollecting flashtext==2.7\n",
            "  Downloading flashtext-2.7.tar.gz (14 kB)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.7/dist-packages (from Questgen==1.0.0) (1.1.5)\n",
            "Collecting botocore<1.18.0,>=1.17.40\n",
            "  Downloading botocore-1.17.63-py2.py3-none-any.whl (6.6 MB)\n",
            "\u001b[K     |████████████████████████████████| 6.6 MB 19.5 MB/s \n",
            "\u001b[?25hCollecting s3transfer<0.4.0,>=0.3.0\n",
            "  Downloading s3transfer-0.3.7-py2.py3-none-any.whl (73 kB)\n",
            "\u001b[K     |████████████████████████████████| 73 kB 2.2 MB/s \n",
            "\u001b[?25hCollecting jmespath<1.0.0,>=0.7.1\n",
            "  Downloading jmespath-0.10.0-py2.py3-none-any.whl (24 kB)\n",
            "Requirement already satisfied: decorator>=4.3.0 in /usr/local/lib/python3.7/dist-packages (from networkx==2.4.0->Questgen==1.0.0) (4.4.2)\n",
            "Collecting PyYAML>=5.1\n",
            "  Downloading PyYAML-6.0-cp37-cp37m-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (596 kB)\n",
            "\u001b[K     |████████████████████████████████| 596 kB 74.6 MB/s \n",
            "\u001b[?25hRequirement already satisfied: tqdm>=4.41.0 in /usr/local/lib/python3.7/dist-packages (from pytorch_lightning==0.8.1->Questgen==1.0.0) (4.62.3)\n",
            "Requirement already satisfied: tensorboard>=1.14 in /usr/local/lib/python3.7/dist-packages (from pytorch_lightning==0.8.1->Questgen==1.0.0) (2.6.0)\n",
            "Requirement already satisfied: catalogue>=0.0.4 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.3->Questgen==1.0.0) (1.0.0)\n",
            "Requirement already satisfied: srsly>=0.2.0 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.3->Questgen==1.0.0) (1.0.5)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.3->Questgen==1.0.0) (0.8.2)\n",
            "Requirement already satisfied: importlib-metadata>=0.20 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.3->Questgen==1.0.0) (4.8.1)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (3.0.5)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (2.23.0)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (7.4.0)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (57.4.0)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (0.4.1)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (2.0.5)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (1.0.5)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.7/dist-packages (from spacy==2.2.4->Questgen==1.0.0) (1.1.3)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/dist-packages (from torch==1.9.0->Questgen==1.0.0) (3.7.4.3)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.7/dist-packages (from transformers==3.0.2->Questgen==1.0.0) (3.3.0)\n",
            "Collecting sacremoses\n",
            "  Downloading sacremoses-0.0.46-py3-none-any.whl (895 kB)\n",
            "\u001b[K     |████████████████████████████████| 895 kB 32.9 MB/s \n",
            "\u001b[?25hCollecting sentencepiece!=0.1.92\n",
            "  Downloading sentencepiece-0.1.96-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.2 MB)\n",
            "\u001b[K     |████████████████████████████████| 1.2 MB 43.9 MB/s \n",
            "\u001b[?25hRequirement already satisfied: packaging in /usr/local/lib/python3.7/dist-packages (from transformers==3.0.2->Questgen==1.0.0) (21.0)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.7/dist-packages (from transformers==3.0.2->Questgen==1.0.0) (2019.12.20)\n",
            "Collecting tokenizers==0.8.1.rc1\n",
            "  Downloading tokenizers-0.8.1rc1-cp37-cp37m-manylinux1_x86_64.whl (3.0 MB)\n",
            "\u001b[K     |████████████████████████████████| 3.0 MB 32.0 MB/s \n",
            "\u001b[?25hRequirement already satisfied: urllib3<1.26,>=1.20 in /usr/local/lib/python3.7/dist-packages (from botocore<1.18.0,>=1.17.40->boto3==1.14.40->Questgen==1.0.0) (1.24.3)\n",
            "Collecting docutils<0.16,>=0.10\n",
            "  Downloading docutils-0.15.2-py3-none-any.whl (547 kB)\n",
            "\u001b[K     |████████████████████████████████| 547 kB 80.8 MB/s \n",
            "\u001b[?25hRequirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata>=0.20->sense2vec==1.0.3->Questgen==1.0.0) (3.6.0)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy==2.2.4->Questgen==1.0.0) (3.0.4)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy==2.2.4->Questgen==1.0.0) (2021.5.30)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy==2.2.4->Questgen==1.0.0) (2.10)\n",
            "Requirement already satisfied: absl-py>=0.4 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (0.12.0)\n",
            "Requirement already satisfied: tensorboard-plugin-wit>=1.6.0 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (1.8.0)\n",
            "Requirement already satisfied: werkzeug>=0.11.15 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (1.0.1)\n",
            "Requirement already satisfied: grpcio>=1.24.3 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (1.41.0)\n",
            "Requirement already satisfied: google-auth<2,>=1.6.3 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (1.35.0)\n",
            "Requirement already satisfied: markdown>=2.6.8 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (3.3.4)\n",
            "Requirement already satisfied: wheel>=0.26 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (0.37.0)\n",
            "Requirement already satisfied: google-auth-oauthlib<0.5,>=0.4.1 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (0.4.6)\n",
            "Requirement already satisfied: tensorboard-data-server<0.7.0,>=0.6.0 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (0.6.1)\n",
            "Requirement already satisfied: protobuf>=3.6.0 in /usr/local/lib/python3.7/dist-packages (from tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (3.17.3)\n",
            "Requirement already satisfied: pyasn1-modules>=0.2.1 in /usr/local/lib/python3.7/dist-packages (from google-auth<2,>=1.6.3->tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (0.2.8)\n",
            "Requirement already satisfied: rsa<5,>=3.1.4 in /usr/local/lib/python3.7/dist-packages (from google-auth<2,>=1.6.3->tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (4.7.2)\n",
            "Requirement already satisfied: cachetools<5.0,>=2.0.0 in /usr/local/lib/python3.7/dist-packages (from google-auth<2,>=1.6.3->tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (4.2.4)\n",
            "Requirement already satisfied: requests-oauthlib>=0.7.0 in /usr/local/lib/python3.7/dist-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (1.3.0)\n",
            "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /usr/local/lib/python3.7/dist-packages (from pyasn1-modules>=0.2.1->google-auth<2,>=1.6.3->tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (0.4.8)\n",
            "Requirement already satisfied: oauthlib>=3.0.0 in /usr/local/lib/python3.7/dist-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard>=1.14->pytorch_lightning==0.8.1->Questgen==1.0.0) (3.1.1)\n",
            "Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from packaging->transformers==3.0.2->Questgen==1.0.0) (2.4.7)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==3.0.2->Questgen==1.0.0) (7.1.2)\n",
            "Building wheels for collected packages: Questgen, flashtext, future\n",
            "  Building wheel for Questgen (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for Questgen: filename=Questgen-1.0.0-py3-none-any.whl size=8650 sha256=84c8b63bae95ae82407e0955f326afaccc755fc25531db244bfbc6f03782cd09\n",
            "  Stored in directory: /tmp/pip-ephem-wheel-cache-5ohdykni/wheels/40/0a/eb/4da53249b8be090573b5775ae929efbbf2b2b58fc3b9f14cec\n",
            "  Building wheel for flashtext (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for flashtext: filename=flashtext-2.7-py2.py3-none-any.whl size=9310 sha256=43c67942537099e1de12daae166cbbc862257a05a9e36f76d71b1a3895c7c764\n",
            "  Stored in directory: /root/.cache/pip/wheels/cb/19/58/4e8fdd0009a7f89dbce3c18fff2e0d0fa201d5cdfd16f113b7\n",
            "  Building wheel for future (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for future: filename=future-0.18.2-py3-none-any.whl size=491070 sha256=e534805a21d5f7e9419dbccf671e651794bf45f81815082b3663373ab0898dea\n",
            "  Stored in directory: /root/.cache/pip/wheels/56/b0/fe/4410d17b32f1f0c3cf54cdfb2bc04d7b4b8f4ae377e2229ba0\n",
            "Successfully built Questgen flashtext future\n",
            "Installing collected packages: python-dateutil, jmespath, docutils, joblib, botocore, tokenizers, sentencepiece, sacremoses, s3transfer, PyYAML, pytz, future, unidecode, transformers, strsim, sense2vec, pytorch-lightning, networkx, flashtext, boto3, Questgen\n",
            "  Attempting uninstall: python-dateutil\n",
            "    Found existing installation: python-dateutil 2.8.2\n",
            "    Uninstalling python-dateutil-2.8.2:\n",
            "      Successfully uninstalled python-dateutil-2.8.2\n",
            "  Attempting uninstall: docutils\n",
            "    Found existing installation: docutils 0.17.1\n",
            "    Uninstalling docutils-0.17.1:\n",
            "      Successfully uninstalled docutils-0.17.1\n",
            "  Attempting uninstall: joblib\n",
            "    Found existing installation: joblib 1.0.1\n",
            "    Uninstalling joblib-1.0.1:\n",
            "      Successfully uninstalled joblib-1.0.1\n",
            "  Attempting uninstall: PyYAML\n",
            "    Found existing installation: PyYAML 3.13\n",
            "    Uninstalling PyYAML-3.13:\n",
            "      Successfully uninstalled PyYAML-3.13\n",
            "  Attempting uninstall: pytz\n",
            "    Found existing installation: pytz 2018.9\n",
            "    Uninstalling pytz-2018.9:\n",
            "      Successfully uninstalled pytz-2018.9\n",
            "  Attempting uninstall: future\n",
            "    Found existing installation: future 0.16.0\n",
            "    Uninstalling future-0.16.0:\n",
            "      Successfully uninstalled future-0.16.0\n",
            "  Attempting uninstall: networkx\n",
            "    Found existing installation: networkx 2.6.3\n",
            "    Uninstalling networkx-2.6.3:\n",
            "      Successfully uninstalled networkx-2.6.3\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "datascience 0.10.6 requires folium==0.2.1, but you have folium 0.8.3 which is incompatible.\n",
            "albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.2.9 which is incompatible.\u001b[0m\n",
            "Successfully installed PyYAML-6.0 Questgen-1.0.0 boto3-1.14.40 botocore-1.17.63 docutils-0.15.2 flashtext-2.7 future-0.18.2 jmespath-0.10.0 joblib-0.14.1 networkx-2.4 python-dateutil-2.8.1 pytorch-lightning-0.8.1 pytz-2020.1 s3transfer-0.3.7 sacremoses-0.0.46 sense2vec-1.0.3 sentencepiece-0.1.96 strsim-0.0.3 tokenizers-0.8.1rc1 transformers-3.0.2 unidecode-1.1.1\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "dateutil",
                  "pytz"
                ]
              }
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting sense2vec==1.0.2\n",
            "  Downloading sense2vec-1.0.2.tar.gz (54 kB)\n",
            "\u001b[?25l\r\u001b[K     |██████                          | 10 kB 21.6 MB/s eta 0:00:01\r\u001b[K     |████████████                    | 20 kB 25.2 MB/s eta 0:00:01\r\u001b[K     |██████████████████              | 30 kB 12.2 MB/s eta 0:00:01\r\u001b[K     |████████████████████████        | 40 kB 9.5 MB/s eta 0:00:01\r\u001b[K     |██████████████████████████████  | 51 kB 5.0 MB/s eta 0:00:01\r\u001b[K     |████████████████████████████████| 54 kB 2.0 MB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy<3.0.0,>=2.2.3 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.2) (2.2.4)\n",
            "Requirement already satisfied: srsly>=0.2.0 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.2) (1.0.5)\n",
            "Requirement already satisfied: catalogue>=0.0.4 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.2) (1.0.0)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.2) (0.8.2)\n",
            "Requirement already satisfied: numpy>=1.15.0 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.2) (1.19.5)\n",
            "Requirement already satisfied: importlib_metadata>=0.20 in /usr/local/lib/python3.7/dist-packages (from sense2vec==1.0.2) (4.8.1)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib_metadata>=0.20->sense2vec==1.0.2) (3.6.0)\n",
            "Requirement already satisfied: typing-extensions>=3.6.4 in /usr/local/lib/python3.7/dist-packages (from importlib_metadata>=0.20->sense2vec==1.0.2) (3.7.4.3)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (2.0.5)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (2.23.0)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (4.62.3)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (3.0.5)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (57.4.0)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (7.4.0)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (1.1.3)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (0.4.1)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.7/dist-packages (from spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (1.0.5)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (3.0.4)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (2.10)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (2021.5.30)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.0.0,>=2.2.3->sense2vec==1.0.2) (1.24.3)\n",
            "Building wheels for collected packages: sense2vec\n",
            "  Building wheel for sense2vec (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for sense2vec: filename=sense2vec-1.0.2-py2.py3-none-any.whl size=35011 sha256=ff1f23cbdc38d79a456afc52da1636520dc838340f83549eabb368e830d837b1\n",
            "  Stored in directory: /root/.cache/pip/wheels/cf/d3/93/fe8e871b410c5456a7b06be0f154ad6bab298462471551f39d\n",
            "Successfully built sense2vec\n",
            "Installing collected packages: sense2vec\n",
            "  Attempting uninstall: sense2vec\n",
            "    Found existing installation: sense2vec 1.0.3\n",
            "    Uninstalling sense2vec-1.0.3:\n",
            "      Successfully uninstalled sense2vec-1.0.3\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "questgen 1.0.0 requires sense2vec==1.0.3, but you have sense2vec 1.0.2 which is incompatible.\u001b[0m\n",
            "Successfully installed sense2vec-1.0.2\n",
            "Collecting git+https://github.com/boudinfl/pke.git\n",
            "  Cloning https://github.com/boudinfl/pke.git to /tmp/pip-req-build-83xqa99w\n",
            "  Running command git clone -q https://github.com/boudinfl/pke.git /tmp/pip-req-build-83xqa99w\n",
            "Requirement already satisfied: nltk in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (3.2.5)\n",
            "Requirement already satisfied: networkx in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (2.4)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (1.19.5)\n",
            "Requirement already satisfied: scipy in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (1.4.1)\n",
            "Requirement already satisfied: spacy in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (2.2.4)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (1.15.0)\n",
            "Requirement already satisfied: sklearn in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (0.0)\n",
            "Requirement already satisfied: unidecode in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (1.1.1)\n",
            "Requirement already satisfied: future in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (0.18.2)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.7/dist-packages (from pke==1.8.1) (0.14.1)\n",
            "Requirement already satisfied: decorator>=4.3.0 in /usr/local/lib/python3.7/dist-packages (from networkx->pke==1.8.1) (4.4.2)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.7/dist-packages (from sklearn->pke==1.8.1) (0.22.2.post1)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (2.0.5)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (1.0.5)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (1.0.0)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (1.0.5)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (0.8.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (57.4.0)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (2.23.0)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (3.0.5)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (4.62.3)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (1.1.3)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (7.4.0)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy->pke==1.8.1) (0.4.1)\n",
            "Requirement already satisfied: importlib-metadata>=0.20 in /usr/local/lib/python3.7/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy->pke==1.8.1) (4.8.1)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata>=0.20->catalogue<1.1.0,>=0.0.7->spacy->pke==1.8.1) (3.6.0)\n",
            "Requirement already satisfied: typing-extensions>=3.6.4 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata>=0.20->catalogue<1.1.0,>=0.0.7->spacy->pke==1.8.1) (3.7.4.3)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy->pke==1.8.1) (2.10)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy->pke==1.8.1) (2021.5.30)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy->pke==1.8.1) (1.24.3)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy->pke==1.8.1) (3.0.4)\n",
            "Building wheels for collected packages: pke\n",
            "  Building wheel for pke (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for pke: filename=pke-1.8.1-py3-none-any.whl size=8764035 sha256=0e83f0f258251198777ee6d3b6e6bdf69a5b840c7c52e05d73c9dff1f77f6593\n",
            "  Stored in directory: /tmp/pip-ephem-wheel-cache-i1t1kzdj/wheels/fa/b3/09/612ee93bf3ee4164bcd5783e742942cdfc892a86039d3e0a33\n",
            "Successfully built pke\n",
            "Installing collected packages: pke\n",
            "Successfully installed pke-1.8.1\n",
            "/usr/lib/python3.7/runpy.py:125: RuntimeWarning: 'nltk.downloader' found in sys.modules after import of package 'nltk', but prior to execution of 'nltk.downloader'; this may result in unpredictable behaviour\n",
            "  warn(RuntimeWarning(msg))\n",
            "[nltk_data] Downloading package universal_tagset to /root/nltk_data...\n",
            "[nltk_data]   Unzipping taggers/universal_tagset.zip.\n",
            "Collecting en_core_web_sm==2.2.5\n",
            "  Downloading https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.5/en_core_web_sm-2.2.5.tar.gz (12.0 MB)\n",
            "\u001b[K     |████████████████████████████████| 12.0 MB 5.2 MB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy>=2.2.2 in /usr/local/lib/python3.7/dist-packages (from en_core_web_sm==2.2.5) (2.2.4)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (1.0.5)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (1.1.3)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (2.0.5)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (57.4.0)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (0.4.1)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (2.23.0)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (3.0.5)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (1.0.0)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (0.8.2)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (1.0.5)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (4.62.3)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (7.4.0)\n",
            "Requirement already satisfied: numpy>=1.15.0 in /usr/local/lib/python3.7/dist-packages (from spacy>=2.2.2->en_core_web_sm==2.2.5) (1.19.5)\n",
            "Requirement already satisfied: importlib-metadata>=0.20 in /usr/local/lib/python3.7/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.2->en_core_web_sm==2.2.5) (4.8.1)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata>=0.20->catalogue<1.1.0,>=0.0.7->spacy>=2.2.2->en_core_web_sm==2.2.5) (3.6.0)\n",
            "Requirement already satisfied: typing-extensions>=3.6.4 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata>=0.20->catalogue<1.1.0,>=0.0.7->spacy>=2.2.2->en_core_web_sm==2.2.5) (3.7.4.3)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.2->en_core_web_sm==2.2.5) (2.10)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.2->en_core_web_sm==2.2.5) (2021.5.30)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.2->en_core_web_sm==2.2.5) (1.24.3)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.2->en_core_web_sm==2.2.5) (3.0.4)\n",
            "\u001b[38;5;2m✔ Download and installation successful\u001b[0m\n",
            "You can now load the model via spacy.load('en_core_web_sm')\n",
            "\u001b[38;5;2m✔ Linking successful\u001b[0m\n",
            "/usr/local/lib/python3.7/dist-packages/en_core_web_sm -->\n",
            "/usr/local/lib/python3.7/dist-packages/spacy/data/en\n",
            "You can now load the model via spacy.load('en')\n",
            "--2021-10-17 14:07:49--  https://github.com/explosion/sense2vec/releases/download/v1.0.0/s2v_reddit_2015_md.tar.gz\n",
            "Resolving github.com (github.com)... 140.82.112.4\n",
            "Connecting to github.com (github.com)|140.82.112.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://github-releases.githubusercontent.com/50261113/52126080-0993-11ea-8190-8f0e295df22a?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20211017%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20211017T140750Z&X-Amz-Expires=300&X-Amz-Signature=1b8fa433adc8cb187c67938b4f617c4484f8c172e874cdc629ec857eaca82d55&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=50261113&response-content-disposition=attachment%3B%20filename%3Ds2v_reddit_2015_md.tar.gz&response-content-type=application%2Foctet-stream [following]\n",
            "--2021-10-17 14:07:50--  https://github-releases.githubusercontent.com/50261113/52126080-0993-11ea-8190-8f0e295df22a?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20211017%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20211017T140750Z&X-Amz-Expires=300&X-Amz-Signature=1b8fa433adc8cb187c67938b4f617c4484f8c172e874cdc629ec857eaca82d55&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=50261113&response-content-disposition=attachment%3B%20filename%3Ds2v_reddit_2015_md.tar.gz&response-content-type=application%2Foctet-stream\n",
            "Resolving github-releases.githubusercontent.com (github-releases.githubusercontent.com)... 185.199.108.154, 185.199.109.154, 185.199.110.154, ...\n",
            "Connecting to github-releases.githubusercontent.com (github-releases.githubusercontent.com)|185.199.108.154|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 600444501 (573M) [application/octet-stream]\n",
            "Saving to: ‘s2v_reddit_2015_md.tar.gz’\n",
            "\n",
            "s2v_reddit_2015_md. 100%[===================>] 572.63M  98.4MB/s    in 6.0s    \n",
            "\n",
            "2021-10-17 14:07:56 (95.7 MB/s) - ‘s2v_reddit_2015_md.tar.gz’ saved [600444501/600444501]\n",
            "\n",
            "./._s2v_old\n",
            "./s2v_old/\n",
            "./s2v_old/._freqs.json\n",
            "./s2v_old/freqs.json\n",
            "./s2v_old/._vectors\n",
            "./s2v_old/vectors\n",
            "./s2v_old/._cfg\n",
            "./s2v_old/cfg\n",
            "./s2v_old/._strings.json\n",
            "./s2v_old/strings.json\n",
            "./s2v_old/._key2row\n",
            "./s2v_old/key2row\n",
            "cfg  freqs.json  key2row  strings.json\tvectors\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Vn6FIywhFg1Q",
        "outputId": "992dffa4-d48e-4b3f-e3e3-716dca1ef6ec"
      },
      "source": [
        "import pdfplumber\n",
        "from nltk.stem import PorterStemmer\n",
        "from nltk.stem import LancasterStemmer\n",
        "from nltk.tokenize import sent_tokenize, word_tokenize\n",
        "import nltk\n",
        "import gensim\n",
        "from gensim.summarization import summarize\n",
        "from nltk.stem import WordNetLemmatizer \n",
        "import re\n",
        "from pprint import pprint\n",
        "nltk.download('stopwords')\n",
        "from Questgen import main\n",
        "from transformers import GPT2Tokenizer,GPT2LMHeadModel\n",
        "from gensim.parsing.preprocessing import remove_stopwords\n",
        "from collections import Counter\n",
        "#https://stackabuse.com/removing-stop-words-from-strings-in-python/\n",
        "nltk.download('wordnet')\n",
        "nltk.download('punkt')\n",
        "nltk.download('averaged_perceptron_tagger')\n",
        "pdf = pdfplumber.open('/content/f.pdf')\n",
        "page = pdf.pages[0] #select page\n",
        "lines = page.extract_text() #download\n",
        "foo=lines #make copy\n",
        "lines=re.sub('[^A-Za-z]+', ' ', lines) #strip\n",
        "lemmatizer = WordNetLemmatizer()\n",
        "word_list = nltk.word_tokenize(lines)\n",
        "lines = ' '.join([lemmatizer.lemmatize(w) for w in word_list])\n",
        "word_list = nltk.word_tokenize(lines)\n",
        "lines=lines.lower()\n",
        "lines = remove_stopwords(lines)\n",
        "lines=lines.split('\\n')\n",
        "def listToString(s): \n",
        "    \n",
        "    # initialize an empty string\n",
        "    str1 = \"\" \n",
        "    \n",
        "    # traverse in the string  \n",
        "    for ele in s: \n",
        "        str1 += ele  \n",
        "    \n",
        "    # return string  \n",
        "    return str1 \n",
        "\n",
        "lines=listToString(lines)\n",
        "from earthy.nltk_wrappers import lemmatize_sent\n",
        "lines, lemmas, tags = zip(*lemmatize_sent(lines))\n",
        "#lines=lines.split(' ')\n"
      ],
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
            "[nltk_data]   Package stopwords is already up-to-date!\n",
            "[nltk_data] Downloading package wordnet to /root/nltk_data...\n",
            "[nltk_data]   Package wordnet is already up-to-date!\n",
            "[nltk_data] Downloading package punkt to /root/nltk_data...\n",
            "[nltk_data]   Package punkt is already up-to-date!\n",
            "[nltk_data] Downloading package averaged_perceptron_tagger to\n",
            "[nltk_data]     /root/nltk_data...\n",
            "[nltk_data]   Package averaged_perceptron_tagger is already up-to-\n",
            "[nltk_data]       date!\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "bFjaHJLYMQIo"
      },
      "source": [
        "lemmas = [i for i in lemmas if len(i) > 1]"
      ],
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "oLSh8J5yjyET",
        "outputId": "494229eb-d428-4da5-f187-da2f338a0681"
      },
      "source": [
        "\n",
        "x=Counter(lemmas)\n",
        "x.most_common(5)\n",
        "\n",
        "#words, lemmas, tags = zip(*lemmatize_sent(lines))"
      ],
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[('cell', 23), ('organism', 6), ('life', 5), ('single', 3), ('diversity', 3)]"
            ]
          },
          "metadata": {},
          "execution_count": 5
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Sncczc45UXKm"
      },
      "source": [
        "words = str(foo).split('\\n')\n",
        "words\n",
        "matches = [match for match in words if 'cell' in match]     \n",
        "# Driver code    \n",
        "miao=listToString(matches)\n",
        "#len(matches)"
      ],
      "execution_count": 6,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "uaN4pHFDaVR2",
        "outputId": "a72d37e3-e3ea-4881-8c18-962a1f48980b"
      },
      "source": [
        "matches"
      ],
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['A single ~200 micrometer (!m) cell,the human egg,with',\n",
              " 'sperm,which are also single cells.From the union of an egg',\n",
              " 'and sperm will arise the 10 trillion cells of a human body.',\n",
              " 'Like ourselves, the individual cells that form our bodies 1.1 The Diversity and Commonality ',\n",
              " 'actions. These abilities define life. We and other multicellular',\n",
              " 'organisms contain billions or trillions of cells organized into',\n",
              " 'cell. Even simple unicellular organisms exhibit all the hall-',\n",
              " 'mark properties of life, indicating that the cell is the funda-',\n",
              " 'cells but is an absolute requirement for others. Most cells in',\n",
              " 'multicellular organisms are intimately involved with other',\n",
              " 'an explosion of new data about the components of cells,',\n",
              " 'cells. Although some unicellular organisms live in isolation,',\n",
              " 'particularly about how information flows through cells and',\n",
              " 'Molecular cell biology is a rich, integrative science that',\n",
              " 'the multifaceted story of the birth, life, and death of cells. We 1.3  The Work of Cells',\n",
              " 'cells, their basic constituents and critical functions, and what',\n",
              " 'we can learn from the various ways of studying cells.  1.5  A Genome Perspective on Evolution']"
            ]
          },
          "metadata": {},
          "execution_count": 7
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 171
        },
        "id": "DRXAVnbSv4ra",
        "outputId": "1db08dce-09c1-4357-e203-4eccec552d94"
      },
      "source": [
        "miao"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'A single ~200 micrometer (!m) cell,the human egg,withsperm,which are also single cells.From the union of an eggand sperm will arise the 10 trillion cells of a human body.Like ourselves, the individual cells that form our bodies 1.1 The Diversity and Commonality actions. These abilities define life. We and other multicellularorganisms contain billions or trillions of cells organized intocell. Even simple unicellular organisms exhibit all the hall-mark properties of life, indicating that the cell is the funda-cells but is an absolute requirement for others. Most cells inmulticellular organisms are intimately involved with otheran explosion of new data about the components of cells,cells. Although some unicellular organisms live in isolation,particularly about how information flows through cells andMolecular cell biology is a rich, integrative science thatthe multifaceted story of the birth, life, and death of cells. We 1.3  The Work of Cellscells, their basic constituents and critical functions, and whatwe can learn from the various ways of studying cells.  1.5  A Genome Perspective on Evolution'"
            ]
          },
          "metadata": {},
          "execution_count": 8
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 69
        },
        "id": "OH_DcepjXoDg",
        "outputId": "081c2dd2-3c41-4100-f215-6323f4fb9eb2"
      },
      "source": [
        "\n",
        "\n",
        "# Passing the text corpus to summarizer\n",
        "summary_by_word_count=summarize(miao)\n",
        "summary_by_word_count"
      ],
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'Although some unicellular organisms live in isolation,particularly about how information flows through cells andMolecular cell biology is a rich, integrative science thatthe multifaceted story of the birth, life, and death of cells.'"
            ]
          },
          "metadata": {},
          "execution_count": 9
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "QDjYBIulXqPr"
      },
      "source": [
        "'''import gensim\n",
        "from gensim.summarization import summarize\n",
        "\n",
        "# Passing the text corpus to summarizer\n",
        "summary_by_word_count=summarize(miao,word_count=30)\n",
        "print(summary_by_word_count)'''\n",
        "\n",
        "!pip install transformers --q"
      ],
      "execution_count": 10,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "HDb4HK48YxfW"
      },
      "source": [
        "#https://www.machinelearningplus.com/nlp/text-summarization-approaches-nlp-example/\n",
        "\n",
        "# Importing requirements\n",
        "from transformers import T5Tokenizer, T5Config, T5ForConditionalGeneration\n"
      ],
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Syqg68rZZSyp"
      },
      "source": [
        "!pip install sentencepiece --q"
      ],
      "execution_count": 12,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 388,
          "referenced_widgets": [
            "2de69b90e7f243809da7b5acd13e45bf",
            "90b76275fa9840dc97b91904f763b057",
            "26091ac6b4be4a21851474bd3138a567",
            "f00eaec31d0145ab88572031f2f415e3",
            "967fa4f219884e1a9f22291285c4ab48",
            "72ba7e2b27ab45ce9cd86c301ebbcd16",
            "364b2883f6504ddfafe1e97fe727c532",
            "8f791efaf30144bbab36d69793a385d1",
            "363863b9cf7341728155364bc60fa548",
            "72cf8f1169f1482795a34c54aa54b714",
            "1f2b253bb7774cb6b83cb284295bb71a",
            "b6773000e0624414be62c5142e155a6e",
            "56700317de2e4dfd973a85cfaaa2a65f",
            "462d770a48694fa9b7eaa706df4e7653",
            "0a02e7475bb04caca20b8fb86b8d8a69",
            "c0867c825231497bafefc4c15a9a7352",
            "e772a734fa6340daa0d2d66f1de3d80d",
            "ce33bdc7f68f45a9a1145733830e7a40",
            "504ab56e5f074b38b444b39dab1265af",
            "a289d7713a6c4fa8aed49c6b5f57acb9",
            "996172a784f141cfa025ed4a8782153f",
            "dc1878fd97ce471cb7af600d7c296674"
          ]
        },
        "id": "pwhq-fSKY6AM",
        "outputId": "1bd2b640-c1f5-4c6f-d54b-5b8e9eaedc5d"
      },
      "source": [
        "my_model = T5ForConditionalGeneration.from_pretrained('t5-small')\n",
        "tokenizer = T5Tokenizer.from_pretrained('t5-small')"
      ],
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "2de69b90e7f243809da7b5acd13e45bf",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "Downloading:   0%|          | 0.00/1.20k [00:00<?, ?B/s]"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "b6773000e0624414be62c5142e155a6e",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "Downloading:   0%|          | 0.00/242M [00:00<?, ?B/s]"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "error",
          "ename": "KeyboardInterrupt",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-13-95db19cd822c>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mmy_model\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mT5ForConditionalGeneration\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfrom_pretrained\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m't5-small'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0mtokenizer\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mT5Tokenizer\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfrom_pretrained\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m't5-small'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/transformers/modeling_utils.py\u001b[0m in \u001b[0;36mfrom_pretrained\u001b[0;34m(cls, pretrained_model_name_or_path, *model_args, **kwargs)\u001b[0m\n\u001b[1;32m    650\u001b[0m                     \u001b[0mproxies\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mproxies\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    651\u001b[0m                     \u001b[0mresume_download\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mresume_download\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 652\u001b[0;31m                     \u001b[0mlocal_files_only\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mlocal_files_only\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    653\u001b[0m                 )\n\u001b[1;32m    654\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mresolved_archive_file\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/transformers/file_utils.py\u001b[0m in \u001b[0;36mcached_path\u001b[0;34m(url_or_filename, cache_dir, force_download, proxies, resume_download, user_agent, extract_compressed_file, force_extract, local_files_only)\u001b[0m\n\u001b[1;32m    569\u001b[0m             \u001b[0mresume_download\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mresume_download\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    570\u001b[0m             \u001b[0muser_agent\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0muser_agent\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 571\u001b[0;31m             \u001b[0mlocal_files_only\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mlocal_files_only\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    572\u001b[0m         )\n\u001b[1;32m    573\u001b[0m     \u001b[0;32melif\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mexists\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0murl_or_filename\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/transformers/file_utils.py\u001b[0m in \u001b[0;36mget_from_cache\u001b[0;34m(url, cache_dir, force_download, proxies, etag_timeout, resume_download, user_agent, local_files_only)\u001b[0m\n\u001b[1;32m    748\u001b[0m             \u001b[0mlogger\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0minfo\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"%s not found in cache or force_download set to True, downloading to %s\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0murl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtemp_file\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    749\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 750\u001b[0;31m             \u001b[0mhttp_get\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0murl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtemp_file\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mproxies\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mproxies\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mresume_size\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mresume_size\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0muser_agent\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0muser_agent\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    751\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    752\u001b[0m         \u001b[0mlogger\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0minfo\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"storing %s in cache at %s\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0murl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcache_path\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/transformers/file_utils.py\u001b[0m in \u001b[0;36mhttp_get\u001b[0;34m(url, temp_file, proxies, resume_size, user_agent)\u001b[0m\n\u001b[1;32m    641\u001b[0m         \u001b[0mdisable\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mbool\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlogger\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgetEffectiveLevel\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mlogging\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mNOTSET\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    642\u001b[0m     )\n\u001b[0;32m--> 643\u001b[0;31m     \u001b[0;32mfor\u001b[0m \u001b[0mchunk\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mresponse\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0miter_content\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchunk_size\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m1024\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    644\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mchunk\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# filter out keep-alive new chunks\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    645\u001b[0m             \u001b[0mprogress\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mupdate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchunk\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/requests/models.py\u001b[0m in \u001b[0;36mgenerate\u001b[0;34m()\u001b[0m\n\u001b[1;32m    749\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mhasattr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mraw\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'stream'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    750\u001b[0m                 \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 751\u001b[0;31m                     \u001b[0;32mfor\u001b[0m \u001b[0mchunk\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mraw\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstream\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchunk_size\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdecode_content\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    752\u001b[0m                         \u001b[0;32myield\u001b[0m \u001b[0mchunk\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    753\u001b[0m                 \u001b[0;32mexcept\u001b[0m \u001b[0mProtocolError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/urllib3/response.py\u001b[0m in \u001b[0;36mstream\u001b[0;34m(self, amt, decode_content)\u001b[0m\n\u001b[1;32m    494\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    495\u001b[0m             \u001b[0;32mwhile\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mis_fp_closed\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_fp\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 496\u001b[0;31m                 \u001b[0mdata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mamt\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mamt\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdecode_content\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mdecode_content\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    497\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    498\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mdata\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/urllib3/response.py\u001b[0m in \u001b[0;36mread\u001b[0;34m(self, amt, decode_content, cache_content)\u001b[0m\n\u001b[1;32m    442\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    443\u001b[0m                 \u001b[0mcache_content\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mFalse\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 444\u001b[0;31m                 \u001b[0mdata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_fp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mamt\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    445\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mamt\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;36m0\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mdata\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# Platform-specific: Buggy versions of Python.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    446\u001b[0m                     \u001b[0;31m# Close the connection when no data is returned\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/lib/python3.7/http/client.py\u001b[0m in \u001b[0;36mread\u001b[0;34m(self, amt)\u001b[0m\n\u001b[1;32m    463\u001b[0m             \u001b[0;31m# Amount is given, implement using readinto\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    464\u001b[0m             \u001b[0mb\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbytearray\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mamt\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 465\u001b[0;31m             \u001b[0mn\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreadinto\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mb\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    466\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mmemoryview\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mb\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0mn\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtobytes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    467\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/lib/python3.7/http/client.py\u001b[0m in \u001b[0;36mreadinto\u001b[0;34m(self, b)\u001b[0m\n\u001b[1;32m    507\u001b[0m         \u001b[0;31m# connection, and the user is reading more bytes than will be provided\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    508\u001b[0m         \u001b[0;31m# (for example, reading in 1k chunks)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 509\u001b[0;31m         \u001b[0mn\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreadinto\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mb\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    510\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mn\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mb\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    511\u001b[0m             \u001b[0;31m# Ideally, we would raise IncompleteRead if the content-length\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/lib/python3.7/socket.py\u001b[0m in \u001b[0;36mreadinto\u001b[0;34m(self, b)\u001b[0m\n\u001b[1;32m    587\u001b[0m         \u001b[0;32mwhile\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    588\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 589\u001b[0;31m                 \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_sock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_into\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mb\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    590\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    591\u001b[0m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_timeout_occurred\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/lib/python3.7/ssl.py\u001b[0m in \u001b[0;36mrecv_into\u001b[0;34m(self, buffer, nbytes, flags)\u001b[0m\n\u001b[1;32m   1069\u001b[0m                   \u001b[0;34m\"non-zero flags not allowed in calls to recv_into() on %s\"\u001b[0m \u001b[0;34m%\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1070\u001b[0m                   self.__class__)\n\u001b[0;32m-> 1071\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnbytes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbuffer\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1072\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1073\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0msuper\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_into\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mbuffer\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnbytes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mflags\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/lib/python3.7/ssl.py\u001b[0m in \u001b[0;36mread\u001b[0;34m(self, len, buffer)\u001b[0m\n\u001b[1;32m    927\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    928\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mbuffer\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 929\u001b[0;31m                 \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_sslobj\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbuffer\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    930\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    931\u001b[0m                 \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_sslobj\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6OBo7mEKZDDt"
      },
      "source": [
        "text = \"summarize:\" + miao\n",
        "text"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 147,
          "referenced_widgets": [
            "d32080a6580c4ea398d1bc1d96b1afc0",
            "969ecbd87b22438a9e4c7caba7cce2e9",
            "312c0e164ed44fc68796c2c42c00739b",
            "5f64044bc9784ee5ac8f89a1e55d66c2",
            "ef300e4fbffb472388351be84b864ca3",
            "3e7b56e425ac46a8a90665d470ba560c",
            "fda549c0c69041a08a55b7159ae1bcc2",
            "58a9cd2ed3bb458985c9272d06a1a34f",
            "4833b72c2d794421abff6a70dbb1db00",
            "45c446fe7b2a4fd89c4a28054d3885db",
            "7f3430a9f4764594b634817825d73a0e",
            "7b4995c7b6fc46f0be8b94160d6c05ec",
            "76bd40872c984fb2ba76167f3bc0d2e4",
            "847f4d8e817f4271b38ce5e0ef651b38",
            "328012e7262440efa79e3995c68aa007",
            "084f670d569a4fefb3efe23195103ccf",
            "9f4b247f4a304400a5de85a50a76f8cc",
            "59fbf2f35f7849e6a7d2ca0ddf006da5",
            "459971060eca4aec95697a8aa4770ec3",
            "bce914497da740fc8e2aa187f3055bfd",
            "523bec92145c4c6aaba2c8a459285eb8",
            "8b780a15640d41dfb74cde32118eab68",
            "a2a372f5d68f472e85961cffaa43871a",
            "956186e628b94fc4b284eb0fb5b77266",
            "68fa6f6936684a328fbbd94971364912",
            "7218a364fbc04955b9a9c9bb9d228638",
            "ce64ca612f9a40c8a6e174b3e616a36b",
            "86ffc8df091a49e997d6468023e2fd46",
            "c0a9c0b4e29a4bc8857f67a80a7f833e",
            "86c0919aa9254fc2b2f785cb3ee9312d",
            "7a20de4590634881b19778b40b39b483",
            "f8bef635edd44941ad7f3578e0991b4a",
            "a2dc661c88bd484fa624cc763f8a2c11"
          ]
        },
        "id": "mX8jMYRfM_kY",
        "outputId": "e14922f3-c161-447e-f3dc-ac3f299b0651"
      },
      "source": [
        "#question generation\n",
        "#https://towardsdatascience.com/questgen-an-open-source-nlp-library-for-question-generation-algorithms-1e18067fcdc6\n",
        "\n",
        "nltk.download('stopwords')\n",
        "\n",
        "qe= main.BoolQGen()\n"
      ],
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
            "[nltk_data]   Package stopwords is already up-to-date!\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "d32080a6580c4ea398d1bc1d96b1afc0",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "Downloading:   0%|          | 0.00/792k [00:00<?, ?B/s]"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "7b4995c7b6fc46f0be8b94160d6c05ec",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "Downloading:   0%|          | 0.00/1.21k [00:00<?, ?B/s]"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "a2a372f5d68f472e85961cffaa43871a",
              "version_minor": 0,
              "version_major": 2
            },
            "text/plain": [
              "Downloading:   0%|          | 0.00/892M [00:00<?, ?B/s]"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3hafc9qZNh7V",
        "outputId": "ac2781dd-cd81-426d-856e-c7dd6bfb50e2"
      },
      "source": [
        "payload = {\n",
        "            \"input_text\": summary_by_word_count }\n",
        "output = qe.predict_boolq(payload)\n",
        "pprint (output)"
      ],
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/torch/_tensor.py:575: UserWarning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.\n",
            "To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (Triggered internally at  /pytorch/aten/src/ATen/native/BinaryOps.cpp:467.)\n",
            "  return torch.floor_divide(self, other)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'Boolean Questions': ['Is cell biology the same as cellular biology?',\n",
            "                       'Is cell biology and cellular biology the same?',\n",
            "                       'Is cell biology and cellular biology the same thing?'],\n",
            " 'Count': 4,\n",
            " 'Text': 'Although some unicellular organisms live in isolation,particularly '\n",
            "         'about how information flows through cells andMolecular cell biology '\n",
            "         'is a rich, integrative science thatthe multifaceted story of the '\n",
            "         'birth, life, and death of cells.'}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Qn4n1R7InMpP"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "LP03aw2knNKw"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "bSx-q5RXnNd1"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "cqjNb34JnNvT"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "v2wGErSInN7F"
      },
      "source": [
        "##EXTRA"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8jcLuU1FQj1Q"
      },
      "source": [
        "###topic modeller\n",
        "##https://towardsdatascience.com/topic-modelling-in-python-with-nltk-and-gensim-4ef03213cd21\n",
        "\n",
        "\n",
        "\n",
        "import spacy\n",
        "spacy.load('en')\n",
        "from spacy.lang.en import English\n",
        "parser = English()\n",
        "def tokenize(text):\n",
        "    lda_tokens = []\n",
        "    tokens = parser(text)\n",
        "    for token in tokens:\n",
        "        if token.orth_.isspace():\n",
        "            continue\n",
        "        elif token.like_url:\n",
        "            lda_tokens.append('URL')\n",
        "        elif token.orth_.startswith('@'):\n",
        "            lda_tokens.append('SCREEN_NAME')\n",
        "        else:\n",
        "            lda_tokens.append(token.lower_)\n",
        "    return lda_tokens"
      ],
      "execution_count": 16,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "aTpKv0tbQuq1",
        "outputId": "c4ff0979-d755-40be-af93-a4dbb574857a"
      },
      "source": [
        "import nltk\n",
        "nltk.download('wordnet')\n",
        "from nltk.corpus import wordnet as wn\n",
        "def get_lemma(word):\n",
        "    lemma = wn.morphy(word)\n",
        "    if lemma is None:\n",
        "        return word\n",
        "    else:\n",
        "        return lemma\n",
        "    \n",
        "from nltk.stem.wordnet import WordNetLemmatizer\n",
        "def get_lemma2(word):\n",
        "    return WordNetLemmatizer().lemmatize(word)"
      ],
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package wordnet to /root/nltk_data...\n",
            "[nltk_data]   Package wordnet is already up-to-date!\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RTUqG-DeQ2H2",
        "outputId": "4c0fc534-0cdb-4f13-c305-cc767b001026"
      },
      "source": [
        "nltk.download('stopwords')\n",
        "en_stop = set(nltk.corpus.stopwords.words('english'))"
      ],
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
            "[nltk_data]   Package stopwords is already up-to-date!\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "oDKL-nMbQ7MG"
      },
      "source": [
        "def prepare_text_for_lda(text):\n",
        "    tokens = tokenize(text)\n",
        "    tokens = [token for token in tokens if len(token) > 4]\n",
        "    tokens = [token for token in tokens if token not in en_stop]\n",
        "    tokens = [get_lemma(token) for token in tokens]\n",
        "    return tokens"
      ],
      "execution_count": 19,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 111
        },
        "id": "dgdoxYUbRTcW",
        "outputId": "8813f412-a746-4207-a0db-7cdeed32b186"
      },
      "source": [
        "import pandas as pd\n",
        "df = pd.DataFrame([x.split(';') for x in summary_by_word_count.split('.')])\n",
        "df.to_csv('miao.csv')\n",
        "df"
      ],
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>0</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Although some unicellular organisms live in is...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td></td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "                                                   0\n",
              "0  Although some unicellular organisms live in is...\n",
              "1                                                   "
            ]
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "sVaj2LB8RAY0"
      },
      "source": [
        "\n",
        "text_data = []\n",
        "with open('miao.csv') as f:\n",
        "    for line in f:\n",
        "        tokens = prepare_text_for_lda(line)\n",
        "        text_data.append(tokens)"
      ],
      "execution_count": 21,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "liBxEU8bWpEE",
        "outputId": "86be9cd2-c888-480d-966c-6979e9cb4f3f"
      },
      "source": [
        "text_data"
      ],
      "execution_count": 22,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[[],\n",
              " ['0,\"although',\n",
              "  'unicellular',\n",
              "  'organism',\n",
              "  'isolation',\n",
              "  'particularly',\n",
              "  'information',\n",
              "  'flow',\n",
              "  'cell',\n",
              "  'andmolecular',\n",
              "  'biology',\n",
              "  'integrative',\n",
              "  'science',\n",
              "  'thatthe',\n",
              "  'multifaceted',\n",
              "  'story',\n",
              "  'birth',\n",
              "  'death',\n",
              "  'cell'],\n",
              " []]"
            ]
          },
          "metadata": {},
          "execution_count": 22
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SqRlDyC3RO8W"
      },
      "source": [
        "from gensim import corpora\n",
        "dictionary = corpora.Dictionary(text_data)\n",
        "corpus = [dictionary.doc2bow(text) for text in text_data]\n",
        "import pickle\n",
        "pickle.dump(corpus, open('corpus.pkl', 'wb'))\n",
        "dictionary.save('dictionary.gensim')"
      ],
      "execution_count": 23,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "LyajujeGShns"
      },
      "source": [
        "\n",
        "import gensim\n",
        "NUM_TOPICS = 1\n",
        "ldamodel = gensim.models.ldamodel.LdaModel(corpus, num_topics = NUM_TOPICS, id2word=dictionary, passes=15)\n",
        "ldamodel.save('model5.gensim')"
      ],
      "execution_count": 24,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4l_2djt2SkP6",
        "outputId": "ad7b0920-26b1-475a-9626-eb409af743c3"
      },
      "source": [
        "topics = ldamodel.print_topics(num_words=5)\n",
        "for topic in topics:\n",
        "    print(topic)"
      ],
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "(0, '0.086*\"cell\" + 0.057*\"story\" + 0.057*\"particularly\" + 0.057*\"science\" + 0.057*\"multifaceted\"')\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 80
        },
        "id": "XEnh-Q8BYBCV",
        "outputId": "735755d3-c706-4e44-df03-7274da0be16c"
      },
      "source": [
        "###TEST###\n",
        "bar='In vertebrates, the p53 response evolved to induce apoptosis in the face of extensive DNA damage, presumably to prevent the accumulation of multiple mutations that might convert a normal cell into a cancer cell'\n",
        "df = pd.DataFrame([x.split(';') for x in bar.split('\\n')])\n",
        "df.to_csv('bar.csv')\n",
        "df"
      ],
      "execution_count": 26,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>0</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>In vertebrates, the p53 response evolved to in...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "                                                   0\n",
              "0  In vertebrates, the p53 response evolved to in..."
            ]
          },
          "metadata": {},
          "execution_count": 26
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wf3BK7pJYQjl"
      },
      "source": [
        "text_data = []\n",
        "with open('bar.csv') as f:\n",
        "    for line in f:\n",
        "        tokens = prepare_text_for_lda(line)\n",
        "        text_data.append(tokens)\n",
        "\n",
        "dictionary = corpora.Dictionary(text_data)\n",
        "corpus = [dictionary.doc2bow(text) for text in text_data]\n",
        "pickle.dump(corpus, open('corpus.pkl', 'wb'))\n",
        "dictionary.save('dictionary.gensim')\n",
        "\n"
      ],
      "execution_count": 27,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "X8HDu7Z3ZqfW"
      },
      "source": [
        "import gensim\n",
        "NUM_TOPICS = 1\n",
        "ldamodel = gensim.models.ldamodel.LdaModel(corpus, num_topics = NUM_TOPICS, id2word=dictionary, passes=15)\n",
        "ldamodel.save('model5.gensim')\n"
      ],
      "execution_count": 28,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "zQxm5CyCZtFR",
        "outputId": "ce31185c-0a70-4be7-b959-92c51c63cbcc"
      },
      "source": [
        "topics = ldamodel.print_topics(num_words=5)\n",
        "for topic in topics:\n",
        "    print(topic)"
      ],
      "execution_count": 29,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "(0, '0.059*\"normal\" + 0.059*\"prevent\" + 0.059*\"mutation\" + 0.059*\"might\" + 0.059*\"presumably\"')\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "QBlaM5WBWIbW"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "2j9DaDYVWIlY"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TyUIslr_WIq_"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "K2awIRz8WItu"
      },
      "source": [
        "###### Chapter indexing####"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lJcr2qLjX9-D",
        "outputId": "8e626263-af77-40b3-c2ad-de00ac02ede2"
      },
      "source": [
        "!pip install fitz --q"
      ],
      "execution_count": 30,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[K     |████████████████████████████████| 78 kB 3.6 MB/s \n",
            "\u001b[K     |████████████████████████████████| 3.2 MB 25.1 MB/s \n",
            "\u001b[K     |████████████████████████████████| 129 kB 71.0 MB/s \n",
            "\u001b[K     |████████████████████████████████| 5.1 MB 38.4 MB/s \n",
            "\u001b[K     |████████████████████████████████| 421 kB 52.2 MB/s \n",
            "\u001b[K     |████████████████████████████████| 407 kB 54.5 MB/s \n",
            "\u001b[K     |████████████████████████████████| 45 kB 3.4 MB/s \n",
            "\u001b[K     |████████████████████████████████| 6.3 MB 26.1 MB/s \n",
            "\u001b[?25h  Building wheel for configobj (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Building wheel for pyxnat (setup.py) ... \u001b[?25l\u001b[?25hdone\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lBRWiowdYREq",
        "outputId": "029f31c8-efad-4673-9715-ed53f7f49ad6"
      },
      "source": [
        "!pip install PyMuPDF --q"
      ],
      "execution_count": 31,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[K     |████████████████████████████████| 8.7 MB 1.1 MB/s \n",
            "\u001b[?25h"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "A9Tm1P0zpHih"
      },
      "source": [
        "#This is from your link"
      ],
      "execution_count": 32,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3V0FBHsyXK0_",
        "outputId": "dc4c9702-2e90-4e5b-cf21-58b95901f7b7"
      },
      "source": [
        "from operator import itemgetter\n",
        "import fitz\n",
        "import json\n",
        "\n",
        "\n",
        "def fonts(doc, granularity=False):\n",
        "    \"\"\"Extracts fonts and their usage in PDF documents.\n",
        "    :param doc: PDF document to iterate through\n",
        "    :type doc: <class 'fitz.fitz.Document'>\n",
        "    :param granularity: also use 'font', 'flags' and 'color' to discriminate text\n",
        "    :type granularity: bool\n",
        "    :rtype: [(font_size, count), (font_size, count}], dict\n",
        "    :return: most used fonts sorted by count, font style information\n",
        "    \"\"\"\n",
        "    styles = {}\n",
        "    font_counts = {}\n",
        "\n",
        "    for page in doc:\n",
        "        blocks = page.getText(\"dict\")[\"blocks\"]\n",
        "        for b in blocks:  # iterate through the text blocks\n",
        "            if b['type'] == 0:  # block contains text\n",
        "                for l in b[\"lines\"]:  # iterate through the text lines\n",
        "                    for s in l[\"spans\"]:  # iterate through the text spans\n",
        "                        if granularity:\n",
        "                            identifier = \"{0}_{1}_{2}_{3}\".format(s['size'], s['flags'], s['font'], s['color'])\n",
        "                            styles[identifier] = {'size': s['size'], 'flags': s['flags'], 'font': s['font'],\n",
        "                                                  'color': s['color']}\n",
        "                        else:\n",
        "                            identifier = \"{0}\".format(s['size'])\n",
        "                            styles[identifier] = {'size': s['size'], 'font': s['font']}\n",
        "\n",
        "                        font_counts[identifier] = font_counts.get(identifier, 0) + 1  # count the fonts usage\n",
        "\n",
        "    font_counts = sorted(font_counts.items(), key=itemgetter(1), reverse=True)\n",
        "\n",
        "    if len(font_counts) < 1:\n",
        "        raise ValueError(\"Zero discriminating fonts found!\")\n",
        "\n",
        "    return font_counts, styles\n",
        "\n",
        "\n",
        "def font_tags(font_counts, styles):\n",
        "    \"\"\"Returns dictionary with font sizes as keys and tags as value.\n",
        "    :param font_counts: (font_size, count) for all fonts occuring in document\n",
        "    :type font_counts: list\n",
        "    :param styles: all styles found in the document\n",
        "    :type styles: dict\n",
        "    :rtype: dict\n",
        "    :return: all element tags based on font-sizes\n",
        "    \"\"\"\n",
        "    p_style = styles[font_counts[0][0]]  # get style for most used font by count (paragraph)\n",
        "    p_size = p_style['size']  # get the paragraph's size\n",
        "\n",
        "    # sorting the font sizes high to low, so that we can append the right integer to each tag\n",
        "    font_sizes = []\n",
        "    for (font_size, count) in font_counts:\n",
        "        font_sizes.append(float(font_size))\n",
        "    font_sizes.sort(reverse=True)\n",
        "\n",
        "    # aggregating the tags for each font size\n",
        "    idx = 0\n",
        "    size_tag = {}\n",
        "    for size in font_sizes:\n",
        "        idx += 1\n",
        "        if size == p_size:\n",
        "            idx = 0\n",
        "            size_tag[size] = '<p>'\n",
        "        if size > p_size:\n",
        "            size_tag[size] = '<h{0}>'.format(idx)\n",
        "        elif size < p_size:\n",
        "            size_tag[size] = '<s{0}>'.format(idx)\n",
        "\n",
        "    return size_tag\n",
        "\n",
        "\n",
        "def headers_para(doc, size_tag):\n",
        "    \"\"\"Scrapes headers & paragraphs from PDF and return texts with element tags.\n",
        "    :param doc: PDF document to iterate through\n",
        "    :type doc: <class 'fitz.fitz.Document'>\n",
        "    :param size_tag: textual element tags for each size\n",
        "    :type size_tag: dict\n",
        "    :rtype: list\n",
        "    :return: texts with pre-prended element tags\n",
        "    \"\"\"\n",
        "    header_para = []  # list with headers and paragraphs\n",
        "    first = True  # boolean operator for first header\n",
        "    previous_s = {}  # previous span\n",
        "\n",
        "    for page in doc:\n",
        "        blocks = page.getText(\"dict\")[\"blocks\"]\n",
        "        for b in blocks:  # iterate through the text blocks\n",
        "            if b['type'] == 0:  # this block contains text\n",
        "\n",
        "                # REMEMBER: multiple fonts and sizes are possible IN one block\n",
        "\n",
        "                block_string = \"\"  # text found in block\n",
        "                for l in b[\"lines\"]:  # iterate through the text lines\n",
        "                    for s in l[\"spans\"]:  # iterate through the text spans\n",
        "                        if s['text'].strip():  # removing whitespaces:\n",
        "                            if first:\n",
        "                                previous_s = s\n",
        "                                first = False\n",
        "                                block_string = size_tag[s['size']] + s['text']\n",
        "                            else:\n",
        "                                if s['size'] == previous_s['size']:\n",
        "\n",
        "                                    if block_string and all((c == \"|\") for c in block_string):\n",
        "                                        # block_string only contains pipes\n",
        "                                        block_string = size_tag[s['size']] + s['text']\n",
        "                                    if block_string == \"\":\n",
        "                                        # new block has started, so append size tag\n",
        "                                        block_string = size_tag[s['size']] + s['text']\n",
        "                                    else:  # in the same block, so concatenate strings\n",
        "                                        block_string += \" \" + s['text']\n",
        "\n",
        "                                else:\n",
        "                                    header_para.append(block_string)\n",
        "                                    block_string = size_tag[s['size']] + s['text']\n",
        "\n",
        "                                previous_s = s\n",
        "\n",
        "                    # new block started, indicating with a pipe\n",
        "                    block_string += \"|\"\n",
        "\n",
        "                header_para.append(block_string)\n",
        "\n",
        "    return header_para\n",
        "\n",
        "\n",
        "def main():\n",
        "\n",
        "    document = 'f.pdf'\n",
        "    doc = fitz.open(document)\n",
        "\n",
        "    font_counts, styles = fonts(doc, granularity=False)\n",
        "\n",
        "    size_tag = font_tags(font_counts, styles)\n",
        "\n",
        "    elements = headers_para(doc, size_tag)\n",
        "\n",
        "    with open(\"doc.json\", 'w') as json_out:\n",
        "        json.dump(elements, json_out)\n",
        "\n",
        "\n",
        "if __name__ == '__main__':\n",
        "    main()"
      ],
      "execution_count": 33,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Deprecation: 'getText' removed from class 'Page' after v1.19.0 - use 'get_text'.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Epvew6gWXPoA"
      },
      "source": [
        "#Conversion of tokens in txt\n",
        "\n",
        "import pandas as pd\n",
        "df = pd.read_json('doc.json')\n",
        "df#22\n",
        "df.to_csv('x.txt', sep=',', index=False)"
      ],
      "execution_count": 34,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wMw7DERDcpZA"
      },
      "source": [
        "#Read back\n",
        "f = open(\"x.txt\", \"r\")\n",
        "f=f.readlines()\n"
      ],
      "execution_count": 35,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wlRbjYQ7dI_k"
      },
      "source": [
        "#https://stackoverflow.com/questions/55893314/how-to-search-for-a-keyword-in-a-list-of-strings-and-return-that-string\n",
        "#Find token font\n",
        "l1 = [k for k in f if '<h22>CHA' in  k]\n"
      ],
      "execution_count": 36,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "y0SLbhQ_dgM0",
        "outputId": "c88ed8c6-1926-4268-fa1a-ae33a5fd6acb"
      },
      "source": [
        "#Clean everything else\n",
        "lista=set(l1)\n",
        "index=list(lista)\n",
        "import re\n",
        "hpp=[re.sub(r'<h22>CHA P TE R  ', 'CHAPTER ', i)for i in index]\n",
        "hpp=[re.sub(r'\"', '', i)for i in hpp]\n",
        "hbb=  [i.split(\"•\", 1)[0] for i in hpp]\n",
        "\n",
        "hbb"
      ],
      "execution_count": 66,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['CHAPTER 22 ', 'CHAPTER 21 ']"
            ]
          },
          "metadata": {},
          "execution_count": 66
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7O4NUCwzf_t-"
      },
      "source": [
        "#Download entire corpus (Takes a minute for 2 chapters)\n",
        "import os\n",
        "import pdfplumber\n",
        "\n",
        "directory = r'/content/'\n",
        "\n",
        "for filename in os.listdir(directory):\n",
        "    if filename.endswith('.pdf'):\n",
        "        fullpath = os.path.join(directory, filename)\n",
        "        #print(fullpath)\n",
        "        all_text = \"\"\n",
        "        with pdfplumber.open(fullpath) as pdf:\n",
        "            for page in pdf.pages:\n",
        "                text = page.extract_text()\n",
        "                #print(text)\n",
        "                all_text += '\\n' + text\n",
        "        with open('all_text.txt', 'w') as f:\n",
        "          f.write(all_text)"
      ],
      "execution_count": 38,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ST12hsb1nKbc"
      },
      "source": [
        "numberChapter=0 #choose chapter (this is 21 from my machine, because i am working with a reduced corpus (only chapter 21 and 22))\n"
      ],
      "execution_count": 67,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 171
        },
        "id": "QyZlwKWhq3Vs",
        "outputId": "63cfb13b-40a4-4105-eddf-90d7d5c90d5d"
      },
      "source": [
        "all_textstart=all_text[all_text.find(hbb[0]):]\n",
        "ChapterIndex=all_textstart.split(hbb[0+1], 1)[0]\n",
        "\n",
        "#ChapterIndex\n",
        "\n",
        "all_textstart\n",
        " "
      ],
      "execution_count": 69,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'CHAPTER 22 • Cell Birth, Lineage, And Death\\n!FIGURE 22-1 Overview of the birth,lineage,and death\\nof cells. Following growth, cells are “born” as the result of\\nsymmetric or asymmetric cell division. (a) The two daughter cells\\nCell growth\\nresulting from symmetric division are essentially identical to each\\nother and to the parental cell. Such daughter cells subsequently (a) (b)\\ncan have different fates if they are exposed to different signals.\\nSymmetric\\nThe two daughter cells resulting from asymmetric division differ\\ndivision Cell\\nfrom birth and consequently have different fates. Asymmetric birth\\ndivision commonly is preceded by the localization of regulatory\\nmolecules (green) in one part of the parent cell. (b) A series of Cell growthFPO\\nsymmetric and/or asymmetric cell divisions, called a cell lineage,\\ngives birth to each of the specialized cell types found in a\\nAsymmetric\\nmulticellular organism. The pattern of cell lineage can be under Cell\\ndivision \\ndeath\\ntight genetic control. Programmed cell death occurs during\\nnormal development (e.g., in the webbing that initially develops\\nwhen fingers grow) and also in response to infection or poison. \\nA series of specific programmed events, called apoptosis, is\\nactivated is these situations.\\nCell lineage\\ncoordinately activate, or repress, batteries of genes that com- produced by such asymmetric cell division may differ in\\nplete the differentiation process. For instance, a few key reg- size, shape, and/or composition, or their genes may be in\\nulatory transcription factors create the different mating types different states of activity or potential activity. The differ-\\nof budding yeast and coordinate the numerous genes neces- ences in these internal signals confer different fates on the\\nsary to turn a precursor cell into a muscle cell, two exam- two cells. \\nples that we discuss in this chapter.  Here we discuss some general features of how different\\nTypically we think of cell fates in terms of the differenti- cell types are generated, culminating with the best-understood\\nated cell types that are formed. A quite different cell fate, complex cell lineage, that of the nematode Caenorhabditis \\nprogrammed cell death,also is absolutely crucial in the for- elegans. In later sections, we focus on examples of the mo-\\nmation and maintenance of many tissues. A precise genetic lecular mechanisms that determine particular cell types in\\nregulatory system, with checks and balances, controls cell yeast, Drosophila, and mammals. \\ndeath just as other genetic programs control cell differentia-\\ntion. In this chapter, then, we consider the life cycle of cells—\\nStem Cells Give Rise to Stem Cells \\ntheir birth, their patterns of division (lineage), and their\\nand to Differentiating Cells \\ndeath. These aspects of cell biology converge with develop-\\nmental biology and are among the most important processes Stem cells, which give rise to the specialized cells composing\\nregulated by various signaling pathways discussed in earlier the tissues of the body, exhibit several patterns of cell divi-\\nchapters.  sion. A stem cell may divide symmetrically to yield two\\ndaughter stem cells identical to itself (Figure 22-2a). Alter-\\nnatively, a stem cell may divide asymmetrically to generate a\\n22.1 The Birth of Cells copy of itself and a derivative stem cell that has more-\\nrestricted capabilities, such as dividing for a limited period of\\nMany descriptions of cell division imply that the parental time or giving rise to fewer types of progeny compared with\\ncell gives rise to two daughter cells that look and behave the parental stem cell (Figure 22-2b). A pluripotent(or multi-\\nexactly like the parental cell, that is, cell division is sym- potent) stem cell has the capability of generating a number of\\nmetric, and the progeny do not change their properties. But different cell types, but not all. For instance, a pluripotent\\nif this were always the case, none of the hundreds of dif- blood stem cell will form more of itself plus multiple types of\\nferentiated cell types would ever be formed. Differences blood cells, but never a skin cell. In contrast, a unipotent\\namong cells can arise when two initially identical daughter stem cell divides to form a copy of itself plus a cell that can\\ncells diverge by receiving distinct developmental or envi- form only one cell type. In many cases, asymmetric division\\nronmental signals. Alternatively, the two daughter cells of a stem cell generates a progenitor cell, which embarks on\\nmay differ from “birth,” with each inheriting different a path of differentiation, or even a terminally differentiating\\nparts of the parental cell (see Figure 22-1). Daughter cells cell (Figure 22-2c, d). \\n22.1 • The Birth of Cells 3\\nproperties. For example, the early mouse embryo passes\\nStem\\ncell through an eight-cell stage in which each cell can give rise\\nto any cell type of the embryo. If the eight cells are experi-\\n(a) mentally separated and individually implanted into a suit-\\nSymmetric able foster mother, each can form a whole mouse with no\\ndivision\\nparts missing. This experiment shows that the eight cells\\nStem\\ncell are all able to form every tissue; that is, they are totipotent.\\nThus the subdivision of body parts and tissue fates among\\nthe early embryonic cells has not irreversibly occurred at\\n(b)\\nAsymmetric the eight-cell stage. At the 16-cell stage, this is no longer\\ndivision\\nMore-restricted true; some of the cells are committed to particular differ-\\npotential stem  entiation paths. \\nFPO\\ncell\\nQuite different specialized cell types can arise from a\\n(c) common precursor cell. A hematopoietic stem cell can give\\nAsymmetric\\nrise to all the multifarious types of blood cell. However, a\\ndivision\\nwhole series of cell divisions is not required; a single divi-\\nProgenitor cell\\nsion of a pluripotent precursor cell can yield distinct prog-\\neny. For instance, lineage studies in which cells are marked\\nby stable infection with a detectable retrovirus have shown\\n(d)\\nAsymmetric that neurons and glial cells arise from a single division of a\\ndivision\\nTerminally particular precursor cell. These cell types are quite differ-\\ndifferentiating ent: neurons propagating and transmitting electrical sig-\\ncell\\nnals and glial cells providing electrical insulation and\\nsupport. The precursor that generates neurons and glial\\n▲FIGURE 22-2 Patterns of stem-cell division. Symmetric\\ncells is not a stem cell, since it is incapable of self-\\ndivision of a stem cell (a) produces two identical daughter cells\\nrenewal; presumably the neuron–glial cell precursor arises\\nwith the same potential as the parental stem cell. Asymmetric\\nfrom a stem cell further back in the lineage. Recent obser-\\ndivision of a stem cell always generates one daughter that is a\\nfunctional duplicate of the parental cell and one daughter that vations have raised the interesting possibility that stem\\nmay be a stem cell with a more-restricted set of possible cells for one tissue may be induced under certain condi-\\nprogeny cells (b), a progenitor (precursor) cell that undergoes tions to act as stem cells for a rather different tissue. As we\\nmore divisions before the progeny begin differentiating (c), or a discuss below, postnatal animals contain stem cells for\\ncell that begins differentiating immediately with no further cell many tissues including the blood, intestine, skin, ovaries\\ndivisions (d). and testes, muscle, liver, and even the adult brain where lit-\\ntle cell division normally occurs. In muscle and liver, stem\\ncells are most important in healing, as relatively little cell\\ndivision occurs in the adult tissues otherwise. \\nThe two critical properties of stem cells that distin-\\nguish them from all other cells are the ability to reproduce\\nthemselves indefinitely by symmetric division, often called\\nself-renewal,and the ability to divide asymmetrically to Cultured Embryonic Stem Cells Can Differentiate \\nform one daughter stem cell identical to itself and one into Various Cell Types\\ndaughter  cell  that  is  different  and  usually  of  more-\\nrestricted potential. In this way, mitotic division of stem Embryonic stem (ES) cells can be isolated from early em-\\ncells preserves a population of undifferentiated cells while bryos and grown in culture (Figure 22-3a). Cultured ES cells\\nsteadily producing a stream of differentiating cells. Al- can differentiate into a wide range of cell types, either in\\nthough some types of precursor cells can divide symmet- vitro or after reinsertion into a host embryo. When grown\\nrically to form more of themselves, they do so only for in suspension culture, human ES cells first differentiate into\\nlimited periods of time. Moreover, in contrast to stem multicellular aggregates, called embryoid bodies, that resem-\\ncells, the asymmetric division of a precursor cell generates ble early embryos. When these are subsequently transferred\\ntwo distinct daughter cells, neither of which is identical to to a solid medium, they grow into confluent cell sheets con-\\nthe parental precursor cell.  taining a variety of differentiated cell types including neural\\nThe fertilized egg, or zygote, is a single cell with the  cells and pigmented and nonpigmented epithelial cells (Fig-\\ncapability to generate all the cell types of the body. Al- ure 22-3b). Under other conditions, ES cells have been in-\\nthough not technically a stem cell because it is not self- duced to differentiate into precursors for various types of\\nrenewing, the zygote does give rise to cells with stem-cell blood cells.\\n4 CHAPTER 22 • Cell Birth, Lineage, And Death\\n!EXPERIMENTAL FIGURE 22-3 (a) (b)\\nCleavage\\nEmbryonic stem (ES) cells can be stage embryo\\nmaintained in culture and form\\ndifferentiated cell types.(a) Human\\nblastocysts are grown from cleavage-stage\\nInner\\nembryos produced by in vitro fertilization.\\ncell mass FPO\\nThe inner cell mass is separated from the Cultured\\nsurrounding extra-embryonic tissues and Blastocoel blastocyst\\nplated onto a layer of fibroblast cells that\\nhelp to nourish the embryonic cells. Trophoblast\\nIndividual cells are replated and form\\nEmbryoid body\\ncolonies of ES cells, which can be\\nmaintained for many generations and can Isolated inner\\ncell mass\\nbe stored frozen. (b) In suspension culture,\\nhuman ES cells differentiate into\\nmulticellular aggregates (embryoid bodies) Fibroblast\\nfeeder cells\\n(top). After embryoid bodies are\\ntransferred to a gelatinized solid medium, FPO\\nthey differentiate further into confluent cell\\nFPO\\nsheets containing a variety of\\ndifferentiated cell types including neural Dissociate cells\\ncells (middle), and pigmented and\\nNeurons\\nnonpigmented epithelial cells (bottom).\\nNew feeder\\n[Parts (a) and (b) adapted from J. S. Odorico  cells\\nReplate\\net al., 2001, Stem Cells19:193–204.]\\nFPO\\nEstablished\\nES cell cultures\\nEpithelial cells\\nThe possibility of using stem cells therapeutically the feasibility of using stem cells for such purposes can be as-\\nto restore or replace damaged tissue is fueling sessed adequately. ❚\\nmuch research on how to recognize and culture\\nthese remarkable cells from embryos and from various tis- Apart from their possible benefit in treating disease, ES\\nsues in postnatal (adult) animals. For example, if neurons cells have already proven invaluable for producing mouse\\nthat produce the neurotransmitter dopamine could be gen- mutants useful in studying a wide range of diseases, devel-\\nerated from stem cells grown in culture, it might be possible opmental mechanisms, behavior, and physiology. By tech-\\nto treat people with Parkinson’s disease who have lost such niques described in Chapter 9, it is possible to eliminate or\\nneurons. For such an approach to succeed, a way must be modify the function of a specific gene in ES cells (see Figure\\nfound to direct a population of embryonic or other stem cells 9-38). If the disrupted gene is not required for growth of ES\\nto form the right type of dopamine-producing neurons, and cells, then the mutated ES cells can be employed to produce\\nrejection by the immune system nust be prevented. In one on- mice with a gene knockout(see Figure 9-39). Analysis of the\\ngoing study in which embryonic neurons were transplanted effects caused by deleting or modifying a gene in this way\\ninto more than 300 Parkinson’s patients, some of the inserted often provides clues about the normal function of the gene\\ncells have survived for more than 12 years and have provided and its encoded protein. \\nsignificant clinical improvement. However, the fetal tissue\\nused in this study is scarce and its use is controversial. Stem\\nTissues Are Maintained by Associated\\ncells grown from very early embryos are another option for\\nPopulations of Stem Cells \\ntreating Parkinson’s disease and perhaps other neurodegen-\\nerative conditions such as Alzheimer’s disease. Similar pos- Many differentiated cell types are sloughed from the body\\nsibilities exist for generating blood, pancreas, and other cell or have life spans that are shorter than that of the organism.\\ntypes. Many important questions must be answered before Disease and trauma also can lead to loss of differentiated\\n22.1 • The Birth of Cells 5\\ncells. Since these cells generally do not divide, they must be continuously regenerate from a stem-cell population located\\nreplenished from nearby stem-cell populations.  deep in the intestinal wall in pits called crypts(Figure 22-4a).\\nOur skin, for instance, is a multilayered epithelium(the The stem cells produce precursor cells that proliferate and dif-\\nepidermis) underlain by a layer of stem cells that give rise ferentiate as they ascend the sides of crypts to form the sur-\\nboth to more of themselves and to keratinocytes,the major face layer of the finger-like gut projections called villi,across\\ncell type in skin. As keratinocytes move toward the outer sur- which intestinal absorption occurs. Pulse-chase labeling ex-\\nface, they become increasingly flattened and filled with ker- periments have shown that the time from cell birth in the\\natin intermediate filaments. It normally takes about 15–30 crypts to the loss of cells at the tip of the villi is only about 2\\ndays for a newly “born” keratinocyte in the lowest layer to to 3 days (Figure 22-4b). Thus enormous numbers of cells\\ndifferentiate and move to the topmost layer. The “cells” must be produced continually to keep the epithelium intact.\\nforming the topmost layer are actually dead and are contin- The production of new cells is precisely controlled: too little\\nually shed from the surface.  division would eliminate villi and lead to breakdown of the in-\\nIn contrast to epidermis, the epithelium lining the small testinal surface; too much division would create an excessively\\nintestine is a single cell thick (Figure 6-4). This thin layer keeps large epithelium and also might be a step toward cancer.\\ntoxins and pathogens from entering our bodies and also trans- Specific signals are required for creating and maintaining\\nports nutrients essential for survival from the intestinal lumen stem-cell populations. In the skin and the intestinal epithelium,\\ninto the body (Chapter 7). The cells of the intestinal epithelium stem-cell growth is regulated in part by !-catenin,a protein\\nthat helps link certain cell-cell junctions to the cytoskeleton\\n(see Figure 6-7) and also functions as a signal transducer in \\n(a)\\nthe Wnt pathway (see Figure 15-32). Overproduction of active\\nDirection of !-catenin leads to excess proliferation of the intestinal epithe-\\ncell migration lium and of hair follicles, which are specialized structures de-\\nrived from skin stem cells and embedded in the epidermis.\\nVillus\\nBlocking the function of !-catenin by interfering with the TCF\\ntranscription factor that it activates abolishes the stem cells in\\nDifferentiated\\nCrypt-villus region the intestine. In the absence of !-catenin function, skin stem\\njunction cells form epidermis, but the hair follicles and hair do not\\nform. It seems likely that Wnt signaling, or at least compo-\\nFPO nents of the intracellular pathway, is critical for forming, main-\\ntaining, or activating stem cells in a variety of tissues. \\nProliferative\\nregion\\nCrypt\\n\"EXPERIMENTAL FIGURE 22-4 Regeneration of the\\nintestinal epithelium from stem cells can be demonstrated in\\nStem cell pulse-chase experiments.(a) Schematic drawing of the lining of\\nregion the small intestine, which contains numerous villi formed from a\\nStem cells column of cells. These epithelial cells are born near the base of\\nPaneth cells\\npits (crypts) located between the villi. Located at the very bottom\\nof crypts are Paneth cells, a type of support cell; just above these\\n(b) are four to six stem cells, which divide about once a day, forming\\nprecursor cells that also actively divide. As the differentiated cells\\nenter the epithelium of a villus, they stop dividing and begin\\ntaking up nutrients from the gut. (b) Results from a pulse-chase\\nexperiment in which radioactively labeled thymidine (the pulse)\\nwas added to a tissue culture of the intestinal epithelium. \\nVilli\\nDividing cells incorporated the labeled thymidine into their newly\\nFPO synthesized DNA. The labeled thymidine was washed away and\\nreplaced with nonlabeled thymidine (the chase) after a brief\\nperiod; cells that divided after the chase did not become labeled.\\nThese micrographs show that 40 minutes after labeling, all the\\nlabel is in cells near the base of the crypt. At later times, the\\nlabeled cells are seen progressively farther away from their point\\nCrypt of birth in the crypt. Cells at the top are shed. This process\\nensures contant replenishment of the gut epithelium with new\\ncells. [Part (a) adapted from C. S. Potten, 1998, Philos. Trans. R. Soc.\\nLondon, Ser. B353:821. Part (b) from P. Kaur and C. S. Potten, 1986, \\n40 min 1 day 2 days 3 days Cell Tiss. Kinet.19:601.]\\n6 CHAPTER 22 • Cell Birth, Lineage, And Death\\nSkin also contains dendritic epidermal T cells, an factor \"(TGF\"), and TGF!. Discovering how all these sig-\\nimmune-system cell that produces a certain form nals work together to control growth and stimulate healing\\nof the T-cell receptor (see Table 14-1). When den- is a substantial challenge that will advance our understand-\\ndritic epidermal T cells are genetically modified so they do ing of diseases such as psoriasis and skin cancer and perhaps\\nnot produce T-cell receptors, wound healing is slow and less pave the way for effective treatments. ❚\\ncomplete than in normal skin. Normal healing is restored by\\naddition of keratinocyte growth factor. The current hypoth- Another continuously replenished tissue is the blood,\\nesis is that when dendritic epidermal T cells recognize anti- whose stem cells are located in the bone marrow in adult an-\\ngens on cells in damaged tissue, they respond by producing imals. The various types of blood cells all derive from a sin-\\nstimulating proteins, such as keratinocyte growth factor, that gle type of pluripotent hematopoietic stem cell,which gives\\npromote keratinocyte growth and wound healing. Many rise to the more-restricted myeloid and lymphoid stem cells\\nother signals also control the growth of skin cells, including (Figure 22-5). The frequency of hematopoietic stem cells is\\nWnt/!-catenin, Hedgehog, calcium, transforming growth about 1 cell per 104bone marrow cells, even lower than the\\nG-CSF\\nGranulocytes\\nIL-3, GM-CSF, SCF, IL-6\\nGM-CFC M-CSF\\nMonocytes\\nMyeloid\\nstem cell IL-3, GM-CSF\\nFPO\\nEo-CFC Eosinophils\\nEpo, SCF Epo\\nGM-CSF\\nIL-3\\nBFU-E CFU-E Erythrocytes\\nPluripotent\\nhematopoietic SCF, Tpo, IL-3, GM-CSF\\nstem cell\\nMegakaryocytes\\nCFC-mega\\n(platelet-forming cells)\\nLymphoid\\nstem cell\\nIIL-2, IL-7, IL-12, SDF-1,\\nFLT-3 ligand, TNF-α,\\nTGF-β1\\nT-CFC B-CFC T and B cells of the\\nimmune system\\n▲FIGURE 22-5 Formation of differentiated blood cells from eliminated and the precursor cells introduced. Further \\nhematopoietic stem cells in the bone marrow. Pluripotent cytokine-induced proliferation, commitment, and differentiation of\\nstem cells may divide symmetrically to self-renew (curved arrow) the precursor cells give rise to the various types of blood cells.\\nor divide asymmetrically to form myeloid and lymphoid stem Some of the cytokines that support this process are indicated\\ncells (light green). Although these stem cells are capable of  (red labels). GM # granulocyte-macrophage; Eo # eosinophil; \\nself-renewal, they are committed to one of the two major E # erythrocyte; mega # megakaryocyte; T #T-cell; B # B-cell;\\nhematopoietic lineages. Depending on the types and amounts of CFU # colony-forming unit; CSF # colony-stimulating factor; \\ncytokines present, the myeloid and lymphoid stem cells generate IL # interleukin; SCF # stem cell factor; Epo # erythropoietin;\\nvarious precursor cells (dark green), which are incapable of  Tpo # thrombopoietin; TNF # tumor necrosis factor; TGF #\\nself-renewal. These cells are detected by their ability to form transforming growth factor; SDF # stromal cell–derived factor;\\ncolonies containing the differentiated cell types shown at right, FLT-3 ligand # ligand for fms-like tyrosine kinase receptor 3.\\nmeasured as “colony-forming cells (CFCs).” The colonies are [Adapted from M. Socolovsky et al., 1998, Proc. Nat’l. Acad. Sci. USA\\ndetected on the spleen of animals that have had their own cells 95:6573.]\\n22.1 • The Birth of Cells 7\\nfrequency of intestinal stem cells in crypts. Numerous extra- Numerous genes have been found to regulate the forma-\\ncellular growth factors called cytokinesregulate proliferation tion, maintenance, and properties of meristems. Many of\\nand differentiation of the precursor cells for various blood- these genes encode transcription factors. For instance, a hier-\\ncell lineages. For example, erythropoietincan activate several archy of regulators, particularly transcription factors, con-\\ndifferent intracellular signal-transduction pathways, leading trols the separation of differentiating cells from SAMs as\\nto changes in gene expression that promote formation of  leaves form; similarly, three types of regulators control for-\\nerythrocytes (see Figure 14-7).  mation of the floral organs from floral meristems (Chapter\\nThe hematopoietic lineage originally was worked out by 15). In both cases, a cascade of gene interactions occurs, with\\ninjecting the various types of precursor cells into mice whose earlier transcription factors causing production of later ones.\\nprecursor cells had been wiped out by irradiation. By ob- At the same time, cells are dividing and the differentiating\\nserving which blood cells were restored in these transplant ones are spreading away from their original birth sites. ❚\\nexperiments, researchers could infer which precursors or ter-\\nminally differentiated cells (e.g., erythrocytes, monocytes)\\narise from a particular type of precursor. The first step in\\n(a) (d)\\nthese experiments was separation of the different types of\\nhematopoietic precursors. This is possible because each type 1\\n1 3\\nproduces unique combinations of cell-surface proteins that 1 3 3\\n1 6 5 3\\ncan serve as type-specific markers. If bone marrow extracts 6 8 5\\n1 8 7 3\\nare treated with fluorochrome-labeled antibodies for these 10 x 8 picas 6 i 5\\nmarkers, cells with different surface markers can be sepa- 4 i 7 2\\n6 7 5\\nrated in a fluorescence-activated cell sorter (see Figure 5-34).  4 6 5 2\\n4 4 2\\n4 2\\n2\\nFPO\\nTo date, bone marrow transplants represent the\\n(b)\\nmost successful and widespread use of stem cells\\nin medicine. The stem cells in the transplanted\\nmarrow can generate new, functional blood cells in pa-\\ntients with certain hereditary blood diseases and in cancer\\npatients who have received irradiation and/or chemother- 10 x 8 picas\\ni\\napy, both of which destroy the bone marrow cells as well\\nas cancer cells. Recent work is directed at exploring\\nwhether embryonic stem cells can be induced to differen-\\ntiate into cells types that would be useful therapeutically.\\nFor example, mouse stem cells treated with inhibitors of (c)\\nphosphatidylinositol-3 kinase, a regulator in one of the 7 8\\nphosphoinositide signaling pathways (Chapter 14), turn\\ninto cells that resemble pancreatic ! cells in their produc- 1\\n6\\ntion of insulin, their sensitivity to glucose levels, and their\\naggregation into structures reminiscent of pancreas struc- 10 x 8 picas 4\\n3\\ntures. Implantation of these cells into diabetic mice re-\\n2\\nstored their growth, weight, glucose levels, and survival 5\\nrates to normal. ❚\\nStem cells in plants are located in meristems,pop- ▲FIGURE 22-6 Cell fates in meristems of Arabidopsis.In\\nulations of undifferentiated cells found at the tips these longitudinal sections through a shoot apical meristem, cell\\nof growing shoots. Shoot apical meristems (SAMs) nuclei are revealed by staining with propidium iodide, which\\nproduce leaves and shoots, and of course more stem cells binds to DNA. (a) The shoot apical meristem (SAM) produces\\nthat constitute the nearly immortal meristems. Meristems shoots, leaves, and more meristem. Flower production occurs\\nwhen the meristem switches from leaf/shoot production to\\ncan persist for thousands of years in long-lived species such\\nflower production, concomitant with an increase in the number\\nas redwood trees and bristlecone pines. As a plant grows, the\\nof meristem cells to form floral meristems (FMs), as shown\\ncells “left behind” the meristems are encased in rigid walls\\nhere. (b) Cells in a SAM exhibit different fates and behaviors.\\nand can no longer grow. SAMs can split to form branches,\\nCells divide rapidly in the peripheral zone (PZ) to produce leaves\\neach branch with its own SAM, or be converted into floral\\nand in the rib zone (Rib) to produce central shoot structures.\\nmeristems (Figure 22-6). Floral meristems give rise to the\\nCells in the central zone (CZ) divide more slowly, producing an\\nfour floral organs—sepals, stamens, carpels, and petals—that\\nongoing source of meristem and contributing cells to the PZ and\\nform flowers. Unlike SAMs, floral meristems are gradually Rib. Scale bars, 50 $m. [See E. Meyerowitz, 1997, Cell88:299;\\ndepleted as they give rise to the floral organs. micrographs courtesy Elliot Meyerowitz.]\\n8 CHAPTER 22 • Cell Birth, Lineage, And Death\\nCell Fates Are Progressively Restricted  genome comes from the successful production of cloned an-\\nDuring Development imals by nuclear-transfer cloning. In this procedure, the \\nnucleus of an adult (somatic) cell is introduced into an egg\\nThe eight cells resulting from the first three divisions of a that lacks its nucleus; the manipulated egg, which contains\\nmammalian zygote (fertilized egg) all look the same. As the diploid number of chromosomes and is equivalent to a \\ndemonstrated experimentally in mice, each of the cells has zygote, then is implanted into a foster mother. The only\\nthe potential to give rise to a complete animal. Additional di- source of genetic information to guide development of the\\nvisions produce a mass, composed of ≈64 cells, that separates embryo is the nuclear genome of the donor somatic cell. The\\ninto two cell types: trophectoderm, which will form extra- frequent failure of such cloning experiments, however, raises\\nembryonic tissues like the placenta, and the inner cell mass,\\nwhich gives rise to the embryo proper. The inner cell mass (a) Primitive node Primitive streak\\neventually forms three germ layers, each with distinct fates.\\nOne layer, ectoderm,will make neural and epidermal cells; Epiblast Amnioblasts\\nanother,mesoderm,will make muscle and connective tissue;\\nthe third layer, endoderm, will make gut epithelia (Figure \\n22-7). If cells from any of these layers are transplanted into\\nYolk sac\\none of the other layers, they do not give rise to cells appro-\\npriate to their new location. This conclusion is based on ex-\\nperiments with chimeric animals composed of chicken and Invaginating Hypoblast\\nmesoderm cells\\nquail cells. Embryos composed of cells from both bird species\\ndevelop fairly normally, yet the cells derived from each donor (b)\\nare distinguishable under the microscope. Thus the contri-\\nbutions of the different donor cells to the final bird can be \\nascertained. \\nOnce the three germ layers are established, they subse-\\nquently divide into cell populations with different fates. For\\ninstance, the ectoderm becomes divided into those cells that\\nare precursors to the skin epithelium and those that are pre-\\ncursors to the nervous system. There appears to be a pro- FPO\\ngressive restriction in the range of cell types that can be\\nformed from stem cells and precursor cells as development (c)\\nproceeds. An early embryonic stem cell, as we’ve seen, can ECTODERM MESODERM ENDODERM\\nform every type of cell; an ectodermal cell has a choice be- Central nervous system Skull Liver\\ntween neural and epidermal fates; and a keratinocyte pre- Retina and lens  Head, skeletal muscle Pancreas\\ncursor can form skin but not neurons.  Cranial and sensory  Skeleton Urinary bladder\\nGanglia and nerves Dermis of skin Epithelial parts of: \\nAnother restriction that occurs early in animal develop-\\nPigment cells Connective tissue    trachea\\nment is the setting aside of cells that will form the germ line, Head connective tissue Urigential system    lungs\\nthat is, the stem cells and precursor cells that eventually will Epidermis Heart    pharynx\\nHair Blood, lymph cells    thyroid\\ngive rise to eggs in a female and sperm in a male. Only the\\nMammary glands Spleen\\ngenome of the germ line will ever be passed on to progeny.\\nThe setting aside of germ-line cells early in development has ▲FIGURE 22-7 Fates of the germ layers in animals. During\\nbeen hypothesized to protect chromosomes from damage by the early period of mammalian development, cells migrate to\\nreducing the number of rounds of replication they undergo create the three germ layers: ectoderm, mesoderm, and\\nor by allowing special packaging of the cells that are critical endoderm. Cells in the different layers have largely distinct fates\\nto heredity. Whatever the reason, the early segregation of the and therefore represent distinct cell lineages. The detailed cell\\ngerm line is widespread (though not universal) among ani- lineage, however, is not exactly the same in different individuals.\\n(a) Shown here is a sketch of a human embryo about 16 days\\nmals. In contrast, plants do nothing of the sort; most meri-\\nafter fertilization of an egg. The first cells to move from the\\nstems can give rise to germ-line cells. \\nepiblast into the interior form endoderm, and they are followed by\\nOne consequence of the early segregation of germ-line\\ninvaginating cells that become mesoderm. The remaining epiblast\\ncells is that the loss or rearrangement of genes in somatic\\ncells become ectoderm. Hypoblast #covering layer. Amnioblasts #\\ncells would not affect the inherited genome of a future zy-\\ncells that will line amniotic cavity. Yolk sac #reminder of our \\ngote. Although segments of the genome are rearranged and\\negg-bound ancestors. (b) Scanning electron micrograph of a \\nlost during development of lymphocytes from hematopoi-\\ncross section of a similar-stage embryo. (c) Some of the tissue\\netic precursors, most somatic cells seem to have an intact derivatives of the three germ layers are listed. [Part (b) from \\ngenome, equivalent to that in the germ line. Evidence that at T. W. Sadler, 2000, Langman’s Medical Embryology,8th ed. (Baltimore:\\nleast some somatic cells have a complete and functional Lippincott Williams and Wilkins), pp. 64–65, Figure 4.3).]\\n22.1 • The Birth of Cells 9\\nquestions about how many adult somatic cells do in fact individuals. The best-documented example of a reproducible\\nhave a complete functional genome. Even the successes, like pattern of cell divisions comes from the nematode C. elegans.\\nthe famous cloned sheep “Dolly,” appear to have some med- Scientists have traced the lineage of all the somatic cells in\\nical problems. Even if differentiated cells have a physically C. elegansfrom the fertilized egg to the mature worm by fol-\\ncomplete genome, clearly only portions of it are transcrip- lowing the development of live worms using Nomarski in-\\ntionally active (Chapter 10). Whether the genome of a dif- terference microscopy (Figure 22-8).\\nferentiated cell can revert to having the full developmental About 10 rounds of cell division, or fewer, create the \\npotential characteristic of an embryonic cell is a matter of adult worm, which is about 1 mm long and 70 $m in di-\\nconsiderable debate. A cell could, for example, have an intact ameter. The adult worm has 959 somatic cell nuclei (her-\\ngenome, but be unable to properly reactivate it due to inher- maphrodite form) or 1031 (male). The number of somatic \\nited chromatin states.  cells is somewhat fewer than the number of nuclei because \\nThese observations raise two important questions: How some cells contain multiple nuclei (i.e., they are syncytia). \\nare cell fates progressively restricted during development? Remarkably, the pattern of cell divisions starting from a \\nAre these restrictions irreversible? In addressing these ques- C. elegansfertilized egg is nearly always the same. As we \\ntions, it is important to remember that what a cell in its nor- discuss later in the chapter, many cells that are generated \\nmal in vivo location will do may differ from what a cell is during development undergo programmed cell death and \\ncapable of doing if it is manipulated experimentally. Thus the are missing in the adult worm. The consistency of the \\nobserved limits to what a cell can do may result from natural C. eleganscell lineage does not result entirely from each \\nregulatory mechanisms or may reflect a failure to find con- newly born cell inheriting specific information about its \\nditions that reveal the cell’s full potential. destiny. In some cases, various signals direct initially iden-\\nAlthough our focus in this chapter is on how cells be- tical cells to particular fates, and the outcomes of these sig-\\ncome different, their ability to remain the same also is criti- nals are consistent from one animal to the next. That is, \\ncal to the functioning of tissues and the whole organism. at their birth cells are not necessarily “hard wired” by \\nNondividing differentiated cells with particular characteris- their own internal inherited instructions to follow a par-\\ntics must retain them, sometimes for many decades. Stem ticular path of differentiation. \\ncells that divide regularly, such as a skin stem cell, must pro- The first few cell divisions in C. elegansproduce six dif-\\nduce one daughter cell with the properties of the parental ferent founder cells,each with a separate fate as shown in\\ncell, retaining its characteristic composition, shape, behavior, Figure 22-9a, b. The initial division is asymmetric, giving rise\\nand responses to specific external signals. Meanwhile, the to P1 and the AB founder cell. Further divisions in the P lin-\\nother daughter cell with its own distinct inheritance, as the eage form the other five founder cells. Some of the signals\\nresult of asymmetric cell division, embarks on a differentia- controlling the asymmetry are known. For example, Wnt\\ntion pathway, which may be determined by the signals the signals from the P2 precursor control the asymmetric divi-\\ncell receives.  sion of the EMS cell into E and MS founder cells. Wnt sig-\\nnaling (see Figure 15-32) is also used in other asymmetric\\ndivisions in worms. \\nThe Complete Cell Lineage of C. elegans\\nThe bilateral symmetry of the worm implies duplication\\nIs Known \\nof lineages on the two sides; curiously, though, functionally\\nIn the development of some organisms, cell lineages are equivalent cells on each side can arise from a pattern of \\nunder tight genetic control and thus are identical in all indi- division that is different on the two sides. Some of the em-\\nviduals of a species. In other organisms the exact number bryonic cells function as stem cells, dividing repeatedly to\\nand arrangement of cells vary substantially among different form more of themselves or another type of stem cell, while\\nDorsal nerve\\nPharynx cord Intestinal lumen\\nMouth Intestine Intestinal nuclei\\nExcretory Anus\\ncell Gonad\\nVentral nerve cord Rectum\\nVentral\\nnuclei\\nnerve cord 20 $m\\n▲FIGURE 22-8 Newly hatched larva of C.elegans.Many of in this micrograph obtained by Nomarski interference microscopy.\\nthe 959 somatic cell nuclei in this hermaphrodite form are visualized [From J. E. Sulston and H. R. Horvitz, 1977, Devel. Biol.56:110.]\\n10 CHAPTER 22 • Cell Birth, Lineage, And Death\\n(a) (b)\\nP0\\nAB P1\\nneurons\\nhypodermis EMS P2\\npharyngeal muscle\\nbody muscle MS E\\nother body muscle gut P3 C FPO\\npharyngeal muscle hypodermis\\nneurons body muscle\\nglands neurons\\nsomatic gonad P4 D\\nother germ cells body muscle\\n(c) Zygote\\ns\\nsi\\ne\\nn Germ\\nge FPO line\\no Pharynx Intestine\\ny\\nr\\nb\\nm\\nE\\ns\\nn\\na\\ng\\ne\\nel\\nC.  Neurons\\nf \\no\\ng \\nn\\ngi\\nS a\\nm\\nN\\nI\\nO e \\ns Vulva\\nI p Epidermis\\nT a Somatic gonad\\nL\\nC -\\ne\\nE m ▲FIGURE 22-9 C.eleganslineage.(a) Pattern of the aside very early, as in most animals. All the other lineages give\\nN Ti first few divisions starting with P0 (the zygote) and leading rise to somatic cells. (b) Light micrographs of the first few\\nN n:  to formation of the six founder cells (yellow highlights). The divisions of the embryo that generate the founder cells with cells\\nO o\\nC ati first division, into P1 and AB, is asymmetric. Further labeled as in part (a). The texture of the cells shows the presence\\nm divisions in the P lineage generates the other founder of organelles and is revealed by differential interference contrast\\n \\nA ni cells. Note that more than one lineage can lead to the microscopy, sometimes called Nomarski microscopy. (c) Full\\nDI o A same tissue type (e.g., muscle or neurons). The EMS cell lineage of the entire body of the worm, showing some of the\\nE e is so named because it is the precursor to most of the tissues formed. Note that any particular cell undergoes relatively\\nd\\nM Vi endoderm and mesoderm. The lineage beginning with the few divisions, typically fewer than 15. [Einhard Schierenberg\\nP4 cell gives rise to all of the germ-line cells, which are set Zoologisches Institut, Universität Köln.]\\nspinning off differentiating cells that give rise to a particular of development occurs too early (precocious development) or\\ntissue. In one pattern of division, one of the daughter cells too late (retarded development). An example of the former is\\nfrom a division initiates a repeat of the lineage pattern of the a premature cell division to yield a cell that differentiates and\\nprevious few divisions. The complete lineage of C. elegans a cell that dies; as a result, the lineage that should have fol-\\nis shown in Figure 22-9c. This organism has been a powerful lowed from the dead cell never happens. In the latter case,\\nmodel system for genetic studies to identify the regulators the delayed occurrence of a lineage causes juvenile structures\\nthat control cell lineages in time and space. to be produced, incorrectly, in more mature animals. In both\\ncases, the character of a parental cell is, in essence, changed\\nto the character of a cell at a different stage of development. \\nHeterochronic Mutants Provide Clues \\nOne example of precocious development in C. elegans\\nAbout Control of Cell Lineage \\ncomes from loss-of-function mutations in the lin-14gene,\\nIntriguing evidence for the genetic control of cell lineage has which cause premature formation of the PDNB neuroblast\\ncome from isolation and analysis of heterochronicmutants. (Figure 22-10a). The lin-14gene and several others found to\\nIn these mutants, a developmental event typical of one stage be defective in heterochronic worm mutants encode RNA-\\n22.1 • The Birth of Cells 11\\n(b) L1 L2 L3/4 Adult\\nlin-14, lin-28 lin-14, lin-28 lin-14, lin-28 lin-14, lin-28\\nmRNA mRNA mRNA mRNA\\nlin-4 lin-4 lin-4 let-7\\n(a) RNA RNA RNA RNA\\nFPO\\nWild type lin-14 mutant\\nLIN-14, LIN-28\\nV5 V5 proteins lin-41 mRNA\\nL1\\nlin-41 mRNA lin-41 mRNA lin-41 mRNA\\nPDNB let-7\\nRNA\\nX\\nLIN-41 LIN-41 LIN-41\\nprotein protein protein lin-29 mRNA\\nL2 PDNB\\n_ _ _\\nlin-29 mRNA lin-29 mRNA lin-29 mRNA LIN-29\\nprotein\\nn\\nei LIN-14, LIN-28\\not\\nr\\np LIN-41\\nr \\nA oel lin-4 let-7 LIN-29\\nNv\\nRe\\nl\\n▲FIGURE 22-10 Timing of specific types of cell division RNA to the 3% untranslated regions (UTRs) of lin-14and lin-28\\nduring development of C.elegans.(a) The pattern of cell mRNAs prevents translation of these mRNAs into protein. This\\ndivision for the V5 cell of C. elegansis shown for normal worms occurs following the first larval (L1) stage, permitting\\nand for a heterochronic mutant called lin-14.In the lin-14mutant, development to proceed to the later larval stages. Starting in the\\nthe pattern of cell division (red arrows) that normally occurs only fourth larval stage (L4), production of let-7RNA begins. It\\nin the second larval stage (L2) occurs in the first larval stage (L1), hybridizes to lin-14,lin-28, and lin-41mRNAs, preventing their\\ncausing the PDNB neuroblast to be generated prematurely. In  translation. LIN-41 protein is an inhibitor of translation of the \\nthe mutant, the V5 cell behaves during L1 like cell “X” (purple) lin-29mRNA, so the appearance of let-7RNA allows production\\nnormally does in L2. The inference is that the LIN-14 protein of LIN-29 protein, which is needed for generation of adult cell\\nprevents L2-type cell divisions, although precisely how it does so lineages. LIN-4 may also bind to lin-41RNA at later stages. Only\\nis unknown. (b) Two small regulatory RNAs, lin-4and let-7,serve the 3% UTRs of the mRNAs are depicted. [Adapted from B. J.\\nas coordinating timers of gene expression. Binding of the lin-4 Reinhart et al., 2000, Nature403:901.]\\nbinding or DNA-binding proteins, which presumably coordi- other miRNAs during the life cycle of C. elegansserve as a regu-\\nnate expression of other genes. However, two other genes (lin- latory clock for cell lineage. Molecules related to let-7RNA have\\n4and let-7) involved in regulating the timing of cell divisions been identified in many other animals including vertebrates and\\nwere initially extremely puzzling, as they appeared not to en- insects; since their production is temporally regulated in these an-\\ncode any protein. To discover the products of these genes, sci- imals as well, they may serve a similar function in their develop-\\nentists first determined which pieces of genomic DNA could ment as in C. elegans. How production of these regulatory\\nrestore gene function, and therefore proper cell lineage, to mu- miRNAs is temporally controlled is not yet known, but they have\\ntants defective in each gene. They then did the same thing with turned out to play many roles in regulating gene expression\\ngenomic DNA from the corresponding genomic regions of dif- (Chapter 12). \\nferent species of worm. Comparison of the “rescuing” frag-\\nments from the different species revealed that they shared\\ncommon short sequences with little protein-coding potential.  KEY CONCEPTS OF SECTION 22.1\\nThe short RNA molecules encoded by lin-4and let-7were\\nThe Birth of Cells\\nsubsequently shown to inhibit translation of the mRNAs en-\\ncoded by lin-14and other heterochronic genes (Figure 22-10b). ■ In asymmetric cell division, two different types of daugh-\\nThese small RNAs, or micro RNAs (miRNAs),are complemen- ter cells are formed from one mother cell. In contrast, both\\ntary to sequences in the 3%untranslated parts of target mRNAs daughter cells formed in symmetric divisions are identical\\nand are believed to control translation of the mNRAs by hy- but may have different fates if they are exposed to differ-\\nbridizing them. Temporal changes in the production of these and ent external signals (see Figure 22-1). \\n12 CHAPTER 22 • Cell Birth, Lineage, And Death\\n■ Pluripotent stem cells can produce more than one type Our first example of cell-type specification comes from\\nof descendant cell, including in some cases a stem cell with the budding yeast, S. cerevisiae.We introduced this useful\\na more-restricted potential to produce differentiated cell unicellular eukaryote way back in Chapter 1 and have en-\\ntypes (see Figure 22-2).  countered it in several other chapters.S. cerevisiaeforms\\nthree cell types: haploid aand \"cells and diploid a/\"cells.\\n■ Cultured embryonic stem cells (ES cells) are capable of\\nEach type has its own distinctive set of active genes; many\\ngiving rise to many kinds of differentiated cell types. They\\nother genes are active in all three cell types. In a pattern com-\\nare useful in production of genetically altered mice and \\nmon to many organisms and tissues, cell-type specification in\\noffer potential for therapeutic uses. \\nyeast is controlled by a small number of transcription factors\\n■ Populations of stem cells associated with most tissues that coordinate the activities of many other genes. Similar\\n(e.g., skin, intestinal epithelium, blood) regenerate differ-\\nregulatory features are found in the responses of higher eu-\\nentiated tissue cells that are damaged or sloughed or be-\\nkaryotic cells to environmental signals and in the specifica-\\ncome aged (see Figures 22-4 and 22-5). \\ntion and patterning of cells and tissues during development. \\n■ Stem cells are prevented from differentiating by specific Recent DNA microarraystudies of transcription patterns\\ncontrols. A high level of !-catenin, a component of the in S. cerevisiaehave provided a genome-wide picture of the\\nWnt signaling pathway, has been implicated in preserving fluctuations of gene expression in the different cell types and\\nstem cells in the skin and intestine.  different stages of the yeast life cycle (see Figure 9-35 for an ex-\\nplanation of the DNA microarray technique). Among other\\n■ Plant stem cells persist for the life of the plant in the\\nthings, these studies identified 32 genes that are transcribed\\nmeristem. Meristem cells can give rise to a broad spectrum\\nat more than twofold higher levels in \"cells than in acells. An-\\nof cell types and structures. \\nother 50 genes are transcribed at more than twofold higher\\n■ During development, precursor cells generally lose po- levels in acells than in \"cells. The products of these 82 genes,\\ntential; that is, they become progressively restricted in the\\nwhich initially are activated by cell-type specification tran-\\nnumber of different cell types they can form. \\nscription regulators, convey many of the critical differences be-\\n■ Early in animal development, the three germ layers— tween the two cell types. Since transcription of more than\\nectoderm, mesoderm, and endoderm—form. Each gives 6000 genes was assessed in these experiments, the results\\nrise to specific tissues and organs (see Figure 22-7). clearly demonstrate that changes in expression of only a small\\nfraction of the genome can significantly alter the behavior and\\n■ Germ-line cells give rise to eggs or sperm. By definition,\\nproperties of cells. Transcription of a much larger number of\\nall other cells are somatic cells. \\ngenes, about 25 percent of the total assayed, differed substan-\\n■ Embryonic development of C. elegansbegins with asym- tially in diploid cells compared with haploid cells. A powerful\\nmetric division of the fertilized egg (zygote). The lineage\\nadvantage of the microarray approach is that it systematically\\nof all the cells in adult worms is known and is highly re-\\nidentifies a large fraction of the relevant genes controlling var-\\nproducible (see Figure 22-9). \\nious processes, allowing scientists to concentrate on the most\\n■ Short regulatory RNAs control the timing of developmen- important players that regulate differences in cell types. \\ntal cell divisions by preventing translation of mRNAs whose\\nencoded proteins control cell lineages (see Figure 22-10).\\nTranscription Factors Encoded at the MATLocus\\nAct in Concert with MCM1 to Specify Cell Type\\n22.2 Cell-Type Specification in Yeast Each of the three S. cerevisiaecell types expresses a unique set\\nof regulatory genes that is responsible for all the differences\\nIn  the  previous  section,  we  saw  that  stem  cells  and  among the three cell types. All haploid cells express certain\\nprecursor/progenitor cells produce progeny that embark haploid-specific genes; in addition, acells express a-specific\\nupon specific differentiation paths. The elegant regulatory genes, and \"cells express \"-specific genes. In a/\"diploid cells,\\nmechanisms whereby cells become different is referred to as diploid-specific genes are expressed, whereas haploid-specific,\\ncell-type specification.Specification usually involves a com- a-specific, and \"-specific genes are not. As illustrated in Fig-\\nbination of external signals with internal signal-transduction ure 22-11, three cell type–specific transcription factors (\"1,\\nmechanisms like those described in Chapters 14 and 15. The \"2, and a1) encoded at the MAT locusin combination with\\ntransition from an undifferentiated cell to a differentiating a general transcription factor called MCM1,which is ex-\\none often involves the production of one or a small number pressed in all three cell types, mediate cell type–specific gene\\nof transcription factors. The newly produced transcription expression in S. cerevisiae. Thus the actions of just three tran-\\nfactors serve as powerful switches that trigger the activation scription factors can set the yeast cell on a specific differenti-\\n(and sometimes repression) of large batteries of subservient ation pathway culminating in a particular cell type. From the\\ngenes. Thus an initially modest change can cause massive DNA microarray experiments we know one outcome: the ac-\\nchanges in gene expression that confer a new character on tivation or repression of many dozens of genes that control\\nthe cell.  cell characteristics.\\n22.2 • Cell-Type Specification in Yeast 13\\n(a) a cell MCM1 was the first member of the MADS familyof\\na1 αsg transcription factors to be discovered. (MADS is an acronym\\nMCM1 for the initial four factors identified in this family.) The\\na1\\nMATa DNA-binding proteins composing this family dimerize and\\n+ + asg\\ncontain a similar N-terminal MADS domain. In Section 22.3\\nwe will encounter other MADS transcription factors that\\nFPO\\n+ + hsg participate in development of skeletal muscle. MADS tran-\\nscription factors also specify cell types in floral organs (see\\nFigure 15-28). MCM1 exhibits different activity in haploid a\\n(b)  α cell\\n+ and \"cells due to its association with \"1 or \"2 protein in \"\\nα2 α1 α1 + cells. As a result of this combinatorial action, MCM1 pro-\\nαsg\\nmotes transcription of \"-specific genes and represses tran-\\nMATα2 MATα1\\nscription of a-specific genes in \"cells. Acting alone, MCM1\\n+ − asg\\nactivates transcription of a-specific genes in a cells and of\\nα2 haploid-specific genes in both \" and a cells (see Figure \\n22-11a, b). Now let’s take a closer look at how MCM1 and\\n+ + hsg\\nthe MAT-encoded proteins exert their effects. \\n(c) a/α cell\\nMCM1 and !1-MCM1 Complexes Activate \\nα2 α2\\n− αsg Gene Transcription\\nMATα2 MATα1 + In acells, homodimeric MCM1 binds to the so-called P box\\n− asg sequence in the upstream regulatory sequences (URSs) of \\nα2 a-specific  genes,  stimulating  their  transcription  (Figure \\na1 a1\\n22-12a). Transcription of \"-specific genes is controlled by\\n− hsg\\ntwo adjacent sequences—the P box and the Q box—located\\nMATa\\nin the URSs associated with these genes. Although MCM1\\nalone binds to the P box in a-specific URSs, it does not bind\\n▲FIGURE 22-11 Transcriptional control of cell type–specific\\nto the P box in \"-specific URSs. Thus a cells do not tran-\\ngenes in S.cerevisiae.The coding sequences carried at the\\nscribe \"-specific genes.\\nMATlocus differ in haploid \" and a cells and in diploid cells.\\nIn \"cells, which produce the \"1 transcription factor en-\\nThree type-specific transcription factors (\"1, \"2, and a1) encoded\\ncoded by MAT\", the simultaneous binding of MCM1 and\\nat the MATlocus act with MCM1, a constitutive transcription\\n\"1 to PQ sites occurs with high affinity (Figure 22-12b). This\\nfactor produced by all three cell types, to produce a distinctive\\npattern of gene expression in each of the three cell types.  binding turns on transcription of \"-specific genes. Therefore\\nasg# a-specific genes/mRNAs; \"sg#\"-specific genes/mRNAs; a-specific transcription is a simple matter of a single tran-\\nhsg# haploid-specific genes/mRNAs. scription factor binding to its target genes, while \"-specific\\n\"FIGURE 22-12 Activity of MCM1 in a\\nand ! yeast cells. MCM1 binds as a dimer\\n(a)  a cells MCM1 dimer\\nto the P site in \"-specific and a-specific\\nupstream regulatory sequences (URSs),\\nwhich control transcription of \"-specific\\nP Q αsg α2 P α2 asg\\ngenes and a-specific genes, respectively. \\nα-specific URS No transcription of a-specific URS Transcription of\\n(a) In a cells, MCM1 stimulates transcription\\nα-specific genes a-specific genes\\nof a-specific genes. MCM1 does not bind\\nFPO efficiently to the P site in \"-specific URSs in\\n(b)  α cells α2 dimer the absence of \"1 protein. (b) In \" cells, the\\nactivity of MCM1 is modified by its\\nassociation with \"1 or \"2. The \"1-MCM1\\ncomplex stimulates transcription of \"-specific\\nα1 genes, whereas the \"2-MCM1 complex\\nP Q αsg α2 P α2 asg blocks transcription of a-specific genes. The\\nα-specific URS Transcription of a-specific URS No transcription of \"2-MCM1 complex also is produced in\\nα-specific genes a-specific genes diploid cells, where it has the same blocking\\neffect on transcription of a-specific genes\\n(see Figure 22-11c).\\n14 CHAPTER 22 • Cell Birth, Lineage, And Death\\ntranscription requires a combination of two factors—neither\\na factor\\nof which can activate target genes alone. α factor\\na α\\n!2-MCM1 and !2-a1 Complexes \\nRepress Transcription\\nFlanking the P box in each a-specific URS are two \"2- α-factor\\nbinding sites. Both MCM1 and \"2 can bind independently receptor a-factor receptor\\nto an a-specific URS with relatively low affinity. However, in\\n\"cells only the highly cooperative, simultaneous binding of\\nPheremone signaling\\nboth \"2 and MCM1 proteins occurs with high affinity. This\\nhigh-affinity binding represses transcription of a-specific\\ngenes, ensuring that they are not expressed in \" cells and\\ndiploid cells (see Figure 22-12b, right). MCM1 promotes a α\\nbinding of \"2 to an a-specific URS by orienting the two DNA-\\nbinding domains of the \"2 dimer to the \"2-binding sequences\\nin this URS. Since a dimeric \"2 molecule binds to both sites in\\nan \"-specific URS, each DNA site is referred to as a half-site. Cell-cycle arrest\\nThe relative positions of both half-sites and their orientation Morphogenesis\\nare highly conserved among different a-specific URSs.\\nThe presence of numerous \"2-binding sites in the genome\\na α\\nand the “relaxed” specificity of \"2 protein may expand the\\nrange of genes that it can regulate. For instance, in a/\" FPO\\ndiploid cells, \"2 forms a heterodimer with a1 that represses\\nboth haploid-specific genes and the gene encoding \"1 (see\\nCell fusion\\nFigure 22-11c). The example of \"2 suggests that relaxed\\nspecificity may be a general strategy for increasing the regu-\\nlatory range of a single transcription factor. Highly specific\\nbinding, then, occurs as a consequence of the interaction of\\n\"2 with other transcription factors at different sites in DNA.\\nNuclear fusion\\nPheromones Induce Mating of ! and a Cells \\nto Generate a Third Cell Type \\na/α\\nDiploid cell\\nAn important feature of the yeast life cycle is the ability of\\nhaploid aand \"cells to mate, that is, attach and fuse giving\\nrise to a diploid a/\"cell (see Figure 1-5). Each haploid cell + Nutrients − Nutrients\\ntype secretes a different mating factor,a small polypeptide\\npheromone,and expresses a cell-surface G protein–coupled Mitotic growth Meiosis\\nreceptor that recognizes the pheromone secreted by cells of\\nthe other type. Thus aand \"cells both secrete and respond to a/α a/α\\npheromones (Figure 22-13). Binding of the mating factors\\nSporulation\\nto their receptors induces expression of a set of genes encod-\\ning proteins that direct arrest of the cell cycle in G and pro- a α\\n1\\nmote attachment/fusion of haploid cells to form diploid cells.\\nIn the presence of sufficient nutrients, the diploid cells will a α\\ncontinue to grow. Starvation, however, induces diploid cells\\nto progress through meiosis, each yielding four haploid\\n▲FIGURE 22-13 Pheromone-induced mating of haploid\\nspores. If the environmental conditions become conducive to\\nyeast cells.The \" cells produce \" mating factor and a receptor;\\nvegetative growth, the spores will germinate and undergo \\nthe a cells produce a factor and \"receptor. Binding of the\\nmitotic division.\\nmating factors to their cognate receptors on cells of the opposite\\nStudies with yeast mutants have provided insights into\\ntype leads to gene activation, resulting in mating and production\\nhow the a and \"pheromones induce mating. For instance, of diploid cells. In the presence of sufficient nutrients, these cells\\nhaploid yeast cells carrying mutations in the sterile 12 will grow as diploids. Without sufficient nutrients, cells will\\n(STE12)locus cannot respond to pheromones and do not undergo meiosis and form four haploid spores.\\n22.3 • Specification and Differentiation of Muscle 15\\nmate. The STE12gene encodes a transcription factor that biologists do not yet know the complete set of molecules that\\nbinds to a DNA sequence referred to as the pheromone- distinguishes any one cell type (e.g., muscle) from all the\\nresponsive element,which is present in many different a- and other cell types in a multicellular organism. The extensive\\n\"-specific URSs. Binding of mating factors to cell-surface re- cell specification and differentiation that occur during \\nceptors induces a cascade of signaling events, resulting in development of animals and plants depend on both quanti-\\nphosphorylation of various proteins including the Ste12 pro- tative and qualitative differences in gene expression, con-\\ntein (see Figure 14-24). This rapid phosphorylation is corre- trolled at the level of transcription, as well as on cell\\nlated with an increase in the ability of Ste12 to stimulate structures and protein activity states. \\ntranscription. It is not yet known, however, whether Ste12 An impressive array of molecular strategies, some analo-\\nmust be phosphorylated to stimulate transcription in re- gous to those found in yeast cell-type specification, have\\nsponse to pheromone.  evolved to carry out the complex developmental pathways\\nInteraction of Ste12 protein with DNA has been studied that characterize multicellular organisms. Muscle cells have\\nmost extensively at the URS controlling transcription of been the focus of many such studies because their develop-\\nSTE2,an a-specific gene encoding the receptor for the \" ment can be studied in cultured cells as well as in intact ani-\\npheromone. Adjacent to this a-specific URS is a pheromone- mals. Early advances in understanding the formation of\\nresponsive element that binds Ste12. When acells are treated muscle cells (myogenesis) came from discovery of regulatory\\nwith \"pheromone, transcription of the STE2gene increases genes that could convert cultured cells into muscle cells.\\nin  a  process  that  requires  Ste12  protein.  Presumably, Then mouse mutations affecting those genes were created\\npheromone-induced production of the \"receptor encoded by and studied to learn the functions of the proteins encoded\\nSTE2increases the efficiency of the mating process. Ste12 by these genes, following which scientists have investigated\\nprotein has been found to bind most efficiently to the how the muscle regulatory genes control other genes.\\npheromone-responsive element in the STE2 URS when DNA microarray analysis has been applied to under-\\nMCM1 is simultaneously bound to the adjacent P site. We stand the function of twistand other regulatory genes in\\nsaw previously that MCM1 can act as an activator or a  muscle development. For example, the expression pattern of\\nrepressor at different URSs depending on whether it com- normal Drosophilaembryos recently has been compared\\nplexes with \"1 or \"2. In this case, the function of MCM1  with that of mutant embryos in which the twistgene is de-\\nas an activator is stimulated by the binding of yet another fective. Previous research had shown that the Twist protein\\ntranscription factor, Ste12, whose activity is modified by  encoded by this gene is a transcription factor necessary to\\nextracellular signals. create muscle cells in flies and other animals. Twist turns\\non production of other transcription factors that in turn ac-\\ntivate genes encoding myosin, actin, and other muscle-\\nKEY CONCEPTS OF SECTION 22.2 specific proteins. To assess how many genes are needed to\\nspecify muscle, researchers determined the expression of\\nCell-Type Specification in Yeast\\nabout 4000 fly genes (about 30 percent of the total) in nor-\\n■ Specification of each of the three yeast cell types—the a mal fly embryos and in twistmutants. Of the genes included\\nand \"haploid cells and the diploid a/\"cells—is determined in the microarray, about 130 (3.3 percent), including many\\nby a unique set of transcription factors acting in different known muscle differentiation genes, were transcribed at\\ncombinations  at  specific  regulatory  sites  in  the  yeast lower levels (or not at all) in the twistmutants. These results\\ngenome (see Figure 22-11). suggest that transcriptional changes in at least several hun-\\n■ Some transcription factors can act as repressors or acti- dred genes are associated with differentiation of a highly\\nvators depending on the specific regulatory sites they bind specialized cell type such as muscle. \\nand the presence or absence of other transcription factors Other recent microarray studies have looked for genes\\nbound to neighboring sites. whose transcription differs in various subtypes of muscle in\\nmice. These have identified 49 genes out of 3000 genes ex-\\n■ Binding of mating-type pheromones by haploid yeast amined that are transcribed at substantially different levels in\\ncells activates expression of genes encoding proteins that\\nred (endurance) muscle and white (fast response) muscle.\\nmediate mating, thereby generating the third yeast cell type\\nClues to the molecular basis of the functional differences be-\\n(see Figure 22-13).\\ntween red and white muscle are likely to come from studying\\nthose 49 genes and their products. \\nHere we examine the role of certain transcription factors\\n22.3 Specification and Differentiation\\nin creating skeletal muscle in vertebrates. These muscle reg-\\nof Muscle  ulators illustrate how coordinated transcription of sets of\\ntarget genes can produce differentiated cell types and how a\\nAs indicated by global expression patterns, yeast cells of dif- cascade of transcriptional events and signals is necessary to\\nferent mating types are still rather similar. Developmental  coordinate cell behaviors and functions. \\n16 CHAPTER 22 • Cell Birth, Lineage, And Death\\nEmbryonic Somites Give Rise to Myoblasts,  siently. These signals trigger production of intracellular fac-\\nthe Precursors of Skeletal Muscle Cells  tors that maintain the myogenic program after the inducing\\nsignals are gone. We discuss the identification and functions\\nVertebrate  skeletal  myogenesis  proceeds  through  three\\nof these myogenic proteins, and their interactions, in the next\\nstages: determination of the precursor muscle cells, called\\nseveral sections.\\nmyoblasts;proliferation and in some cases migration of \\nmyoblasts; and their terminal differentiation into mature\\nmuscle (Figure 22-14). In the first stage, myoblasts arise from Myogenic Genes Were First Identified in Studies\\nblocks of mesoderm cells, called somites,that are located with Cultured Fibroblasts\\nnext to the neural tube in the embryo. Specific signals from\\nIn vitro studies with the fibroblast cell line designated C3H \\nsurrounding tissue play an important role in determining\\n10T1⁄ have played a central role in dissecting the transcription-\\nwhere myoblasts will form in the developing somite. At the 2\\ncontrol mechanisms regulating skeletal myogenesis. When \\nmolecular level, the decision of a mesoderm cell to adopt a\\nthese cells are incubated in the presence of 5-azacytidine, a \\nmuscle cell fate reflects the activation of genes encoding par-\\ncytidine derivative that cannot be methylated and therefore \\nticular transcription factors.\\nalters transcription, they differentiate into myotubes. Upon\\nAs myoblasts proliferate and migrate, say to a developing\\nentry into cells, 5-azacytidine is converted to 5-azadeoxycytidine\\nlimb bud, they become aligned, stop dividing, and fuse to\\ntriphosphate and then is incorporated into DNA in place of\\nform a syncytium(a cell containing many nuclei but sharing\\ndeoxycytidine. Because methylated deoxycytidine residues\\na common cytoplasm). We refer to this multinucleate cell as\\ncommonly are present in transcriptionally inactive DNA re-\\na myotube.Concomitant with cell fusion is a dramatic rise in\\ngions, replacement of cytidine residues with a derivative that\\nthe expression of genes necessary for further muscle devel-\\ncannot be methylated may permit activation of genes previ-\\nopment and function. \\nously repressed by methylation. \\nThe specific extracellular signals that induce determina-\\nThe high frequency at which azacytidine-treated C3H\\ntion of each group of myoblasts are expressed only tran-\\n10T1⁄ cells are converted into myotubes suggested to early\\n2\\nworkers that reactivation of one or a small number of closely\\nlinked genes is sufficient to drive a myogenic program. To test\\n!EXPERIMENTAL FIGURE 22-15 Myogenic genes\\nisolated from azacytidine-treated cells can drive myogenesis\\nMyotome\\nwhen transfected into other cells.(a) When C3H 10T1⁄2 cells \\n(a fibroblast cell line) are treated with azacytidine, they develop\\nSomite\\ninto myotubes at high frequency. To isolate the genes responsible\\nMyoD/Myf5 1 Determination for converting azacytidine-treated cells into myotubes, all the\\nmRNAs from treated cells first were isolated from cell extracts\\nFPO on an oligo-dT column. Because of their poly(A) tails, mRNAs are\\nselectively retained on this column. Steps  1 and  2: The \\nisolated mRNAs were converted to radiolabeled cDNAs. Step  3:\\nMyoblasts When the cDNAs were mixed with mRNAs from untreatedC3H\\nMyogenin and MEFs 2 Proliferation and/or migration 10T1⁄2 cells, only cDNAs derived from mRNAs (light red)\\nproduced by both azacytidine-treated cells and untreated cells\\nhybridized. The resulting double-stranded DNA was separated\\nPremuscle masses\\nfrom the unhybridized cDNAs (dark blue) produced only by\\nazacytidine-treated cells. Step  4: The cDNAs specific for\\n3 Differentiation and cell fusion\\nazacytidine-treated cells then were used as probes to screen a\\ncDNA library from azacytidine-treated cells (Chapter 9). At least\\nsome of the clones identified with these probes correspond to\\ngenes required for myogenesis. (b) Each of the cDNA clones\\nMyotube (muscle cell)\\nidentified in part (a) was incorporated into a plasmid carrying a\\n▲FIGURE 22-14 Three stages in development of  strong promoter. Steps  1 and  2: C3H 10T1⁄2 cells were\\nvertebrate skeletal muscle. Somites are epithelial spheres of cotransfected with each recombinant plasmid plus a second\\nembryonic mesoderm cells, some of which (the myotome) plasmid carrying a gene conferring resistance to an antibiotic\\nbecome determined as myoblasts after receiving signals from called G418; only cells that have incorporated the plasmids will\\nother tissues (1). After the myoblasts proliferate and migrate  grow on a medium containing G418. One of the selected clones,\\nto the limb buds and elsewhere (2), they undergo terminal designated myoD,was shown to drive conversion of C3H 10T1⁄2\\ndifferentiation into multinucleate skeletal muscle cells, called cells into muscle cells, identified by their binding of labeled\\nmyotubes (3). Key transcription factors that help drive the antibodies against myosin, a muscle-specific protein (step  3).\\nmyogenic program are highlighted in yellow. See also Figure 22-17. [See R. L. Davis et al., 1987, Cell51:987.]\\n22.3 • Specification and Differentiation of Muscle 17\\n(a)  Screen for myogenic genes this hypothesis, researchers isolated DNA from C3H 10T1⁄2\\ncells grown in the presence of 5-azacytidine and transfected it\\ninto untreated cells. The observation that 1 in 104cells trans-\\nfected with this DNA was converted into myotubes is consis-\\ntent with the hypothesis that one or a small set of closely linked\\nTotal mRNA from azacytidine-treated cells genes is responsible for converting fibroblasts into myotubes. \\n1 Incubate with reverse Subsequent studies led to the isolation and characteriza-\\ntranscriptase and tion of four genes that can convert C3H 10T1⁄ cells into\\n2\\n[32P]dNTPs\\nmuscle. Figure 22-15 outlines the experimental protocol \\n2 Remove mRNAs for identifying and assaying one of these genes, called the\\nmyogenic determination (myoD)gene. C3H 10T1⁄ cells\\n2\\ntransfected with myoDcDNA and those treated with 5-\\nazacytidine both formed myotubes. The myoDcDNA also\\nwas able to convert a number of other cultured cell lines into\\nmuscle. Based on these findings, the myoDgene was pro-\\n32P-labeled cDNAs\\nposed to play a key role in muscle development. A similar ap-\\n3 Hybridize with proach identified three other genes—myogenin, myf5,and\\nDiscard\\nexcess of mRNAs mrf4—that also function in muscle development. \\nfrom untreated\\nC3H 10T1/2 cells\\n(                 ) Muscle-Regulatory Factors (MRFs) \\nand Myocyte-Enhancing Factors (MEFs) Act \\nin Concert to Confer Myogenic Specificity \\n32P-labeled azamyoblast- The four myogenic proteins—MyoD, Myf5, myogenin, and\\nspecific cDNAs\\nMRF4—are  all  members  of  the  basic  helix-loop-helix\\n(bHLH) family of DNA-binding transcription factors (see \\nScreen cDNA library Figure 11-22b). Near the center of these proteins is a DNA-\\n4\\nfrom treated cells binding basic (B) region adjacent to the HLH domain, which\\nmediates dimer formation. Flanking this central DNA-\\nClones enriched\\nin myogenic genes binding/dimerization region are two activation domains. We\\nrefer to the four myogenic bHLH proteins collectively as\\nFPO muscle regulatory factors,or MRFs(Figure 22-16a). \\n(a) Structure of muscle-regulatory factors (MRFs)\\nDNA binding/dimerization\\n(b)  Assay for myogenic activity of myoD cDNA\\nH2N Transactivation B HLH Transactivation COOH\\nC3H 10T1/2 cell MEF-interacting domain\\nFPO\\n1 Transfect with a plasmid (b) Structure of myocyte-enhancing factors (MEFs)\\ncarrying myoD cDNA and a\\ngene conferring resistance DNA binding/dimerization\\nto G418\\n2 Select on G418-containing H2N MADS MEF Transactivation COOH\\nmedium\\nMyogenin-interacting domain\\n▲FIGURE 22-16 General structures of two classes of\\n3 Stain with transcription factors that participate in myogenesis. MRFs\\nlabeled antimyosin (muscle regulatory factors) are bHLH (basic helix-loop-helix)\\nantibody (   )  proteins produced only in developing muscle. MEFs (myocyte-\\nenhancing factors), which are produced in several tissues in\\naddition to developing muscle, belong to the MADS family. \\nThe myogenic activity of MRFs is enhanced by their interaction\\nwith MEFs.\\n18 CHAPTER 22 • Cell Birth, Lineage, And Death\\nbHLH proteins form homo- and heterodimers that bind scribed at several levels. First, the muscle regulators are ac-\\nto a 6-bp DNA site with the consensus sequence CANNTG tivated only in mesoderm cells, at the right place and time\\n(N = any nucleotide). Referred to as the E box, this sequence in the embryo, in response to spatial regulators of the sorts\\nis present in many different locations within the genome  that are described in Chapter 15. Other proteins mediate ad-\\n(on a purely random basis the E box will be found every 256 ditional mechanisms for assuring tight control over myoge-\\nnucleotides). Thus some mechanism(s) must ensure that nesis: chromatin-remodeling proteins are needed to make\\nMRFs specifically regulate muscle-specific genes and not target genes accessible to MRFs; inhibitory proteins can re-\\nother genes containing E boxes in their transcription- strict when MRFs act; and antagonistic relations between\\ncontrol regions. One clue to how this myogenic specificity is cell-cycle regulators and differentiation factors like MRFs en-\\nachieved was the finding that the DNA-binding affinity of sure that differentiating cells will not divide. All these factors\\nMyoD is tenfold greater when it binds as a heterodimer com- control when and where muscles form.\\nplexed with E2A, another bHLH protein, than when it binds\\nas a homodimer. Moreover, in azacytidine-treated C3H Activating Chromatin-Remodeling Proteins MRF proteins\\n10T1⁄2 cells, MyoD is found as a heterodimer complexed control batteries of muscle-specific genes, but can do so only\\nwith E2A, and both proteins are required for myogenesis in if chromatin factors allow access. Remodeling of chromatin,\\nthese cells. The DNA-binding domains of E2A and MyoD which usually is necessary for gene activation, is carried out\\nhave similar but not identical amino acid sequences, and by large protein complexes (e.g., Swi/Snf complex) that have\\nboth proteins recognize E box sequences. The other MRFs ATPase and perhaps helicase activity. These complexes are\\nalso form heterodimers with E2A that have properties simi- thought to recruit histone acetylases that modify chromatin\\nlar to MyoD-E2A complexes.  to make genes accessible to transcription factors (Chapter\\nSince E2A is expressed in many tissues, the requirement 11). The need for remodeling complexes to help myogenic\\nfor E2A is not sufficient to confer myogenic specificity. Sub- factors was tested using dominant-negative versions of the\\nsequent studies suggested that specific amino acids in the ATPase proteins that form the cores of these complexes. (Re-\\nbHLH domain of all the MRFs may confer myogenic speci- call from Chapter 9 that a dominant-negative mutation pro-\\nficity by allowing MRF-E2A complexes to bind specifically duces a mutant phenotype even when a normal allele of the\\nto another family of DNA-binding proteins called myocyte- gene also is present.) When genes carrying these dominant-\\nenhancing factors,or MEFs.MEFs were considered excellent negative mutations were transfected into C3H 10T1⁄ cells,\\n2\\ncandidates for interaction with MRFs for two reasons. First, the subsequent introduction of myogenic genes no longer\\nmany muscle-specific genes contain recognition sites for both converted the cells into myotubes. In addition, a muscle-\\nMEFs and MRFs. Second, although MEFs cannot induce specific gene that is normally activated did not exhibit its\\nmyogenic conversion of azacytidine-treated C3H 10T1⁄2 usual pattern of chromatin changes in the doubly transfected\\ncells, they enhance the ability of MRFs to do so. This en- C3H 10T1⁄ cells. These results indicate that transcription ac-\\n2\\nhancement requires physical interaction between a MEF and tivation by myogenic proteins depends on a suitable chro-\\nMRF-E2A heterodimer. Later, MEFs were found to belong to matin structure in the regions of muscle-specific genes.\\nthe MADS family of transcription factors and to contain a\\nMEF domain, adjacent to the MADS domain, that mediates Inhibitory Proteins Screens for genes related to myoDled to\\ninteraction with myogenin (Figure 22-16b). The synergistic identification of a related protein that retains the dimeriza-\\naction of the MEF homodimer and MRF-E2A heterodimer is tion region but lacks the basic DNA-binding region and\\nthought to drive high-level expression of muscle-specific hence is unable to bind to E box sequences in DNA. By bind-\\ngenes.  ing to MyoD or E2A, this protein inhibits formation of\\nKnockout mice and Drosophilamutants have been used MyoD-E2A heterodimers and hence their high-affinity bind-\\nto explore the roles of MRF and MEF proteins in conferring ing to DNA. Accordingly, this protein is referred to as Id, for\\nmyogenic specificity in intact animals rather than cell cul- inhibitor of DNA binding. Id prevents cells that produce\\nture. These experiments demonstrated the importance of MyoD and E2A from activating transcription of the muscle-\\nthree of the MRF proteins and of MEF proteins for distinct specific gene encoding creatine kinase. This finding pre-\\nsteps in muscle development (see Figure 22-14). The function sumably reflects the formation of MyoD-Id or E2A-Id \\nof the fourth myogenic protein, Mrf4, is not entirely clear; complexes, which cannot bind to and activate transcription\\nit may be expressed later and help maintain differentiated of muscle-specific genes. As a result, the cells remain in a\\nmuscle cells.  proliferative growth state. When these cells are induced to\\ndifferentiate into muscle (for instance, by the removal of\\nserum, which contains the growth factors required for pro-\\nTerminal Differentiation of Myoblasts Is Under\\nliferative growth), the Id concentration falls. MyoD-E2A\\nPositive and Negative Control \\ndimers now can form and bind to the regulatory regions of\\nPowerful developmental regulators like the MRFs cannot be target genes, driving differentiation of C3H 10T1⁄2cells into\\nallowed to run rampant. In fact, their actions are circum- myoblast-like cells. \\n22.3 • Specification and Differentiation of Muscle 19\\nRecent work shows that histone acetylases and deacety- Signals Are Crucial for Muscle Cell-Fate\\nlases are crucial for regulating muscle-specific genes. As ex- Determination and Myoblast Migration \\nplained in Chapter 11, acetylation of histones in chromatin \\nAs noted already, after myoblasts arise from somites, they\\nis necessary to activate many genes; in contrast, histone\\nmust not only proliferate but also move to their proper lo-\\ndeacetylases  cause  transcriptional  repression  (see  Figure \\ncations and form the correct attachments as they differ-\\n11-32). MEF2 recruits histone acetylases such as p300/CBP,\\nentiate into muscle cells (Figure 22-17). Myogenic gene \\nthrough another protein that serves as a mediator, to activate\\nexpression often follows elaborate organizational events that\\ntranscription of target genes. Chromatin immunoprecipitation\\ntell certain somite cells to delaminate from the somite ep-\\nexperiments with antibodies against acetylated histone H4\\nithelium and which way to move. A transcription factor,\\nshow that the acetylated histone level associated with MEF2-\\nPax3, is produced in the subset of somite cells that will form\\nregulated genes is higher in differentiated myotubes than in\\nmuscle and appears to be at the top of the regulatory hierar-\\nmyoblasts (see Figure 11-31). The role of histone deacetylases\\nchy controlling muscle formation in the body wall and limbs.\\nin muscle development was revealed in experiments in which\\nMyoblasts that will migrate, but not cells that remain be-\\nscientists first introduced extra myoDgenes into cultured C3H\\nhind, also produce a transcription factor called Lbx1. If Pax3\\n10T1⁄ cells to raise the level of MyoD. This resulted in in-\\n2\\nis not functional, Lbx1 transcripts are not seen and myo-\\ncreased activation of target genes and more rapid differentia-\\nblasts do not migrate. Both Pax3 and Lbx1 can affect ex-\\ntion of the cells into myotubes. However, when genes encoding\\npression of myoD.\\nhistone deacetylases also were introduced into the C3H 10T1⁄\\n2\\nDeparture of myoblasts from somites depends upon a \\ncells, the muscle-inducing effect of MyoD was blocked and the\\nsecreted protein signal appropriately called scatter factor,\\ncells did not differentiate into myotubes. \\nor hepatocyte growth factor (SF/HGF).This signal is pro-\\nThe explanation for how histone deacetylases inhibit\\nduced by embryonic connective tissue cells (mesenchyme) in\\nMyoD-induced muscle differentiation came from the sur-\\nthe limb buds to which myoblasts migrate. Production of\\nprising finding that MEF2 can bind, through its MADS do-\\nSF/HGF is previously induced by other external signals such\\nmain, to a histone deacetylase. This interaction, which can\\nas fibroblast growth factor and Sonic hedgehog, which are\\nprevent MEF2 function and muscle differentiation, is nor-\\ncritical to limb development (Chapter 15). The cell-surface\\nmally blocked during differentiation because the histone\\nreceptor for SF/HGF, which is expressed by myoblasts, be-\\ndeacetylase is phosphorylated by a calcium/calmodulin-\\nlongs to the receptor tyrosine kinase (RTK) class of receptors\\ndependent protein kinase; the phosphorylated deacetylase\\n(Chapter 14). Cells migrate from the somites at the head-to-\\nthen is moved from the nucleus to the cytoplasm. Taken to-\\ntail body levels where limbs will form, and not elsewhere,\\ngether, these results indicate that activation of muscle genes\\nby MyoD and MEF2 is in competition with inactivation of\\nmuscle genes by repressive chromatin structures and that nu-\\nclear versus cytoplasmic localization of chromatin factors is Neural tube\\na key regulatory step. Dermomyotome\\n(gives rise to\\ndermis of skin)\\nCell-Cycle Proteins The onset of terminal differentiation in Epidermis\\nmany cell types is associated with arrest of the cell cycle, Myoblasts\\nmost commonly in G , suggesting that the transition from (migrate to FPO\\n1\\nlimb bud to\\nthe determined to differentiated state may be influenced by\\nform skeletal\\ncell-cycle proteins including cyclins and cyclin-dependent  muscle)\\nkinases(Chapter 21). For instance, certain inhibitors of cyclin-\\ndependent kinases can induce muscle differentiation in cell\\nculture, and the amounts of these inhibitors are markedly\\nNotochord\\nhigher in differentiating muscle cells than in nondifferentiat-\\nSclerotome \\ning ones in vivo. Conversely, differentiation of cultured  (gives rise to skeletal structures \\nmyoblasts can be inhibited by transfecting the cells with such as vertebrae)\\nDNA encoding cyclin D1 under the control of a constitutively\\n▲FIGURE 22-17 Embryonic determination and migration\\nactive promoter. Expression of cyclin D1, which normally\\nof myoblasts in mammals. After formation of the neural tube,\\noccurs only during G , is induced by mitogenic factors in\\n1 each somite forms sclerotome, which develops into skeletal\\nmany cell types and drives the cell cycle (see Figure 21-28).\\nstructures, and dermomyotome. The dermomyotome gives rise to\\nThe ability of cyclin D1 to prevent myoblast differentiation\\nthe dermis of the skin and to the muscles. Lateral myoblasts\\nin vitro may mimic aspects of the in vivo signals that antag-\\nmigrate to the limb bud; medial myoblasts develop into the trunk\\nonize the differentiation pathway. The antagonism between muscles. The remainder of a dermomyotome gives rise to the\\nnegative and positive regulators of G1progression is likely to connective tissue of the skin. [Adapted from M. Buckingham, 1992,\\nplay an important role in controlling myogenesis in vivo. Trends Genet.8:144.]\\n20 CHAPTER 22 • Cell Birth, Lineage, And Death\\ndue to the presence of SF/HGF at limb locations and not teins, designated Da and Emc, are analogous in structure and\\nelsewhere. If the SF/HGF signal or its receptor is not func- function to vertebrate E2A and Id, respectively. For example,\\ntional, somite cells will produce Lbx1 but not go on to mi- heterodimeric complexes of Da with Ac or Sc bind to DNA\\ngrate; thus no muscles will form in the limbs. Expression of better than the homodimeric forms of Ac and Sc. Emc, like\\nthe myogeningene, which is necessary for myotube forma- Id, lacks a DNA-binding basic domain; it binds to Ac and\\ntion, and of mrf4, which is necessary for muscle fiber differ- Sc proteins, thus inhibiting their association with Da and\\nentiation, does not begin until migrating myoblasts approach binding to DNA. The similar functions of these myogenic\\ntheir limb-bud destination; these steps in terminal muscle dif- and neurogenic proteins are depicted in Figure 22-18. \\nferentiation are presumably not compatible with migration.  A family of bHLH proteins related to the Drosophila\\nWe have touched on just a few of the many external sig- Achaete and Scute proteins has been identified in vertebrates.\\nnals and transcription factors that participate in develop- One of these, called neurogenin,which has been identified in\\nment of a properly patterned muscle. The function of all the rat, mouse, and frog, appears to control the formation of\\nthese regulatory molecules must be coordinated both in neuroblasts. In situ hybridization experiments showed that\\nspace and in time during myogenesis.  neurogenin is produced at an early stage in the developing nerv-\\nous system and may induce production of NeuroD, another\\nbHLH protein that acts later (Figure 22-19a). Injection of large\\nbHLH Regulatory Proteins Function in Creation \\namounts of neurogeninmRNA into Xenopusembryos further\\nof Other Tissues\\ndemonstrated the ability of neurogenin to induce neurogenesis\\nFour bHLH transcription factors that are remarkably similar (Figure 22-19b). These studies suggest that the function of neu-\\nto the myogenic bHLH proteins control neurogenesis in rogenin is analogous to that of the Achaete and Scute in\\nDrosophila. Similar proteins appear to function in neurogen- Drosophila;likewise, NeuroD and Asense may have analogous\\nesis in vertebrates and perhaps in the determination and dif- functions in vertebrates and Drosophila,respectively.\\nferentiation of hematopoietic cells.  In addition to neurons, the nervous system contains a\\nThe neurogenic DrosophilabHLH proteins are encoded large number of glial cells, which also arise from the neuro-\\nby an ≈100-kb stretch of genomic DNA, termed the achaete- ectoderm. Glial cells support and insulate neurons; they also\\nscute complex (AS-C), containing four genes designated provide guidance and contact surfaces for migrating neurons\\nachaete (ac),scute (sc),lethal of scute (l’sc),and asense (a). during development and send a signal to neurons that pro-\\nAnalysis of the effects of loss-of-function mutations indicate motes formation of synapses. Neurogenins control the fates\\nthat the Achaete (Ac) and Scute (Sc) proteins participate in of precursor cells that are capable of making either neurons\\ndetermination of neuronal precursor cells, called neuroblasts, or glial cells by promoting neural development and repress-\\nwhile the Asense (As) protein is required for differentiation ing glial development. Neurogenins are therefore switches\\nof these cells into neurons. These functions are analogous to that control the decision between two alternative cell fates,\\nthe roles of MyoD and Myf5 in muscle determination and just as the yeast mating-type regulators select among three\\nof myogenin in differentiation. Two other Drosophilapro- cell types. \\nFly neurogenesis\\nac, sc mRNA Da(general) as mRNA\\n+\\nachaete asense\\nAc, Sc Asense Neurons\\nscute (others)\\n−\\nProneural Neural\\ngenes Emc precursor\\nFPgenesO\\n!FIGURE 22-18 Comparison of genes\\nthat regulate Drosophilaneurogenesis\\nVertebrate myogenesis\\nand mammalian myogenesis. bHLH\\nmyf5, myoD \\ntranscription factors have analogous mRNA myogenin mRNA\\nE2A(general)\\nfunctions in determination of precursor\\n+\\ncells (i.e., neuroblasts and myoblasts) and myf5 myogenin\\ntheir subsequent differentiation into mature myoD Myf5, MyoD mrf4 (?) Myogenin Muscle\\nneurons and muscle cells. In both cases, (others) (others)\\n−\\nthe proteins encoded by the earliest-acting Promuscular Muscle precursor\\nId\\ngenes (left) are under both positive and genes genes\\nnegative control by other related proteins\\n(blue type). [Adapted from Y. N. Jan and L. Y. Precursor determination Differentiation\\nJan, 1993, Cell75:827.]\\n22.4 • Regulation of Asymmetric Cell Division 21\\n(b)\\n(a)\\nFPO\\nneurogenin mRNA  neuroD mRNA  β-tubulin mRNA  neurogenin mRNA \\n▲EXPERIMENTAL FIGURE 22-19 In situ hybridization  ventricular zone (other arrow). (b) One of the two cells in early\\nand injection experiments demonstrate that neurogenin acts Xenopusembryos was injected with neurogeninmRNA (inj) and\\nbefore NeuroD in vertebrate neurogenesis.(a) Sections of rat then stained with a probe specific for neuron-specific mRNAs\\nneural tube were treated with a probe specific for neurogenin encoding !-tubulin (left) or NeuroD (right).The region of the\\nmRNA (left) or neuroDmRNA (right).The open space in the embryo derived from the uninjected cell served as a control\\ncenter is the ventricle, and the cells lining this cavity are in the (con). The neurogeninmRNA induced a massive increase in the\\nventricular layer. All the neural cells are born in the ventricular number of neuroblasts expressing neuroDmRNA and neurons\\nlayer and then migrate outward. As illustrated in these expressing !-tubulinmRNA in the region of the neural tube\\nmicrographs, neurogeninmRNA is produced in proliferating derived from the injected cell. [From Q. Ma et al., 1996, Cell87:43;\\nneuroblasts in the ventricular layer (arrow), whereas neuroD courtesy of D. J. Anderson.]\\nmRNA is present in migrating neuroblasts that have left the\\n■ Migration of myoblasts to the limb buds is induced by\\nKEY CONCEPTS OF SECTION 22.3 scatter factor/hepatocyte growth factor (SF/HGF), a protein\\nsignal secreted by mesenchymal cells (see Figure 22-17).\\nSpecification and Differentiation of Muscle\\nMyoblasts must express both the Pax3 and Lbx1 tran-\\n■ Development of skeletal muscle begins with the signal- scription factors to migrate. \\ninduced determination of certain mesoderm cells in somites\\nas myoblasts. Following their proliferation and migration, ■ Terminal differentiation of myoblasts and induction of\\nmuscle-specific proteins do not occur until myoblasts stop\\nmyoblasts stop dividing and differentiate into multinucle-\\ndividing and begin migrating. \\nate muscle cells (myotubes) that express muscle-specific\\nproteins (see Figure 22-14). ■ Neurogenesis in Drosophiladepends upon a set of four neu-\\nrogenic bHLH proteins that are conceptually and structurally\\n■ Four  myogenic  bHLH  transcription  factors—MyoD, similar to the vertebrate myogenic proteins (see Figure 22-18). \\nmyogenin, Myf5, and MRF4, called muscle-regulatory fac-\\ntors (MRFs)—associate with E2A and MEFs to form large ■ A related vertebrate protein, neurogenin, is required for\\nformation of neural precursors and also controls their di-\\ntranscriptional complexes that drive myogenesis and ex-\\nvision into neurons or glial cells. \\npression of muscle-specific genes. \\n■ Dimerization of bHLH transcription factors with dif- 22.4 Regulation of Asymmetric \\nferent partners modulates the specificity or affinity of their\\nbinding to specific DNA regulatory sites, and also may pre- Cell Division \\nvent their binding entirely.\\nDuring embryogenesis,the earliest stage in animal develop-\\n■ The myogenic program driven by MRFs depends on the ment, asymmetric cell division often creates the initial diver-\\nSwi/Snf chromatin-remodeling complex, which makes tar-\\nsity that ultimately culminates in formation of specific\\nget genes accessible. \\ndifferentiated cell types. Even in bacteria, cell division may\\n■ The myogenic program is inhibited by binding of Id pro- yield unequal daughter cells, for example, one that remains\\ntein to MyoD, thereby blocking binding of MyoD to DNA, attached to a stalk and one that develops flagella used for\\nand by histone deacetylases, which repress activation of swimming. Essential to asymmetric cell division is polariza-\\ntarget genes by MRFs. tionof the parental cell and then differential incorporation \\n22 CHAPTER 22 • Cell Birth, Lineage, And Death\\nCytoplasm Interestingly, some haploid yeast cells can switch repeat-\\nProtein\\nor mRNA edly between the \" and a types. Mating-type switching\\noccurs when the \"allele occupying the MATlocus is replaced\\nby the aallele, or vice versa. The first step in this process is\\ncatalyzed by HO endonuclease, which is expressed in mother\\nPolarized parental cell cells but not in daughter cells. Thus mating-type switch-\\ning occurs only in mother cells (Figure 22-21). Transcription\\nFPO of the HOgene is dependent on the Swi/Snf chromatin-\\nremodeling complex(see Figure 11-37), the same complex\\nthat we encountered earlier in our discussion of myogenesis.\\nDaughter yeast cells arising by budding from mother cells\\ncontain a protein called Ash1 that prevents recruitment of\\nMitotic the Swi/Snf complex to the HOgene, thereby preventing its\\nspindle\\ntranscription. The absence of Ash1 from mother cells allows\\nthem to transcribe the HOgene. \\nRecent experiments have revealed how the asymmetry in\\nthe distribution of Ash1 between mother and daughter cells\\nis established. ASH1mRNA accumulates in the growing bud\\nAsymmetric daughter cells that will form a daughter cell due to the action of a myosin\\nmotor protein (Chapter 19). This motor protein, called\\n▲FIGURE 22-20 General features of asymmetric cell Myo4p, moves the ASH1mRNA, as a ribonucleoprotein\\ndivision.Various mechanisms can lead to asymmetric distribution\\ncomplex, along actin filaments in one direction only, toward\\nof cytoplasmic components, such as particular proteins or\\nthe bud (Figure 22-22). Two connector proteins tether ASH1\\nmRNAs (red dots) to form a polarized parental cell. Division of a\\nmRNA to the motor protein. By the time the bud separates\\npolarized cell will be asymmetric if the mitotic spindle is oriented\\nfrom the mother cell, the mother cell is largely depleted of\\nso that the localized cytoplasmic components are distributed\\nASH1mRNA and thus can switch mating type in the fol-\\nunequally to the two daughter cells, as shown here. However, if\\nlowing G before additional ASH1mRNA is produced and\\nthe spindle is positioned differently relative to the localized 1\\nbefore DNA replication in the S phase.\\ncytoplasmic components, division of a polarized cell may produce\\nequivalent daughter cells. \\nBud\\n\"\\nof parts of the parental cell into the two daughters (Figure\\n22-20). A variety of molecular mechanisms are employed to Division\\ncreate and propagate the initial asymmetry that polarizes the\\nparental cell. In addition to being different, the daughter cells \" \"\\nmust often be placed in a specific orientation with respect to\\n− Ash1 M D + Ash1\\nsurrounding structures. HO transcriptiFon      PO     No HO transcription\\nWe begin with an especially well-understood example of Switching                   No switching\\nasymmetric cell division, the budding of yeast cells, and\\nmove on to recently discovered protein complexes important a a \" \"\\nfor asymmetric cell divisions in multicellular organisms. We\\nM D M D\\nsee in this example an elegant system that links asymmetric\\ndivision to the process of controlling cell type. \\n\" \" a a\\nM D M D\\nYeast Mating-Type Switching Depends \\nupon Asymmetric Cell Division ▲FIGURE 22-21 Specificity of mating-type switching in\\nhaploid yeast cells. Division by budding forms a larger mother\\nWhether a haploid yeast cell exhibits the \"or amating type\\ncell (M) and smaller daughter cell (D), both of which have the\\nis determined by which genes are present at the MATlocus\\nsame mating type as the original cell (\" in this example). The\\n(see Figure 22-11). As described in Chapter 11, the MAT\\nmother can switch mating type during G of the next cell cycle\\nlocus in the S. cerevisiaegenome is flanked by two “silent,” 1\\nand then divide again, producing two cells of the opposite a\\ntranscriptionally inactive loci containing the alternative \"or\\ntype. Switching depends on transcription of the HOgene, which\\na sequences (see Figure 11-28). A specific DNA rearrange-\\noccurs only in the absence of Ash1 protein. The smaller daughter\\nment brings the genes that encode the \"-specific or a-specific cells, which produce Ash1 protein, cannot switch; after growing\\ntranscription factors from these silent loci to the active MAT in size through interphase, they divide to form a mother cell and\\nlocus where they can be transcribed. daughter cell. Orange cells and arrows indicate switch events.\\n22.4 • Regulation of Asymmetric Cell Division 23\\n!FIGURE 22-22 Model for restriction of mating-\\ntype switching to mother cells in S.cerevisiae. Ash1\\nprotein prevents a cell from transcribing the HOgene AV M\\nwhose encoded protein initiates the DNA rearrangement ASH1 SHide E\\nttsohe apata .r rSeatsweuisltt csfhr oiinnmg m  oaa ctncineugwrs-tl yyo pnbeluy ds idwne ittdhc hedi anmuggo fhtrhtoeemrr   ccaeelltllo,,   ba\"fetceoarr  ui\"tse mRNA Bud 1 mRNo Anim DIA \\nAsh1 protein is present in the daughter cell but not in She3p Aa C\\nltohcea lmizaottihoenr  ocfe lAl.s Thh1e i sm tohlee counlaer- wbaasyi str afonrs pthoirst  doiff fAerSeHn1tial  Location:  ON\\nmRNA into the bud. A linking protein, She2p, binds  She2Fp POMyo4p Actin liz N\\na E\\nto specific 3% untranslated sequences in the ASH1 t\\nio C\\nmRNA and also binds to She3p protein. This protein in n T\\nturn binds to a myosin motor, Myo4p, which moves I\\nO\\nalong actin filaments into the bud. [See S. Koon and \\nN\\nB. J. Schnapp, 2001, Curr. Biology11:R166–R168.]\\nS\\nBudding yeasts use a relatively simple mechanism to  dermal fate (see Figures 14-29 and 15-36). The delaminat-\\ncreate molecular differences between the two cells formed by ing cells move inside and become spherical neuroblasts,\\ndivision. In higher organisms, polarization of the parental while the prospective epidermal cells remain behind and\\ncell involves many more participants, and, in addition, the close up to form a tight sheet.\\nmitotic spindle must be oriented in such a way that each Once formed, the neuroblasts undergo asymmetric divi-\\ndaughter cell receives its own set of cytoplasmic components. sions, at each division recreating themselves and producing a\\nTo illustrate these complexities, we focus on asymmetric  ganglion mother cell (GMC)at the basal side of the neuro-\\ndivision of neuroblasts in Drosophila. Genetic studies in  blast (Figure 22-23). A single neuroblast will produce several\\nC. elegansand Drosophilahave revealed the key participants, GMCs; each GMC in turn forms two neurons. Depending \\na first step in understanding at the molecular level how asym- on where they form in the embryo and consequent regula-\\nmetric cell division is regulated in multicellular organisms. tory events, neuroblasts may form more or fewer GMCs.\\nLarge Protein Complexes Are Localized \\nat Opposite Ends of Dividing Neuroblasts  1 Ectodermal\\ncells\\nin Drosophila\\nDV M\\nFly neuroblasts arise from a sheet of ectoderm cells that is 2 roside E\\noo D\\none cell thick. As in vertebrates, the Drosophilaectoderm ph A I\\nfeoctromdse rbmo tche llesp hidaevrem thise  panodte nthtiea ln teor avsosuusm sey esittehmer,  aa nndeu mraal noyr  3 ilanim A \\nEa C\\neopnildye rinm acel rfatatein.  Ucneldlse,r  sthoem ceo notfr tohl eo fc eglelns eisn tchraeta sbee cionm siez aec atinvde Neuroblast mbrtion ON\\nbegin to loosen from the ectodermal layer. At this point, the 4 yo: E N\\nFPO c\\ndelaminating cells use Notch signaling to mediate lateral  to E\\ninhibitionof their neighbors, causing them to retain the epi- GMC de C\\nr T\\nm\\nI\\n5 al C ON\\n!FIGURE 22-23 Asymmetric cell division during Drosophila GMC ell  S\\nneurogenesis.The ectodermal sheet (1) of the early embryo gives rise to both D\\nepidermal cells and neural cells. Neuroblasts, the stem cells for the nervous iv\\nis\\nsystem, are formed when ectoderm cells enlarge, separate from the ectodermal 6 io\\nepithelium, and move into the interior of the embryo (2–4). Each neuroblast n\\ns\\nthat arises divides asymmetrically to recreate itself and produce a ganglion Neurons  in\\nmother cell (GMC) (5). Subsequent divisions of a neuroblast produce more  t\\nGMC h\\nGMCs, creating a stack of these precursor cells (6). Each GMC divides once  e\\n \\nto give rise to two neurons (7). Neuroblasts and their GMC descendants can\\n7\\nhave different fates depending on their location.\\n24 CHAPTER 22 • Cell Birth, Lineage, And Death\\nNeuroblasts and GMCs in different locations exhibit differ- in the process of division. This complex is already localized\\nent patterns of gene expression, an indicator of their fates. in ectoderm cells and stays at the apical end as a prospective\\nAnalysis of fly mutants led to the discovery of key proteins neuroblast delaminates from the ectodermal sheet (Figure\\nthat participate in creating specific neuroblasts, inhibiting 22-24a, left). Three proteins compose the apical complex:\\nepidermal cells from becoming neuroblasts, and directing Baz, Par6, and PKC3, the last being an isoform of protein \\nneuroblasts to divide asymmetrically. kinase C. The Baz/Par6/PKC3 complex persists at the apical\\nend from late interphase (G stage) until late anaphase in mi-\\n2\\nApical Baz/Par6/PKC3 Complex A ternary complex of pro- tosis and then disperses or is destroyed (see Figure 20-29 for\\nteins, referred to as the apical complex,is located at the end mitotic stages). After each neuroblast division, the complex\\nof a fly neuroblast that will become the “new” neuroblast re-forms at the apical end of the progeny neuroblast in the\\nnext interphase. It remains unclear how polarity information\\nis preserved through telophase, since no known proteins are\\nlocalized during that time. As noted previously, the very first\\nApical (Baz, \\nProtein complexes: Miranda Numb\\nPar6, PKC3) C. eleganscell division after fertilization is asymmetric, form-\\ning an AB cell and P1 cell, which have quite different fates\\n(see Figure 22-9). Remarkably, a protein complex like the\\nfly Baz/Par6/PKC3 complex controls the asymmetry of the\\n(a) Neuroblast P1-AB division; similar protein complexes exist in mammals\\nas well.\\nFPO Two mechanisms control the apical localization of the\\nDrosophilaBaz/Par6/PKC3 complex: the first operates in the\\nGMC\\nectoderm prior to neuroblast delamination; the second is ac-\\ntive during the repeating divisions of delaminated neuro-\\nINTERPHASE ANAPHASE blasts. The first mechanism involves another multiprotein\\nNEUROBLAST NEUROBLAST complex composed of at least three proteins, Scribble (Scrib),\\nCentromere Discs-large (Dlg), and Lethal giant larvae (Lgl). This complex\\nis located in the cortical region of ectoderm cells (i.e., the re-\\ngion just below the plasma membrane), but it is not polar-\\n(b) ized along the apical-basal axis of these cells. Thus, though\\nNeuroblast\\nnecessary for the apical localization of the Baz/Par6/PKC3\\ncomplex, the Scrib/Dlg/Lgl complex probably is not suffi-\\ncient. Once a neuroblast has separated from the ectoderm,\\nGMC two other proteins called Inscuteable and Partner of Inscute-\\nCytoskeleton: able join the Baz/Par6/PKC3 complex, forming a five-protein\\nActin microfilaments complex. These two additional proteins probably stabilize\\nChromosomes\\nMicrotubules the Baz/Par6/PKC3 complex. Reassembly of the entire com-\\nplex at the apical end following each neuroblast division re-\\n▲FIGURE 22-24 Localized protein complexes that control quires microtubules and perhaps microfilaments (Figure\\nasymmetric cell division in the Drosophilaneuroblast.(a) The 22-24b). This finding suggests that motor protein–mediated\\napical complex (blue), which consists of three proteins (Baz, Par6, transport of these protein complexes along cytoskeletal ele-\\nand PKC3), is localized in ectoderm cells and in delaminating ments plays a role in their apical localization. \\nneuroblasts. As a neuroblast begins dividing, the Miranda\\ncomplex (orange) assembles at the apical side and then moves Basal Complexes In a dividing neuroblast two basal com-\\nto the basal side, where it will be incorporated into the ganglion plexesare found at the end that will form a ganglion mother\\nmother cell (GMC). The second basal complex (green), containing cell (GMC) at division (see Figure 22-24a, right). Localiza-\\nthe Numb protein, is detected only basally. Mutations in genes tion of these complexes, called the Miranda complex and the\\nthat encode polarized proteins disrupt asymmetric cell division.\\nNumb complex, at the basal end of neuroblasts requires the\\n(b) Motor protein–mediated transport along cytoskeletal filaments\\napical Baz/Par6/PKC3 complex. The Numb complex is lo-\\nis thought to localize the apical and basal complexes. Actin\\ncated throughout the cell during interphase; at prophase it\\nmicrofilaments (red) lie just under the cell surface at all times.\\nbecomes basally located. In contrast, the Miranda complex\\nMicrotubules (blue) radiate from the centrosome during\\ninitially assembles at the apical end; it then moves to the\\ninterphase and then assemble into the mitotic spindle, attached\\nbasal end at prophase. The apical Baz/Par6/PKC3 complex\\nto the duplicated centrosomes, during cell division. Note that in\\nthe neuroblast-GMC division the centrosome is skewed to the forms even when either of the basal complexes cannot as-\\nneuroblast end of the cell, and the chromosomes (light gray) are semble due to mutations. Mutations in any of the compo-\\ndiagrammed lined up on the spindle. [Adapted from C. Q. Doe and nents of the apical complex, however, prevent the Miranda\\nB. Bowerman, 2001, Curr. Opin. Cell Biol.13:68.] and Numb complexes from moving to the basal region. \\n22.4 • Regulation of Asymmetric Cell Division 25\\nInstead they accumulate in the cortical region or in clumps blasts, the mitotic spindle first aligns perpendicular to the\\nthroughout the cell. The simplest idea, that the basal com- apical-basal axis and then turns 90 degrees to align with it\\nplexes form wherever the apical complex is absent, is prob- at the same time that the basal complexes become localized\\nably wrong, since parts of the cell contain neither of these to the basal side (Figure 22-25). The apical complex, which\\ncomplexes.  is already in place before spindle rotation, controls the final\\nThe basal Miranda complex has four components: Pros- orientation of the spindle. This is supported by the finding\\npero protein, a presumed transcription factor that contains that mutations in any of the components of the apical com-\\na DNA-binding homeodomain; Miranda protein, which has plex eliminate the coordination of the spindle with apical-\\na coiled-coil structure; Staufen, an RNA-binding protein; basal polarity, causing the spindle orientation to become\\nand prosperomRNA. During each round of neuroblast di- random. \\nvision, new Prospero protein is made during interphase and Spindle orientation is linked to actin and myosin-related\\nincorporated into the Miranda basal complex, which is dis- proteins. Mammalian Par6, a component of the apical com-\\ntributed to the GMC at the time of cleavage. In the absence plex, can bind two small GTPase proteins, Cdc42 and\\nof Prospero, GMCs do not transcribe the appropriate genes Rac1, that control the arrangement of actin microfilaments\\nand do not develop into normal neurons. The participation in the cytoskeleton (see Figure 19-29). The protein Lethal\\nof a transcription factor in a localized cytoplasmic complex giant larvae (Lgl), a component of the Scrib/Dlg/Lgl asym-\\nprovides a link between the asymmetry of the cytoplasm and metry complex, helps to localize the Miranda complex\\nsubsequent cell-fate determination. Since the apical and basal basally in fly neuroblasts, although Lgl itself is localized\\ncomplexes are presumably localized by the cytoskeleton in apically. Lgl binds to myosin II, which functions in cyto-\\nconcert with motor proteins, the involvement of a transcrip- kinesis (see Figure 19-20), suggesting that the location of\\ntion factor in one of the complexes potentially explains how the cleavage furrow may be coordinated with localization\\na transcription factor is preferentially distributed to one of of the apical complex. \\ntwo daughter cells. The involvement of an mRNA and a Two yeast proteins related to Lgl also bind to myosin II;\\nRNA-binding protein in the Miranda complex suggests that they have been implicated in exocytosis and secretion, specif-\\nit also mediates translational control in GMCs.  ically in the docking of post-Golgi vesicles with the plasma\\nmembrane (Chapter 17). Mutations in the two yeast proteins\\nsuppress mutations in the myosin IIgene. This observation is\\nOrientation of the Mitotic Spindle Is Linked \\ninterpreted to mean that the function of myosin II in spindle\\nto Cytoplasmic Cell-Asymmetry Factors\\norientation is opposite to that of the Lgl-like proteins: re-\\nFor localized protein complexes to be differentially incorpo- duction of one protein’s function is ameliorated by reduction\\nrated into two daughter cells requires that the plane of cell of the other, restoring a semblance of the normal balance. \\ndivision be appropriately oriented. In dividing fly neuro- In this case, therefore, myosin and Lgl probably act in \\nFPO inV M\\n Dide E\\nroo D\\nso A I\\npn A\\n0 s 32 s 64 s 80 s hilaima  C\\nt\\nNio O\\nen N\\nu: \\nrM N\\no\\nbit E\\n▲imEagXiPngE RreIMveEaNlsT rAoLta tFioIGn UoRf Eth 2e 2m-2it5otiTci mspei-nladples ein fluorescence fauren cvtiisoinb laes o tnh eo psppoinsditlee  spiodleess ;o af st hme itcoeslils.  Tphreoscee eds the last Cell otic Spin CTIO\\nDd N\\nasymmetrically dividing neuroblasts. Early Drosophilaembryos microtubules forming the mitotic spindle are assembled ivle S\\nwere injected with a hybrid gene composed of the gene from the poles (see Figure 20-29). In successive  is A\\nencoding green fluorescent protein (GFP) fused to the gene images (at 32, 64, and 80 seconds), the bipolar spindle ionssy\\nencoding Tau, a protein that binds to microtubules. At the top  can be seem to form and rotate 90 degrees to align  m\\ne\\nare time-lapse images of a single dividing neuroblast in a live with the apical-basal axis, as schematically depicted at t\\nr\\nembryo. The basal side is at the top, and the apical side at the the bottom. [From J. A. Kaltschmidt et al, 2000, Nature Cell  y \\nbottom. At time 0, equivalent to prophase, the two centrosomes Biol.2:7–12.]\\n26 CHAPTER 22 • Cell Birth, Lineage, And Death\\nopposite directions: myosin II moving Miranda or other ma- organisms require signals to stay alive. In the absence of such\\nterials to control spindle orientation and Lgl restraining it.  survival signals, frequently referred to as trophic factors,cells\\nFurther evidence for the importance of myosin in spin- activate a “suicide” program. Second, in some developmental\\ndle orientation comes the finding that myosin VI binds di- contexts, including the immune system, specific signals induce\\nrectly to Miranda. In addition, mutations in myosin VI in a “murder” program that kills cells. Whether cells commit sui-\\nflies prevent basal targeting of Miranda and simultaneously cide for lack of survival signals or are murdered by killing sig-\\nblock proper spindle orientation. These various findings give nals from other cells, recent studies suggest that death is\\nus the beginning of a picture of how localized protein com- mediated by a common molecular pathway. In this section, we\\nplexes are coordinated with cell division, so that each daugh- first distinguish programmed cell death from death due to tis-\\nter cell receives the appropriate amounts of the various sue injury, then consider the role of trophic factors in neuronal\\ncomplexes.  development, and finally describe the evolutionarily conserved\\neffector pathway that leads to cell suicide or murder.\\nKEY CONCEPTS OF SECTION 22.4\\nProgrammed Cell Death Occurs \\nRegulation of Asymmetric Cell Division Through Apoptosis\\n■ Asymmetric cell division requires polarization of the di- The demise of cells by programmed cell death is marked by\\nviding cell, which usually entails localization of some cyto- a well-defined sequence of morphological changes, collec-\\nplasmic components, and then the unequal distribution of tively referred to as apoptosis, a Greek word that means\\nthese components to the daughter cells (see Figure 22-20).  “dropping off” or “falling off,” as in leaves from a tree.\\nDying cells shrink and condense and then fragment, releas-\\n■ In the asymmetric division of budding yeasts, a myosin-\\ning small membrane-bound apoptotic bodies, which gen-\\ndependent transport system carries ASH1mRNA into the\\nerally are engulfed by other cells (Figure 22-26; see also\\nbud (see Figure 22-22). \\nFigure 1-19). The nuclei condense and the DNA is frag-\\n■ Ash1 protein is produced in the daughter cell soon after mented. Importantly, the intracellular constituents are not\\ndivision  and  prevents  expression  of  HO  endonuclease,\\nreleased into the extracellular milieu where they might have\\nwhich is necessary for mating-type switching. Thus a daugh-\\ndeleterious effects on neighboring cells. The highly stereo-\\nter cell cannot switch mating type, whereas the mother cell\\ntyped changes accompanying apoptosis suggested to early\\nfrom which it arises can (see Figure 22-21). \\nworkers that this type of cell death was under the control of\\n■ Asymmetric cell division in Drosophilaneuroblasts depends a strict program. This program is critical during both em-\\non  localization  of  the  apical  protein  complex  (Baz/Par6/ bryonic and adult life to maintain normal cell number and\\nPKC3) and two basal complexes. The basal complexes are in- composition.\\ncorporated into the ganglion mother cell (GMC) and contain The genes involved in controlling cell death encode pro-\\nproteins that determine cell fate (see Figure 22-24).  teins with three distinct functions: \\n■ The apical Baz/Par6/PKC3 complex also controls asym- ■ “Killer” proteins are required for a cell to begin the\\nmetric cell division in C. elegansand perhaps mammals.  apoptotic process. \\n■ Asymmetry factors exert their influence at least in part ■ “Destruction” proteins do things like digest DNA in a\\nby controlling the orientation of the mitotic spindle, so that dying cell. \\nasymmetrically localized proteins and structures are differ-\\nentially incorporated into the two daughter cells. Myosin ■ “Engulfment” proteins are required for phagocytosis of\\nthe dying cell by another cell. \\nproteins bind to proteins that control asymmetry factors of\\ncells to control spindle orientation. At first glance, engulfment seems to be simply an after-death\\ncleanup process, but some evidence suggests that it is part\\nof the final death decision. For example, mutations in killer\\n22.5 Cell Death and Its Regulation  genes always prevent cells from initiating apoptosis, whereas\\nmutations that block engulfment sometimes allow cells to sur-\\nProgrammed cell death is a cell fate, an odd sort of cell fate vive that would normally die. That is, cells with engulfment-\\nbut nonetheless one that is essential. Cell death keeps our gene mutations can initiate apoptosis but then sometimes\\nhands from being webbed, our embryonic tails from persist- recover. \\ning, our immune system from responding to our own pro- In contrast to apoptosis, cells that die in response to tis-\\nteins, and our brain from being filled with useless electrical sue damage exhibit very different morphological changes,\\nconnections. In fact, the majority of cells generated during referred to as necrosis. Typically, cells that undergo this\\nbrain development also die during development. process swell and burst, releasing their intracellular con-\\nCellular interactions regulate cell death in two fundamen- tents that can damage surrounding cells and frequently\\ntally different ways. First, most cells, if not all, in multicellular cause inflammation.\\n22.5 • Cell Death and Its Regulation 27\\n!FIGURE 22-26 Ultrastructural\\nfeatures of cell death by apoptosis.\\n(a) Schematic drawings illustrating  F M\\no\\nthe progression of morphologic c E\\nu\\nchanges observed in apoptotic  s D\\ncells. Early in apoptosis, dense  A I\\nn A\\nchromosome condensation occurs im  \\nalong the nuclear periphery. The cell a C\\nbody also shrinks, although most tio O\\nn N\\norganelles remain intact. Later both : A N\\nthe nucleus and cytoplasm fragment, p\\no E\\nforming apoptotic bodies, which are p C\\nt\\nphagocytosed by surrounding cells. o T\\ns\\n(b) Photomicrographs comparing  is I\\nO\\na normal cell (top) and apoptotic cell\\nN\\n(bottom).Clearly visible in the latter\\nS\\nare dense spheres of compacted\\nchromatin as the nucleus begins to\\nfragment. [Part (a) adapted from J. Kuby,\\n28p (w) X 34p (d)\\n1997, Immunology,3d ed., W. H. Freeman\\n& Co., p. 53. Part (b) from M. J. Arends\\nand A. H. Wyllie, 1991, Int’l. Rev. Exp.\\nPathol.32:223.]\\nNeurotrophins Promote Survival of Neurons cremental increases in the target-field size are accompanied\\nby commensurate incremental increases in the number of\\nThe earliest studies demonstrating the importance of trophic neurons innervating the target field. This relation was found\\nfactors in cellular development came from analyses of the de- to result from the selective survival of neurons rather than\\nveloping nervous system. When neurons grow to make con- changes in their differentiation or proliferation. The obser-\\nnections to other neurons or to muscles, sometimes over vation that many sensory and motor neurons die after reach-\\nconsiderable distances, more cells grow than will eventually ing their peripheral target field suggested that these neurons\\nsurvive. Those that make connections prevail and survive; compete for survival factors produced by the target tissue. \\nthose that fail to connect die.  Subsequent to these early observations, scientists discov-\\nIn the early 1900s the number of neurons innervating the ered that transplantation of a mouse sarcoma tumor into a\\nperiphery was shown to depend upon the size of the tissue to chick led to a marked increase in the numbers of certain\\nwhich they would connect, the so-called “target field.” For in- types of neurons. This finding implicated the tumor as a rich\\nstance, removal of limb buds from the developing chick em- source of the presumed trophic factor. To isolate and purify\\nbryo leads to a reduction in the number of sensory neurons this factor, known simply as nerve growth factor (NGF), sci-\\nand motoneurons innervating the bud (Figure 22-27). Con- entists used an in vitro assay in which outgrowth of neurites\\nversely, grafting additional limb tissue to a limb bud leads  from sensory ganglia (nerves) was measured. Neurites are ex-\\nto an increase in the number of neurons in corresponding  tensions of the cell cytoplasm that can grow to become the\\nregions of the spinal cord and sensory ganglia. Indeed, in- long wires of the nervous system, the axons and dendrites\\n28 CHAPTER 22 • Cell Birth, Lineage, And Death\\nBDNF\\nNGF NT-3 NT-4/5 NT-3 NT-3\\nFPO\\nExterior\\nATP ATP ATP ATP ATP ATP\\nP P P P P P\\n21p (w) X 28p (d) ADP ADP ADP ADP ADP ADP\\nCytosol Tyrosine\\nkinase\\nTrkA TrkB TrkC\\n▲FIGURE 22-28 Specificity of neurotrophins for Trks,a\\nfamily of receptor tyrosine kinases. Each neurotrophin binds\\nwith high affinity to one Trk receptor indicated by the solid arrow\\nfrom the ligand to the receptor. NT-3 also can bind with lower\\naffinity to both TrkA and TrkB as indicated by the dashed arrow. \\nIn addition, neurotrophins bind to a distinct receptor called\\np75NTR either alone or in combination with Trks. NGF # nerve\\ngrowth factor; BDNF # brain-derived neurotrophic factor; \\nNT-3 # neurotrophin-3. [Adapted from W. D. Snider, 1994, Cell77:627.]\\nNeurotrophin Receptors Neurotrophins bind to and activate\\na family of receptor tyrosine kinases called Trks(pronounced\\n“tracks”). (The general structure of receptor tyrosine kinases\\n▲EXPERIMENTAL FIGURE 22-27 The survival of motor\\nand the intracellular signaling pathways they activate are cov-\\nneurons depends on the size of the muscle target field they\\nered in Chapter 14.) As shown in Figure 22-28, NGF binds to\\ninnervate.(a) Removal of a limb bud from one side of a chick\\nTrkA; BDNF, to TrkB; and NT-3, to TrkC. Binding of these fac-\\nembryo at about 2.5 days results in a marked decrease in the\\ntors to their receptors provides a survival signal for different\\nnumber of motor neurons on the affected side. In an amputated\\nembryo, normal numbers of motor neurons are generated on classes of neurons. A second type of receptor called p75NTR\\nboth sides (middle).Later in development, many fewer motor (NTR #neurotrophin receptor) also binds to neurotrophins,\\nneurons remain on the side of the spinal cord with the missing but with lower affinity. However, p75NTRforms heteromulti-\\nlimb than on the normal side (bottom).Note that only about 50 meric complexes with the different Trk receptors; this associa-\\npercent of the motor neurons that originally are generated tion increases the affinity of Trks for their ligands. Some studies\\nnormally survive. (b) Transplantation of an extra limb bud into an indicate that the binding of NGF to p75NTRin the absence of\\nearly chick embryo produces the opposite effect, more motor TrkA may promote cell death rather than prevent it.\\nneurons on the side with additional target tissue than on the\\nnormal side. [Adapted from D. Purves, 1988, Body and Brain: A Trophic Knockouts of Neurotrophins and Their Receptors To criti-\\nTheory of Neural Connections(Cambridge, MA: Harvard University Press), cally address the role of the neurotrophins in development, sci-\\nand E. R. Kandel, J. H. Schwartz, and T. M. Jessell, 2000, Principles of entists produced mice with knockout mutations in each of the\\nNeural Science,4th ed. (New York: McGraw-Hill), p. 1054, Figure 53-11.] neurotrophins and their receptors. These studies revealed that\\ndifferent neurotrophins and their corresponding receptors are\\nrequired for the survival of different classes of sensory neurons,\\n(see Figure 7-29). The later discovery that the submaxillary which carry signals from peripheral sensory systems to the\\ngland in the mouse also produces large quantities of NGF en- brain (Figure 22-29). For instance, pain-sensitive (nociceptive)\\nabled biochemists to purify it to homogeneity and to se- neurons, which express TrkA, are selectively lost from the dor-\\nquence it. A homodimer of two 118-residue polypeptides, sal root ganglion of knockout mice lacking NGF or TrkA,\\nNGF belongs to a family of structurally and functionally re- whereas TrkB- and TrkC-expressing neurons are unaffected in\\nlated trophic factors collectively referred to as neurotrophins. such knockouts. In contrast, TrkC-expressing proprioceptive\\nBrain-derived neurotrophic factor (BDNF) and neurotro- neurons, which detect the position of the limbs, are missing\\nphin-3 (NT-3) also are members of this protein family. from the dorsal root ganglion in TrkCand NT-3mutants.\\n22.5 • Cell Death and Its Regulation 29\\nWild Type Mutant Mutant \"EXPERIMENTAL FIGURE 22-29\\nDifferent classes of sensory neurons are\\nNGF −/− NT-3 −/− lost in knockout mice lacking different\\nSpinal Mechanoreceptors or or\\ntrophic factors or their receptors.In\\ncord Dorsal root TrkA −/− TrkC −/− animals lacking nerve growth factor (NGF)\\nganglion or its receptor TrkA, small nociceptive \\nPropioceptive (pain-sensing) neurons (blue) that innervate\\nneurons the skin are missing. These neurons\\nNociceptive express TrkA receptor and innervate \\nneurons NGF-producing targets. In animals lacking\\neither neurotrophin-3 (NT-3) or its receptor\\nTrkC, large propioceptive neurons (red)\\ninnervating muscle spindles are missing.\\nMuscle produces NT-3 and the\\nSkin\\npropioceptive neurons express TrkC.\\nMechanoreceptors (brown), another class\\nof sensory neurons in the dorsal root\\nganglion, are unaffected in these mutants.\\nMuscle Motoneurons\\n[Adapted from W. D. Snider, 1994, Cell77:627.]\\nA Cascade of Caspase Proteins Functions  (a)\\nin the Apoptotic Pathway \\nV M\\nid\\nKey insights into the molecular mechanisms regulating cell e E\\no D\\ndeath came from genetic studies in C. elegans.As we noted   A I\\nin Section 22.1, scientists have traced the lineage of all the nn A\\nsomatic cells in C. elegansfrom the fertilized egg to the  im  C\\nmature worm (see Figure 22-9). Of the 947 nongonadal cells at O\\ngenerated during development of the adult hermaphrodite ion N\\nform, 131 cells undergo programmed cell death.  : P N\\nr E\\nSpecific mutations have identified a variety of genes o\\ng C\\nwhose encoded proteins play an essential role in controlling ra T\\nprogrammed cell death during C. elegansdevelopment. For (b) m I\\nm O\\ninstance, programmed cell death does not occur in worms FPO e N\\nd\\ncarrying loss-of-function mutations in the ced-3gene or the  C S\\nced-4gene; as a result, the 131 “doomed” cells survive (Fig- e\\nure 22-30). CED-4 is a protease-activating factor that ll D\\ne\\ncleaves the CED-3 precursor protein, creating an active a\\nt\\nCED-3 protease that initiates cell death. In contrast, in ced- h\\n9mutants, all cells die during embryonic life, so the adult  in\\n C\\nform never develops. These genetic studies indicate that the . e\\nCED-3 and CED-4 proteins are required for cell death, that le\\ng\\nCED-9 suppresses apoptosis, and that the apoptotic path- a\\nn\\nway can be activated in all cells. Moreover, the finding that s\\ncell death does not occur in ced-9/ced-3double mutants sug- ▲EXPERIMENTAL FIGURE 22-30 Mutations in the Em\\ngests that CED-9 acts “upstream” of CED-3 to suppress the ced-3gene block programmed cell death in C.elegans. b\\nr\\napoptotic pathway. (a) Newly hatched larva carrying a mutation in the ced-1 yo\\nThe confluence of genetic studies in worms and studies gene. Because mutations in this gene prevent engulfment nic\\non human cancer cells first suggested that an evolutionarily of dead cells, highly refractile dead cells accumulate  D\\nconserved pathway mediates apoptosis. The first apoptotic (arrows), facilitating their visualization. (b) Newly hatched ev\\nlarva with mutations in both the ced-1and ced-3genes. e\\ngene to be cloned, bcl-2,was isolated from human follicular lo\\nlymphomas. A mutant form of this gene, created in lym- The absence of refractile dead cells in these double p\\nm\\nmutants indicates that no cell deaths occurred. Thus CED-3\\nphoma cells by a chromosomal rearrangement, was shown to e\\nprotein is required for programmed cell death. [From H. M. n\\nact as an oncogenethat promoted cell survival rather than t\\nEllis and H. R. Horvitz, 1986, Cell91:818; courtesy of Hilary Ellis.]\\ncell death (Chapter 23). The human Bcl-2 protein and worm\\nCED-9 protein are homologous, and a bcl-2transgene can\\n30 CHAPTER 22 • Cell Birth, Lineage, And Death\\nblock the extensive cell death found in ced-9mutant worms. For instance, expression of C. elegansCED-4 in a human\\nThus both proteins act as regulators that suppress the apop- kidney cell line induces rapid apoptosis. This can be blocked\\ntotic pathway (Figure 22-31). In addition, both proteins con- by co-expression of the negative regulator CED-9 (or mam-\\ntain a single transmembrane domain and are localized to the malian Bcl-2), indicating that CED-9 opposes CED-4 action.\\nouter mitochondrial, nuclear, and endoplasmic reticulum In addition, CED-9 has been shown to directly bind to \\nmembranes, where they serve as sensors that control the CED-4 and move it from the cytosol to intracellular mem-\\napoptotic pathway in response to external stimuli. As we dis- branes. Thus the pro-apoptotic function of CED-4 is directly\\ncuss below, other regulators promote apoptosis. suppressed by the anti-apoptotic function of CED-9. CED-4\\nThe effector proteinsin the apoptotic pathway are en- also binds directly to the CED-3 caspase (and related mam-\\nzymes called caspasesin vertebrates,so named because they malian caspases) and promotes activation of its protease ac-\\ncontain a key cysteine residue in the catalytic site and selec- tivity, as noted previously. Biochemical studies have shown\\ntively cleave proteins at sites just C-terminal to aspartate that CED-4 can simultaneously bind to both CED-9 and\\nresidues. Humans have 15 different caspases, all of which are CED-3. These results fit with the genetics, which shows that\\ninitially made as procaspases that must be cleaved to become the absence of CED-9 has no effect if CED-3 is also missing\\nactive. Such proteolytic processing of proproteins is used re- (ced-3/ced-9double mutants have no cell death, like ced-3\\npeatedly in blood clotting, generation of digestive enzymes, mutants).\\nand generation of hormones (Chapter 17). Initiator caspases\\n(e.g., caspase-9) are activated by autoproteolysis induced by\\nPro-Apoptotic Regulators Permit Caspase\\nother types of proteins, which help the initiators to aggre-\\nActivation in the Absence of Trophic Factors \\ngate. Activated initiator caspases cleave effector caspases\\n(e.g., caspase-3) and thus quickly amplify the total caspase Having introduced the major participants in the apoptotic\\nactivity level in the dying cell. The various effector caspases pathway, we now take a closer look at how the effector cas-\\nrecognize and cleave short amino acid sequences and differ in pases are activated in mammalian cells. Although the normal\\ntheir preferred target sequences. Their specific intracellular function of CED-9 or Bcl-2 is to suppress the cell-death path-\\ntargets include proteins of the nuclear lamina and cytoskele- way, other intracellular regulatory proteins promote apopto-\\nton whose cleavage leads to the demise of a cell. Seven lines sis. The first pro-apoptotic regulator to be identified, named\\nof knockout mice have been made, each lacking a particular Bax, was found associated with Bcl-2 in extracts of cells ex-\\ncaspase function. Some homozygotes die as embryos; others pressing high levels of Bcl-2. Sequence analysis demonstrated\\nare viable but with subtle defects. Strikingly, some exhibit that Bax is related in sequence to CED-9 and Bcl-2, but over-\\nan excessive number of neurons (i.e., neuronal hyperplasia), production of Bax induces cell death rather than protecting\\nreaffirming the importance of cell death in development of cells from apoptosis, as CED-9 and Bcl-2 do. Thus this family\\nthe nervous system; other mutants have cells that are resis- of regulatory proteins comprises both anti-apoptoticmem-\\ntant to apoptosis. The principal effector protease inC. elegans bers (e.g., CED-9, Bcl-2) and pro-apoptoticmembers (e.g.,\\nis the CED-3 caspase.  Bax). All members of this family, which we refer to as the Bcl-\\nCell-culture studies have yielded important insights about 2 family,are single-pass transmembrane proteins and can par-\\na group of adapter proteinsthat coordinate the action of reg- ticipate in oligomeric interactions. In mammals, six Bcl-2\\nulators and effectors to control apoptosis (see Figure 22-31). family members prevent apoptosis and nine promote it. Thus\\nthe fate of a given cell—survival or death—may reflect the\\nparticular spectrum of Bcl-2 family members made by that\\nRegulator Adapter Effector\\ncell and the intracellular signaling pathways regulating them.\\nFPO\\nSome Bcl-2 family members preserve or disrupt the in-\\nC. elegans CED-9 CED-4 CED-3 Death tegrity of mitochondria, thereby controlling release of mito-\\nchondrial proteins such as cytochrome c. In normal healthy\\nVertebrates Bcl-2 Apaf-1 Casp9 Casp3 Death\\ncells, cytochrome cis localized between the inner and outer\\nmitochondrial membrane, but in cells undergoing apoptosis,\\n▲FIGURE 22-31 Overview of the evolutionarily conserved cytochrome cis released into the cytosol. This release can be\\napoptotis pathway in C.elegansand vertebrates.Three blocked by overproduction of Bcl-2; conversely, overproduc-\\ngeneral types of proteins are critical in this conserved pathway.\\ntion of Bax promotes release of cytochrome cinto the cytosol\\nRegulators either promote or suppress apoptosis; two\\nand apoptosis. Moreover, injection of cytochrome cinto the\\northologous regulators shown here, CED-9 and Bcl-2, suppress\\ncytosol of cells induces apoptosis. A variety of death-inducing\\napoptosis in the presence of trophic factors. Adapters interact\\nstimuli cause Bax monomers to move from the cytosol to the\\nwith both regulators and effectors; in the absence of trophic\\nouter mitochondrial membrane where they oligomerize. Bax\\nfactors, they promote activation of effectors. Proteases called\\nhomodimers, but not Bcl-2 homodimers or Bcl-2/Bax het-\\ncaspases serve as effector proteins; their activation leads to\\ndegradation of various intracellular substrates and eventually  erodimers, permit influx of ions through the mitochondrial\\ncell death. [Adapted from D. L. Vaux and S. J. Korsemeyer, 1999,  membrane. It remains unclear how this ion influx triggers the\\nCell96:245.] release of cytochrome c.Once cytochrome cis released into\\n22.5 • Cell Death and Its Regulation 31\\nthe cytosol, it binds to the adapter protein Apaf-1 (the mam- Some Trophic Factors Induce Inactivation \\nmalian homolog of CED-4) and promotes activation of a cas- of a Pro-Apoptotic Regulator\\npase cascade leading to cell death (Figure 22-32a). Bax can\\ncause cell death by a second pathway that triggers mitochon- We saw earlier that neurotrophins such as nerve growth fac-\\ndrial dysfunction independently of any caspase action, at least tor (NGF) protect neurons from cell death. The intracellular\\nin part by stimulating mitochondrial depolarization. signaling pathways linking such survival factors to inactiva-\\nGene knockout experiments have dramatically confirmed tion of the cell-death machinery are quite elaborate. The\\nthe importance of both pro-apoptotic and anti-apoptotic  finding that trophic factors appear to work largely inde-\\nBcl-2 family members in neuronal development. Mice lacking pendent of protein synthesis suggested that these external\\nthe bcl-xlgene, which encodes an anti-apoptotic protein, have signals lead to changes in the activities of preexisting proteins\\nmassive defects in nervous system development with wide- rather than to activation of gene expression. Scientists\\nspread cell death in the spinal cord, dorsal root ganglion, and demonstrated that in the absence of trophic factors, the non-\\nbrain of developing embryos. In contrast, baxknockouts ex- phosphorylated form of Bad is associated with Bcl-2/Bcl-xl at\\nhibit a marked increase in neurons in some regions of the the mitochondrial membrane (see Figure 22-32a). Binding of\\nnervous system. These knockouts also demonstrate that the Bad inhibits the anti-apoptotic function of Bcl-2/Bcl-xl,\\neffects of Bcl-2 family members generally are tissue specific. thereby promoting cell death. Phosphorylated Bad, however,\\n(a)  Absence of trophic factor: Caspase activation (b)  Presence of trophic factor: Inhibition of caspase activation\\nTrophic factor\\nTrophic factor receptor\\nF M\\n1 o\\nc E\\nu\\nPlasma membrane s D\\n A I\\nDeath n A\\nim  \\nCleavage of substrates a C\\ntio O\\nn N\\nPI-3 kinase Procaspase 3 Caspase 3 PI-3 kinase Procaspase 3 : A N\\np\\no E\\np C\\n14-3-3 Caspase 9 to T\\n14-3-3 ATP s\\nAkt FPO P ADP Akt is IO\\n4\\nN\\nBad\\nProcaspase 9 2 Procaspase 9 S\\nApaf 1 Apaf 1\\nCyt c\\n3\\nBad\\n1 Bcl-2 Bcl-xl Bax Bax mOuetmerb mrainteochondrial Bcl-2Bcl-xl BaxBax\\n2 Cyt c 3\\nCyt c Ions\\n▲FIGURE 22-32 Proposed intracellular pathways leading adapter protein Apaf-1 (3), promoting a caspase cascade that\\nto cell death by apoptosis or to trophic factor–mediated cell leads to cell death (4). (b) In some cells, binding of a trophic\\nsurvival in mammalian cells.(a) In the absence of a trophic factor (e.g., NGF) stimulates PI-3 kinase activity, leading to\\nfactor, Bad, a soluble pro-apoptotic protein, binds to the  activation of the downstream kinase Akt, which phosphorylates\\nanti-apoptotic proteins Bcl-2 and Bcl-xl, which are inserted into Bad (1). Phosphorylated Bad then forms a complex with the \\nthe mitochondrial membrane (1). Bad binding prevents the  14-3-3 protein (2). With Bad sequestered in the cytosol, the \\nanti-apoptotic proteins from interacting with Bax, a membrane- anti-apoptotic Bcl-2/Bcl-xl proteins can inhibit the activity of\\nbound pro-apoptotic protein. As a consequence, Bax forms Bax (3), thereby preventing the release of cytochrome cand\\nhomo-oligomeric channels in the membrane that mediate ion flux activation of the caspase cascade. [Adapted from B. Pettman \\n(2). Through an as-yet-unknown mechanism, this leads to the and C. E. Henderson, 1998, Neuron20:633.]\\nrelease of cytochrome cinto the cytosol, where it binds to the\\n32 CHAPTER 22 • Cell Birth, Lineage, And Death\\ncannot bind to Bcl-2/Bcl-xl and is found in the cytosol com- ■ Genetic studies in C. elegansdefined an evolutionarily\\nplexed to the phosphoserine-binding protein 14-3-3. Hence, conserved apoptotic pathway with three major compo-\\nsignaling pathways leading to Bad phosphorylation would be nents: regulatory proteins, adapter proteins, and effector\\nparticularly attractive candidates for transmitting survival proteases called caspases in vertebrates (see Figure 22-31). \\nsignals.\\n■ Once activated, apoptotic proteases cleave specific intra-\\nA number of trophic factors including NGF have been\\ncellular substrates leading to the demise of a cell. Adapter\\nshown to trigger the PI-3 kinase signaling pathway, leading\\nproteins (e.g., Apaf-1), which bind both regulatory proteins\\nto activation of a downstream kinase called PKB (see Figure\\nand caspases, are required for caspase activation. \\n14-27). Activated PKB  phosphorylates Bad at sites known to\\ninhibit its pro-apoptotic activity. Moreover, a constitutively ■ Pro-apoptotic regulator proteins (e.g., Bax, Bad) pro-\\nactive form of PKB can rescue cultured neurotrophin- mote  caspase  activation,  and  anti-apoptotic  regulators\\ndeprived neurons, which otherwise would undergo apoptosis (e.g., Bcl-2) suppress activation. Direct interactions be-\\nand die. These findings support the mechanism for the sur- tween pro-apoptotic and anti-apoptotic proteins lead to\\nvival action of trophic factors depicted in Figure 22-32b. In cell death in the absence of trophic factors. Binding of ex-\\nother cell types, different trophic factors may promote cell tracellular trophic factors can trigger changes in these in-\\nsurvival through post-translational modification of other teractions, resulting in cell survival (see Figure 22-32).\\ncomponents of the cell-death machinery. \\n■ The Bcl2 family contains both pro-apoptotic and anti-\\napoptotic proteins; all are single-pass transmembrane pro-\\nTumor Necrosis Factor and Related Death  teins and engage in protein-protein interactions.\\nSignals Promote Cell Murder by Caspases  ■ Binding of extracellular death signals, such as tumor\\nnecrosis factor and Fas ligand, to their receptors activates\\nAlthough cell death can arise as a default in the absence of\\nan associated protein (FADD) that in turn triggers the cas-\\nsurvival factors, apoptosis can also be stimulated by posi-\\npase cascade leading to cell murder. \\ntively acting “death” signals.For instance, tumor necrosis\\nfactor (TNF),which is released by macrophages, triggers the\\ncell death and tissue destruction seen in certain chronic in-\\nflammatory diseases. Another important death-inducing sig- PERSPECTIVES FOR THE FUTURE\\nnal, the Fas ligand,is a cell-surface protein expressed by\\nactivated natural killer cells and cytotoxic T lymphocytes. Cell birth, lineage, and death lie at the heart of the growth\\nThis signal can trigger death of virus-infected cells, some of an organism and are also central to disease processes,\\ntumor cells, and foreign graft cells.  most notably cancer. Few transformations seem more \\nBoth  TNF  and  Fas  ligand  act  through  cell-surface remarkable than the blooming of cell types during develop-\\n“death” receptors that have a single transmembrane domain ment. A lineage beginning with a fertilized egg, a “plain\\nand are activated when ligand binding brings three receptor vanilla” 200-$m sphere, produces neurons a yard long,\\nmolecules into close proximity. The trimeric receptor com- pulsating multinucleate muscle cells, exquisitely light-\\nplex attracts a protein called FADD (Fas-associated death sensitive retina cells, ravenous macrophages that recognize\\ndomain), which serves as an adapter to recruit and in some and engulf germs, and all the hundreds of other cell types.\\nway activate caspase-8, an initiator caspase, in cells receiving Regulators of cell lineage produce this rich variety by \\na death signal. The death domain found in FADD is a se- controlling two critical decisions: (1) when and where to\\nquence that is present in a number of proteins involved in activate the cell division cycle (Chapter 21) and (2) whether\\napoptosis. Once activated, caspase-8 activates other caspases the two daughter cells will be the same or different. A \\nand the amplification cascade begins. To test the ability of cell may be just like its parent, or it may embark on a new\\nthe Fas receptor to induce cell death, researchers incubated path. \\ncells with antibodies against the receptor. These antibodies, Cell birth is normally carefully restricted to specific lo-\\nwhich bind and cross-link Fas receptors, were found to stim- cales and times, such as the basal layer of the skin or the root\\nulate cell death, indicating that activation of the Fas recep- meristem. Liver regenerates when there is injury, but liver\\ntor is sufficient to trigger apoptosis.  cancer is prevented by restricting unnecessary growth at\\nother times. Cell lineage is patterned by the asymmetric dis-\\ntribution of key regulators to the daughter cells of a division.\\nKEY CONCEPTS OF SECTION 22.5 Some of these regulators are intrinsic to the parent cell, be-\\ncoming asymmetrically distributed during polarization of the\\nCell Death and Its Regulation\\ncell; other regulators are external signals that differentially\\n■ All cells require trophic factors to prevent apoptosis and reach the daughter cells. Asymmetry of cells becomes asym-\\nthus survive. In the absence of these factors, cells commit metry of tissues and whole organisms. Our left and right\\nsuicide.  hands differ only as a result of cell asymmetry. \\nReview the Concepts 33\\nSome cells persist for the life of the organism, but  embryonic stem (ES)  mesoderm 000\\nothers such as blood and intestinal cells turn over rapidly. cells 000 micro RNAs \\nMany cells live for awhile and are then programmed to die endoderm 000 (miRNAs) 000\\nand be replaced by others arising from a stem-cell popula-\\nganglion mother  muscle regulatory \\ntion. Programmed cell death is also the basis for the metic-\\ncell (GMC) 000 factors (MRFs) 000\\nulous elimination of potentially harmful cells, such as\\ngerm line 000 neurotrophins 000\\nautoreactive immune cells, which attack the body’s own\\nheterochronic mutants 000 pluripotent 000\\ncells, or neurons that have failed to properly connect. Cell-\\ndeath programs have also evolved as a defense against in- MATlocus 000 precursor/progenitor \\nfection, and virus-infected cells are selectively murdered in mating factor 000 cells 000\\nresponse to death signals. Viruses, in turn, devote much of mating-type  somatic cells 000\\ntheir effort to evading host defenses. For example, p53, a switching 000 stem cells 000\\ntranscription factor that senses cell stresses and damage\\nmeristems 000 trophic factors 000\\nand activates transcription of pro-apoptotic members of the\\nbcl-2gene family, is inhibited by the adenovirus E1B pro-\\ntein. It has been estimated that about a third of the aden-\\nREVIEW THE CONCEPTS\\novirus genome is directed at evading host defenses. Cell\\ndeath is relevant to toxic chemicals as well as viral infec-\\n1. What two properties define a stem cell? Distinguish be-\\ntions; malformations due to poisons often originate from\\ntween a totipotential stem cell, a pluripotent stem cell, and\\nexcess apoptosis.\\na progenitor cell.\\nFailures of programmed cell death can lead to uncon-\\ntrolled cancerous growth (Chapter 23). The proteins that re- 2. Where are stem cells located in plants? Where are stem\\nstrain the death of cancer cells therefore become possible cells located in adult animals? How does the concept of stem\\ntargets for drugs. A tumor may contain a mixture of cells, cell differ between animal and plant systems?\\nsome capable of seeding new tumors or continued uncon-\\n3. In 1997, Dolly the sheep was cloned by a technique\\ntrolled growth, and some capable only of growing in place or\\ncalled somatic cell nuclear transfer. A nucleus from an\\nfor a limited time. In this sense the tumor has its own stem\\nadult mammary cell was transferred into an egg from\\ncells, and they must be found and studied, so they become\\nwhich the nucleus had been removed. The egg was allowed\\nvulnerable to our medicine. One option is to manipulate the\\nto divide several times in culture, then the embryo was\\ncell-death pathway to our own advantage, to send the signals\\ntransferred to a surrogate mother who gave birth to \\nthat will make cancer cells destroy themselves.\\nDolly. Dolly died in 2003 after mating and giving birth\\nMuch attention is now being given to the regulation of\\nherself to viable offspring. What does the creation of Dolly\\nstem cells in an effort to understand how dividing popula-\\ntell us about the potential of nuclear material derived from\\ntions of cells are created and maintained. This has clear im-\\na fully differentiated adult cell? Does the creation of Dolly\\nplications for repair of tissue, for example, to help damaged\\ntell us anything about the potential of an intact, fully dif-\\neyes, torn cartilage, degenerating brain tissue, or failing or-\\nferentiated adult cell? Name three types of information\\ngans. One interesting possibility is that some populations of\\nthat function to preserve cell type. Which of these types\\nstem cells with the potential to generate or regenerate tissue\\nof information was shown to be reversible by the Dolly \\nare normally eliminated by cell death during later develop-\\nexperiment?\\nment. If so, finding ways to selectively block the death of the\\ncells could make regeneration more likely. Could the elimi-\\n4. The roundworm C. eleganshas proven to be a valuable\\nnation of such cells during mammalian development be the\\nmodel organism for studies of cell birth, cell lineage, and cell\\ndifference between an amphibian capable of limb regenera-\\ndeath. What properties of C. elegansrender it so well suited\\ntion and a mammal that is not?\\nfor these studies? Why is so much information from C. elegans\\nexperiments of use to investigators interested in mammalian\\ndevelopment?\\nKEY TERMS\\n5. In the budding yeast S. cerevisiae,what is the role of the\\nMCM1 protein in the following:\\napical complex 000 caspases 000\\napoptosis 000 cell lineage 000 a. transcription of a-specific genes in acells\\nasymmetric cell  death signals 000 b. blocking transcription of \"-specific genes in acells\\ndivision 000 determination 000\\nc. transcription of \"-specific genes in \"cells\\n!-catenin 000 differentiation 000\\nBcl-2 family 000 ectoderm 000 d. blocking transcription of a-specific genes in \"cells\\n34 CHAPTER 22 • Cell Birth, Lineage, And Death\\n6. In S. cerevisiae, what ensures that a and \" cells mate One common characteristic of cancer cells is a loss of func-\\nwith one another rather than with cells of the same mating tion in the apoptotic pathway. Which of the mutations listed\\ntype (i.e., awith aor \"with \")? might you expect to find in some cancer cells?\\n7. Exposure of C3H 10T1/2 cells to 5-azacytidine, a nu-\\ncleotide analog, is a model system for muscle differentiation.\\nANALYZE THE DATA\\nHow was 5-azacytidine treatment used to isolate the genes\\ninvolved in muscle differentiation? \\nTo better understand the potential of adult stem cells to dif-\\n8. Through the experiments on C3H 10T1/2 cells treated ferentiate into various cell types, the following studies were\\nwith 5-azacytidine, MyoD was identified as a key transcrip- performed.\\ntion factor in regulating the differentiation of muscle. To Bone marrow was harvested from adult male mice and\\nwhat general class of DNA-binding proteins does MyoD be- then transplanted into isogeneic female recipient mice that\\nlong? How do the interactions of MyoD with the following had been treated with a level of irradiation sufficient to\\nproteins affect its function? (a) E2A, (b) MEFs, (c) Id destroy their own hematopoetic stem cells (see the figure).\\nAlthough the dose of irradiation given was lethal to mice\\n9. The mechanisms that regulate muscle differentiation in\\nthat did not receive a transplant, the majority of the re-\\nmammals and neural differentiation in Drosophila (and\\ncipient mice receiving the transplant survived. After 4\\nprobably mammals as well) bear remarkable similarities.\\nweeks, peripheral blood from the recipient mice was ana-\\nWhat proteins function analogous to MyoD, myogenin, Id,\\nlyzed. The composition of the blood was normal with re-\\nand E2A in neural cell differentiation in Drosophila? Based\\nspect to all blood cell types. Every blood cell examined\\non these analogies, predict the effect of microinjection of\\nwas determined to be positive for the presence of the Y\\nMyoD mRNA on the development of Xenopusembryos.\\nchromosome.\\n10. Predict the effect of the following mutations on the abil-\\nity of mother and daughter cells of S. cerevisaeto undergo male mouse\\nmating type switching following cell division: \\na. loss-of-function mutation in the HO endonuclease Harvest marrow\\nb. gain-of-function  mutation  that  renders  HO  endo-\\nnuclease gene constitutively expressed independent of Isogeneic female mouse Bone\\nSWI/SNF marrow\\ntransplant\\nc. gain-of-function mutation in SWI/SNF that renders it in-\\nLethal irradiation\\nsensitive to Ash1\\nFPO\\n11. Asymmetric cell division often relies on cytoskeletal el-\\nements to generate or maintain asymmetric distribution of\\ncellular factors. In S. cerevisiae,what factor is localized to\\n\"labeled\" mouse\\nthe bud by myosin motors? In Drosophilaneuroblasts, what\\nfactors are localized apically by microtubules?\\n12. How do studies of brain development in knockout mice\\nsupport the statement that apoptosis is a default pathway in\\nneuronal cells?\\na. What was the purpose of using male mice as the bone\\n13. What morphologic features distinguish programmed cell marrow donors and female mice as the recipients? Would the\\ndeath and necrotic cell death? TNF and Fas ligand bind cell converse experiment (female donors, male recipients) have\\nsurface receptors to trigger cell death. Although the death worked? \\nsignal is generated external to the cell, why do we consider\\nb. What was the purpose of irradiating the recipient mice\\nthe death induced by these molecules to be apoptotic rather\\nprior to transplantation? What outcome might you expect if\\nthan necrotic?\\nbone marrow cells were transplanted to nonirradiated mice? \\n14. Predict the effects of the following mutations on the\\nc. What method could have been used to purify hematopo-\\nability of a cell to undergo apoptosis:\\netic stem cells from bone marrow before transplanting these\\na. mutation in Bad such that it cannot be phosphorylated by cells into the recipient mouse? How might a purification step\\nAkt have affected the outcome of the experiment?\\nb. overexpression of Bcl-2\\nAfter 8 weeks, some of the recipient mice were sacrificed and\\nc. mutation in Bax such that it cannot form homodimers histochemical analysis was performed on tissue sections of'"
            ]
          },
          "metadata": {},
          "execution_count": 69
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 35
        },
        "id": "GacTTONFq84C",
        "outputId": "49b3497e-1970-44ce-9b26-b572d0680bd0"
      },
      "source": [
        "hbb[0]"
      ],
      "execution_count": 61,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'\"CHAPTER 22 '"
            ]
          },
          "metadata": {},
          "execution_count": 61
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 35
        },
        "id": "qW5WUzMKpPPM",
        "outputId": "284e90e3-359a-4d48-eb26-7bdcb040d3c7"
      },
      "source": [
        "hbb[1]"
      ],
      "execution_count": 57,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'CHAPTER 21 '"
            ]
          },
          "metadata": {},
          "execution_count": 57
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "5PPLgkTznbl6"
      },
      "source": [
        "\n",
        "with open('ggggggg.txt', 'w') as f:\n",
        "    f.write(all_text)\n",
        "    "
      ],
      "execution_count": 42,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 171
        },
        "id": "7_bWIQFKpIUj",
        "outputId": "a7775b9e-f32f-4265-a5fe-385fab6e3534"
      },
      "source": [
        "all_text"
      ],
      "execution_count": 55,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'\\n1\\nLIFE BEGINS \\nWITH CELLS\\nA single ~200 micrometer (!m) cell,the human egg,with\\nsperm,which are also single cells.From the union of an egg\\nand sperm will arise the 10 trillion cells of a human body.\\n[Photo Researchers, Inc.]\\nLike ourselves, the individual cells that form our bodies 1.1 The Diversity and Commonality \\ncan grow, reproduce, process information, respond to\\nof Cells\\nstimuli, and carry out an amazing array of chemical re-\\nactions. These abilities define life. We and other multicellular\\nCells come in an amazing variety of sizes and shapes (Figure\\norganisms contain billions or trillions of cells organized into\\n1-1). Some move rapidly and have fast-changing structures, as\\ncomplex structures, but many organisms consist of a single\\nwe can see in movies of amoebae and rotifers. Others are\\ncell. Even simple unicellular organisms exhibit all the hall-\\nlargely stationary and structurally stable. Oxygen kills some\\nmark properties of life, indicating that the cell is the funda-\\ncells but is an absolute requirement for others. Most cells in\\nmental unit of life. As the twenty-first century opens, we face\\nmulticellular organisms are intimately involved with other\\nan explosion of new data about the components of cells,\\ncells. Although some unicellular organisms live in isolation,\\nwhat structures they contain, how they touch and influence\\nothers form colonies or live in close association with other\\neach other. Still, an immense amount remains to be learned,\\ntypes of organisms, such as the bacteria that help plants to ex-\\nparticularly about how information flows through cells and\\ntract nitrogen from the air or the bacteria that live in our in-\\nhow they decide on the most appropriate ways to respond. \\ntestines and help us digest food. Despite these and numerous\\nMolecular cell biology is a rich, integrative science that\\nbrings together biochemistry, biophysics, molecular biology,\\nmicroscopy, genetics, physiology, computer science, and de-\\nO UTLINE\\nvelopmental biology. Each of these fields has its own em-\\nphasis  and  style  of  experimentation.  In  the  following 1.1  The Diversity and Commonality of Cells\\nchapters, we will describe insights and experimental ap-\\n1.2  The Molecules of a Cell\\nproaches drawn from all of these fields, gradually weaving\\nthe multifaceted story of the birth, life, and death of cells. We 1.3  The Work of Cells\\nstart in this prologue chapter by introducing the diversity of\\n1.4  Investigating Cells and Their Parts\\ncells, their basic constituents and critical functions, and what\\nwe can learn from the various ways of studying cells.  1.5  A Genome Perspective on Evolution\\n1\\n21.6 • Cell-Cycle Control in Mammalian Cells 33\\nsee Table 21-1.) Disruption of cyclin A function inhibits\\nCyclin E\\nDNA synthesis in mammalian cells, suggesting that cyclin\\nCyclin D\\nCDK2 A-CDK2 complexes may function like S. cerevisiaeS-phase\\nCDK4/6 cyclin-CDK complexes to trigger initiation of DNA synthe-\\nFPOP P P\\nsis. There is also evidence that cyclin E-CDK2 may con-\\nRb E2F Rb + E2F tribute to activation of pre-replication complexes. \\nThree related CDK inhibitory proteins, or CIPs(p27KIP1,\\nMid G1 Late G1 p57KIP2, and p21CIP), appear to share the function of the \\n▲ FIGURE 21-30 Regulation of Rb and E2F activities in S. cerevisiaeS-phase inhibitor Sic1 (see Figure 21-24). Phos-\\nphorylation of p27KIP1by cyclin E-CDK2 targets it for poly-\\nmid-late G . Stimulation of G cells with mitogens induces\\n1 0\\nexpression of CDK4, CDK6, D-type cyclins, and the E2F ubiquitination by the mammalian SCF complex (see Figure\\ntranscription factors, all encoded by delayed-response genes. Rb 21-2, step 5). The SCF subunit that targets p27KIP1is syn-\\nprotein initially inhibits E2F activity. When signaling from thesized as cells approach the G → S transition. The mecha-\\n1\\nmitogens is sustained, the resulting cyclin D-CDK4/6 complexes nism for degrading p21CIP and p57KIP2 is less well under-\\nbegin phosphorylating Rb, releasing some E2F, which stimulates stood. The activity of mammalian cyclin-CDK2 complexes is\\ntranscription of the genes encoding cyclin E, CDK2, and E2F also regulated by phosphorylation and dephosphorylation\\n(autostimulation). The cyclin E-CDK4 complexes further mechanisms similarly to those controlling the S. pombe\\nphosphorylate Rb, resulting in positive feedback loops (blue mitosis-promoting factor, MPF (see Figure 21-14). The\\narrows) that lead to a rapid rise in the expression and activity of\\nCdc25A phosphatase,which removes the inhibitory phos-\\nboth E2F and cyclin E-CDK2 as the cell approaches the G → S\\n1 phate from CDK2, is a mammalian equivalent of S. pombe\\ntransition.\\nCdc25 except that it functions at the G → S transition rather\\n1\\nthan the G → M transition. The mammalian phosphatase\\n2\\nnormally is activated late in G , but is degraded in the re-\\n1\\nRb protein is one of the most significant substrates of sponse of mammalian cells to DNA damage to prevent the\\nmammalian G1cyclin-CDK complexes. Phosphorylation of cells from entering S phase (see Section 21.7).\\nRb protein at multiple sites prevents its association with Once cyclin A-CDK2 is activated by Cdc25A and the \\nE2Fs, thereby permitting E2Fs to activate transcription of S-phase inhibitors have been degraded, DNA replication \\ngenes required for entry into S phase. As shown in Figure  is initiated at pre-replication complexes. The general mech-\\n21-30, phosphorylation of Rb protein is initiated by cyclin anism is thought to parallel that in S. cerevisiae(see Figure\\nD-CDK4 and cyclin D-CDK6 in mid G1. Once cyclin E and 21-26), although small differences are found in vertebrates.\\nCDK2 are induced by phosphorylation of some Rb, the re- As in yeast, phosphorylation of certain initiation factors by\\nsulting cyclin E-CDK2 further phosphorylates Rb in late G1. cyclin A-CDK2 most likely promotes initiation of DNA\\nWhen cyclin E-CDK2 accumulates to a critical threshold replication and prevents reassembly of pre-replication com-\\nlevel, further phosphorylation of Rb by cyclin E-CDK2 con- plexes until the cell passes through mitosis, thereby assuring\\ntinues even when cyclin D-CDK4/6 activity is removed. This that replication from each origin occurs only once during\\nis one of the principle biochemical events responsible for pas- each cell cycle. In metazoans, a second small protein, gemi-\\nsage through the restriction point. At this point, further nin, contributes to the inhibition of re-initiation at origins\\nphosphorylation of Rb by cyclin E-CDK2 occurs even when until cells complete a full cell cycle.\\nmitogens are withdrawn and cyclin D and CDK4/6 levels The principle mammalian CDK in G and mitosis is\\n2\\nfall. Since E2F stimulates its own expression and that of cy- CDK1 (see Figure 21-28). This CDK, which is highly ho-\\nclin E and CDK2, positive cross-regulation of E2F and cyclin mologous with S. pombeCdc2, associates with cyclins A and\\nE-CDK2 produces a rapid rise of both activities in late G1.  B. The mRNAs encoding either of these mammalian cyclins\\nAs they accumulate, S-phase cyclin-CDK and mitotic  can promote meiotic maturation when injected into Xenopus\\ncyclin-CDK complexes maintain Rb protein in the phospho- oocytes arrested in G (see Figure 21-6), demonstrating that\\n2\\nrylated state throughout the S, G2, and early M phases. After they function as mitotic cyclins. Thus mammalian cyclin \\ncells complete anaphase and enter early G1or G0, the fall in A-CDK1 and cyclin B-CDK1 are functionally equivalent to\\ncyclin-CDK levels leads to dephosphorylation of Rb by un- the S. pombeMPF (mitotic cyclin-CDK). The kinase activ-\\nopposed phosphatases. As a consequence, hypophosphory- ity of these mammalian complexes also appears to be regu-\\nlated Rb is available to inhibit E2F activity during early G1of lated by proteins analogous to those that control the activity\\nthe next cycle and in G0-arrested cells. of the S. pombeMPF (see Figure 21-14). The inhibitory\\nphosphate on CDK1 is removed by Cdc25C phosphatase,\\nwhich is analogous to S. pombeCdc25 phosphatase. \\nCyclin A Is Required for DNA Synthesis \\nIn cycling mammalian cells, cyclin B is first synthesized\\nand CDK1 for Entry into Mitosis\\nlate in the S phase and increases in concentration as cells pro-\\nHigh levels of E2Fs activate transcription of the cyclin A ceed through G , peaking during metaphase and dropping\\n2\\ngene as mammalian cells approach the G → S transition. after late anaphase. This parallels the time course of cyclin B\\n1 \\n(Despite its name, cyclin A is a B-type cyclin, not a G cyclin; expression in Xenopuscycling egg extracts (see Figure 21-9).\\n1\\n34 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nIn human cells, cyclin B first accumulates in the cytosol and ■ Mammalian cells use several CDKs and cyclins to regu-\\nthen enters the nucleus just before the nuclear envelope late passage through the cell cycle. Cyclin D-CDK4/6 func-\\nbreaks down early in mitosis. Thus MPF activity is con- tion in mid to late G ; cyclin E-CDK2 in late G and early\\n1 1\\ntrolled not only by phosphorylation and dephosphorylation S; cyclin A-CDK2 in S; and cyclin A/B-CDK1 in G and\\n2\\nbut also by regulation of the nuclear transport of cyclin B. M through anaphase (see Figure 21-28).\\nIn fact, cyclin B shuttles between the nucleus and cytosol,\\n■ Unphosphorylated Rb protein binds to E2Fs, converting\\nand the change in its localization during the cell cycle results\\nthem into transcriptional repressors. Phosphorylation of\\nfrom a change in the relative rates of import and export. As\\nRb by cyclin D-CDK4/6 in mid G liberates E2Fs to acti-\\nin Xenopuseggs and S. cerevisiae,cyclins A and B are poly- 1\\nvate transcription of genes encoding cyclin E, CDK2, and\\nubiquitinated by the anaphase-promoting complex (APC)\\nother proteins required for the S phase. E2Fs also auto-\\nduring late anaphase and then are degraded by proteasomes\\nstimulate transcription of their own genes. \\n(see Figure 21-2, step 9).\\n■ Cyclin E-CDK2 further phosphorylates Rb, further ac-\\ntivating E2Fs. Once a critical level of cyclin E-CDK2 has\\nTwo Types of Cyclin-CDK Inhibitors Contribute \\nbeen expressed, a positive feedback loop with E2F results\\nto Cell-Cycle Control in Mammals \\nin a rapid rise of both activities that drives passage through\\nAs noted above, three related CIPs—p21CIP, p27KIP2, and the restriction print (see Figure 21-30). \\np57KIP2—inhibit cyclin A-CDK2 activity and must be de-\\n■ The activity of cyclin A-CDK2, induced by high E2F ac-\\ngraded before DNA replication can begin. These same CDK\\ntivity, initially is held in check by CIPs, which function like\\ninhibitory proteins also can bind to and inhibit the other\\nan S-phase inhibitor, and by the presence of an inhibitory\\nmammalian cyclin-CDK complexes involved in cell-cycle\\nphosphate on CDK2. Proteasomal degradation of the in-\\ncontrol. As we discuss later, p21CIP plays a role in the re-\\nhibitors and activation of the Cdc25A phosphatase, as cells\\nsponse of mammalian cells to DNA damage. Experiments\\napproach the G → S transition, generate active cyclin \\nwith knockout mice lacking p27KIP2 have shown that this 1\\nA-CDK2.  This  complex  activates  pre-replication  com-\\nCIP is particularly important in controlling generalized cell\\nplexes to initiate DNA synthesis by a mechanism similar\\nproliferation soon after birth. Although p27KIP2 knockouts\\nto that in S. cerevisiae(see Figure 21-26). \\nare larger than normal, most develop normally otherwise.\\nIn contrast, p57KIP2 knockouts exhibit defects in cell differ- ■ Cyclin A/B-CDK1 induce the events of mitosis through\\nentiation and most die shortly after birth due to defective de- early anaphase. Cyclins A and B are polyubiquitinated by\\nvelopment of various organs.  the  anaphase-promoting  complex  (APC)  during  late\\nA second class of cyclin-CDK inhibitors called INK4s anaphase and then are degraded by proteasomes.\\n(inhibitors of kinase 4) includes several small, closely related\\n■ The activity of mammalian mitotic cyclin-CDK com-\\nproteins that interact only with CDK4 and CDK6 and thus\\nplexes also are regulated by phosphorylation and dephos-\\nfunction specifically in controlling the mid-G phase. Binding\\n1 phorylation similar to the mechanism in S. pombe, with\\nof INK4s to CDK4/6 blocks their interaction with cyclin D\\nthe Cdc25C phosphatase removing inhibitory phosphates\\nand hence their protein kinase activity. The resulting de-\\n(see Figure 21-14). \\ncreased phosphorylation of Rb protein prevents transcrip-\\ntional activation by E2Fs and entry into the S phase. One ■ The activities of mammalian cyclin-CDK complexes also\\nINK4 called p16 is a tumor suppressor, like Rb protein dis- are regulated by CDK inhibitors (CIPs), which bind to and\\ncussed earlier. The presence of two mutant p16 alleles in a inhibit each of the mammalian cyclin-CDK complexes, and\\nlarge fraction of human cancers is evidence for the important INK4 proteins, which block passage through G1by specif-\\nrole of p16 in controlling the cell cycle (Chapter 23).  ically inhibiting CDK4 and CDK6. \\nKEY CONCEPTS OF SECTION 21.6 21.7 Checkpoints in Cell-Cycle\\nCell-Cycle Control in Mammalian Cells Regulation\\n■ Various  polypeptide  growth  factors  called  mitogens Catastrophic genetic damage can occur if cells progress to\\nstimulate cultured mammalian cells to proliferate by in-\\nthe next phase of the cell cycle before the previous phase\\nducing expression of early-response genes. Many of these\\nis properly completed. For example, when S-phase cells\\nencode transcription factors that stimulate expression of\\nare induced to enter mitosis by fusion to a cell in mitosis,\\ndelayed-response genes encoding the G CDKs, G cyclins,\\n1 1 the MPF present in the mitotic cell forces the chromo-\\nand E2F transcription factors.\\nsomes of the S-phase cell to condense. However, since the\\n■ Once cells pass the restriction point, they can enter the replicating chromosomes are fragmented by the conden-\\nS phase and complete S, G , and mitosis in the absence of sation process, such premature entry into mitosis is dis-\\n2\\ngrowth factors. astrous for a cell. \\n21.7 • Checkpoints in Cell-Cycle Regulation 35\\nAnother example concerns attachment of kineto- !FIGURE 21-31\\nchores to microtubules of the mitotic spindle during Abnormal Nondisjunction.This\\nspindle abnormality occurs when\\nmetaphase. If anaphase is initiated before both kine-\\nchromosomes segregate \\ntochores of a replicated chromosome become attached to micro-\\nin anaphase before the\\ntubules from opposite spindle poles, daughter cells are produced\\nAnaphase kinetochore of each sister\\nthat have a missing or extra chromosome (Figure 21-31). When\\nchromatid has attached \\nthis process, called nondisjunction,occurs during the meiotic di- FPO\\nto microtubules (red lines)\\nvision that generates a human egg, Down syndrome can occur\\nfrom the opposite spindle\\nfrom trisomy of chromosome 21, resulting in developmental ab-\\npoles. As a result, one\\nnormalities and mental retardation. ❚ daughter cell contains two\\ncopies of one chromosome,\\nTo minimize the occurrence of such mistakes in cell-cycle Cytokinesis\\nwhile the other daughter\\nevents, a cell’s progress through the cycle is monitored at sev-\\ncell lacks that chromosome.\\neral checkpoints (Figure 21-32). Control mechanisms that\\n[Adapted from A. Murray and \\noperate at these checkpoints ensure that chromosomes are\\nT. Hunt, 1993, The Cell Cycle:\\nintact and that each stage of the cell cycle is completed before\\nAn Introduction,W. H. Freeman\\nthe following stage is initiated. Our understanding of these\\nand Company.]\\ncontrol mechanisms at the molecular level has advanced con- Missing Extra\\nsiderably in recent years. chromosome chromosome\\n3\\n2 Chromosome-segregation\\nSpindle-assembly checkpoint\\ncheckpoint\\nAPC-Cdh1\\nMad2\\npolyubiquitination Cdc14 Sic1\\nof B-type cyclins\\nAPC-Cdc20\\nATM/R ATM/R\\npolyubiquitination\\nTelophase\\nof securin \\n4a\\n4d\\np53 p53 DNA-damage\\nDNA-damage\\nAnaphase checkpoint\\ncheckpoint\\np21CIP M G1 Cyclin D−CDK4/6 p21CIP\\nFPO\\nCyclin A/B−CDK1 M-phase\\nentry\\nG\\nCdc25C 2\\nS-phase Cyclin E/A−CDK2\\nS\\nentry\\n1\\nChk1\\nUnreplicated-DNA p21CIP Cdc25A\\ncheckpoint Cyclin A−CDK2\\nATR 4b\\np53\\nDNA-damage\\np21CIP Cdc25A\\ncheckpoint\\n4c ATM/R Chk1/2\\np53\\nDNA-damage\\ncheckpoint\\nATM/R Chk1/2\\n▲ FIGURE 21-32 Overview of checkpoint controls in the Cdc14 phosphatase from nucleoli, thereby blocking activation of\\ncell cycle.The unreplicated-DNA checkpoint (1) prevents the APC specificity factor (Cdh1) required for polyubiquitination of\\nactivation of cyclin A-CDK1 and cyclin B-CDK1 (i.e., mitosis- B-type cyclins as well as induction of Sic1. As a result, the\\npromoting factor, MPF) by activation of an ATR-Chk1 protein decrease in MPF activity required for the events of telophase\\nkinase cascade that phosphorylates and inactivates Cdc25C, does not occur. In the initial phase of the DNA-damage\\nthereby inhibiting entry into mitosis. In the spindle-assembly checkpoint (4), the ATM or ATR protein kinase (ATM/R) is\\ncheckpoint (2), Mad2 and other proteins inhibit activation of the activated. The active kinases then trigger two pathways: the Chk-\\nAPC specificity factor (Cdc20) required for polyubiquitination of Cdc25A pathway (4a and 4c), blocking entry into or through the\\nsecurin, thereby preventing entry into anaphase. The S phase, and the p53-p21CIP pathway, leading to arrest in G , S,\\n1\\nchromosome-segregation checkpoint (3) prevents release of the and G (4a–4b). See text for further discussion.\\n2\\n36 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nThe Presence of Unreplicated DNA Prevents lived activated form that can interact with and inhibit\\nEntry into Mitosis Cdc20. Microtubule attachment prevents this activation of\\nMad2. Consequently, once all kinetochore complexes bind\\nCells that fail to replicate all their chromosomes do not enter\\na microtubule, generation of the activated form of Mad2\\nmitosis. Operation of the unreplicated-DNA checkpointcon-\\nceases, the inhibition of Cdc20 is relieved, and Cdc20 is free\\ntrol involves the recognition of unreplicated DNA and inhi-\\nto direct the APC to polyubiquitinate securin, thereby initi-\\nbition of MPF activation (see Figure 21-32,  1). Recent\\nating the onset of anaphase.\\ngenetic studies in S. pombeand biochemical studies with\\nXenopusegg extracts suggest that the ATR and Chk1 protein\\nkinases, which also function in the DNA-damage check- Proper Segregation of Daughter Chromosomes Is\\npoint, inhibit entry into mitosis by cells that have not com- Monitored by the Mitotic Exit Network \\npleted DNA synthesis. \\nOnce chromosomes have segregated properly, telophase\\nThe association of ATR with replication forks is thought\\ncommences. The various events of telophase and subsequent\\nto activate its protein kinase activity, leading to the phos-\\ncytokinesis, collectively referred to as the exit from mitosis,\\nphorylation and activation of the Chk1 kinase. Active Chk1\\nrequire inactivation of MPF. As discussed earlier, dephos-\\nthen phosphorylates and inactivates the Cdc25 phosphatase\\nphorylation of the APC specificity factor Cdh1 by the Cdc14\\n(Cdc25C in vertebrates), which normally removes the in-\\nphosphatase leads to degradation of mitotic cyclins and loss\\nhibitory phosphate from CDKs that function during mito-\\nof MPF activity late in anaphase (see Figure 21-10). During\\nsis. As a result, the cyclin A/B-CDK1 complexes remain\\ninterphase and early mitosis, Cdc14 is sequestered in the nu-\\ninhibited and cannot phosphorylate targets required to ini-\\ncleolus and inactivated. The chromosome-segregation check-\\ntiate mitosis. ATR continues to initiate this protein kinase\\npoint, which  monitors  the  location  of  the  segregating\\ncascade until all replication forks complete DNA replication\\ndaughter chromosomes at the end of anaphase, determines\\nand disassemble.\\nwhether active Cdc14 is available to promote exit from mi-\\ntosis (see Figure 21-32, 3). \\nOperation of this checkpoint in S. cerevisiaedepends on a\\nImproper Assembly of the Mitotic Spindle\\nset of proteins referred to as the mitotic exit network.A key\\nPrevents the Initiation of Anaphase \\ncomponent is a small (monomeric) GTPase, called Tem1. This\\nThe  spindle-assembly  checkpoint prevents  entry  into member of the GTPase superfamilyof switch proteins controls\\nanaphase when just a single kinetochore of one chromatid the activity of a protein kinase cascade similarly to the way\\nfails to associate properly with spindle microtubules. Clues Ras controls MAP kinase pathways (Chapter 14). During\\nabout how this checkpoint operates has come from isolation anaphase, Tem1 becomes associated with the spindle pole\\nof yeast mutants in the presence of benomyl, a microtubule- body (SPB) closest to the daughter cell bud. (The SPB, from\\ndepolymerizing  drug.  Low  concentrations  of  benomyl  which spindle microtubules originate, is equivalent to the cen-\\nincrease the time required for yeast cells to assemble the  trosome in higher eukaryotes.) At the SPB, Tem1 is maintained\\nmitotic spindle and attach kinetochores to microtubules. in the inactive GDP-bound state by a specific GAP (GTPase-\\nWild-type cells exposed to benomyl do not begin anaphase accelerating  protein).  The  GEF  (guanosine  nucleotide–\\nuntil these processes are completed and then proceed on exchange factor) that activates Tem1 is localized to the cortex\\nthrough mitosis, producing normal daughter cells. In con- of the bud and is absent from the mother cell. When spindle\\ntrast, mutants defective in the spindle-assembly checkpoint microtubule elongation at the end of anaphase has correctly\\nproceed through anaphase before assembly of the spindle positioned segregating daughter chromosomes into the bud,\\nand attachment of kinetochores is complete; consequently, Tem1 comes into contact with its GEF and is converted into\\nthey mis-segregate their chromosomes, producing abnormal the active GTP-bound state. The terminal kinase in the cas-\\ndaughter cells that die.  cade triggered by Tem1 · GTP then phosphorylates the nu-\\nAnalysis of these mutants identified a protein called Mad2 cleolar anchor that binds and inhibits Cdc14, releasing it into\\nand other proteins that regulate Cdc20, the specificity factor the cytoplasm and nucleoplasm in both the bud and mother\\nrequired to target the APC to securin (see Figure 21-32, 2). cell (Figure 21-33, 1). Once active Cdc14 is available, a cell\\nRecall that APC-mediated polyubiquitination of securin and can proceed through telophase and cytokinesis. If daughter\\nits  subsequent  degradation  is  required  for  entry  into chromosomes fail to segregate into the bud, Tem1 remains in\\nanaphase (see Figure 21-19). Mad2 has been shown to asso- its inactive state, Cdc14 is not released from the nucleolus,\\nciate with kinetochores that are unattached to microtubules. and mitotic exit is blocked (Figure 21-33, 2).\\nExperiments with Mad2 fused to green fluorescent protein In the fission yeast S. pombe,formation of the septum\\n(GFP) indicate that kinetochore-bound Mad2 rapidly ex- that divides daughter cells is regulated by proteins homolo-\\nchanges with a soluble form of Mad2. Current models pro- gous to those that constitute the mitotic exit network in \\npose that when Mad2 associates with a kinetochore complex S. cerevisiae. Genes encoding similar proteins also have been\\nthat is not bound by a microtubule, it is converted to a short- found in higher organisms where the homologs probably\\n21.7 • Checkpoints in Cell-Cycle Regulation 37\\nTem1-GEF Arrest in G and S prevents copying of damaged bases, which\\n1\\nBud would fix mutations in the genome. Replication of damaged\\nTem1 . GDP\\nDNA also promotes chromosomal rearrangements that can\\n(inactive)\\nTem1-GAP GDP Linker protein contribute to the onset of cancer. Arrest in G2allows DNA\\ndouble-stranded breaks to be repaired before mitosis. If a\\nNucleus\\nNucleolus double-stranded break is not repaired, the broken distal por-\\nSpindle\\nCdc14 microtubule tion of the damaged chromosome is not properly segregated\\n(inactive) because it is not physically linked to a centromere, which is\\nSPB\\nMother pulled toward a spindle pole during anaphase. \\nAs we discuss in detail in Chapter 23, inactivation of\\nProper chromosome Improper chromosome\\nsegregation segregation tumor-suppressor genes contributes to the development of\\n1 2\\ncancer. The proteins encoded by several tumor-suppressor\\ngenes, including ATMand Chk2,normally function in the\\nGTP Tem1 . GTP DNA-damage checkpoint. Patients with mutations in both\\n(active) copies of ATMor Chk2develop cancers far more frequently\\nFPO\\nthan normal. Both of these genes encode protein kinases. \\nDNA damage due to UV light somehow activates the\\nATM kinase, which phosphorylates Chk2, thereby activating\\nG\\nCdc14 PD its kinase activity. Activated Chk2 then phosphorylates the\\n(active) Cdc25A phosphatase, marking it for polyubiquitination by\\nan as-yet undetermined ubiquitin ligase and subsequent pro-\\nteasomal degradation. Recall that removal of the inhibitory\\nphosphate from mammalian CDK2 by Cdc25A is required\\nExit from mitosis Late mitotic arrest for onset of and passage through the S phase mediated by cy-\\nclin E-CDK2 and cyclin A-CDK2. Degradation of Cdc25A\\n▲ FIGURE 21-33 Operation of the chromosome-segregation\\nresulting from activation of the ATM-Chk2 pathway in G\\ncheckpoint.In S. cerevisiae,Cdc14 phosphatase activity is 1\\nor S-phase cells thus leads to G or S arrest (see Figure 21-32, \\nrequired for the exit from mitosis. (Top) During interphase and 1\\n4b and 4c). A similar pathway consisting of the protein \\nearly mitosis, Cdc14 is sequestered and inactivated in the\\nnucleolus.Inactive Tem1 · GDP (purple) associates with the kinases ATR and Chk1 leads to phosphorylation and poly-\\nspindle pole body (SPB) nearest to the bud early in anaphase ubiquitination of Cdc25A in response to !-radiation. As dis-\\nwith the aid of a linker protein (green) and is maintained in the cussed earlier for the unreplicated-DNA checkpoint, Chk1\\ninactive state by a specific GAP (GTPase-accelerating protein). If also inactivates Cdc25C, preventing the activation of CDK1\\nchromosome segregation occurs properly (1), extension of the and entry into mitosis.\\nspindle microtubules inserts the daughter SPB into the bud, Another tumor-suppressor protein, p53, contributes to\\ncausing Tem1 to come in contact with a specific GEF (guanine arrest of cells with damaged DNA. Cells with functional p53\\nnucleotide–exchange factor) localized to the cortex of the bud arrest in G and G when exposed to !-irradiation, whereas\\n1 2\\n(brown). This GEF converts inactive Tim1 · GDP to active \\ncells lacking functional p53 do not arrest in G . Although the\\nTem1 · GTP, which triggers a protein kinase cascade leading to 1\\np53 protein is a transcription factor, under normal condi-\\nrelease of active Cdc14 and exit from mitosis. If the spindle\\ntions it is extremely unstable and generally does not accu-\\napparatus fails to place the daughter SPB in the bud (2), Tem1\\nmulate to high enough levels to stimulate transcription. The\\nremains in the inactive GDP-bound state and Cdc14 remains\\ninstability of p53 results from its polyubiquitination by a\\nassociated with nucleoli. Arrest in late mitosis results. [Adapted\\nubiquitin ligase called Mdm2 and subsequent proteasomal\\nfrom G. Pereira and E. Schiebel, 2001, Curr. Opin. Cell Biol.13:762.]\\ndegradation. The rapid degradation of p53 is inhibited by\\nATM and probably ATR, which phosphorylate p53 at a site\\nfunction in an analogous checkpoint that leads to arrest in that interferes with binding by Mdm2. This and other mod-\\nlate mitosis when daughter chromosomes do not segregate ifications of p53 in response to DNA damage greatly increase\\nproperly.  its ability to activate transcription of specific genes that help\\nthe cell cope with DNA damage. One of these genes encodes\\np21CIP, a generalized CIP that binds and inhibits all mam-\\nCell-Cycle Arrest of Cells with Damaged DNA\\nmalian cyclin-CDK complexes. As a result, cells are arrested\\nDepends on Tumor Suppressors\\nin G and G until the DNA damage is repaired and p53 and\\n1 2\\nThe DNA-damage checkpointblocks progression subsequently p21CIPlevels fall (see Figure 21-32, 4a–4d). \\nthrough the cell cycle until the damage is repaired. Under some circumstances, such as when DNA damage\\nDamage to DNA can result from chemical agents is extensive, p53 also activates expression of genes that lead\\nand from irradiation with ultraviolet (UV) light or !-rays. to apoptosis, the process of programmed cell death that \\n38 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nnormally occurs in specific cells during the development of \"FIGURE 21-34 Meiosis. Premeiotic cells have two copies\\nmulticellular animals. In vertebrates, the p53 response of each chromosome (2n), one derived from the paternal parent\\nevolved to induce apoptosis in the face of extensive DNA and one from the maternal parent. For simplicity, the paternal\\ndamage, presumably to prevent the accumulation of multiple and maternal homologs of only one chromosome are\\nmutations that might convert a normal cell into a cancer cell. diagrammed. Step  1: All chromosomes are replicated during the\\nS phase before the first meiotic division, giving a 4n\\nThe dual role of p53 in both cell-cycle arrest and the induc-\\nchromosomal complement. Cohesin complexes (not shown) link\\ntion of apoptosis may account for the observation that nearly\\nthe sister chromatids composing each replicated chromosome\\nall cancer cells have mutations in both alleles of the p53gene\\nalong their full lengths. Step  2: After chromosomes condense\\nor in the pathways that stabilize p53 in response to DNA\\nduring the first meiotic prophase, replicated homologs become\\ndamage (Chapter 23). The consequences of mutations in\\npaired as the result of at least one crossover event between a\\np53, ATM,and Chk2provide dramatic examples of the sig-\\npaternal and a maternal chromatid. This pairing of replicated\\nnificance of cell-cycle checkpoints to the health of a multi-\\nhomologous chromosomes is called synapsis. At metaphase,\\ncellular organism. ❚ shown here, both chromatids of one chromosome associate with\\nmicrotubules emanating from one spindle pole, but each member\\nof a homologous chromosome pair associates with microtubules\\nKEY CONCEPTS OF SECTION 21.7 emanating from opposite poles. Step  3: During anaphase of\\nmeiosis I, the homologous chromosomes, each consisting of two\\nCheckpoints in Cell-Cycle Regulation\\nchromatids, are pulled to opposite spindle poles. Step  4:\\n■ Checkpoint controls function to ensure that chromo- Cytokinesis yields the two daughter cells (now 2n), which enter\\nsomes are intact and that critical stages of the cell cycle meiosis II without undergoing DNA replication. At metaphase of\\nare completed before the following stage is initiated.  meiosis II, shown here, the chromatids composing each\\nreplicated chromosome associate with spindle microtubules from\\n■ The unreplicated-DNA checkpoint operates during S and opposite spindle poles, as they do in mitosis. Steps  5 and  6:\\nG to prevent the activation of MPF before DNA synthe-\\n2 Segregation of chromatids to opposite spindle poles during the\\nsis is complete by inhibiting the activation of CDK1 by second meiotic anaphase followed by cytokinesis generates\\nCdc25C (see Figure 21-32, 1). haploid germ cells (1n) containing one copy of each chromosome\\n■ The spindle-assembly checkpoint, which prevents pre- (referred to as chromatids earlier).\\nmature initiation of anaphase, utilizes Mad2 and other pro-\\nteins to regulate the APC specificity factor Cdc20 that tar-\\ngets securin for polyubiquitination (see Figures 21-32, 2, 21.8 Meiosis: A Special Type \\nand 21-19).\\nof Cell Division \\n■ The chromosome-segregation checkpoint prevents telophase\\nand cytokinesis until daughter chromosomes have been prop- In nearly all eukaryotes, meiosisgenerates haploid germ cells\\nerly segregated, so that the daughter cell has a full set of chro- (eggs and sperm), which can then fuse to generate a diploid\\nmosomes (see Figure 21-32, 3). zygote (Figure 21-34). During meiosis, a single round of\\n■ In the chromosome-segregation checkpoint, the small DNA replication is followed by two cycles of cell division,\\nGTPase Tem1 controls the availability of Cdc14 phos- termed meiosis I and meiosis II. Crossing over of chro-\\nphatase, which in turn activates the APC specificity factor matids, visible in the first meiotic metaphase, produces \\nCdh1 that targets B-type cyclins for degradation, causing recombination between parental chromosomes. This in-\\ninactivation of MPF (see Figure 21-10). creases the genetic diversity among the individuals of a\\nspecies. During meiosis I, both chromatids of each homolo-\\n■ The DNA-damage checkpoint arrests the cell cycle in re-\\ngous chromosome segregate together to opposite spindle\\nsponse to DNA damage until the damage is repaired. Three\\npoles, so that each of the resulting daughter cells contains\\ntumor-suppressor proteins (ATM/ATR, Chk1, and p53) are\\none homologous chromosome consisting of two chromatids.\\ncritical to this checkpoint.\\nDuring meiosis II, which resembles mitosis, the chromatids\\n■ Activation of the ATM or ATR protein kinases in re- of one chromosome segregate to opposite spindle poles, gen-\\nsponse to DNA damage due to UV light or !-irradiation erating haploid germ cells. Meiosis generates four haploid\\nleads to arrest in G1 and the S phase via a pathway that germ cells from one diploid premeiotic cell.\\nleads to loss of Cdc25A phosphatase activity. A second\\npathway from activated ATM/R stabilizes p53, which stim-\\nulates expression of p21CIP. Subsequent inhibition of mul- Repression of G1 Cyclins and Meiosis-Specific\\ntiple CDK-cyclin complexes by p21CIP causes prolonged Ime2 Prevents DNA Replication in Meiosis II \\narrest in G and G (see Figure 21-32, 4a–4d). \\n1 2 In S. cerevisiaeand S. pombe, depletion of nitrogen and car-\\n■ In response to extensive DNA damage, p53 also acti- bon sources induces diploid cells to undergo meiosis, yielding\\nvates genes that induce apoptosis. haploid spores (see Figure 1-5). This process is analogous to\\n21.8 • Meiosis: A Special Type of Cell Division 39\\nPaternal the formation of germ cells in higher eukaryotes. Multiple\\nhomolog yeast mutants that cannot form spores have been isolated,\\nPremeiotic\\ncell (2n) Maternal and the wild-type proteins encoded by these genes have been\\nhomolog analyzed. These studies have identified specialized cell-cycle\\nproteins that are required for meiosis. \\nUnder starvation conditions, expression of G cyclins\\n1 1\\n(Cln1/2/3) in S. cerevisiaeis repressed, blocking the normal\\nprogression of G in cells as they complete mitosis. Instead, a\\n1\\nset of early meiotic proteins are induced. Among these is\\nReplicated\\nIme2,a protein kinase that performs the essential G cyclin-\\nchromosomes 1\\n(4n) CDK function of phosphorylating the S-phase inhibitor Sic1,\\nleading to release of active S-phase cyclin-CDK complexes\\nand the onset of DNA replication in meiosis I (see Figure \\n2 21-34, step 1). The absence of Ime2 expression during meio-\\nsis II and the continued repression of Cln expression account\\nfor the failure to replicate DNA during the second meiotic di-\\nCrossing over vision (steps 5 and 6).\\nand synapsis\\n(metaphase I)\\nCrossing Over and Meiosis-Specific Rec8 \\ns I Are Necessary for Specialized Chromosome\\nsi 3\\no Segregation in Meiosis I \\nei\\nM\\nRecall that in mitosis, sister chromatids replicated during \\nthe S phase are initially linked by cohesin complexes at mul-\\ntiple positions along their full length (see Figure 21-19, left). \\nAnaphase I\\nAs chromosomes condense, cohesin complexes become re-\\nstricted to the region of the centromere, and at metaphase, \\nthe sister chromatids composing each (replicated) chromo-\\nsome associate with microtubules emanating from opposite \\nspindle poles. Although motor proteins pull sister chro-\\n4 matids toward opposite spindle poles, their movement \\ninitially is resisted by the cohesin complexes linking them \\nFPO at the centromere. The subsequent separase-catalyzed cleav-\\nDaughter cells\\nage of the Scc1 cohesin subunit permits movement of \\nin metaphase II\\n(2n) the chromatids toward the spindle poles to begin, herald-\\ning the onset of anaphase (Figure 21-35a; see also Figure \\n21-19, right).\\n5 In metaphase of meiosis I, both sister chromatids in one\\n(replicated) chromosome associate with microtubules ema-\\nnating from the samespindle pole, rather than from opposite\\npoles as they do in mitosis. Two physical links between ho-\\ns II mologous chromosomes are thought to resist the pulling\\nosi Anaphase II force of the spindle until anaphase: (a) crossing over between\\nei chromatids, one from each pair of homologous chromo-\\nM\\nsomes, and (b) cohesin cross-links between chromatids distal\\nto the crossover point. \\nEvidence for the role of crossing over in meiosis in \\n6 S. cerevisiaecomes from the observation that when re-\\ncombination is blocked by mutations in proteins essential\\nfor the process, chromosomes segregate randomly during\\nmeiosis I; that is, homologous chromosomes do not nec-\\nessarily segregate to opposite spindle poles. Such segrega-\\ntion to opposite spindle poles normally occurs because\\nboth chromatids of homologous chromosome pairs associ-\\n1n 1n 1n 1n ate with spindle fibers emanating from opposite spindle\\n40 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\n(a) Mitosis chromatid linking through cohesin complexes contribute\\nto synapsis in meiosis I.\\nMetaphase Anaphase\\nDuring meiosis I, the cohesin cross-links between chro-\\nmosome arms are cleaved by separase, allowing the homol-\\nogous chromosomes to separate, but cohesin complexes at\\nthe centromere remain linked (Figure 21-35b, top). The\\nSeparase\\nmaintenance of centromeric cohesion during meiosis I is\\nnecessary for the proper segregation of chromatids during\\nScc1 meiosis II. Recent studies with a S. pombemutant have\\nshown that a specialized cohesin subunit, Rec8,maintains\\ncentromeric cohesion between sister chromatids during\\n(b) Meiosis meiosis I. Expressed only during meiosis, Rec8 is homolo-\\nMetaphase I Anaphase I gous to Scc1, the cohesin subunit that forms the actual\\nbridge between sister chromatids in mitosis (see Figure 21-\\n19). Immunolocalization experiments in S. pombehave re-\\nvealed that during early anaphase of meiosis I, Rec8 is lost\\nfrom chromosome arms but is retained at centromeres.\\nSeparase\\nFPO\\nHowever, during early anaphase of meiosis II, centromeric\\nRec8 is degraded by separase, so the chromatids can segre-\\ngate, as they do in mitosis (Figure 21-35b, bottom). A spe-\\nMetaphase II Anaphase II cific protein expressed during meiosis I, but not during\\nmeiosis II, protects centromeric Rec8 from separase cleav-\\nage in meiosis I. \\nS. cerevisiaeRec8 has been shown to localize and function\\nsimilarly to S. pombeRec8.Homologs of Rec8 also have been\\nSeparase\\nidentified in higher organisms, and RNA interference (RNAi)\\nexperiments in C. elegans(Chapter 9) indicate that the Rec8\\nhomolog in that organism has a similar function. Recent \\n▲ FIGURE 21-35 Cohesin function during mitosis and micromanipulation experiments during grasshopper sper-\\nmeiosis.(a) During mitosis, sister chromatids generated by DNA matogenesis support the hypothesis that kinetochore-bound\\nreplication in the S phase are initially associated by cohesin\\nproteins protect centromeric Rec8 from cleavage during\\ncomplexes along the full length of the chromatids. During\\nmeiosis I but not during meiosis II and also direct kinetochore\\nchromosome condensation, cohesin complexes (gold) become\\nattachment to microtubules emanating from the correct spin-\\nrestricted to the region of the centromere at metaphase, as\\ndle pole (Figure 21-36). Thus crossing over, Rec8, and spe-\\ndepicted here. Once separase cleaves the Scc1 cohesin subunit,\\ncial kinetochore-associated proteins appear to function in\\nsister chromatids can separate, marking the onset of anaphase\\nmeiosis in all eukaryotes. \\n(see Figure 21-19). (b) In metaphase of meiosis I, crossing over\\nRecent DNA microarrayanalyses in S. cerevisiaehave re-\\nbetween maternal and paternal chromatids produces synapsis of\\nhomologous parental chromosomes. The chromatids of each vealed other proteins that are required for meiosis. As dis-\\nreplicated chromosome are cross-linked by cohesin complexes cussed  in  other  chapters,  researchers  can  monitor\\nalong their full length. Rec8, a meiosis-specific homolog of Scc1, transcription of thousands of genes, indeed entire genomes,\\nis cleaved in chromosome arms but not in the centromere, with DNA microarrays (see Figure 9-35). One of the multi-\\nallowing homologous chromosome pairs to segregate to ple genes found to be expressed in S. cerevisiaecells during\\ndaughter cells. Centromeric Rec8 is cleaved during meiosis II, meiosis but not during mitosis is MAM1,which encodes a\\nallowing individual chromatids to segregate to daughter cells. protein called monopolin.Subsequent gene-specific mutage-\\n[Modified from F. Uhlmann, 2001, Curr. Opin. Cell Biol.13:754.] nesis studies revealed that deletion of MAM1causes sister\\nchromatids in metaphase of meiosis I to associate with the\\nfirst meiotic spindle as though they were mitotic chromatids\\npoles (see Figure 21-34, step 3). This in turn requires that or chromatids in meiosis II. That is, kinetochores of the sis-\\nhomologous chromosomes pair during meiosis I, a process ter chromatids composing a single (replicated) chromosome\\ncalled synapsisthat can be visualized microscopically in attached to microtubules emanating from opposite spindle\\neukaryotes with large chromosomes. Consequently, the poles rather than from the same spindle pole. This result in-\\nfinding that mutations that block recombination also dicates that monopolin is required for formation of a spe-\\nblock proper segregation in meiosis I implies that recom- cialized kinetochore in meiosis I responsible for the unique\\nbination is required for synapsis in S. cerevisiae.In higher co-orientation of sister chromatids of synapsed homologous\\neukaryotes, processes in addition to recombination and chromosomes in the first meiotic division.\\n21.8 • Meiosis: A Special Type of Cell Division 41\\nUnmanipulated Manipulated\\nmeiosis I meiosis I\\nsynapsed synapsed\\nchromosomes chromosomes\\nManipulated Unmanipulated\\nmeiosis II meiosis II\\nchromosomes chromosomes\\nMeiosis I spindle Meiosis II spindle\\n70 min\\nFPO\\n▲EXPERIMENTAL FIGURE 21-36 Anaphase movements pair toward one spindle pole and the other homologous pair\\nand cohesion of meiotic chromosomes are determined by toward the other) whether attached to the meiosis I spindle (left)\\nproteins associated with the chromosomes. Grasshopper or meiosis II spindle (right).Similarly, the meiosis II\\nspermatocytes in meiosis I and II were fused so that both types chromosomes (red) separated normally (i.e., one chromatid\\nof spindles with their associated chromosomes were present in a toward one spindle pole and the other chromatid toward the\\nsingle fused cell. Then a micromanipulation needle was used to opposite pole) independent of which spindle they were attached\\nmove some meiosis I chromosomes (blue) and meiosis II to. These results indicate that the association of kinetochores\\nchromosomes (red) from one spindle to another; other with spindle microtubules and the stability of cohesins linking\\nchromosomes were left attached to their normal spindles. After chromosomes are determined by factors that are associated with\\n70 minutes, both spindles with their attached chromosomes had the chromosomes and not by the spindles or soluble\\ncompleted anaphase movements. The synapsed meiosis I components of cells in meiosis I and II. [See L. V. Paliulis and R. B.\\nchromosomes (blue) separated normally (i.e., one homologous Nicklas, 2000, J. Cell Biol.150:1223.]\\nforms the function of G cyclin-CDK complexes in pro-\\n1\\nKEY CONCEPTS OF SECTION 21.8 moting initiation of DNA replication during meiosis I.\\nDNA replication does not occur during meiosis II because\\nMeiosis: A Special Type of Cell Division \\nneither Ime2 nor G cyclins are expressed. \\n1\\n■ Meiosis involves one cycle of chromosome replication fol-\\nlowed by two cycles of cell division to produce haploid germ ■ In S. cerevisiae,recombination (crossing over) between\\nchromatids of homologous parental chromatids and cohesin\\ncells from a diploid premeiotic cell (see Figure 21-34). Dur-\\ncross-links between chromatids distal to the crossover are\\ning meiosis I, replicated homologous chromosomes pair\\nresponsible for synapsis of homologous chromosomes dur-\\nalong their lengths in a process called synapsis.\\ning prophase and metaphase of meiosis I. A specialized \\n■ At least one recombination between chromatids of ho- cohesin subunit, Rec8, replaces the Scc1 cohesin subunit\\nmologous chromosomes almost invariably occurs. \\nduring meiosis. \\n■ Most of the cell-cycle proteins that function in mitoti- ■ During early anaphase of meiosis I, Rec8 in the chro-\\ncally dividing cells also function in cells undergoing meio-\\nmosome arms is cleaved, but a meiosis-specific protein as-\\nsis, but some proteins are unique to meiosis. \\nsociated with the kinetochore protects Rec8 in the region\\n■ In S. cerevisiae, expression of G1 cyclins (Clns) is re- of the centromere from cleavage. As a result, the chro-\\npressed  throughout  meiosis.  Meiosis-specific  Ime2  per- matids of homologous chromosomes remain associated\\n42 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nduring segregation in meiosis I. Cleavage of centromeric trols and checkpoints, new strategies for treating p53-minus\\nRec8 during anaphase of meiosis II allows individual chro- cancers might be possible. For instance, some chemothera-\\nmatids to segregate into germ cells (see Figure 21-35b). peutic agents inhibit microtubule function, interfering with\\nmitosis. Resistant cells selected during the course of treatment\\n■ Monopolin, another meiosis-specific protein, is required\\nmay be defective in the spindle-assembly checkpoint as the re-\\nfor both chromatids of homologous chromosomes to as-\\nsult of mutations in the genes encoding the proteins involved.\\nsociate with microtubules emanating from the same spin-\\nPerhaps the loss of this checkpoint could be turned to thera-\\ndle poles during meiosis I.\\npeutic advantage. Only better understanding of the molecular\\nprocesses involved will tell. ❚\\nPERSPECTIVES FOR THE FUTURE\\nKEY TERMS\\nThe remarkable pace of cell-cycle research over the last 25\\nyears has led to the model of eukaryotic cell-cycle control anaphase-promoting \\noutlined in Figure 21-2. A beautiful logic underlies these mo- complex (APC) 000 interphase 000\\nlecular controls. Each regulatory event has two important\\nAPC specificity  meiosis 000\\nfunctions: to activate a step of the cell cycle and to prepare\\nfactors 000 mitogens 000\\nthe cell for the next event of the cycle. This strategy ensures\\nCdc25 phosphatase 000 mitosis-promoting \\nthat the phases of the cycle occur in the proper order. \\ncheckpoints 000 factor (MPF) 000\\nAlthough the general logic of cell-cycle regulation now\\nseems well established, many critical details remain to be dis- CIPs 000 mitotic cyclin 000\\ncovered. For instance, although researchers have identified cohesins 000 p53 protein 000\\nsome components of the pre-replication complex that must condensin 000 quiescent cells 000\\nbe phosphorylated by S-phase CDK-cyclin complexes to ini-\\ncrossing over 000 Rb protein 000\\ntiate DNA replication, other components remain to be de-\\ncyclin-dependent  restriction point 000\\ntermined. Also still largely unknown are the substrates\\nkinases (CDKs) 000 S-phase inhibitor 000\\nphosphorylated by mitotic cyclin-CDK complexes, leading to\\nchromosome condensation and the remarkable reorganiza- destruction box 000 S phase–phase promoting \\ntion of microtubules that results in assembly of the beauti- DNA-damage  factor (SPF) 000\\nful mitotic spindle. Relatively little is known about how the checkpoint 000 securin 000\\nactivities of Wee1 kinase and Cdc25 phosphatase are con- E2F proteins 000 synapsis 000\\ntrolled; these proteins in turn regulate the kinase activity of\\nG1 cyclins 000 Wee1 protein-tyrosine \\nthe CDK subunit in most cyclin-CDK complexes. \\nkinase 000\\nMuch has been discovered recently about operation of\\nthe cell-cycle checkpoints, but the mechanisms that activate\\nREVIEW THE CONCEPTS\\nATM and ATR in the DNA-damage checkpoint are poorly\\nunderstood. Likewise, much remains to be learned about the\\n1. What strategy ensures that passage through the cell cycle\\ncontrol and mechanism of Mad2 in the spindle-assembly\\nis unidirectional and irreversible? What is the molecular \\ncheckpoint and of Cdc14 in the chromosome-segregation\\nmachinery that underlies this strategy?\\ncheckpoint in higher cells. As we learn in the next chapter,\\nasymmetric cell division plays a critical role in the normal de- 2. When fused with an S-phase cell, cells in which of the\\nvelopment of multicellular organisms. Many questions re- following phases of the cell cycle will initiate DNA replica-\\nmain about how the plane of cytokinesis and the localization tion prematurely—G1? G2? M? Predict the effect of fusing\\nof daughter chromosomes are determined in cells that divide a cell in G1 and a cell in G2 with respect to the timing of \\nasymmetrically. Similarly, the mechanisms that underlie the S phase in each cell. \\nunique segregation of chromatids during meiosis I have not\\n3. In 2001, the Nobel Prize in Physiology or Medicine was\\nbeen elucidated yet. \\nawarded to three cell cycle scientists. Sir Paul Nurse was rec-\\nognized for his studies with the fission yeast S. pombe,in\\nUnderstanding these detailed aspects of cell-cycle\\nparticular for the discovery and characterization of the wee1\\ncontrol will have significant consequences, partic-\\ngene. What is the wee phenotype? What did the characteri-\\nularly for the treatment of cancers. Radiation and\\nzation of the wee1gene tell us about cell cycle control?\\nmany forms of chemotherapy cause DNA damage and cell-\\ncycle arrest in the target cells, leading to their apoptosis. But 4. Tim Hunt shared the 2001 Nobel Prize for his work in the\\nthis induction of apoptosis depends on p53 function. For this discovery and characterization of cyclin proteins in eggs and\\nreason, human cancers associated with mutations of both p53 embryos. What experimental evidence indicates that cyclin B\\nalleles, which is fairly common, are particularly resistant to is required for a cell to enter mitosis? What evidence indicates\\nthese therapies. If more were understood about cell-cycle con- that cyclin B must be destroyed for a cell to exit mitosis? \\nAnalyze the Data 43\\n5. Leeland Hartwell, the third recipient of the 2001 Nobel how the following mutations, which are found in some can-\\nPrize, was acknowledged for his characterization of cell cycle cer cells, lead to a bypass of restriction point controls: \\ncheckpoints in the budding yeast S. cerevisiae. What is a cell (a) overexpression of cyclin D, (b) loss of Rb function, \\ncycle checkpoint? Where do checkpoints occur in the cell (c) infection by retroviruses encoding v-Fos and v-Jun.\\ncycle? How do cell cycle checkpoints help to preserve the \\n13. Individuals with the hereditary disorder ataxia telan-\\nfidelity of the genome?\\ngiectasia suffer from neurodegeneration, immunodeficiency,\\n6. In Xenopus,one of the substrates of MPF is the Cdc25 and increased incidence of cancer. The genetic basis for\\nphosphatase. When phosphorylated by MPF, Cdc25 is acti- ataxia telangiectasia is a loss-of-function mutation in the\\nvated. What is the substrate of Cdc25? How does this infor- ATM gene (ATM \" ataxia telangiectasia-mutated). Name\\nmation explain the autocatalytic nature of MPF as described two substrates of ATM. How does the phosphorylation of\\nin Figure 21-6?  these substrates lead to inactivation of CDKs to enforce cell\\ncycle arrest at a checkpoint?\\n7. Explain how CDK activity is modulated by the follow-\\ning proteins: (a) cyclin, (b) CAK, (c) Wee1, (d) p21. 14. Meiosis and mitosis are overall analogous processes in-\\nvolving many of the same proteins. However, some proteins\\n8. Three known substrates of MPF or kinases regulated by\\nfunction uniquely in each of these cell division events. Ex-\\nMPF are the nuclear lamins, subunits of condensin, and myosin\\nplain the meiosis-specific function of the following: (a) Ime2,\\nlight chain. Describe how the phosphorylation of each of these\\n(b) Rec8, (c) monopolin.\\nproteins affects its function and progression through mitosis.\\n9. Describe the series of events by which the APC promotes\\nthe separation of sister chromatids at anaphase.\\nANALYZE THE DATA\\n10. A common feature of cell cycle regulation is that the\\nevents of one phase ensure progression into a subsequent As genes encoding cyclin-dependent kinases were cloned\\nphase. In S. cerevisiae, Cdc28-Clns catalyze progression from fission yeast, budding yeast, and animal cells, investi-\\nthrough G1, and Cdc28-Clbs catalyze progression through gators working in plant systems designed experiments to de-\\nS/G2/M. Name three ways in which the activity of Cdc28- termine whether plants also possessed CDKs that functioned\\nClns promotes the activation of Cdc28-Clbs. as the catalysts for cell cycle progression. To isolate a cDNA\\nencoding a CDK in maize, investigators aligned the predicted\\n11. For S phase to be completed in a timely manner, DNA\\namino acid sequences of human cdc2, S. pombe cdc2, and\\nreplication initiates from multiple origins in eukaryotes. In \\nS. cerevisiae cdc28genes (see the figure below). Degenerate\\nS. cerevisiae, what role do S-phase CDK-cyclin complexes\\noligonucleotide PCR primers corresponding to the boxed\\nplay to ensure that the entire genome is replicated once and\\namino acids were generated and a DNA fragment of the ex-\\nonly once per cell cycle?\\npected size was synthesized by PCR using these primers and\\n12. What is the functional definition of the restriction point? maize cDNA library as a template. Why were the boxed re-\\nCancer cells typically lose restriction point controls. Explain gions selected as the basis for primer design?\\n44 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nThe PCR product was then used as a probe to screen the Fang, G., H. Yu, and M. W. Kirschner. 1999. Control of mitotic\\nmaize cDNA library, and a full-length cDNA clone was iso- transitions by the anaphase-promoting complex. Philos. Trans. R.\\nSoc. London Ser. B354:1583–1590.\\nlated. The clone was sequenced, and the predicted amino\\nR. Jessberger. 2002. The many functions of SMC proteins in\\nacid sequence was 64% identical to human cdc2and 63%\\nchromosome dynamics. Nature Rev. Mol. Cell Biol.3:67–78.\\nidentical to S. pombe cdc2and toS. cerevisiae cdc28. What\\nY. Masui. 2001. From oocyte maturation to the in vitro cell cy-\\ndoes the level of sequence similarity suggest about the evo- cle: the history of discoveries of Maturation-Promoting Factor (MPF)\\nlution of cdc2genes? and Cytostatic Factor (CSF). Differentiation69:1–17.\\nA complementation experiment was performed with \\nS. cerevisiaecells possessing a temperature-sensitive cdc28 Genetic Studies with S. pombe\\nmutation. Wild-type cells, cdc28ts cells, and cdc28ts cells\\nNurse, P. 2002. Cyclin dependent kinases and cell cycle control\\ntransformed with the maize cdc2cDNA under the influence (Nobel Lecture). Chembiochem.3:596–603.\\nof a strong promoter were grown at the permissive (25 °C) or\\nrestrictive (37 °C) temperatures. Cell proliferation was mon- Molecular Mechanisms for Regulating Mitotic Events\\nitored by the growth of colonies on the culture plates (shown\\nBurke, B., and J. Ellenberg. 2002. Remodeling the walls of the\\nin the figure below).\\nnucleus. Nature Rev. Mol. Cell Biol.3:487–497.\\nNelson, W. J. 2000. W(h)ither the Golgi during mitosis? J. Cell\\nBiol.149:243–248.\\nNigg, E. A. 2001. Mitotic kinases as regulators of cell division\\nand its checkpoints. Nature. Rev. Mol. Cell Biol.2:21–32.\\nGenetic Studies with S. cerevisiae\\nBell, S. P., and A. Dutta. 2002. DNA replication in eukaryotic\\ncells. Ann. Rev. Biochem.71:333–374.\\nDeshaies, R. J. 1999. SCF and Cullin/Ring H2-based ubiquitin\\nligases. Ann. Rev. Cell Devel. Biol.15:435–467.\\nDiffley, J. F., and K. Labib. 2002. The chromosome replication\\ncycle. J. Cell Sci.115:869–872.\\nHartwell, L. H. 2002. Yeast and cancer (Nobel Lecture). Biosci.\\nRep.22:373–394.\\nKelly, T. J., and G. W. Brown. 2000. Regulation of chromosome\\nreplication. Ann. Rev. Biochem.69:829–880.\\nKitagawa, K., and P. Hieter. 2001. Evolutionary conservation be-\\ntween budding yeast and human kinetochores. Nature Rev. Mol. Cell\\nBiol.2:678–687.\\nLei, M., and B. K. Tye. 2001. Initiating DNA synthesis: from re-\\ncruiting to activating the MCM complex. J. Cell Sci.114:1447–1454.\\nNasmyth, K. 2001. Disseminating the genome: joining, resolv-\\ning, and separating sister chromatids during mitosis and meiosis.\\nAnn. Rev. Genet.35:673–745.\\nNasmyth, K. 2002. Segregating sister genomes: the molecular bi-\\nology of chromosome separation. Science297:559–565.\\nUhlmann, F. 2001. Chromosome cohesion and segregation in mi-\\nWhy do the cdc28tscells form colonies at 25 °C but not tosis and meiosis. Curr. Opin. Cell Biol.13:754–761.\\n37 °C? What is the significance of colony formation of the\\ncdc28ts#maize cdc2cells at 37 °C? What does this experi- Cell-Cycle Control in Mammalian Cells\\nment tell us about the functional homology of cyclin- Ekholm, S. V., and S. I. Reed. 2000. Regulation of G(1) cyclin-\\ndependent kinase genes among eukaryotic species? dependent kinases in the mammalian cell cycle. Curr. Opin. Cell Biol.\\n12:676–684.\\nHarper, J. W., J. L. Burton, and M. J. Solomon. 2002. The\\nanaphase-promoting complex: it’s not just for mitosis any more.\\nREFERENCES\\nGenes Devel.16:2179–2206.\\nSears, R. C., and J. R. Nevins. 2002. Signaling networks that\\nOverview of the Cell Cycle and Its Control link cell proliferation and cell fate. J. Biol. Chem.277:11617–11620.\\nNasmyth, K. 2001. A prize for proliferation. Cell107: 689–701. Sherr, C. J. 2001. The INK4a/ARF network in tumour suppres-\\nsion. Nature Rev. Mol. Cell Biol.2:731–737.\\nBiochemical Studies with Oocytes, Eggs, and Early Embryos\\nCheckpoints in Cell-Cycle Regulation\\nDoree, M., and T. Hunt. 2002. From Cdc2 to Cdk1: when did\\nthe  cell  cycle  kinase  join  its  cyclin  partner? J. Cell Sci. 115: Bardin, A. J., and A. Amon. 2001. Men and sin: what’s the dif-\\n2461–2464. ference? Nature Rev. Mol. Cell Biol.2:815–826.\\nReferences 45\\nOsborn, A. J., S. J. Elledge, and L. Zou. 2002. Checking on the Meiosis: A Special Type of  Cell Division\\nfork: the DNA-replication stress-response pathway. Trends Cell Biol.\\nLee, B., and A. Amon. 2001. Meiosis: how to create a special-\\n12:509–516.\\nized cell cycle. Curr. Opin. Cell Biol.13:770–777. \\nPereira, G., and E. Schiebel. 2001. The role of the yeast spindle\\nPetronczki, M., M. F. Siomos, and K. Nasmyth. 2003. Un menage\\npole body and the mammalian centrosome in regulating late mitotic\\na quatre: the molecular biology of chromosome segregation in meio-\\nevents. Curr. Opin. Cell Biol.13:762–769.\\nsis. Cell112:423–440.\\nShah, J. V., and D. W. Cleveland. 2000. Waiting for anaphase:\\nMad2 and the spindle assembly checkpoint. Cell103:997–1000.\\n22\\nCELL BIRTH, \\nLINEAGE, \\nAND DEATH\\nCells being born in the developing cerebellum.All nuclei are\\nlabeled in red;the green cells are dividing and migrating into\\ninternal layers of the neural tissue.[Tal Raveh, Matthew Scott, and\\nJane Johnson.]\\nDuring the evolution of multicellular organisms, new cell lineage ultimately culminates in formation of terminally\\nmechanisms arose to diversify cell types, to coordi- differentiated cells such as skin cells, neurons, or muscle\\nnate their production, to regulate their size and num- cells. Terminal differentiation generally is irreversible, and\\nber, to organize them into functioning tissues, and to the resulting highly specialized cells often cannot divide; they\\neliminate extraneous or aged cells. Signaling between cells survive, carry out their functions for varying lengths of time,\\nbecame even more important than it is for single-celled  and then die. \\norganisms. The mode of reproduction also changed, with Many cell lineages contain intermediate cells, referred to\\nsome cells becoming specialized as germ cells (e.g., eggs, as precursor cellsor progenitor cells¸whose potential to form\\nsperm), which give rise to new organisms, as distinct from all different kinds of differentiated cells is more limited than\\nother body cells, called somatic cells. Under normal condi- that of the stem cells from which they arise.(Although some\\ntions somatic cells will never be part of a new individual. researchers distinguish between precursor and progenitor\\nThe formation of working tissues and organs during  cells, we will use these terms interchangeably.) Once a new\\ndevelopmentof multicellular organisms depends on specific precursor cell type is created, it often produces transcription\\npatterns of mitotic cell division. A series of such cell divisions factors characteristic of its fate. These transcription factors\\nakin to a family tree is called a cell lineage,which traces the\\nprogressive differentiationof cells into specialized cell types.\\nCell lineages are controlled by intrinsic (internal) factors—cells\\nO UTLINE\\nacting according to their history and internal regulators—\\nas well as by extrinsic (external) factors such as cell-cell sig- 22.1 The Birth of Cells \\nnals and environmental inputs (Figure 22-1). A cell lineage\\n22.2 Cell-Type Specification in Yeast\\nbegins with stem cells, unspecialized cells that can poten-\\ntially reproduce themselves and generate more-specialized 22.3 Specification and Differentiation of Muscle\\ncells indefinitely. Their name comes from the image of a\\n22.4 Regulation of Asymmetric Cell Division\\nplant stem, which grows upward, continuing to form more\\nstem, while sending off leaves and branches to the side. A 22.5 Cell Death and Its Regulation\\n1\\n2 CHAPTER 22 • Cell Birth, Lineage, And Death\\n!FIGURE 22-1 Overview of the birth,lineage,and death\\nof cells. Following growth, cells are “born” as the result of\\nsymmetric or asymmetric cell division. (a) The two daughter cells\\nCell growth\\nresulting from symmetric division are essentially identical to each\\nother and to the parental cell. Such daughter cells subsequently (a) (b)\\ncan have different fates if they are exposed to different signals.\\nSymmetric\\nThe two daughter cells resulting from asymmetric division differ\\ndivision Cell\\nfrom birth and consequently have different fates. Asymmetric birth\\ndivision commonly is preceded by the localization of regulatory\\nmolecules (green) in one part of the parent cell. (b) A series of Cell growthFPO\\nsymmetric and/or asymmetric cell divisions, called a cell lineage,\\ngives birth to each of the specialized cell types found in a\\nAsymmetric\\nmulticellular organism. The pattern of cell lineage can be under Cell\\ndivision \\ndeath\\ntight genetic control. Programmed cell death occurs during\\nnormal development (e.g., in the webbing that initially develops\\nwhen fingers grow) and also in response to infection or poison. \\nA series of specific programmed events, called apoptosis, is\\nactivated is these situations.\\nCell lineage\\ncoordinately activate, or repress, batteries of genes that com- produced by such asymmetric cell division may differ in\\nplete the differentiation process. For instance, a few key reg- size, shape, and/or composition, or their genes may be in\\nulatory transcription factors create the different mating types different states of activity or potential activity. The differ-\\nof budding yeast and coordinate the numerous genes neces- ences in these internal signals confer different fates on the\\nsary to turn a precursor cell into a muscle cell, two exam- two cells. \\nples that we discuss in this chapter.  Here we discuss some general features of how different\\nTypically we think of cell fates in terms of the differenti- cell types are generated, culminating with the best-understood\\nated cell types that are formed. A quite different cell fate, complex cell lineage, that of the nematode Caenorhabditis \\nprogrammed cell death,also is absolutely crucial in the for- elegans. In later sections, we focus on examples of the mo-\\nmation and maintenance of many tissues. A precise genetic lecular mechanisms that determine particular cell types in\\nregulatory system, with checks and balances, controls cell yeast, Drosophila, and mammals. \\ndeath just as other genetic programs control cell differentia-\\ntion. In this chapter, then, we consider the life cycle of cells—\\nStem Cells Give Rise to Stem Cells \\ntheir birth, their patterns of division (lineage), and their\\nand to Differentiating Cells \\ndeath. These aspects of cell biology converge with develop-\\nmental biology and are among the most important processes Stem cells, which give rise to the specialized cells composing\\nregulated by various signaling pathways discussed in earlier the tissues of the body, exhibit several patterns of cell divi-\\nchapters.  sion. A stem cell may divide symmetrically to yield two\\ndaughter stem cells identical to itself (Figure 22-2a). Alter-\\nnatively, a stem cell may divide asymmetrically to generate a\\n22.1 The Birth of Cells copy of itself and a derivative stem cell that has more-\\nrestricted capabilities, such as dividing for a limited period of\\nMany descriptions of cell division imply that the parental time or giving rise to fewer types of progeny compared with\\ncell gives rise to two daughter cells that look and behave the parental stem cell (Figure 22-2b). A pluripotent(or multi-\\nexactly like the parental cell, that is, cell division is sym- potent) stem cell has the capability of generating a number of\\nmetric, and the progeny do not change their properties. But different cell types, but not all. For instance, a pluripotent\\nif this were always the case, none of the hundreds of dif- blood stem cell will form more of itself plus multiple types of\\nferentiated cell types would ever be formed. Differences blood cells, but never a skin cell. In contrast, a unipotent\\namong cells can arise when two initially identical daughter stem cell divides to form a copy of itself plus a cell that can\\ncells diverge by receiving distinct developmental or envi- form only one cell type. In many cases, asymmetric division\\nronmental signals. Alternatively, the two daughter cells of a stem cell generates a progenitor cell, which embarks on\\nmay differ from “birth,” with each inheriting different a path of differentiation, or even a terminally differentiating\\nparts of the parental cell (see Figure 22-1). Daughter cells cell (Figure 22-2c, d). \\n22.1 • The Birth of Cells 3\\nproperties. For example, the early mouse embryo passes\\nStem\\ncell through an eight-cell stage in which each cell can give rise\\nto any cell type of the embryo. If the eight cells are experi-\\n(a) mentally separated and individually implanted into a suit-\\nSymmetric able foster mother, each can form a whole mouse with no\\ndivision\\nparts missing. This experiment shows that the eight cells\\nStem\\ncell are all able to form every tissue; that is, they are totipotent.\\nThus the subdivision of body parts and tissue fates among\\nthe early embryonic cells has not irreversibly occurred at\\n(b)\\nAsymmetric the eight-cell stage. At the 16-cell stage, this is no longer\\ndivision\\nMore-restricted true; some of the cells are committed to particular differ-\\npotential stem  entiation paths. \\nFPO\\ncell\\nQuite different specialized cell types can arise from a\\n(c) common precursor cell. A hematopoietic stem cell can give\\nAsymmetric\\nrise to all the multifarious types of blood cell. However, a\\ndivision\\nwhole series of cell divisions is not required; a single divi-\\nProgenitor cell\\nsion of a pluripotent precursor cell can yield distinct prog-\\neny. For instance, lineage studies in which cells are marked\\nby stable infection with a detectable retrovirus have shown\\n(d)\\nAsymmetric that neurons and glial cells arise from a single division of a\\ndivision\\nTerminally particular precursor cell. These cell types are quite differ-\\ndifferentiating ent: neurons propagating and transmitting electrical sig-\\ncell\\nnals and glial cells providing electrical insulation and\\nsupport. The precursor that generates neurons and glial\\n▲FIGURE 22-2 Patterns of stem-cell division. Symmetric\\ncells is not a stem cell, since it is incapable of self-\\ndivision of a stem cell (a) produces two identical daughter cells\\nrenewal; presumably the neuron–glial cell precursor arises\\nwith the same potential as the parental stem cell. Asymmetric\\nfrom a stem cell further back in the lineage. Recent obser-\\ndivision of a stem cell always generates one daughter that is a\\nfunctional duplicate of the parental cell and one daughter that vations have raised the interesting possibility that stem\\nmay be a stem cell with a more-restricted set of possible cells for one tissue may be induced under certain condi-\\nprogeny cells (b), a progenitor (precursor) cell that undergoes tions to act as stem cells for a rather different tissue. As we\\nmore divisions before the progeny begin differentiating (c), or a discuss below, postnatal animals contain stem cells for\\ncell that begins differentiating immediately with no further cell many tissues including the blood, intestine, skin, ovaries\\ndivisions (d). and testes, muscle, liver, and even the adult brain where lit-\\ntle cell division normally occurs. In muscle and liver, stem\\ncells are most important in healing, as relatively little cell\\ndivision occurs in the adult tissues otherwise. \\nThe two critical properties of stem cells that distin-\\nguish them from all other cells are the ability to reproduce\\nthemselves indefinitely by symmetric division, often called\\nself-renewal,and the ability to divide asymmetrically to Cultured Embryonic Stem Cells Can Differentiate \\nform one daughter stem cell identical to itself and one into Various Cell Types\\ndaughter  cell  that  is  different  and  usually  of  more-\\nrestricted potential. In this way, mitotic division of stem Embryonic stem (ES) cells can be isolated from early em-\\ncells preserves a population of undifferentiated cells while bryos and grown in culture (Figure 22-3a). Cultured ES cells\\nsteadily producing a stream of differentiating cells. Al- can differentiate into a wide range of cell types, either in\\nthough some types of precursor cells can divide symmet- vitro or after reinsertion into a host embryo. When grown\\nrically to form more of themselves, they do so only for in suspension culture, human ES cells first differentiate into\\nlimited periods of time. Moreover, in contrast to stem multicellular aggregates, called embryoid bodies, that resem-\\ncells, the asymmetric division of a precursor cell generates ble early embryos. When these are subsequently transferred\\ntwo distinct daughter cells, neither of which is identical to to a solid medium, they grow into confluent cell sheets con-\\nthe parental precursor cell.  taining a variety of differentiated cell types including neural\\nThe fertilized egg, or zygote, is a single cell with the  cells and pigmented and nonpigmented epithelial cells (Fig-\\ncapability to generate all the cell types of the body. Al- ure 22-3b). Under other conditions, ES cells have been in-\\nthough not technically a stem cell because it is not self- duced to differentiate into precursors for various types of\\nrenewing, the zygote does give rise to cells with stem-cell blood cells.\\n4 CHAPTER 22 • Cell Birth, Lineage, And Death\\n!EXPERIMENTAL FIGURE 22-3 (a) (b)\\nCleavage\\nEmbryonic stem (ES) cells can be stage embryo\\nmaintained in culture and form\\ndifferentiated cell types.(a) Human\\nblastocysts are grown from cleavage-stage\\nInner\\nembryos produced by in vitro fertilization.\\ncell mass FPO\\nThe inner cell mass is separated from the Cultured\\nsurrounding extra-embryonic tissues and Blastocoel blastocyst\\nplated onto a layer of fibroblast cells that\\nhelp to nourish the embryonic cells. Trophoblast\\nIndividual cells are replated and form\\nEmbryoid body\\ncolonies of ES cells, which can be\\nmaintained for many generations and can Isolated inner\\ncell mass\\nbe stored frozen. (b) In suspension culture,\\nhuman ES cells differentiate into\\nmulticellular aggregates (embryoid bodies) Fibroblast\\nfeeder cells\\n(top). After embryoid bodies are\\ntransferred to a gelatinized solid medium, FPO\\nthey differentiate further into confluent cell\\nFPO\\nsheets containing a variety of\\ndifferentiated cell types including neural Dissociate cells\\ncells (middle), and pigmented and\\nNeurons\\nnonpigmented epithelial cells (bottom).\\nNew feeder\\n[Parts (a) and (b) adapted from J. S. Odorico  cells\\nReplate\\net al., 2001, Stem Cells19:193–204.]\\nFPO\\nEstablished\\nES cell cultures\\nEpithelial cells\\nThe possibility of using stem cells therapeutically the feasibility of using stem cells for such purposes can be as-\\nto restore or replace damaged tissue is fueling sessed adequately. ❚\\nmuch research on how to recognize and culture\\nthese remarkable cells from embryos and from various tis- Apart from their possible benefit in treating disease, ES\\nsues in postnatal (adult) animals. For example, if neurons cells have already proven invaluable for producing mouse\\nthat produce the neurotransmitter dopamine could be gen- mutants useful in studying a wide range of diseases, devel-\\nerated from stem cells grown in culture, it might be possible opmental mechanisms, behavior, and physiology. By tech-\\nto treat people with Parkinson’s disease who have lost such niques described in Chapter 9, it is possible to eliminate or\\nneurons. For such an approach to succeed, a way must be modify the function of a specific gene in ES cells (see Figure\\nfound to direct a population of embryonic or other stem cells 9-38). If the disrupted gene is not required for growth of ES\\nto form the right type of dopamine-producing neurons, and cells, then the mutated ES cells can be employed to produce\\nrejection by the immune system nust be prevented. In one on- mice with a gene knockout(see Figure 9-39). Analysis of the\\ngoing study in which embryonic neurons were transplanted effects caused by deleting or modifying a gene in this way\\ninto more than 300 Parkinson’s patients, some of the inserted often provides clues about the normal function of the gene\\ncells have survived for more than 12 years and have provided and its encoded protein. \\nsignificant clinical improvement. However, the fetal tissue\\nused in this study is scarce and its use is controversial. Stem\\nTissues Are Maintained by Associated\\ncells grown from very early embryos are another option for\\nPopulations of Stem Cells \\ntreating Parkinson’s disease and perhaps other neurodegen-\\nerative conditions such as Alzheimer’s disease. Similar pos- Many differentiated cell types are sloughed from the body\\nsibilities exist for generating blood, pancreas, and other cell or have life spans that are shorter than that of the organism.\\ntypes. Many important questions must be answered before Disease and trauma also can lead to loss of differentiated\\n22.1 • The Birth of Cells 5\\ncells. Since these cells generally do not divide, they must be continuously regenerate from a stem-cell population located\\nreplenished from nearby stem-cell populations.  deep in the intestinal wall in pits called crypts(Figure 22-4a).\\nOur skin, for instance, is a multilayered epithelium(the The stem cells produce precursor cells that proliferate and dif-\\nepidermis) underlain by a layer of stem cells that give rise ferentiate as they ascend the sides of crypts to form the sur-\\nboth to more of themselves and to keratinocytes,the major face layer of the finger-like gut projections called villi,across\\ncell type in skin. As keratinocytes move toward the outer sur- which intestinal absorption occurs. Pulse-chase labeling ex-\\nface, they become increasingly flattened and filled with ker- periments have shown that the time from cell birth in the\\natin intermediate filaments. It normally takes about 15–30 crypts to the loss of cells at the tip of the villi is only about 2\\ndays for a newly “born” keratinocyte in the lowest layer to to 3 days (Figure 22-4b). Thus enormous numbers of cells\\ndifferentiate and move to the topmost layer. The “cells” must be produced continually to keep the epithelium intact.\\nforming the topmost layer are actually dead and are contin- The production of new cells is precisely controlled: too little\\nually shed from the surface.  division would eliminate villi and lead to breakdown of the in-\\nIn contrast to epidermis, the epithelium lining the small testinal surface; too much division would create an excessively\\nintestine is a single cell thick (Figure 6-4). This thin layer keeps large epithelium and also might be a step toward cancer.\\ntoxins and pathogens from entering our bodies and also trans- Specific signals are required for creating and maintaining\\nports nutrients essential for survival from the intestinal lumen stem-cell populations. In the skin and the intestinal epithelium,\\ninto the body (Chapter 7). The cells of the intestinal epithelium stem-cell growth is regulated in part by !-catenin,a protein\\nthat helps link certain cell-cell junctions to the cytoskeleton\\n(see Figure 6-7) and also functions as a signal transducer in \\n(a)\\nthe Wnt pathway (see Figure 15-32). Overproduction of active\\nDirection of !-catenin leads to excess proliferation of the intestinal epithe-\\ncell migration lium and of hair follicles, which are specialized structures de-\\nrived from skin stem cells and embedded in the epidermis.\\nVillus\\nBlocking the function of !-catenin by interfering with the TCF\\ntranscription factor that it activates abolishes the stem cells in\\nDifferentiated\\nCrypt-villus region the intestine. In the absence of !-catenin function, skin stem\\njunction cells form epidermis, but the hair follicles and hair do not\\nform. It seems likely that Wnt signaling, or at least compo-\\nFPO nents of the intracellular pathway, is critical for forming, main-\\ntaining, or activating stem cells in a variety of tissues. \\nProliferative\\nregion\\nCrypt\\n\"EXPERIMENTAL FIGURE 22-4 Regeneration of the\\nintestinal epithelium from stem cells can be demonstrated in\\nStem cell pulse-chase experiments.(a) Schematic drawing of the lining of\\nregion the small intestine, which contains numerous villi formed from a\\nStem cells column of cells. These epithelial cells are born near the base of\\nPaneth cells\\npits (crypts) located between the villi. Located at the very bottom\\nof crypts are Paneth cells, a type of support cell; just above these\\n(b) are four to six stem cells, which divide about once a day, forming\\nprecursor cells that also actively divide. As the differentiated cells\\nenter the epithelium of a villus, they stop dividing and begin\\ntaking up nutrients from the gut. (b) Results from a pulse-chase\\nexperiment in which radioactively labeled thymidine (the pulse)\\nwas added to a tissue culture of the intestinal epithelium. \\nVilli\\nDividing cells incorporated the labeled thymidine into their newly\\nFPO synthesized DNA. The labeled thymidine was washed away and\\nreplaced with nonlabeled thymidine (the chase) after a brief\\nperiod; cells that divided after the chase did not become labeled.\\nThese micrographs show that 40 minutes after labeling, all the\\nlabel is in cells near the base of the crypt. At later times, the\\nlabeled cells are seen progressively farther away from their point\\nCrypt of birth in the crypt. Cells at the top are shed. This process\\nensures contant replenishment of the gut epithelium with new\\ncells. [Part (a) adapted from C. S. Potten, 1998, Philos. Trans. R. Soc.\\nLondon, Ser. B353:821. Part (b) from P. Kaur and C. S. Potten, 1986, \\n40 min 1 day 2 days 3 days Cell Tiss. Kinet.19:601.]\\n6 CHAPTER 22 • Cell Birth, Lineage, And Death\\nSkin also contains dendritic epidermal T cells, an factor \"(TGF\"), and TGF!. Discovering how all these sig-\\nimmune-system cell that produces a certain form nals work together to control growth and stimulate healing\\nof the T-cell receptor (see Table 14-1). When den- is a substantial challenge that will advance our understand-\\ndritic epidermal T cells are genetically modified so they do ing of diseases such as psoriasis and skin cancer and perhaps\\nnot produce T-cell receptors, wound healing is slow and less pave the way for effective treatments. ❚\\ncomplete than in normal skin. Normal healing is restored by\\naddition of keratinocyte growth factor. The current hypoth- Another continuously replenished tissue is the blood,\\nesis is that when dendritic epidermal T cells recognize anti- whose stem cells are located in the bone marrow in adult an-\\ngens on cells in damaged tissue, they respond by producing imals. The various types of blood cells all derive from a sin-\\nstimulating proteins, such as keratinocyte growth factor, that gle type of pluripotent hematopoietic stem cell,which gives\\npromote keratinocyte growth and wound healing. Many rise to the more-restricted myeloid and lymphoid stem cells\\nother signals also control the growth of skin cells, including (Figure 22-5). The frequency of hematopoietic stem cells is\\nWnt/!-catenin, Hedgehog, calcium, transforming growth about 1 cell per 104bone marrow cells, even lower than the\\nG-CSF\\nGranulocytes\\nIL-3, GM-CSF, SCF, IL-6\\nGM-CFC M-CSF\\nMonocytes\\nMyeloid\\nstem cell IL-3, GM-CSF\\nFPO\\nEo-CFC Eosinophils\\nEpo, SCF Epo\\nGM-CSF\\nIL-3\\nBFU-E CFU-E Erythrocytes\\nPluripotent\\nhematopoietic SCF, Tpo, IL-3, GM-CSF\\nstem cell\\nMegakaryocytes\\nCFC-mega\\n(platelet-forming cells)\\nLymphoid\\nstem cell\\nIIL-2, IL-7, IL-12, SDF-1,\\nFLT-3 ligand, TNF-α,\\nTGF-β1\\nT-CFC B-CFC T and B cells of the\\nimmune system\\n▲FIGURE 22-5 Formation of differentiated blood cells from eliminated and the precursor cells introduced. Further \\nhematopoietic stem cells in the bone marrow. Pluripotent cytokine-induced proliferation, commitment, and differentiation of\\nstem cells may divide symmetrically to self-renew (curved arrow) the precursor cells give rise to the various types of blood cells.\\nor divide asymmetrically to form myeloid and lymphoid stem Some of the cytokines that support this process are indicated\\ncells (light green). Although these stem cells are capable of  (red labels). GM # granulocyte-macrophage; Eo # eosinophil; \\nself-renewal, they are committed to one of the two major E # erythrocyte; mega # megakaryocyte; T #T-cell; B # B-cell;\\nhematopoietic lineages. Depending on the types and amounts of CFU # colony-forming unit; CSF # colony-stimulating factor; \\ncytokines present, the myeloid and lymphoid stem cells generate IL # interleukin; SCF # stem cell factor; Epo # erythropoietin;\\nvarious precursor cells (dark green), which are incapable of  Tpo # thrombopoietin; TNF # tumor necrosis factor; TGF #\\nself-renewal. These cells are detected by their ability to form transforming growth factor; SDF # stromal cell–derived factor;\\ncolonies containing the differentiated cell types shown at right, FLT-3 ligand # ligand for fms-like tyrosine kinase receptor 3.\\nmeasured as “colony-forming cells (CFCs).” The colonies are [Adapted from M. Socolovsky et al., 1998, Proc. Nat’l. Acad. Sci. USA\\ndetected on the spleen of animals that have had their own cells 95:6573.]\\n22.1 • The Birth of Cells 7\\nfrequency of intestinal stem cells in crypts. Numerous extra- Numerous genes have been found to regulate the forma-\\ncellular growth factors called cytokinesregulate proliferation tion, maintenance, and properties of meristems. Many of\\nand differentiation of the precursor cells for various blood- these genes encode transcription factors. For instance, a hier-\\ncell lineages. For example, erythropoietincan activate several archy of regulators, particularly transcription factors, con-\\ndifferent intracellular signal-transduction pathways, leading trols the separation of differentiating cells from SAMs as\\nto changes in gene expression that promote formation of  leaves form; similarly, three types of regulators control for-\\nerythrocytes (see Figure 14-7).  mation of the floral organs from floral meristems (Chapter\\nThe hematopoietic lineage originally was worked out by 15). In both cases, a cascade of gene interactions occurs, with\\ninjecting the various types of precursor cells into mice whose earlier transcription factors causing production of later ones.\\nprecursor cells had been wiped out by irradiation. By ob- At the same time, cells are dividing and the differentiating\\nserving which blood cells were restored in these transplant ones are spreading away from their original birth sites. ❚\\nexperiments, researchers could infer which precursors or ter-\\nminally differentiated cells (e.g., erythrocytes, monocytes)\\narise from a particular type of precursor. The first step in\\n(a) (d)\\nthese experiments was separation of the different types of\\nhematopoietic precursors. This is possible because each type 1\\n1 3\\nproduces unique combinations of cell-surface proteins that 1 3 3\\n1 6 5 3\\ncan serve as type-specific markers. If bone marrow extracts 6 8 5\\n1 8 7 3\\nare treated with fluorochrome-labeled antibodies for these 10 x 8 picas 6 i 5\\nmarkers, cells with different surface markers can be sepa- 4 i 7 2\\n6 7 5\\nrated in a fluorescence-activated cell sorter (see Figure 5-34).  4 6 5 2\\n4 4 2\\n4 2\\n2\\nFPO\\nTo date, bone marrow transplants represent the\\n(b)\\nmost successful and widespread use of stem cells\\nin medicine. The stem cells in the transplanted\\nmarrow can generate new, functional blood cells in pa-\\ntients with certain hereditary blood diseases and in cancer\\npatients who have received irradiation and/or chemother- 10 x 8 picas\\ni\\napy, both of which destroy the bone marrow cells as well\\nas cancer cells. Recent work is directed at exploring\\nwhether embryonic stem cells can be induced to differen-\\ntiate into cells types that would be useful therapeutically.\\nFor example, mouse stem cells treated with inhibitors of (c)\\nphosphatidylinositol-3 kinase, a regulator in one of the 7 8\\nphosphoinositide signaling pathways (Chapter 14), turn\\ninto cells that resemble pancreatic ! cells in their produc- 1\\n6\\ntion of insulin, their sensitivity to glucose levels, and their\\naggregation into structures reminiscent of pancreas struc- 10 x 8 picas 4\\n3\\ntures. Implantation of these cells into diabetic mice re-\\n2\\nstored their growth, weight, glucose levels, and survival 5\\nrates to normal. ❚\\nStem cells in plants are located in meristems,pop- ▲FIGURE 22-6 Cell fates in meristems of Arabidopsis.In\\nulations of undifferentiated cells found at the tips these longitudinal sections through a shoot apical meristem, cell\\nof growing shoots. Shoot apical meristems (SAMs) nuclei are revealed by staining with propidium iodide, which\\nproduce leaves and shoots, and of course more stem cells binds to DNA. (a) The shoot apical meristem (SAM) produces\\nthat constitute the nearly immortal meristems. Meristems shoots, leaves, and more meristem. Flower production occurs\\nwhen the meristem switches from leaf/shoot production to\\ncan persist for thousands of years in long-lived species such\\nflower production, concomitant with an increase in the number\\nas redwood trees and bristlecone pines. As a plant grows, the\\nof meristem cells to form floral meristems (FMs), as shown\\ncells “left behind” the meristems are encased in rigid walls\\nhere. (b) Cells in a SAM exhibit different fates and behaviors.\\nand can no longer grow. SAMs can split to form branches,\\nCells divide rapidly in the peripheral zone (PZ) to produce leaves\\neach branch with its own SAM, or be converted into floral\\nand in the rib zone (Rib) to produce central shoot structures.\\nmeristems (Figure 22-6). Floral meristems give rise to the\\nCells in the central zone (CZ) divide more slowly, producing an\\nfour floral organs—sepals, stamens, carpels, and petals—that\\nongoing source of meristem and contributing cells to the PZ and\\nform flowers. Unlike SAMs, floral meristems are gradually Rib. Scale bars, 50 $m. [See E. Meyerowitz, 1997, Cell88:299;\\ndepleted as they give rise to the floral organs. micrographs courtesy Elliot Meyerowitz.]\\n8 CHAPTER 22 • Cell Birth, Lineage, And Death\\nCell Fates Are Progressively Restricted  genome comes from the successful production of cloned an-\\nDuring Development imals by nuclear-transfer cloning. In this procedure, the \\nnucleus of an adult (somatic) cell is introduced into an egg\\nThe eight cells resulting from the first three divisions of a that lacks its nucleus; the manipulated egg, which contains\\nmammalian zygote (fertilized egg) all look the same. As the diploid number of chromosomes and is equivalent to a \\ndemonstrated experimentally in mice, each of the cells has zygote, then is implanted into a foster mother. The only\\nthe potential to give rise to a complete animal. Additional di- source of genetic information to guide development of the\\nvisions produce a mass, composed of ≈64 cells, that separates embryo is the nuclear genome of the donor somatic cell. The\\ninto two cell types: trophectoderm, which will form extra- frequent failure of such cloning experiments, however, raises\\nembryonic tissues like the placenta, and the inner cell mass,\\nwhich gives rise to the embryo proper. The inner cell mass (a) Primitive node Primitive streak\\neventually forms three germ layers, each with distinct fates.\\nOne layer, ectoderm,will make neural and epidermal cells; Epiblast Amnioblasts\\nanother,mesoderm,will make muscle and connective tissue;\\nthe third layer, endoderm, will make gut epithelia (Figure \\n22-7). If cells from any of these layers are transplanted into\\nYolk sac\\none of the other layers, they do not give rise to cells appro-\\npriate to their new location. This conclusion is based on ex-\\nperiments with chimeric animals composed of chicken and Invaginating Hypoblast\\nmesoderm cells\\nquail cells. Embryos composed of cells from both bird species\\ndevelop fairly normally, yet the cells derived from each donor (b)\\nare distinguishable under the microscope. Thus the contri-\\nbutions of the different donor cells to the final bird can be \\nascertained. \\nOnce the three germ layers are established, they subse-\\nquently divide into cell populations with different fates. For\\ninstance, the ectoderm becomes divided into those cells that\\nare precursors to the skin epithelium and those that are pre-\\ncursors to the nervous system. There appears to be a pro- FPO\\ngressive restriction in the range of cell types that can be\\nformed from stem cells and precursor cells as development (c)\\nproceeds. An early embryonic stem cell, as we’ve seen, can ECTODERM MESODERM ENDODERM\\nform every type of cell; an ectodermal cell has a choice be- Central nervous system Skull Liver\\ntween neural and epidermal fates; and a keratinocyte pre- Retina and lens  Head, skeletal muscle Pancreas\\ncursor can form skin but not neurons.  Cranial and sensory  Skeleton Urinary bladder\\nGanglia and nerves Dermis of skin Epithelial parts of: \\nAnother restriction that occurs early in animal develop-\\nPigment cells Connective tissue    trachea\\nment is the setting aside of cells that will form the germ line, Head connective tissue Urigential system    lungs\\nthat is, the stem cells and precursor cells that eventually will Epidermis Heart    pharynx\\nHair Blood, lymph cells    thyroid\\ngive rise to eggs in a female and sperm in a male. Only the\\nMammary glands Spleen\\ngenome of the germ line will ever be passed on to progeny.\\nThe setting aside of germ-line cells early in development has ▲FIGURE 22-7 Fates of the germ layers in animals. During\\nbeen hypothesized to protect chromosomes from damage by the early period of mammalian development, cells migrate to\\nreducing the number of rounds of replication they undergo create the three germ layers: ectoderm, mesoderm, and\\nor by allowing special packaging of the cells that are critical endoderm. Cells in the different layers have largely distinct fates\\nto heredity. Whatever the reason, the early segregation of the and therefore represent distinct cell lineages. The detailed cell\\ngerm line is widespread (though not universal) among ani- lineage, however, is not exactly the same in different individuals.\\n(a) Shown here is a sketch of a human embryo about 16 days\\nmals. In contrast, plants do nothing of the sort; most meri-\\nafter fertilization of an egg. The first cells to move from the\\nstems can give rise to germ-line cells. \\nepiblast into the interior form endoderm, and they are followed by\\nOne consequence of the early segregation of germ-line\\ninvaginating cells that become mesoderm. The remaining epiblast\\ncells is that the loss or rearrangement of genes in somatic\\ncells become ectoderm. Hypoblast #covering layer. Amnioblasts #\\ncells would not affect the inherited genome of a future zy-\\ncells that will line amniotic cavity. Yolk sac #reminder of our \\ngote. Although segments of the genome are rearranged and\\negg-bound ancestors. (b) Scanning electron micrograph of a \\nlost during development of lymphocytes from hematopoi-\\ncross section of a similar-stage embryo. (c) Some of the tissue\\netic precursors, most somatic cells seem to have an intact derivatives of the three germ layers are listed. [Part (b) from \\ngenome, equivalent to that in the germ line. Evidence that at T. W. Sadler, 2000, Langman’s Medical Embryology,8th ed. (Baltimore:\\nleast some somatic cells have a complete and functional Lippincott Williams and Wilkins), pp. 64–65, Figure 4.3).]\\n22.1 • The Birth of Cells 9\\nquestions about how many adult somatic cells do in fact individuals. The best-documented example of a reproducible\\nhave a complete functional genome. Even the successes, like pattern of cell divisions comes from the nematode C. elegans.\\nthe famous cloned sheep “Dolly,” appear to have some med- Scientists have traced the lineage of all the somatic cells in\\nical problems. Even if differentiated cells have a physically C. elegansfrom the fertilized egg to the mature worm by fol-\\ncomplete genome, clearly only portions of it are transcrip- lowing the development of live worms using Nomarski in-\\ntionally active (Chapter 10). Whether the genome of a dif- terference microscopy (Figure 22-8).\\nferentiated cell can revert to having the full developmental About 10 rounds of cell division, or fewer, create the \\npotential characteristic of an embryonic cell is a matter of adult worm, which is about 1 mm long and 70 $m in di-\\nconsiderable debate. A cell could, for example, have an intact ameter. The adult worm has 959 somatic cell nuclei (her-\\ngenome, but be unable to properly reactivate it due to inher- maphrodite form) or 1031 (male). The number of somatic \\nited chromatin states.  cells is somewhat fewer than the number of nuclei because \\nThese observations raise two important questions: How some cells contain multiple nuclei (i.e., they are syncytia). \\nare cell fates progressively restricted during development? Remarkably, the pattern of cell divisions starting from a \\nAre these restrictions irreversible? In addressing these ques- C. elegansfertilized egg is nearly always the same. As we \\ntions, it is important to remember that what a cell in its nor- discuss later in the chapter, many cells that are generated \\nmal in vivo location will do may differ from what a cell is during development undergo programmed cell death and \\ncapable of doing if it is manipulated experimentally. Thus the are missing in the adult worm. The consistency of the \\nobserved limits to what a cell can do may result from natural C. eleganscell lineage does not result entirely from each \\nregulatory mechanisms or may reflect a failure to find con- newly born cell inheriting specific information about its \\nditions that reveal the cell’s full potential. destiny. In some cases, various signals direct initially iden-\\nAlthough our focus in this chapter is on how cells be- tical cells to particular fates, and the outcomes of these sig-\\ncome different, their ability to remain the same also is criti- nals are consistent from one animal to the next. That is, \\ncal to the functioning of tissues and the whole organism. at their birth cells are not necessarily “hard wired” by \\nNondividing differentiated cells with particular characteris- their own internal inherited instructions to follow a par-\\ntics must retain them, sometimes for many decades. Stem ticular path of differentiation. \\ncells that divide regularly, such as a skin stem cell, must pro- The first few cell divisions in C. elegansproduce six dif-\\nduce one daughter cell with the properties of the parental ferent founder cells,each with a separate fate as shown in\\ncell, retaining its characteristic composition, shape, behavior, Figure 22-9a, b. The initial division is asymmetric, giving rise\\nand responses to specific external signals. Meanwhile, the to P1 and the AB founder cell. Further divisions in the P lin-\\nother daughter cell with its own distinct inheritance, as the eage form the other five founder cells. Some of the signals\\nresult of asymmetric cell division, embarks on a differentia- controlling the asymmetry are known. For example, Wnt\\ntion pathway, which may be determined by the signals the signals from the P2 precursor control the asymmetric divi-\\ncell receives.  sion of the EMS cell into E and MS founder cells. Wnt sig-\\nnaling (see Figure 15-32) is also used in other asymmetric\\ndivisions in worms. \\nThe Complete Cell Lineage of C. elegans\\nThe bilateral symmetry of the worm implies duplication\\nIs Known \\nof lineages on the two sides; curiously, though, functionally\\nIn the development of some organisms, cell lineages are equivalent cells on each side can arise from a pattern of \\nunder tight genetic control and thus are identical in all indi- division that is different on the two sides. Some of the em-\\nviduals of a species. In other organisms the exact number bryonic cells function as stem cells, dividing repeatedly to\\nand arrangement of cells vary substantially among different form more of themselves or another type of stem cell, while\\nDorsal nerve\\nPharynx cord Intestinal lumen\\nMouth Intestine Intestinal nuclei\\nExcretory Anus\\ncell Gonad\\nVentral nerve cord Rectum\\nVentral\\nnuclei\\nnerve cord 20 $m\\n▲FIGURE 22-8 Newly hatched larva of C.elegans.Many of in this micrograph obtained by Nomarski interference microscopy.\\nthe 959 somatic cell nuclei in this hermaphrodite form are visualized [From J. E. Sulston and H. R. Horvitz, 1977, Devel. Biol.56:110.]\\n10 CHAPTER 22 • Cell Birth, Lineage, And Death\\n(a) (b)\\nP0\\nAB P1\\nneurons\\nhypodermis EMS P2\\npharyngeal muscle\\nbody muscle MS E\\nother body muscle gut P3 C FPO\\npharyngeal muscle hypodermis\\nneurons body muscle\\nglands neurons\\nsomatic gonad P4 D\\nother germ cells body muscle\\n(c) Zygote\\ns\\nsi\\ne\\nn Germ\\nge FPO line\\no Pharynx Intestine\\ny\\nr\\nb\\nm\\nE\\ns\\nn\\na\\ng\\ne\\nel\\nC.  Neurons\\nf \\no\\ng \\nn\\ngi\\nS a\\nm\\nN\\nI\\nO e \\ns Vulva\\nI p Epidermis\\nT a Somatic gonad\\nL\\nC -\\ne\\nE m ▲FIGURE 22-9 C.eleganslineage.(a) Pattern of the aside very early, as in most animals. All the other lineages give\\nN Ti first few divisions starting with P0 (the zygote) and leading rise to somatic cells. (b) Light micrographs of the first few\\nN n:  to formation of the six founder cells (yellow highlights). The divisions of the embryo that generate the founder cells with cells\\nO o\\nC ati first division, into P1 and AB, is asymmetric. Further labeled as in part (a). The texture of the cells shows the presence\\nm divisions in the P lineage generates the other founder of organelles and is revealed by differential interference contrast\\n \\nA ni cells. Note that more than one lineage can lead to the microscopy, sometimes called Nomarski microscopy. (c) Full\\nDI o A same tissue type (e.g., muscle or neurons). The EMS cell lineage of the entire body of the worm, showing some of the\\nE e is so named because it is the precursor to most of the tissues formed. Note that any particular cell undergoes relatively\\nd\\nM Vi endoderm and mesoderm. The lineage beginning with the few divisions, typically fewer than 15. [Einhard Schierenberg\\nP4 cell gives rise to all of the germ-line cells, which are set Zoologisches Institut, Universität Köln.]\\nspinning off differentiating cells that give rise to a particular of development occurs too early (precocious development) or\\ntissue. In one pattern of division, one of the daughter cells too late (retarded development). An example of the former is\\nfrom a division initiates a repeat of the lineage pattern of the a premature cell division to yield a cell that differentiates and\\nprevious few divisions. The complete lineage of C. elegans a cell that dies; as a result, the lineage that should have fol-\\nis shown in Figure 22-9c. This organism has been a powerful lowed from the dead cell never happens. In the latter case,\\nmodel system for genetic studies to identify the regulators the delayed occurrence of a lineage causes juvenile structures\\nthat control cell lineages in time and space. to be produced, incorrectly, in more mature animals. In both\\ncases, the character of a parental cell is, in essence, changed\\nto the character of a cell at a different stage of development. \\nHeterochronic Mutants Provide Clues \\nOne example of precocious development in C. elegans\\nAbout Control of Cell Lineage \\ncomes from loss-of-function mutations in the lin-14gene,\\nIntriguing evidence for the genetic control of cell lineage has which cause premature formation of the PDNB neuroblast\\ncome from isolation and analysis of heterochronicmutants. (Figure 22-10a). The lin-14gene and several others found to\\nIn these mutants, a developmental event typical of one stage be defective in heterochronic worm mutants encode RNA-\\n22.1 • The Birth of Cells 11\\n(b) L1 L2 L3/4 Adult\\nlin-14, lin-28 lin-14, lin-28 lin-14, lin-28 lin-14, lin-28\\nmRNA mRNA mRNA mRNA\\nlin-4 lin-4 lin-4 let-7\\n(a) RNA RNA RNA RNA\\nFPO\\nWild type lin-14 mutant\\nLIN-14, LIN-28\\nV5 V5 proteins lin-41 mRNA\\nL1\\nlin-41 mRNA lin-41 mRNA lin-41 mRNA\\nPDNB let-7\\nRNA\\nX\\nLIN-41 LIN-41 LIN-41\\nprotein protein protein lin-29 mRNA\\nL2 PDNB\\n_ _ _\\nlin-29 mRNA lin-29 mRNA lin-29 mRNA LIN-29\\nprotein\\nn\\nei LIN-14, LIN-28\\not\\nr\\np LIN-41\\nr \\nA oel lin-4 let-7 LIN-29\\nNv\\nRe\\nl\\n▲FIGURE 22-10 Timing of specific types of cell division RNA to the 3% untranslated regions (UTRs) of lin-14and lin-28\\nduring development of C.elegans.(a) The pattern of cell mRNAs prevents translation of these mRNAs into protein. This\\ndivision for the V5 cell of C. elegansis shown for normal worms occurs following the first larval (L1) stage, permitting\\nand for a heterochronic mutant called lin-14.In the lin-14mutant, development to proceed to the later larval stages. Starting in the\\nthe pattern of cell division (red arrows) that normally occurs only fourth larval stage (L4), production of let-7RNA begins. It\\nin the second larval stage (L2) occurs in the first larval stage (L1), hybridizes to lin-14,lin-28, and lin-41mRNAs, preventing their\\ncausing the PDNB neuroblast to be generated prematurely. In  translation. LIN-41 protein is an inhibitor of translation of the \\nthe mutant, the V5 cell behaves during L1 like cell “X” (purple) lin-29mRNA, so the appearance of let-7RNA allows production\\nnormally does in L2. The inference is that the LIN-14 protein of LIN-29 protein, which is needed for generation of adult cell\\nprevents L2-type cell divisions, although precisely how it does so lineages. LIN-4 may also bind to lin-41RNA at later stages. Only\\nis unknown. (b) Two small regulatory RNAs, lin-4and let-7,serve the 3% UTRs of the mRNAs are depicted. [Adapted from B. J.\\nas coordinating timers of gene expression. Binding of the lin-4 Reinhart et al., 2000, Nature403:901.]\\nbinding or DNA-binding proteins, which presumably coordi- other miRNAs during the life cycle of C. elegansserve as a regu-\\nnate expression of other genes. However, two other genes (lin- latory clock for cell lineage. Molecules related to let-7RNA have\\n4and let-7) involved in regulating the timing of cell divisions been identified in many other animals including vertebrates and\\nwere initially extremely puzzling, as they appeared not to en- insects; since their production is temporally regulated in these an-\\ncode any protein. To discover the products of these genes, sci- imals as well, they may serve a similar function in their develop-\\nentists first determined which pieces of genomic DNA could ment as in C. elegans. How production of these regulatory\\nrestore gene function, and therefore proper cell lineage, to mu- miRNAs is temporally controlled is not yet known, but they have\\ntants defective in each gene. They then did the same thing with turned out to play many roles in regulating gene expression\\ngenomic DNA from the corresponding genomic regions of dif- (Chapter 12). \\nferent species of worm. Comparison of the “rescuing” frag-\\nments from the different species revealed that they shared\\ncommon short sequences with little protein-coding potential.  KEY CONCEPTS OF SECTION 22.1\\nThe short RNA molecules encoded by lin-4and let-7were\\nThe Birth of Cells\\nsubsequently shown to inhibit translation of the mRNAs en-\\ncoded by lin-14and other heterochronic genes (Figure 22-10b). ■ In asymmetric cell division, two different types of daugh-\\nThese small RNAs, or micro RNAs (miRNAs),are complemen- ter cells are formed from one mother cell. In contrast, both\\ntary to sequences in the 3%untranslated parts of target mRNAs daughter cells formed in symmetric divisions are identical\\nand are believed to control translation of the mNRAs by hy- but may have different fates if they are exposed to differ-\\nbridizing them. Temporal changes in the production of these and ent external signals (see Figure 22-1). \\n12 CHAPTER 22 • Cell Birth, Lineage, And Death\\n■ Pluripotent stem cells can produce more than one type Our first example of cell-type specification comes from\\nof descendant cell, including in some cases a stem cell with the budding yeast, S. cerevisiae.We introduced this useful\\na more-restricted potential to produce differentiated cell unicellular eukaryote way back in Chapter 1 and have en-\\ntypes (see Figure 22-2).  countered it in several other chapters.S. cerevisiaeforms\\nthree cell types: haploid aand \"cells and diploid a/\"cells.\\n■ Cultured embryonic stem cells (ES cells) are capable of\\nEach type has its own distinctive set of active genes; many\\ngiving rise to many kinds of differentiated cell types. They\\nother genes are active in all three cell types. In a pattern com-\\nare useful in production of genetically altered mice and \\nmon to many organisms and tissues, cell-type specification in\\noffer potential for therapeutic uses. \\nyeast is controlled by a small number of transcription factors\\n■ Populations of stem cells associated with most tissues that coordinate the activities of many other genes. Similar\\n(e.g., skin, intestinal epithelium, blood) regenerate differ-\\nregulatory features are found in the responses of higher eu-\\nentiated tissue cells that are damaged or sloughed or be-\\nkaryotic cells to environmental signals and in the specifica-\\ncome aged (see Figures 22-4 and 22-5). \\ntion and patterning of cells and tissues during development. \\n■ Stem cells are prevented from differentiating by specific Recent DNA microarraystudies of transcription patterns\\ncontrols. A high level of !-catenin, a component of the in S. cerevisiaehave provided a genome-wide picture of the\\nWnt signaling pathway, has been implicated in preserving fluctuations of gene expression in the different cell types and\\nstem cells in the skin and intestine.  different stages of the yeast life cycle (see Figure 9-35 for an ex-\\nplanation of the DNA microarray technique). Among other\\n■ Plant stem cells persist for the life of the plant in the\\nthings, these studies identified 32 genes that are transcribed\\nmeristem. Meristem cells can give rise to a broad spectrum\\nat more than twofold higher levels in \"cells than in acells. An-\\nof cell types and structures. \\nother 50 genes are transcribed at more than twofold higher\\n■ During development, precursor cells generally lose po- levels in acells than in \"cells. The products of these 82 genes,\\ntential; that is, they become progressively restricted in the\\nwhich initially are activated by cell-type specification tran-\\nnumber of different cell types they can form. \\nscription regulators, convey many of the critical differences be-\\n■ Early in animal development, the three germ layers— tween the two cell types. Since transcription of more than\\nectoderm, mesoderm, and endoderm—form. Each gives 6000 genes was assessed in these experiments, the results\\nrise to specific tissues and organs (see Figure 22-7). clearly demonstrate that changes in expression of only a small\\nfraction of the genome can significantly alter the behavior and\\n■ Germ-line cells give rise to eggs or sperm. By definition,\\nproperties of cells. Transcription of a much larger number of\\nall other cells are somatic cells. \\ngenes, about 25 percent of the total assayed, differed substan-\\n■ Embryonic development of C. elegansbegins with asym- tially in diploid cells compared with haploid cells. A powerful\\nmetric division of the fertilized egg (zygote). The lineage\\nadvantage of the microarray approach is that it systematically\\nof all the cells in adult worms is known and is highly re-\\nidentifies a large fraction of the relevant genes controlling var-\\nproducible (see Figure 22-9). \\nious processes, allowing scientists to concentrate on the most\\n■ Short regulatory RNAs control the timing of developmen- important players that regulate differences in cell types. \\ntal cell divisions by preventing translation of mRNAs whose\\nencoded proteins control cell lineages (see Figure 22-10).\\nTranscription Factors Encoded at the MATLocus\\nAct in Concert with MCM1 to Specify Cell Type\\n22.2 Cell-Type Specification in Yeast Each of the three S. cerevisiaecell types expresses a unique set\\nof regulatory genes that is responsible for all the differences\\nIn  the  previous  section,  we  saw  that  stem  cells  and  among the three cell types. All haploid cells express certain\\nprecursor/progenitor cells produce progeny that embark haploid-specific genes; in addition, acells express a-specific\\nupon specific differentiation paths. The elegant regulatory genes, and \"cells express \"-specific genes. In a/\"diploid cells,\\nmechanisms whereby cells become different is referred to as diploid-specific genes are expressed, whereas haploid-specific,\\ncell-type specification.Specification usually involves a com- a-specific, and \"-specific genes are not. As illustrated in Fig-\\nbination of external signals with internal signal-transduction ure 22-11, three cell type–specific transcription factors (\"1,\\nmechanisms like those described in Chapters 14 and 15. The \"2, and a1) encoded at the MAT locusin combination with\\ntransition from an undifferentiated cell to a differentiating a general transcription factor called MCM1,which is ex-\\none often involves the production of one or a small number pressed in all three cell types, mediate cell type–specific gene\\nof transcription factors. The newly produced transcription expression in S. cerevisiae. Thus the actions of just three tran-\\nfactors serve as powerful switches that trigger the activation scription factors can set the yeast cell on a specific differenti-\\n(and sometimes repression) of large batteries of subservient ation pathway culminating in a particular cell type. From the\\ngenes. Thus an initially modest change can cause massive DNA microarray experiments we know one outcome: the ac-\\nchanges in gene expression that confer a new character on tivation or repression of many dozens of genes that control\\nthe cell.  cell characteristics.\\n22.2 • Cell-Type Specification in Yeast 13\\n(a) a cell MCM1 was the first member of the MADS familyof\\na1 αsg transcription factors to be discovered. (MADS is an acronym\\nMCM1 for the initial four factors identified in this family.) The\\na1\\nMATa DNA-binding proteins composing this family dimerize and\\n+ + asg\\ncontain a similar N-terminal MADS domain. In Section 22.3\\nwe will encounter other MADS transcription factors that\\nFPO\\n+ + hsg participate in development of skeletal muscle. MADS tran-\\nscription factors also specify cell types in floral organs (see\\nFigure 15-28). MCM1 exhibits different activity in haploid a\\n(b)  α cell\\n+ and \"cells due to its association with \"1 or \"2 protein in \"\\nα2 α1 α1 + cells. As a result of this combinatorial action, MCM1 pro-\\nαsg\\nmotes transcription of \"-specific genes and represses tran-\\nMATα2 MATα1\\nscription of a-specific genes in \"cells. Acting alone, MCM1\\n+ − asg\\nactivates transcription of a-specific genes in a cells and of\\nα2 haploid-specific genes in both \" and a cells (see Figure \\n22-11a, b). Now let’s take a closer look at how MCM1 and\\n+ + hsg\\nthe MAT-encoded proteins exert their effects. \\n(c) a/α cell\\nMCM1 and !1-MCM1 Complexes Activate \\nα2 α2\\n− αsg Gene Transcription\\nMATα2 MATα1 + In acells, homodimeric MCM1 binds to the so-called P box\\n− asg sequence in the upstream regulatory sequences (URSs) of \\nα2 a-specific  genes,  stimulating  their  transcription  (Figure \\na1 a1\\n22-12a). Transcription of \"-specific genes is controlled by\\n− hsg\\ntwo adjacent sequences—the P box and the Q box—located\\nMATa\\nin the URSs associated with these genes. Although MCM1\\nalone binds to the P box in a-specific URSs, it does not bind\\n▲FIGURE 22-11 Transcriptional control of cell type–specific\\nto the P box in \"-specific URSs. Thus a cells do not tran-\\ngenes in S.cerevisiae.The coding sequences carried at the\\nscribe \"-specific genes.\\nMATlocus differ in haploid \" and a cells and in diploid cells.\\nIn \"cells, which produce the \"1 transcription factor en-\\nThree type-specific transcription factors (\"1, \"2, and a1) encoded\\ncoded by MAT\", the simultaneous binding of MCM1 and\\nat the MATlocus act with MCM1, a constitutive transcription\\n\"1 to PQ sites occurs with high affinity (Figure 22-12b). This\\nfactor produced by all three cell types, to produce a distinctive\\npattern of gene expression in each of the three cell types.  binding turns on transcription of \"-specific genes. Therefore\\nasg# a-specific genes/mRNAs; \"sg#\"-specific genes/mRNAs; a-specific transcription is a simple matter of a single tran-\\nhsg# haploid-specific genes/mRNAs. scription factor binding to its target genes, while \"-specific\\n\"FIGURE 22-12 Activity of MCM1 in a\\nand ! yeast cells. MCM1 binds as a dimer\\n(a)  a cells MCM1 dimer\\nto the P site in \"-specific and a-specific\\nupstream regulatory sequences (URSs),\\nwhich control transcription of \"-specific\\nP Q αsg α2 P α2 asg\\ngenes and a-specific genes, respectively. \\nα-specific URS No transcription of a-specific URS Transcription of\\n(a) In a cells, MCM1 stimulates transcription\\nα-specific genes a-specific genes\\nof a-specific genes. MCM1 does not bind\\nFPO efficiently to the P site in \"-specific URSs in\\n(b)  α cells α2 dimer the absence of \"1 protein. (b) In \" cells, the\\nactivity of MCM1 is modified by its\\nassociation with \"1 or \"2. The \"1-MCM1\\ncomplex stimulates transcription of \"-specific\\nα1 genes, whereas the \"2-MCM1 complex\\nP Q αsg α2 P α2 asg blocks transcription of a-specific genes. The\\nα-specific URS Transcription of a-specific URS No transcription of \"2-MCM1 complex also is produced in\\nα-specific genes a-specific genes diploid cells, where it has the same blocking\\neffect on transcription of a-specific genes\\n(see Figure 22-11c).\\n14 CHAPTER 22 • Cell Birth, Lineage, And Death\\ntranscription requires a combination of two factors—neither\\na factor\\nof which can activate target genes alone. α factor\\na α\\n!2-MCM1 and !2-a1 Complexes \\nRepress Transcription\\nFlanking the P box in each a-specific URS are two \"2- α-factor\\nbinding sites. Both MCM1 and \"2 can bind independently receptor a-factor receptor\\nto an a-specific URS with relatively low affinity. However, in\\n\"cells only the highly cooperative, simultaneous binding of\\nPheremone signaling\\nboth \"2 and MCM1 proteins occurs with high affinity. This\\nhigh-affinity binding represses transcription of a-specific\\ngenes, ensuring that they are not expressed in \" cells and\\ndiploid cells (see Figure 22-12b, right). MCM1 promotes a α\\nbinding of \"2 to an a-specific URS by orienting the two DNA-\\nbinding domains of the \"2 dimer to the \"2-binding sequences\\nin this URS. Since a dimeric \"2 molecule binds to both sites in\\nan \"-specific URS, each DNA site is referred to as a half-site. Cell-cycle arrest\\nThe relative positions of both half-sites and their orientation Morphogenesis\\nare highly conserved among different a-specific URSs.\\nThe presence of numerous \"2-binding sites in the genome\\na α\\nand the “relaxed” specificity of \"2 protein may expand the\\nrange of genes that it can regulate. For instance, in a/\" FPO\\ndiploid cells, \"2 forms a heterodimer with a1 that represses\\nboth haploid-specific genes and the gene encoding \"1 (see\\nCell fusion\\nFigure 22-11c). The example of \"2 suggests that relaxed\\nspecificity may be a general strategy for increasing the regu-\\nlatory range of a single transcription factor. Highly specific\\nbinding, then, occurs as a consequence of the interaction of\\n\"2 with other transcription factors at different sites in DNA.\\nNuclear fusion\\nPheromones Induce Mating of ! and a Cells \\nto Generate a Third Cell Type \\na/α\\nDiploid cell\\nAn important feature of the yeast life cycle is the ability of\\nhaploid aand \"cells to mate, that is, attach and fuse giving\\nrise to a diploid a/\"cell (see Figure 1-5). Each haploid cell + Nutrients − Nutrients\\ntype secretes a different mating factor,a small polypeptide\\npheromone,and expresses a cell-surface G protein–coupled Mitotic growth Meiosis\\nreceptor that recognizes the pheromone secreted by cells of\\nthe other type. Thus aand \"cells both secrete and respond to a/α a/α\\npheromones (Figure 22-13). Binding of the mating factors\\nSporulation\\nto their receptors induces expression of a set of genes encod-\\ning proteins that direct arrest of the cell cycle in G and pro- a α\\n1\\nmote attachment/fusion of haploid cells to form diploid cells.\\nIn the presence of sufficient nutrients, the diploid cells will a α\\ncontinue to grow. Starvation, however, induces diploid cells\\nto progress through meiosis, each yielding four haploid\\n▲FIGURE 22-13 Pheromone-induced mating of haploid\\nspores. If the environmental conditions become conducive to\\nyeast cells.The \" cells produce \" mating factor and a receptor;\\nvegetative growth, the spores will germinate and undergo \\nthe a cells produce a factor and \"receptor. Binding of the\\nmitotic division.\\nmating factors to their cognate receptors on cells of the opposite\\nStudies with yeast mutants have provided insights into\\ntype leads to gene activation, resulting in mating and production\\nhow the a and \"pheromones induce mating. For instance, of diploid cells. In the presence of sufficient nutrients, these cells\\nhaploid yeast cells carrying mutations in the sterile 12 will grow as diploids. Without sufficient nutrients, cells will\\n(STE12)locus cannot respond to pheromones and do not undergo meiosis and form four haploid spores.\\n22.3 • Specification and Differentiation of Muscle 15\\nmate. The STE12gene encodes a transcription factor that biologists do not yet know the complete set of molecules that\\nbinds to a DNA sequence referred to as the pheromone- distinguishes any one cell type (e.g., muscle) from all the\\nresponsive element,which is present in many different a- and other cell types in a multicellular organism. The extensive\\n\"-specific URSs. Binding of mating factors to cell-surface re- cell specification and differentiation that occur during \\nceptors induces a cascade of signaling events, resulting in development of animals and plants depend on both quanti-\\nphosphorylation of various proteins including the Ste12 pro- tative and qualitative differences in gene expression, con-\\ntein (see Figure 14-24). This rapid phosphorylation is corre- trolled at the level of transcription, as well as on cell\\nlated with an increase in the ability of Ste12 to stimulate structures and protein activity states. \\ntranscription. It is not yet known, however, whether Ste12 An impressive array of molecular strategies, some analo-\\nmust be phosphorylated to stimulate transcription in re- gous to those found in yeast cell-type specification, have\\nsponse to pheromone.  evolved to carry out the complex developmental pathways\\nInteraction of Ste12 protein with DNA has been studied that characterize multicellular organisms. Muscle cells have\\nmost extensively at the URS controlling transcription of been the focus of many such studies because their develop-\\nSTE2,an a-specific gene encoding the receptor for the \" ment can be studied in cultured cells as well as in intact ani-\\npheromone. Adjacent to this a-specific URS is a pheromone- mals. Early advances in understanding the formation of\\nresponsive element that binds Ste12. When acells are treated muscle cells (myogenesis) came from discovery of regulatory\\nwith \"pheromone, transcription of the STE2gene increases genes that could convert cultured cells into muscle cells.\\nin  a  process  that  requires  Ste12  protein.  Presumably, Then mouse mutations affecting those genes were created\\npheromone-induced production of the \"receptor encoded by and studied to learn the functions of the proteins encoded\\nSTE2increases the efficiency of the mating process. Ste12 by these genes, following which scientists have investigated\\nprotein has been found to bind most efficiently to the how the muscle regulatory genes control other genes.\\npheromone-responsive element in the STE2 URS when DNA microarray analysis has been applied to under-\\nMCM1 is simultaneously bound to the adjacent P site. We stand the function of twistand other regulatory genes in\\nsaw previously that MCM1 can act as an activator or a  muscle development. For example, the expression pattern of\\nrepressor at different URSs depending on whether it com- normal Drosophilaembryos recently has been compared\\nplexes with \"1 or \"2. In this case, the function of MCM1  with that of mutant embryos in which the twistgene is de-\\nas an activator is stimulated by the binding of yet another fective. Previous research had shown that the Twist protein\\ntranscription factor, Ste12, whose activity is modified by  encoded by this gene is a transcription factor necessary to\\nextracellular signals. create muscle cells in flies and other animals. Twist turns\\non production of other transcription factors that in turn ac-\\ntivate genes encoding myosin, actin, and other muscle-\\nKEY CONCEPTS OF SECTION 22.2 specific proteins. To assess how many genes are needed to\\nspecify muscle, researchers determined the expression of\\nCell-Type Specification in Yeast\\nabout 4000 fly genes (about 30 percent of the total) in nor-\\n■ Specification of each of the three yeast cell types—the a mal fly embryos and in twistmutants. Of the genes included\\nand \"haploid cells and the diploid a/\"cells—is determined in the microarray, about 130 (3.3 percent), including many\\nby a unique set of transcription factors acting in different known muscle differentiation genes, were transcribed at\\ncombinations  at  specific  regulatory  sites  in  the  yeast lower levels (or not at all) in the twistmutants. These results\\ngenome (see Figure 22-11). suggest that transcriptional changes in at least several hun-\\n■ Some transcription factors can act as repressors or acti- dred genes are associated with differentiation of a highly\\nvators depending on the specific regulatory sites they bind specialized cell type such as muscle. \\nand the presence or absence of other transcription factors Other recent microarray studies have looked for genes\\nbound to neighboring sites. whose transcription differs in various subtypes of muscle in\\nmice. These have identified 49 genes out of 3000 genes ex-\\n■ Binding of mating-type pheromones by haploid yeast amined that are transcribed at substantially different levels in\\ncells activates expression of genes encoding proteins that\\nred (endurance) muscle and white (fast response) muscle.\\nmediate mating, thereby generating the third yeast cell type\\nClues to the molecular basis of the functional differences be-\\n(see Figure 22-13).\\ntween red and white muscle are likely to come from studying\\nthose 49 genes and their products. \\nHere we examine the role of certain transcription factors\\n22.3 Specification and Differentiation\\nin creating skeletal muscle in vertebrates. These muscle reg-\\nof Muscle  ulators illustrate how coordinated transcription of sets of\\ntarget genes can produce differentiated cell types and how a\\nAs indicated by global expression patterns, yeast cells of dif- cascade of transcriptional events and signals is necessary to\\nferent mating types are still rather similar. Developmental  coordinate cell behaviors and functions. \\n16 CHAPTER 22 • Cell Birth, Lineage, And Death\\nEmbryonic Somites Give Rise to Myoblasts,  siently. These signals trigger production of intracellular fac-\\nthe Precursors of Skeletal Muscle Cells  tors that maintain the myogenic program after the inducing\\nsignals are gone. We discuss the identification and functions\\nVertebrate  skeletal  myogenesis  proceeds  through  three\\nof these myogenic proteins, and their interactions, in the next\\nstages: determination of the precursor muscle cells, called\\nseveral sections.\\nmyoblasts;proliferation and in some cases migration of \\nmyoblasts; and their terminal differentiation into mature\\nmuscle (Figure 22-14). In the first stage, myoblasts arise from Myogenic Genes Were First Identified in Studies\\nblocks of mesoderm cells, called somites,that are located with Cultured Fibroblasts\\nnext to the neural tube in the embryo. Specific signals from\\nIn vitro studies with the fibroblast cell line designated C3H \\nsurrounding tissue play an important role in determining\\n10T1⁄ have played a central role in dissecting the transcription-\\nwhere myoblasts will form in the developing somite. At the 2\\ncontrol mechanisms regulating skeletal myogenesis. When \\nmolecular level, the decision of a mesoderm cell to adopt a\\nthese cells are incubated in the presence of 5-azacytidine, a \\nmuscle cell fate reflects the activation of genes encoding par-\\ncytidine derivative that cannot be methylated and therefore \\nticular transcription factors.\\nalters transcription, they differentiate into myotubes. Upon\\nAs myoblasts proliferate and migrate, say to a developing\\nentry into cells, 5-azacytidine is converted to 5-azadeoxycytidine\\nlimb bud, they become aligned, stop dividing, and fuse to\\ntriphosphate and then is incorporated into DNA in place of\\nform a syncytium(a cell containing many nuclei but sharing\\ndeoxycytidine. Because methylated deoxycytidine residues\\na common cytoplasm). We refer to this multinucleate cell as\\ncommonly are present in transcriptionally inactive DNA re-\\na myotube.Concomitant with cell fusion is a dramatic rise in\\ngions, replacement of cytidine residues with a derivative that\\nthe expression of genes necessary for further muscle devel-\\ncannot be methylated may permit activation of genes previ-\\nopment and function. \\nously repressed by methylation. \\nThe specific extracellular signals that induce determina-\\nThe high frequency at which azacytidine-treated C3H\\ntion of each group of myoblasts are expressed only tran-\\n10T1⁄ cells are converted into myotubes suggested to early\\n2\\nworkers that reactivation of one or a small number of closely\\nlinked genes is sufficient to drive a myogenic program. To test\\n!EXPERIMENTAL FIGURE 22-15 Myogenic genes\\nisolated from azacytidine-treated cells can drive myogenesis\\nMyotome\\nwhen transfected into other cells.(a) When C3H 10T1⁄2 cells \\n(a fibroblast cell line) are treated with azacytidine, they develop\\nSomite\\ninto myotubes at high frequency. To isolate the genes responsible\\nMyoD/Myf5 1 Determination for converting azacytidine-treated cells into myotubes, all the\\nmRNAs from treated cells first were isolated from cell extracts\\nFPO on an oligo-dT column. Because of their poly(A) tails, mRNAs are\\nselectively retained on this column. Steps  1 and  2: The \\nisolated mRNAs were converted to radiolabeled cDNAs. Step  3:\\nMyoblasts When the cDNAs were mixed with mRNAs from untreatedC3H\\nMyogenin and MEFs 2 Proliferation and/or migration 10T1⁄2 cells, only cDNAs derived from mRNAs (light red)\\nproduced by both azacytidine-treated cells and untreated cells\\nhybridized. The resulting double-stranded DNA was separated\\nPremuscle masses\\nfrom the unhybridized cDNAs (dark blue) produced only by\\nazacytidine-treated cells. Step  4: The cDNAs specific for\\n3 Differentiation and cell fusion\\nazacytidine-treated cells then were used as probes to screen a\\ncDNA library from azacytidine-treated cells (Chapter 9). At least\\nsome of the clones identified with these probes correspond to\\ngenes required for myogenesis. (b) Each of the cDNA clones\\nMyotube (muscle cell)\\nidentified in part (a) was incorporated into a plasmid carrying a\\n▲FIGURE 22-14 Three stages in development of  strong promoter. Steps  1 and  2: C3H 10T1⁄2 cells were\\nvertebrate skeletal muscle. Somites are epithelial spheres of cotransfected with each recombinant plasmid plus a second\\nembryonic mesoderm cells, some of which (the myotome) plasmid carrying a gene conferring resistance to an antibiotic\\nbecome determined as myoblasts after receiving signals from called G418; only cells that have incorporated the plasmids will\\nother tissues (1). After the myoblasts proliferate and migrate  grow on a medium containing G418. One of the selected clones,\\nto the limb buds and elsewhere (2), they undergo terminal designated myoD,was shown to drive conversion of C3H 10T1⁄2\\ndifferentiation into multinucleate skeletal muscle cells, called cells into muscle cells, identified by their binding of labeled\\nmyotubes (3). Key transcription factors that help drive the antibodies against myosin, a muscle-specific protein (step  3).\\nmyogenic program are highlighted in yellow. See also Figure 22-17. [See R. L. Davis et al., 1987, Cell51:987.]\\n22.3 • Specification and Differentiation of Muscle 17\\n(a)  Screen for myogenic genes this hypothesis, researchers isolated DNA from C3H 10T1⁄2\\ncells grown in the presence of 5-azacytidine and transfected it\\ninto untreated cells. The observation that 1 in 104cells trans-\\nfected with this DNA was converted into myotubes is consis-\\ntent with the hypothesis that one or a small set of closely linked\\nTotal mRNA from azacytidine-treated cells genes is responsible for converting fibroblasts into myotubes. \\n1 Incubate with reverse Subsequent studies led to the isolation and characteriza-\\ntranscriptase and tion of four genes that can convert C3H 10T1⁄ cells into\\n2\\n[32P]dNTPs\\nmuscle. Figure 22-15 outlines the experimental protocol \\n2 Remove mRNAs for identifying and assaying one of these genes, called the\\nmyogenic determination (myoD)gene. C3H 10T1⁄ cells\\n2\\ntransfected with myoDcDNA and those treated with 5-\\nazacytidine both formed myotubes. The myoDcDNA also\\nwas able to convert a number of other cultured cell lines into\\nmuscle. Based on these findings, the myoDgene was pro-\\n32P-labeled cDNAs\\nposed to play a key role in muscle development. A similar ap-\\n3 Hybridize with proach identified three other genes—myogenin, myf5,and\\nDiscard\\nexcess of mRNAs mrf4—that also function in muscle development. \\nfrom untreated\\nC3H 10T1/2 cells\\n(                 ) Muscle-Regulatory Factors (MRFs) \\nand Myocyte-Enhancing Factors (MEFs) Act \\nin Concert to Confer Myogenic Specificity \\n32P-labeled azamyoblast- The four myogenic proteins—MyoD, Myf5, myogenin, and\\nspecific cDNAs\\nMRF4—are  all  members  of  the  basic  helix-loop-helix\\n(bHLH) family of DNA-binding transcription factors (see \\nScreen cDNA library Figure 11-22b). Near the center of these proteins is a DNA-\\n4\\nfrom treated cells binding basic (B) region adjacent to the HLH domain, which\\nmediates dimer formation. Flanking this central DNA-\\nClones enriched\\nin myogenic genes binding/dimerization region are two activation domains. We\\nrefer to the four myogenic bHLH proteins collectively as\\nFPO muscle regulatory factors,or MRFs(Figure 22-16a). \\n(a) Structure of muscle-regulatory factors (MRFs)\\nDNA binding/dimerization\\n(b)  Assay for myogenic activity of myoD cDNA\\nH2N Transactivation B HLH Transactivation COOH\\nC3H 10T1/2 cell MEF-interacting domain\\nFPO\\n1 Transfect with a plasmid (b) Structure of myocyte-enhancing factors (MEFs)\\ncarrying myoD cDNA and a\\ngene conferring resistance DNA binding/dimerization\\nto G418\\n2 Select on G418-containing H2N MADS MEF Transactivation COOH\\nmedium\\nMyogenin-interacting domain\\n▲FIGURE 22-16 General structures of two classes of\\n3 Stain with transcription factors that participate in myogenesis. MRFs\\nlabeled antimyosin (muscle regulatory factors) are bHLH (basic helix-loop-helix)\\nantibody (   )  proteins produced only in developing muscle. MEFs (myocyte-\\nenhancing factors), which are produced in several tissues in\\naddition to developing muscle, belong to the MADS family. \\nThe myogenic activity of MRFs is enhanced by their interaction\\nwith MEFs.\\n18 CHAPTER 22 • Cell Birth, Lineage, And Death\\nbHLH proteins form homo- and heterodimers that bind scribed at several levels. First, the muscle regulators are ac-\\nto a 6-bp DNA site with the consensus sequence CANNTG tivated only in mesoderm cells, at the right place and time\\n(N = any nucleotide). Referred to as the E box, this sequence in the embryo, in response to spatial regulators of the sorts\\nis present in many different locations within the genome  that are described in Chapter 15. Other proteins mediate ad-\\n(on a purely random basis the E box will be found every 256 ditional mechanisms for assuring tight control over myoge-\\nnucleotides). Thus some mechanism(s) must ensure that nesis: chromatin-remodeling proteins are needed to make\\nMRFs specifically regulate muscle-specific genes and not target genes accessible to MRFs; inhibitory proteins can re-\\nother genes containing E boxes in their transcription- strict when MRFs act; and antagonistic relations between\\ncontrol regions. One clue to how this myogenic specificity is cell-cycle regulators and differentiation factors like MRFs en-\\nachieved was the finding that the DNA-binding affinity of sure that differentiating cells will not divide. All these factors\\nMyoD is tenfold greater when it binds as a heterodimer com- control when and where muscles form.\\nplexed with E2A, another bHLH protein, than when it binds\\nas a homodimer. Moreover, in azacytidine-treated C3H Activating Chromatin-Remodeling Proteins MRF proteins\\n10T1⁄2 cells, MyoD is found as a heterodimer complexed control batteries of muscle-specific genes, but can do so only\\nwith E2A, and both proteins are required for myogenesis in if chromatin factors allow access. Remodeling of chromatin,\\nthese cells. The DNA-binding domains of E2A and MyoD which usually is necessary for gene activation, is carried out\\nhave similar but not identical amino acid sequences, and by large protein complexes (e.g., Swi/Snf complex) that have\\nboth proteins recognize E box sequences. The other MRFs ATPase and perhaps helicase activity. These complexes are\\nalso form heterodimers with E2A that have properties simi- thought to recruit histone acetylases that modify chromatin\\nlar to MyoD-E2A complexes.  to make genes accessible to transcription factors (Chapter\\nSince E2A is expressed in many tissues, the requirement 11). The need for remodeling complexes to help myogenic\\nfor E2A is not sufficient to confer myogenic specificity. Sub- factors was tested using dominant-negative versions of the\\nsequent studies suggested that specific amino acids in the ATPase proteins that form the cores of these complexes. (Re-\\nbHLH domain of all the MRFs may confer myogenic speci- call from Chapter 9 that a dominant-negative mutation pro-\\nficity by allowing MRF-E2A complexes to bind specifically duces a mutant phenotype even when a normal allele of the\\nto another family of DNA-binding proteins called myocyte- gene also is present.) When genes carrying these dominant-\\nenhancing factors,or MEFs.MEFs were considered excellent negative mutations were transfected into C3H 10T1⁄ cells,\\n2\\ncandidates for interaction with MRFs for two reasons. First, the subsequent introduction of myogenic genes no longer\\nmany muscle-specific genes contain recognition sites for both converted the cells into myotubes. In addition, a muscle-\\nMEFs and MRFs. Second, although MEFs cannot induce specific gene that is normally activated did not exhibit its\\nmyogenic conversion of azacytidine-treated C3H 10T1⁄2 usual pattern of chromatin changes in the doubly transfected\\ncells, they enhance the ability of MRFs to do so. This en- C3H 10T1⁄ cells. These results indicate that transcription ac-\\n2\\nhancement requires physical interaction between a MEF and tivation by myogenic proteins depends on a suitable chro-\\nMRF-E2A heterodimer. Later, MEFs were found to belong to matin structure in the regions of muscle-specific genes.\\nthe MADS family of transcription factors and to contain a\\nMEF domain, adjacent to the MADS domain, that mediates Inhibitory Proteins Screens for genes related to myoDled to\\ninteraction with myogenin (Figure 22-16b). The synergistic identification of a related protein that retains the dimeriza-\\naction of the MEF homodimer and MRF-E2A heterodimer is tion region but lacks the basic DNA-binding region and\\nthought to drive high-level expression of muscle-specific hence is unable to bind to E box sequences in DNA. By bind-\\ngenes.  ing to MyoD or E2A, this protein inhibits formation of\\nKnockout mice and Drosophilamutants have been used MyoD-E2A heterodimers and hence their high-affinity bind-\\nto explore the roles of MRF and MEF proteins in conferring ing to DNA. Accordingly, this protein is referred to as Id, for\\nmyogenic specificity in intact animals rather than cell cul- inhibitor of DNA binding. Id prevents cells that produce\\nture. These experiments demonstrated the importance of MyoD and E2A from activating transcription of the muscle-\\nthree of the MRF proteins and of MEF proteins for distinct specific gene encoding creatine kinase. This finding pre-\\nsteps in muscle development (see Figure 22-14). The function sumably reflects the formation of MyoD-Id or E2A-Id \\nof the fourth myogenic protein, Mrf4, is not entirely clear; complexes, which cannot bind to and activate transcription\\nit may be expressed later and help maintain differentiated of muscle-specific genes. As a result, the cells remain in a\\nmuscle cells.  proliferative growth state. When these cells are induced to\\ndifferentiate into muscle (for instance, by the removal of\\nserum, which contains the growth factors required for pro-\\nTerminal Differentiation of Myoblasts Is Under\\nliferative growth), the Id concentration falls. MyoD-E2A\\nPositive and Negative Control \\ndimers now can form and bind to the regulatory regions of\\nPowerful developmental regulators like the MRFs cannot be target genes, driving differentiation of C3H 10T1⁄2cells into\\nallowed to run rampant. In fact, their actions are circum- myoblast-like cells. \\n22.3 • Specification and Differentiation of Muscle 19\\nRecent work shows that histone acetylases and deacety- Signals Are Crucial for Muscle Cell-Fate\\nlases are crucial for regulating muscle-specific genes. As ex- Determination and Myoblast Migration \\nplained in Chapter 11, acetylation of histones in chromatin \\nAs noted already, after myoblasts arise from somites, they\\nis necessary to activate many genes; in contrast, histone\\nmust not only proliferate but also move to their proper lo-\\ndeacetylases  cause  transcriptional  repression  (see  Figure \\ncations and form the correct attachments as they differ-\\n11-32). MEF2 recruits histone acetylases such as p300/CBP,\\nentiate into muscle cells (Figure 22-17). Myogenic gene \\nthrough another protein that serves as a mediator, to activate\\nexpression often follows elaborate organizational events that\\ntranscription of target genes. Chromatin immunoprecipitation\\ntell certain somite cells to delaminate from the somite ep-\\nexperiments with antibodies against acetylated histone H4\\nithelium and which way to move. A transcription factor,\\nshow that the acetylated histone level associated with MEF2-\\nPax3, is produced in the subset of somite cells that will form\\nregulated genes is higher in differentiated myotubes than in\\nmuscle and appears to be at the top of the regulatory hierar-\\nmyoblasts (see Figure 11-31). The role of histone deacetylases\\nchy controlling muscle formation in the body wall and limbs.\\nin muscle development was revealed in experiments in which\\nMyoblasts that will migrate, but not cells that remain be-\\nscientists first introduced extra myoDgenes into cultured C3H\\nhind, also produce a transcription factor called Lbx1. If Pax3\\n10T1⁄ cells to raise the level of MyoD. This resulted in in-\\n2\\nis not functional, Lbx1 transcripts are not seen and myo-\\ncreased activation of target genes and more rapid differentia-\\nblasts do not migrate. Both Pax3 and Lbx1 can affect ex-\\ntion of the cells into myotubes. However, when genes encoding\\npression of myoD.\\nhistone deacetylases also were introduced into the C3H 10T1⁄\\n2\\nDeparture of myoblasts from somites depends upon a \\ncells, the muscle-inducing effect of MyoD was blocked and the\\nsecreted protein signal appropriately called scatter factor,\\ncells did not differentiate into myotubes. \\nor hepatocyte growth factor (SF/HGF).This signal is pro-\\nThe explanation for how histone deacetylases inhibit\\nduced by embryonic connective tissue cells (mesenchyme) in\\nMyoD-induced muscle differentiation came from the sur-\\nthe limb buds to which myoblasts migrate. Production of\\nprising finding that MEF2 can bind, through its MADS do-\\nSF/HGF is previously induced by other external signals such\\nmain, to a histone deacetylase. This interaction, which can\\nas fibroblast growth factor and Sonic hedgehog, which are\\nprevent MEF2 function and muscle differentiation, is nor-\\ncritical to limb development (Chapter 15). The cell-surface\\nmally blocked during differentiation because the histone\\nreceptor for SF/HGF, which is expressed by myoblasts, be-\\ndeacetylase is phosphorylated by a calcium/calmodulin-\\nlongs to the receptor tyrosine kinase (RTK) class of receptors\\ndependent protein kinase; the phosphorylated deacetylase\\n(Chapter 14). Cells migrate from the somites at the head-to-\\nthen is moved from the nucleus to the cytoplasm. Taken to-\\ntail body levels where limbs will form, and not elsewhere,\\ngether, these results indicate that activation of muscle genes\\nby MyoD and MEF2 is in competition with inactivation of\\nmuscle genes by repressive chromatin structures and that nu-\\nclear versus cytoplasmic localization of chromatin factors is Neural tube\\na key regulatory step. Dermomyotome\\n(gives rise to\\ndermis of skin)\\nCell-Cycle Proteins The onset of terminal differentiation in Epidermis\\nmany cell types is associated with arrest of the cell cycle, Myoblasts\\nmost commonly in G , suggesting that the transition from (migrate to FPO\\n1\\nlimb bud to\\nthe determined to differentiated state may be influenced by\\nform skeletal\\ncell-cycle proteins including cyclins and cyclin-dependent  muscle)\\nkinases(Chapter 21). For instance, certain inhibitors of cyclin-\\ndependent kinases can induce muscle differentiation in cell\\nculture, and the amounts of these inhibitors are markedly\\nNotochord\\nhigher in differentiating muscle cells than in nondifferentiat-\\nSclerotome \\ning ones in vivo. Conversely, differentiation of cultured  (gives rise to skeletal structures \\nmyoblasts can be inhibited by transfecting the cells with such as vertebrae)\\nDNA encoding cyclin D1 under the control of a constitutively\\n▲FIGURE 22-17 Embryonic determination and migration\\nactive promoter. Expression of cyclin D1, which normally\\nof myoblasts in mammals. After formation of the neural tube,\\noccurs only during G , is induced by mitogenic factors in\\n1 each somite forms sclerotome, which develops into skeletal\\nmany cell types and drives the cell cycle (see Figure 21-28).\\nstructures, and dermomyotome. The dermomyotome gives rise to\\nThe ability of cyclin D1 to prevent myoblast differentiation\\nthe dermis of the skin and to the muscles. Lateral myoblasts\\nin vitro may mimic aspects of the in vivo signals that antag-\\nmigrate to the limb bud; medial myoblasts develop into the trunk\\nonize the differentiation pathway. The antagonism between muscles. The remainder of a dermomyotome gives rise to the\\nnegative and positive regulators of G1progression is likely to connective tissue of the skin. [Adapted from M. Buckingham, 1992,\\nplay an important role in controlling myogenesis in vivo. Trends Genet.8:144.]\\n20 CHAPTER 22 • Cell Birth, Lineage, And Death\\ndue to the presence of SF/HGF at limb locations and not teins, designated Da and Emc, are analogous in structure and\\nelsewhere. If the SF/HGF signal or its receptor is not func- function to vertebrate E2A and Id, respectively. For example,\\ntional, somite cells will produce Lbx1 but not go on to mi- heterodimeric complexes of Da with Ac or Sc bind to DNA\\ngrate; thus no muscles will form in the limbs. Expression of better than the homodimeric forms of Ac and Sc. Emc, like\\nthe myogeningene, which is necessary for myotube forma- Id, lacks a DNA-binding basic domain; it binds to Ac and\\ntion, and of mrf4, which is necessary for muscle fiber differ- Sc proteins, thus inhibiting their association with Da and\\nentiation, does not begin until migrating myoblasts approach binding to DNA. The similar functions of these myogenic\\ntheir limb-bud destination; these steps in terminal muscle dif- and neurogenic proteins are depicted in Figure 22-18. \\nferentiation are presumably not compatible with migration.  A family of bHLH proteins related to the Drosophila\\nWe have touched on just a few of the many external sig- Achaete and Scute proteins has been identified in vertebrates.\\nnals and transcription factors that participate in develop- One of these, called neurogenin,which has been identified in\\nment of a properly patterned muscle. The function of all the rat, mouse, and frog, appears to control the formation of\\nthese regulatory molecules must be coordinated both in neuroblasts. In situ hybridization experiments showed that\\nspace and in time during myogenesis.  neurogenin is produced at an early stage in the developing nerv-\\nous system and may induce production of NeuroD, another\\nbHLH protein that acts later (Figure 22-19a). Injection of large\\nbHLH Regulatory Proteins Function in Creation \\namounts of neurogeninmRNA into Xenopusembryos further\\nof Other Tissues\\ndemonstrated the ability of neurogenin to induce neurogenesis\\nFour bHLH transcription factors that are remarkably similar (Figure 22-19b). These studies suggest that the function of neu-\\nto the myogenic bHLH proteins control neurogenesis in rogenin is analogous to that of the Achaete and Scute in\\nDrosophila. Similar proteins appear to function in neurogen- Drosophila;likewise, NeuroD and Asense may have analogous\\nesis in vertebrates and perhaps in the determination and dif- functions in vertebrates and Drosophila,respectively.\\nferentiation of hematopoietic cells.  In addition to neurons, the nervous system contains a\\nThe neurogenic DrosophilabHLH proteins are encoded large number of glial cells, which also arise from the neuro-\\nby an ≈100-kb stretch of genomic DNA, termed the achaete- ectoderm. Glial cells support and insulate neurons; they also\\nscute complex (AS-C), containing four genes designated provide guidance and contact surfaces for migrating neurons\\nachaete (ac),scute (sc),lethal of scute (l’sc),and asense (a). during development and send a signal to neurons that pro-\\nAnalysis of the effects of loss-of-function mutations indicate motes formation of synapses. Neurogenins control the fates\\nthat the Achaete (Ac) and Scute (Sc) proteins participate in of precursor cells that are capable of making either neurons\\ndetermination of neuronal precursor cells, called neuroblasts, or glial cells by promoting neural development and repress-\\nwhile the Asense (As) protein is required for differentiation ing glial development. Neurogenins are therefore switches\\nof these cells into neurons. These functions are analogous to that control the decision between two alternative cell fates,\\nthe roles of MyoD and Myf5 in muscle determination and just as the yeast mating-type regulators select among three\\nof myogenin in differentiation. Two other Drosophilapro- cell types. \\nFly neurogenesis\\nac, sc mRNA Da(general) as mRNA\\n+\\nachaete asense\\nAc, Sc Asense Neurons\\nscute (others)\\n−\\nProneural Neural\\ngenes Emc precursor\\nFPgenesO\\n!FIGURE 22-18 Comparison of genes\\nthat regulate Drosophilaneurogenesis\\nVertebrate myogenesis\\nand mammalian myogenesis. bHLH\\nmyf5, myoD \\ntranscription factors have analogous mRNA myogenin mRNA\\nE2A(general)\\nfunctions in determination of precursor\\n+\\ncells (i.e., neuroblasts and myoblasts) and myf5 myogenin\\ntheir subsequent differentiation into mature myoD Myf5, MyoD mrf4 (?) Myogenin Muscle\\nneurons and muscle cells. In both cases, (others) (others)\\n−\\nthe proteins encoded by the earliest-acting Promuscular Muscle precursor\\nId\\ngenes (left) are under both positive and genes genes\\nnegative control by other related proteins\\n(blue type). [Adapted from Y. N. Jan and L. Y. Precursor determination Differentiation\\nJan, 1993, Cell75:827.]\\n22.4 • Regulation of Asymmetric Cell Division 21\\n(b)\\n(a)\\nFPO\\nneurogenin mRNA  neuroD mRNA  β-tubulin mRNA  neurogenin mRNA \\n▲EXPERIMENTAL FIGURE 22-19 In situ hybridization  ventricular zone (other arrow). (b) One of the two cells in early\\nand injection experiments demonstrate that neurogenin acts Xenopusembryos was injected with neurogeninmRNA (inj) and\\nbefore NeuroD in vertebrate neurogenesis.(a) Sections of rat then stained with a probe specific for neuron-specific mRNAs\\nneural tube were treated with a probe specific for neurogenin encoding !-tubulin (left) or NeuroD (right).The region of the\\nmRNA (left) or neuroDmRNA (right).The open space in the embryo derived from the uninjected cell served as a control\\ncenter is the ventricle, and the cells lining this cavity are in the (con). The neurogeninmRNA induced a massive increase in the\\nventricular layer. All the neural cells are born in the ventricular number of neuroblasts expressing neuroDmRNA and neurons\\nlayer and then migrate outward. As illustrated in these expressing !-tubulinmRNA in the region of the neural tube\\nmicrographs, neurogeninmRNA is produced in proliferating derived from the injected cell. [From Q. Ma et al., 1996, Cell87:43;\\nneuroblasts in the ventricular layer (arrow), whereas neuroD courtesy of D. J. Anderson.]\\nmRNA is present in migrating neuroblasts that have left the\\n■ Migration of myoblasts to the limb buds is induced by\\nKEY CONCEPTS OF SECTION 22.3 scatter factor/hepatocyte growth factor (SF/HGF), a protein\\nsignal secreted by mesenchymal cells (see Figure 22-17).\\nSpecification and Differentiation of Muscle\\nMyoblasts must express both the Pax3 and Lbx1 tran-\\n■ Development of skeletal muscle begins with the signal- scription factors to migrate. \\ninduced determination of certain mesoderm cells in somites\\nas myoblasts. Following their proliferation and migration, ■ Terminal differentiation of myoblasts and induction of\\nmuscle-specific proteins do not occur until myoblasts stop\\nmyoblasts stop dividing and differentiate into multinucle-\\ndividing and begin migrating. \\nate muscle cells (myotubes) that express muscle-specific\\nproteins (see Figure 22-14). ■ Neurogenesis in Drosophiladepends upon a set of four neu-\\nrogenic bHLH proteins that are conceptually and structurally\\n■ Four  myogenic  bHLH  transcription  factors—MyoD, similar to the vertebrate myogenic proteins (see Figure 22-18). \\nmyogenin, Myf5, and MRF4, called muscle-regulatory fac-\\ntors (MRFs)—associate with E2A and MEFs to form large ■ A related vertebrate protein, neurogenin, is required for\\nformation of neural precursors and also controls their di-\\ntranscriptional complexes that drive myogenesis and ex-\\nvision into neurons or glial cells. \\npression of muscle-specific genes. \\n■ Dimerization of bHLH transcription factors with dif- 22.4 Regulation of Asymmetric \\nferent partners modulates the specificity or affinity of their\\nbinding to specific DNA regulatory sites, and also may pre- Cell Division \\nvent their binding entirely.\\nDuring embryogenesis,the earliest stage in animal develop-\\n■ The myogenic program driven by MRFs depends on the ment, asymmetric cell division often creates the initial diver-\\nSwi/Snf chromatin-remodeling complex, which makes tar-\\nsity that ultimately culminates in formation of specific\\nget genes accessible. \\ndifferentiated cell types. Even in bacteria, cell division may\\n■ The myogenic program is inhibited by binding of Id pro- yield unequal daughter cells, for example, one that remains\\ntein to MyoD, thereby blocking binding of MyoD to DNA, attached to a stalk and one that develops flagella used for\\nand by histone deacetylases, which repress activation of swimming. Essential to asymmetric cell division is polariza-\\ntarget genes by MRFs. tionof the parental cell and then differential incorporation \\n22 CHAPTER 22 • Cell Birth, Lineage, And Death\\nCytoplasm Interestingly, some haploid yeast cells can switch repeat-\\nProtein\\nor mRNA edly between the \" and a types. Mating-type switching\\noccurs when the \"allele occupying the MATlocus is replaced\\nby the aallele, or vice versa. The first step in this process is\\ncatalyzed by HO endonuclease, which is expressed in mother\\nPolarized parental cell cells but not in daughter cells. Thus mating-type switch-\\ning occurs only in mother cells (Figure 22-21). Transcription\\nFPO of the HOgene is dependent on the Swi/Snf chromatin-\\nremodeling complex(see Figure 11-37), the same complex\\nthat we encountered earlier in our discussion of myogenesis.\\nDaughter yeast cells arising by budding from mother cells\\ncontain a protein called Ash1 that prevents recruitment of\\nMitotic the Swi/Snf complex to the HOgene, thereby preventing its\\nspindle\\ntranscription. The absence of Ash1 from mother cells allows\\nthem to transcribe the HOgene. \\nRecent experiments have revealed how the asymmetry in\\nthe distribution of Ash1 between mother and daughter cells\\nis established. ASH1mRNA accumulates in the growing bud\\nAsymmetric daughter cells that will form a daughter cell due to the action of a myosin\\nmotor protein (Chapter 19). This motor protein, called\\n▲FIGURE 22-20 General features of asymmetric cell Myo4p, moves the ASH1mRNA, as a ribonucleoprotein\\ndivision.Various mechanisms can lead to asymmetric distribution\\ncomplex, along actin filaments in one direction only, toward\\nof cytoplasmic components, such as particular proteins or\\nthe bud (Figure 22-22). Two connector proteins tether ASH1\\nmRNAs (red dots) to form a polarized parental cell. Division of a\\nmRNA to the motor protein. By the time the bud separates\\npolarized cell will be asymmetric if the mitotic spindle is oriented\\nfrom the mother cell, the mother cell is largely depleted of\\nso that the localized cytoplasmic components are distributed\\nASH1mRNA and thus can switch mating type in the fol-\\nunequally to the two daughter cells, as shown here. However, if\\nlowing G before additional ASH1mRNA is produced and\\nthe spindle is positioned differently relative to the localized 1\\nbefore DNA replication in the S phase.\\ncytoplasmic components, division of a polarized cell may produce\\nequivalent daughter cells. \\nBud\\n\"\\nof parts of the parental cell into the two daughters (Figure\\n22-20). A variety of molecular mechanisms are employed to Division\\ncreate and propagate the initial asymmetry that polarizes the\\nparental cell. In addition to being different, the daughter cells \" \"\\nmust often be placed in a specific orientation with respect to\\n− Ash1 M D + Ash1\\nsurrounding structures. HO transcriptiFon      PO     No HO transcription\\nWe begin with an especially well-understood example of Switching                   No switching\\nasymmetric cell division, the budding of yeast cells, and\\nmove on to recently discovered protein complexes important a a \" \"\\nfor asymmetric cell divisions in multicellular organisms. We\\nM D M D\\nsee in this example an elegant system that links asymmetric\\ndivision to the process of controlling cell type. \\n\" \" a a\\nM D M D\\nYeast Mating-Type Switching Depends \\nupon Asymmetric Cell Division ▲FIGURE 22-21 Specificity of mating-type switching in\\nhaploid yeast cells. Division by budding forms a larger mother\\nWhether a haploid yeast cell exhibits the \"or amating type\\ncell (M) and smaller daughter cell (D), both of which have the\\nis determined by which genes are present at the MATlocus\\nsame mating type as the original cell (\" in this example). The\\n(see Figure 22-11). As described in Chapter 11, the MAT\\nmother can switch mating type during G of the next cell cycle\\nlocus in the S. cerevisiaegenome is flanked by two “silent,” 1\\nand then divide again, producing two cells of the opposite a\\ntranscriptionally inactive loci containing the alternative \"or\\ntype. Switching depends on transcription of the HOgene, which\\na sequences (see Figure 11-28). A specific DNA rearrange-\\noccurs only in the absence of Ash1 protein. The smaller daughter\\nment brings the genes that encode the \"-specific or a-specific cells, which produce Ash1 protein, cannot switch; after growing\\ntranscription factors from these silent loci to the active MAT in size through interphase, they divide to form a mother cell and\\nlocus where they can be transcribed. daughter cell. Orange cells and arrows indicate switch events.\\n22.4 • Regulation of Asymmetric Cell Division 23\\n!FIGURE 22-22 Model for restriction of mating-\\ntype switching to mother cells in S.cerevisiae. Ash1\\nprotein prevents a cell from transcribing the HOgene AV M\\nwhose encoded protein initiates the DNA rearrangement ASH1 SHide E\\nttsohe apata .r rSeatsweuisltt csfhr oiinnmg m  oaa ctncineugwrs-tl yyo pnbeluy ds idwne ittdhc hedi anmuggo fhtrhtoeemrr   ccaeelltllo,,   ba\"fetceoarr  ui\"tse mRNA Bud 1 mRNo Anim DIA \\nAsh1 protein is present in the daughter cell but not in She3p Aa C\\nltohcea lmizaottihoenr  ocfe lAl.s Thh1e i sm tohlee counlaer- wbaasyi str afonrs pthoirst  doiff fAerSeHn1tial  Location:  ON\\nmRNA into the bud. A linking protein, She2p, binds  She2Fp POMyo4p Actin liz N\\na E\\nto specific 3% untranslated sequences in the ASH1 t\\nio C\\nmRNA and also binds to She3p protein. This protein in n T\\nturn binds to a myosin motor, Myo4p, which moves I\\nO\\nalong actin filaments into the bud. [See S. Koon and \\nN\\nB. J. Schnapp, 2001, Curr. Biology11:R166–R168.]\\nS\\nBudding yeasts use a relatively simple mechanism to  dermal fate (see Figures 14-29 and 15-36). The delaminat-\\ncreate molecular differences between the two cells formed by ing cells move inside and become spherical neuroblasts,\\ndivision. In higher organisms, polarization of the parental while the prospective epidermal cells remain behind and\\ncell involves many more participants, and, in addition, the close up to form a tight sheet.\\nmitotic spindle must be oriented in such a way that each Once formed, the neuroblasts undergo asymmetric divi-\\ndaughter cell receives its own set of cytoplasmic components. sions, at each division recreating themselves and producing a\\nTo illustrate these complexities, we focus on asymmetric  ganglion mother cell (GMC)at the basal side of the neuro-\\ndivision of neuroblasts in Drosophila. Genetic studies in  blast (Figure 22-23). A single neuroblast will produce several\\nC. elegansand Drosophilahave revealed the key participants, GMCs; each GMC in turn forms two neurons. Depending \\na first step in understanding at the molecular level how asym- on where they form in the embryo and consequent regula-\\nmetric cell division is regulated in multicellular organisms. tory events, neuroblasts may form more or fewer GMCs.\\nLarge Protein Complexes Are Localized \\nat Opposite Ends of Dividing Neuroblasts  1 Ectodermal\\ncells\\nin Drosophila\\nDV M\\nFly neuroblasts arise from a sheet of ectoderm cells that is 2 roside E\\noo D\\none cell thick. As in vertebrates, the Drosophilaectoderm ph A I\\nfeoctromdse rbmo tche llesp hidaevrem thise  panodte nthtiea ln teor avsosuusm sey esittehmer,  aa nndeu mraal noyr  3 ilanim A \\nEa C\\neopnildye rinm acel rfatatein.  Ucneldlse,r  sthoem ceo notfr tohl eo fc eglelns eisn tchraeta sbee cionm siez aec atinvde Neuroblast mbrtion ON\\nbegin to loosen from the ectodermal layer. At this point, the 4 yo: E N\\nFPO c\\ndelaminating cells use Notch signaling to mediate lateral  to E\\ninhibitionof their neighbors, causing them to retain the epi- GMC de C\\nr T\\nm\\nI\\n5 al C ON\\n!FIGURE 22-23 Asymmetric cell division during Drosophila GMC ell  S\\nneurogenesis.The ectodermal sheet (1) of the early embryo gives rise to both D\\nepidermal cells and neural cells. Neuroblasts, the stem cells for the nervous iv\\nis\\nsystem, are formed when ectoderm cells enlarge, separate from the ectodermal 6 io\\nepithelium, and move into the interior of the embryo (2–4). Each neuroblast n\\ns\\nthat arises divides asymmetrically to recreate itself and produce a ganglion Neurons  in\\nmother cell (GMC) (5). Subsequent divisions of a neuroblast produce more  t\\nGMC h\\nGMCs, creating a stack of these precursor cells (6). Each GMC divides once  e\\n \\nto give rise to two neurons (7). Neuroblasts and their GMC descendants can\\n7\\nhave different fates depending on their location.\\n24 CHAPTER 22 • Cell Birth, Lineage, And Death\\nNeuroblasts and GMCs in different locations exhibit differ- in the process of division. This complex is already localized\\nent patterns of gene expression, an indicator of their fates. in ectoderm cells and stays at the apical end as a prospective\\nAnalysis of fly mutants led to the discovery of key proteins neuroblast delaminates from the ectodermal sheet (Figure\\nthat participate in creating specific neuroblasts, inhibiting 22-24a, left). Three proteins compose the apical complex:\\nepidermal cells from becoming neuroblasts, and directing Baz, Par6, and PKC3, the last being an isoform of protein \\nneuroblasts to divide asymmetrically. kinase C. The Baz/Par6/PKC3 complex persists at the apical\\nend from late interphase (G stage) until late anaphase in mi-\\n2\\nApical Baz/Par6/PKC3 Complex A ternary complex of pro- tosis and then disperses or is destroyed (see Figure 20-29 for\\nteins, referred to as the apical complex,is located at the end mitotic stages). After each neuroblast division, the complex\\nof a fly neuroblast that will become the “new” neuroblast re-forms at the apical end of the progeny neuroblast in the\\nnext interphase. It remains unclear how polarity information\\nis preserved through telophase, since no known proteins are\\nlocalized during that time. As noted previously, the very first\\nApical (Baz, \\nProtein complexes: Miranda Numb\\nPar6, PKC3) C. eleganscell division after fertilization is asymmetric, form-\\ning an AB cell and P1 cell, which have quite different fates\\n(see Figure 22-9). Remarkably, a protein complex like the\\nfly Baz/Par6/PKC3 complex controls the asymmetry of the\\n(a) Neuroblast P1-AB division; similar protein complexes exist in mammals\\nas well.\\nFPO Two mechanisms control the apical localization of the\\nDrosophilaBaz/Par6/PKC3 complex: the first operates in the\\nGMC\\nectoderm prior to neuroblast delamination; the second is ac-\\ntive during the repeating divisions of delaminated neuro-\\nINTERPHASE ANAPHASE blasts. The first mechanism involves another multiprotein\\nNEUROBLAST NEUROBLAST complex composed of at least three proteins, Scribble (Scrib),\\nCentromere Discs-large (Dlg), and Lethal giant larvae (Lgl). This complex\\nis located in the cortical region of ectoderm cells (i.e., the re-\\ngion just below the plasma membrane), but it is not polar-\\n(b) ized along the apical-basal axis of these cells. Thus, though\\nNeuroblast\\nnecessary for the apical localization of the Baz/Par6/PKC3\\ncomplex, the Scrib/Dlg/Lgl complex probably is not suffi-\\ncient. Once a neuroblast has separated from the ectoderm,\\nGMC two other proteins called Inscuteable and Partner of Inscute-\\nCytoskeleton: able join the Baz/Par6/PKC3 complex, forming a five-protein\\nActin microfilaments complex. These two additional proteins probably stabilize\\nChromosomes\\nMicrotubules the Baz/Par6/PKC3 complex. Reassembly of the entire com-\\nplex at the apical end following each neuroblast division re-\\n▲FIGURE 22-24 Localized protein complexes that control quires microtubules and perhaps microfilaments (Figure\\nasymmetric cell division in the Drosophilaneuroblast.(a) The 22-24b). This finding suggests that motor protein–mediated\\napical complex (blue), which consists of three proteins (Baz, Par6, transport of these protein complexes along cytoskeletal ele-\\nand PKC3), is localized in ectoderm cells and in delaminating ments plays a role in their apical localization. \\nneuroblasts. As a neuroblast begins dividing, the Miranda\\ncomplex (orange) assembles at the apical side and then moves Basal Complexes In a dividing neuroblast two basal com-\\nto the basal side, where it will be incorporated into the ganglion plexesare found at the end that will form a ganglion mother\\nmother cell (GMC). The second basal complex (green), containing cell (GMC) at division (see Figure 22-24a, right). Localiza-\\nthe Numb protein, is detected only basally. Mutations in genes tion of these complexes, called the Miranda complex and the\\nthat encode polarized proteins disrupt asymmetric cell division.\\nNumb complex, at the basal end of neuroblasts requires the\\n(b) Motor protein–mediated transport along cytoskeletal filaments\\napical Baz/Par6/PKC3 complex. The Numb complex is lo-\\nis thought to localize the apical and basal complexes. Actin\\ncated throughout the cell during interphase; at prophase it\\nmicrofilaments (red) lie just under the cell surface at all times.\\nbecomes basally located. In contrast, the Miranda complex\\nMicrotubules (blue) radiate from the centrosome during\\ninitially assembles at the apical end; it then moves to the\\ninterphase and then assemble into the mitotic spindle, attached\\nbasal end at prophase. The apical Baz/Par6/PKC3 complex\\nto the duplicated centrosomes, during cell division. Note that in\\nthe neuroblast-GMC division the centrosome is skewed to the forms even when either of the basal complexes cannot as-\\nneuroblast end of the cell, and the chromosomes (light gray) are semble due to mutations. Mutations in any of the compo-\\ndiagrammed lined up on the spindle. [Adapted from C. Q. Doe and nents of the apical complex, however, prevent the Miranda\\nB. Bowerman, 2001, Curr. Opin. Cell Biol.13:68.] and Numb complexes from moving to the basal region. \\n22.4 • Regulation of Asymmetric Cell Division 25\\nInstead they accumulate in the cortical region or in clumps blasts, the mitotic spindle first aligns perpendicular to the\\nthroughout the cell. The simplest idea, that the basal com- apical-basal axis and then turns 90 degrees to align with it\\nplexes form wherever the apical complex is absent, is prob- at the same time that the basal complexes become localized\\nably wrong, since parts of the cell contain neither of these to the basal side (Figure 22-25). The apical complex, which\\ncomplexes.  is already in place before spindle rotation, controls the final\\nThe basal Miranda complex has four components: Pros- orientation of the spindle. This is supported by the finding\\npero protein, a presumed transcription factor that contains that mutations in any of the components of the apical com-\\na DNA-binding homeodomain; Miranda protein, which has plex eliminate the coordination of the spindle with apical-\\na coiled-coil structure; Staufen, an RNA-binding protein; basal polarity, causing the spindle orientation to become\\nand prosperomRNA. During each round of neuroblast di- random. \\nvision, new Prospero protein is made during interphase and Spindle orientation is linked to actin and myosin-related\\nincorporated into the Miranda basal complex, which is dis- proteins. Mammalian Par6, a component of the apical com-\\ntributed to the GMC at the time of cleavage. In the absence plex, can bind two small GTPase proteins, Cdc42 and\\nof Prospero, GMCs do not transcribe the appropriate genes Rac1, that control the arrangement of actin microfilaments\\nand do not develop into normal neurons. The participation in the cytoskeleton (see Figure 19-29). The protein Lethal\\nof a transcription factor in a localized cytoplasmic complex giant larvae (Lgl), a component of the Scrib/Dlg/Lgl asym-\\nprovides a link between the asymmetry of the cytoplasm and metry complex, helps to localize the Miranda complex\\nsubsequent cell-fate determination. Since the apical and basal basally in fly neuroblasts, although Lgl itself is localized\\ncomplexes are presumably localized by the cytoskeleton in apically. Lgl binds to myosin II, which functions in cyto-\\nconcert with motor proteins, the involvement of a transcrip- kinesis (see Figure 19-20), suggesting that the location of\\ntion factor in one of the complexes potentially explains how the cleavage furrow may be coordinated with localization\\na transcription factor is preferentially distributed to one of of the apical complex. \\ntwo daughter cells. The involvement of an mRNA and a Two yeast proteins related to Lgl also bind to myosin II;\\nRNA-binding protein in the Miranda complex suggests that they have been implicated in exocytosis and secretion, specif-\\nit also mediates translational control in GMCs.  ically in the docking of post-Golgi vesicles with the plasma\\nmembrane (Chapter 17). Mutations in the two yeast proteins\\nsuppress mutations in the myosin IIgene. This observation is\\nOrientation of the Mitotic Spindle Is Linked \\ninterpreted to mean that the function of myosin II in spindle\\nto Cytoplasmic Cell-Asymmetry Factors\\norientation is opposite to that of the Lgl-like proteins: re-\\nFor localized protein complexes to be differentially incorpo- duction of one protein’s function is ameliorated by reduction\\nrated into two daughter cells requires that the plane of cell of the other, restoring a semblance of the normal balance. \\ndivision be appropriately oriented. In dividing fly neuro- In this case, therefore, myosin and Lgl probably act in \\nFPO inV M\\n Dide E\\nroo D\\nso A I\\npn A\\n0 s 32 s 64 s 80 s hilaima  C\\nt\\nNio O\\nen N\\nu: \\nrM N\\no\\nbit E\\n▲imEagXiPngE RreIMveEaNlsT rAoLta tFioIGn UoRf Eth 2e 2m-2it5otiTci mspei-nladples ein fluorescence fauren cvtiisoinb laes o tnh eo psppoinsditlee  spiodleess ;o af st hme itcoeslils.  Tphreoscee eds the last Cell otic Spin CTIO\\nDd N\\nasymmetrically dividing neuroblasts. Early Drosophilaembryos microtubules forming the mitotic spindle are assembled ivle S\\nwere injected with a hybrid gene composed of the gene from the poles (see Figure 20-29). In successive  is A\\nencoding green fluorescent protein (GFP) fused to the gene images (at 32, 64, and 80 seconds), the bipolar spindle ionssy\\nencoding Tau, a protein that binds to microtubules. At the top  can be seem to form and rotate 90 degrees to align  m\\ne\\nare time-lapse images of a single dividing neuroblast in a live with the apical-basal axis, as schematically depicted at t\\nr\\nembryo. The basal side is at the top, and the apical side at the the bottom. [From J. A. Kaltschmidt et al, 2000, Nature Cell  y \\nbottom. At time 0, equivalent to prophase, the two centrosomes Biol.2:7–12.]\\n26 CHAPTER 22 • Cell Birth, Lineage, And Death\\nopposite directions: myosin II moving Miranda or other ma- organisms require signals to stay alive. In the absence of such\\nterials to control spindle orientation and Lgl restraining it.  survival signals, frequently referred to as trophic factors,cells\\nFurther evidence for the importance of myosin in spin- activate a “suicide” program. Second, in some developmental\\ndle orientation comes the finding that myosin VI binds di- contexts, including the immune system, specific signals induce\\nrectly to Miranda. In addition, mutations in myosin VI in a “murder” program that kills cells. Whether cells commit sui-\\nflies prevent basal targeting of Miranda and simultaneously cide for lack of survival signals or are murdered by killing sig-\\nblock proper spindle orientation. These various findings give nals from other cells, recent studies suggest that death is\\nus the beginning of a picture of how localized protein com- mediated by a common molecular pathway. In this section, we\\nplexes are coordinated with cell division, so that each daugh- first distinguish programmed cell death from death due to tis-\\nter cell receives the appropriate amounts of the various sue injury, then consider the role of trophic factors in neuronal\\ncomplexes.  development, and finally describe the evolutionarily conserved\\neffector pathway that leads to cell suicide or murder.\\nKEY CONCEPTS OF SECTION 22.4\\nProgrammed Cell Death Occurs \\nRegulation of Asymmetric Cell Division Through Apoptosis\\n■ Asymmetric cell division requires polarization of the di- The demise of cells by programmed cell death is marked by\\nviding cell, which usually entails localization of some cyto- a well-defined sequence of morphological changes, collec-\\nplasmic components, and then the unequal distribution of tively referred to as apoptosis, a Greek word that means\\nthese components to the daughter cells (see Figure 22-20).  “dropping off” or “falling off,” as in leaves from a tree.\\nDying cells shrink and condense and then fragment, releas-\\n■ In the asymmetric division of budding yeasts, a myosin-\\ning small membrane-bound apoptotic bodies, which gen-\\ndependent transport system carries ASH1mRNA into the\\nerally are engulfed by other cells (Figure 22-26; see also\\nbud (see Figure 22-22). \\nFigure 1-19). The nuclei condense and the DNA is frag-\\n■ Ash1 protein is produced in the daughter cell soon after mented. Importantly, the intracellular constituents are not\\ndivision  and  prevents  expression  of  HO  endonuclease,\\nreleased into the extracellular milieu where they might have\\nwhich is necessary for mating-type switching. Thus a daugh-\\ndeleterious effects on neighboring cells. The highly stereo-\\nter cell cannot switch mating type, whereas the mother cell\\ntyped changes accompanying apoptosis suggested to early\\nfrom which it arises can (see Figure 22-21). \\nworkers that this type of cell death was under the control of\\n■ Asymmetric cell division in Drosophilaneuroblasts depends a strict program. This program is critical during both em-\\non  localization  of  the  apical  protein  complex  (Baz/Par6/ bryonic and adult life to maintain normal cell number and\\nPKC3) and two basal complexes. The basal complexes are in- composition.\\ncorporated into the ganglion mother cell (GMC) and contain The genes involved in controlling cell death encode pro-\\nproteins that determine cell fate (see Figure 22-24).  teins with three distinct functions: \\n■ The apical Baz/Par6/PKC3 complex also controls asym- ■ “Killer” proteins are required for a cell to begin the\\nmetric cell division in C. elegansand perhaps mammals.  apoptotic process. \\n■ Asymmetry factors exert their influence at least in part ■ “Destruction” proteins do things like digest DNA in a\\nby controlling the orientation of the mitotic spindle, so that dying cell. \\nasymmetrically localized proteins and structures are differ-\\nentially incorporated into the two daughter cells. Myosin ■ “Engulfment” proteins are required for phagocytosis of\\nthe dying cell by another cell. \\nproteins bind to proteins that control asymmetry factors of\\ncells to control spindle orientation. At first glance, engulfment seems to be simply an after-death\\ncleanup process, but some evidence suggests that it is part\\nof the final death decision. For example, mutations in killer\\n22.5 Cell Death and Its Regulation  genes always prevent cells from initiating apoptosis, whereas\\nmutations that block engulfment sometimes allow cells to sur-\\nProgrammed cell death is a cell fate, an odd sort of cell fate vive that would normally die. That is, cells with engulfment-\\nbut nonetheless one that is essential. Cell death keeps our gene mutations can initiate apoptosis but then sometimes\\nhands from being webbed, our embryonic tails from persist- recover. \\ning, our immune system from responding to our own pro- In contrast to apoptosis, cells that die in response to tis-\\nteins, and our brain from being filled with useless electrical sue damage exhibit very different morphological changes,\\nconnections. In fact, the majority of cells generated during referred to as necrosis. Typically, cells that undergo this\\nbrain development also die during development. process swell and burst, releasing their intracellular con-\\nCellular interactions regulate cell death in two fundamen- tents that can damage surrounding cells and frequently\\ntally different ways. First, most cells, if not all, in multicellular cause inflammation.\\n22.5 • Cell Death and Its Regulation 27\\n!FIGURE 22-26 Ultrastructural\\nfeatures of cell death by apoptosis.\\n(a) Schematic drawings illustrating  F M\\no\\nthe progression of morphologic c E\\nu\\nchanges observed in apoptotic  s D\\ncells. Early in apoptosis, dense  A I\\nn A\\nchromosome condensation occurs im  \\nalong the nuclear periphery. The cell a C\\nbody also shrinks, although most tio O\\nn N\\norganelles remain intact. Later both : A N\\nthe nucleus and cytoplasm fragment, p\\no E\\nforming apoptotic bodies, which are p C\\nt\\nphagocytosed by surrounding cells. o T\\ns\\n(b) Photomicrographs comparing  is I\\nO\\na normal cell (top) and apoptotic cell\\nN\\n(bottom).Clearly visible in the latter\\nS\\nare dense spheres of compacted\\nchromatin as the nucleus begins to\\nfragment. [Part (a) adapted from J. Kuby,\\n28p (w) X 34p (d)\\n1997, Immunology,3d ed., W. H. Freeman\\n& Co., p. 53. Part (b) from M. J. Arends\\nand A. H. Wyllie, 1991, Int’l. Rev. Exp.\\nPathol.32:223.]\\nNeurotrophins Promote Survival of Neurons cremental increases in the target-field size are accompanied\\nby commensurate incremental increases in the number of\\nThe earliest studies demonstrating the importance of trophic neurons innervating the target field. This relation was found\\nfactors in cellular development came from analyses of the de- to result from the selective survival of neurons rather than\\nveloping nervous system. When neurons grow to make con- changes in their differentiation or proliferation. The obser-\\nnections to other neurons or to muscles, sometimes over vation that many sensory and motor neurons die after reach-\\nconsiderable distances, more cells grow than will eventually ing their peripheral target field suggested that these neurons\\nsurvive. Those that make connections prevail and survive; compete for survival factors produced by the target tissue. \\nthose that fail to connect die.  Subsequent to these early observations, scientists discov-\\nIn the early 1900s the number of neurons innervating the ered that transplantation of a mouse sarcoma tumor into a\\nperiphery was shown to depend upon the size of the tissue to chick led to a marked increase in the numbers of certain\\nwhich they would connect, the so-called “target field.” For in- types of neurons. This finding implicated the tumor as a rich\\nstance, removal of limb buds from the developing chick em- source of the presumed trophic factor. To isolate and purify\\nbryo leads to a reduction in the number of sensory neurons this factor, known simply as nerve growth factor (NGF), sci-\\nand motoneurons innervating the bud (Figure 22-27). Con- entists used an in vitro assay in which outgrowth of neurites\\nversely, grafting additional limb tissue to a limb bud leads  from sensory ganglia (nerves) was measured. Neurites are ex-\\nto an increase in the number of neurons in corresponding  tensions of the cell cytoplasm that can grow to become the\\nregions of the spinal cord and sensory ganglia. Indeed, in- long wires of the nervous system, the axons and dendrites\\n28 CHAPTER 22 • Cell Birth, Lineage, And Death\\nBDNF\\nNGF NT-3 NT-4/5 NT-3 NT-3\\nFPO\\nExterior\\nATP ATP ATP ATP ATP ATP\\nP P P P P P\\n21p (w) X 28p (d) ADP ADP ADP ADP ADP ADP\\nCytosol Tyrosine\\nkinase\\nTrkA TrkB TrkC\\n▲FIGURE 22-28 Specificity of neurotrophins for Trks,a\\nfamily of receptor tyrosine kinases. Each neurotrophin binds\\nwith high affinity to one Trk receptor indicated by the solid arrow\\nfrom the ligand to the receptor. NT-3 also can bind with lower\\naffinity to both TrkA and TrkB as indicated by the dashed arrow. \\nIn addition, neurotrophins bind to a distinct receptor called\\np75NTR either alone or in combination with Trks. NGF # nerve\\ngrowth factor; BDNF # brain-derived neurotrophic factor; \\nNT-3 # neurotrophin-3. [Adapted from W. D. Snider, 1994, Cell77:627.]\\nNeurotrophin Receptors Neurotrophins bind to and activate\\na family of receptor tyrosine kinases called Trks(pronounced\\n“tracks”). (The general structure of receptor tyrosine kinases\\n▲EXPERIMENTAL FIGURE 22-27 The survival of motor\\nand the intracellular signaling pathways they activate are cov-\\nneurons depends on the size of the muscle target field they\\nered in Chapter 14.) As shown in Figure 22-28, NGF binds to\\ninnervate.(a) Removal of a limb bud from one side of a chick\\nTrkA; BDNF, to TrkB; and NT-3, to TrkC. Binding of these fac-\\nembryo at about 2.5 days results in a marked decrease in the\\ntors to their receptors provides a survival signal for different\\nnumber of motor neurons on the affected side. In an amputated\\nembryo, normal numbers of motor neurons are generated on classes of neurons. A second type of receptor called p75NTR\\nboth sides (middle).Later in development, many fewer motor (NTR #neurotrophin receptor) also binds to neurotrophins,\\nneurons remain on the side of the spinal cord with the missing but with lower affinity. However, p75NTRforms heteromulti-\\nlimb than on the normal side (bottom).Note that only about 50 meric complexes with the different Trk receptors; this associa-\\npercent of the motor neurons that originally are generated tion increases the affinity of Trks for their ligands. Some studies\\nnormally survive. (b) Transplantation of an extra limb bud into an indicate that the binding of NGF to p75NTRin the absence of\\nearly chick embryo produces the opposite effect, more motor TrkA may promote cell death rather than prevent it.\\nneurons on the side with additional target tissue than on the\\nnormal side. [Adapted from D. Purves, 1988, Body and Brain: A Trophic Knockouts of Neurotrophins and Their Receptors To criti-\\nTheory of Neural Connections(Cambridge, MA: Harvard University Press), cally address the role of the neurotrophins in development, sci-\\nand E. R. Kandel, J. H. Schwartz, and T. M. Jessell, 2000, Principles of entists produced mice with knockout mutations in each of the\\nNeural Science,4th ed. (New York: McGraw-Hill), p. 1054, Figure 53-11.] neurotrophins and their receptors. These studies revealed that\\ndifferent neurotrophins and their corresponding receptors are\\nrequired for the survival of different classes of sensory neurons,\\n(see Figure 7-29). The later discovery that the submaxillary which carry signals from peripheral sensory systems to the\\ngland in the mouse also produces large quantities of NGF en- brain (Figure 22-29). For instance, pain-sensitive (nociceptive)\\nabled biochemists to purify it to homogeneity and to se- neurons, which express TrkA, are selectively lost from the dor-\\nquence it. A homodimer of two 118-residue polypeptides, sal root ganglion of knockout mice lacking NGF or TrkA,\\nNGF belongs to a family of structurally and functionally re- whereas TrkB- and TrkC-expressing neurons are unaffected in\\nlated trophic factors collectively referred to as neurotrophins. such knockouts. In contrast, TrkC-expressing proprioceptive\\nBrain-derived neurotrophic factor (BDNF) and neurotro- neurons, which detect the position of the limbs, are missing\\nphin-3 (NT-3) also are members of this protein family. from the dorsal root ganglion in TrkCand NT-3mutants.\\n22.5 • Cell Death and Its Regulation 29\\nWild Type Mutant Mutant \"EXPERIMENTAL FIGURE 22-29\\nDifferent classes of sensory neurons are\\nNGF −/− NT-3 −/− lost in knockout mice lacking different\\nSpinal Mechanoreceptors or or\\ntrophic factors or their receptors.In\\ncord Dorsal root TrkA −/− TrkC −/− animals lacking nerve growth factor (NGF)\\nganglion or its receptor TrkA, small nociceptive \\nPropioceptive (pain-sensing) neurons (blue) that innervate\\nneurons the skin are missing. These neurons\\nNociceptive express TrkA receptor and innervate \\nneurons NGF-producing targets. In animals lacking\\neither neurotrophin-3 (NT-3) or its receptor\\nTrkC, large propioceptive neurons (red)\\ninnervating muscle spindles are missing.\\nMuscle produces NT-3 and the\\nSkin\\npropioceptive neurons express TrkC.\\nMechanoreceptors (brown), another class\\nof sensory neurons in the dorsal root\\nganglion, are unaffected in these mutants.\\nMuscle Motoneurons\\n[Adapted from W. D. Snider, 1994, Cell77:627.]\\nA Cascade of Caspase Proteins Functions  (a)\\nin the Apoptotic Pathway \\nV M\\nid\\nKey insights into the molecular mechanisms regulating cell e E\\no D\\ndeath came from genetic studies in C. elegans.As we noted   A I\\nin Section 22.1, scientists have traced the lineage of all the nn A\\nsomatic cells in C. elegansfrom the fertilized egg to the  im  C\\nmature worm (see Figure 22-9). Of the 947 nongonadal cells at O\\ngenerated during development of the adult hermaphrodite ion N\\nform, 131 cells undergo programmed cell death.  : P N\\nr E\\nSpecific mutations have identified a variety of genes o\\ng C\\nwhose encoded proteins play an essential role in controlling ra T\\nprogrammed cell death during C. elegansdevelopment. For (b) m I\\nm O\\ninstance, programmed cell death does not occur in worms FPO e N\\nd\\ncarrying loss-of-function mutations in the ced-3gene or the  C S\\nced-4gene; as a result, the 131 “doomed” cells survive (Fig- e\\nure 22-30). CED-4 is a protease-activating factor that ll D\\ne\\ncleaves the CED-3 precursor protein, creating an active a\\nt\\nCED-3 protease that initiates cell death. In contrast, in ced- h\\n9mutants, all cells die during embryonic life, so the adult  in\\n C\\nform never develops. These genetic studies indicate that the . e\\nCED-3 and CED-4 proteins are required for cell death, that le\\ng\\nCED-9 suppresses apoptosis, and that the apoptotic path- a\\nn\\nway can be activated in all cells. Moreover, the finding that s\\ncell death does not occur in ced-9/ced-3double mutants sug- ▲EXPERIMENTAL FIGURE 22-30 Mutations in the Em\\ngests that CED-9 acts “upstream” of CED-3 to suppress the ced-3gene block programmed cell death in C.elegans. b\\nr\\napoptotic pathway. (a) Newly hatched larva carrying a mutation in the ced-1 yo\\nThe confluence of genetic studies in worms and studies gene. Because mutations in this gene prevent engulfment nic\\non human cancer cells first suggested that an evolutionarily of dead cells, highly refractile dead cells accumulate  D\\nconserved pathway mediates apoptosis. The first apoptotic (arrows), facilitating their visualization. (b) Newly hatched ev\\nlarva with mutations in both the ced-1and ced-3genes. e\\ngene to be cloned, bcl-2,was isolated from human follicular lo\\nlymphomas. A mutant form of this gene, created in lym- The absence of refractile dead cells in these double p\\nm\\nmutants indicates that no cell deaths occurred. Thus CED-3\\nphoma cells by a chromosomal rearrangement, was shown to e\\nprotein is required for programmed cell death. [From H. M. n\\nact as an oncogenethat promoted cell survival rather than t\\nEllis and H. R. Horvitz, 1986, Cell91:818; courtesy of Hilary Ellis.]\\ncell death (Chapter 23). The human Bcl-2 protein and worm\\nCED-9 protein are homologous, and a bcl-2transgene can\\n30 CHAPTER 22 • Cell Birth, Lineage, And Death\\nblock the extensive cell death found in ced-9mutant worms. For instance, expression of C. elegansCED-4 in a human\\nThus both proteins act as regulators that suppress the apop- kidney cell line induces rapid apoptosis. This can be blocked\\ntotic pathway (Figure 22-31). In addition, both proteins con- by co-expression of the negative regulator CED-9 (or mam-\\ntain a single transmembrane domain and are localized to the malian Bcl-2), indicating that CED-9 opposes CED-4 action.\\nouter mitochondrial, nuclear, and endoplasmic reticulum In addition, CED-9 has been shown to directly bind to \\nmembranes, where they serve as sensors that control the CED-4 and move it from the cytosol to intracellular mem-\\napoptotic pathway in response to external stimuli. As we dis- branes. Thus the pro-apoptotic function of CED-4 is directly\\ncuss below, other regulators promote apoptosis. suppressed by the anti-apoptotic function of CED-9. CED-4\\nThe effector proteinsin the apoptotic pathway are en- also binds directly to the CED-3 caspase (and related mam-\\nzymes called caspasesin vertebrates,so named because they malian caspases) and promotes activation of its protease ac-\\ncontain a key cysteine residue in the catalytic site and selec- tivity, as noted previously. Biochemical studies have shown\\ntively cleave proteins at sites just C-terminal to aspartate that CED-4 can simultaneously bind to both CED-9 and\\nresidues. Humans have 15 different caspases, all of which are CED-3. These results fit with the genetics, which shows that\\ninitially made as procaspases that must be cleaved to become the absence of CED-9 has no effect if CED-3 is also missing\\nactive. Such proteolytic processing of proproteins is used re- (ced-3/ced-9double mutants have no cell death, like ced-3\\npeatedly in blood clotting, generation of digestive enzymes, mutants).\\nand generation of hormones (Chapter 17). Initiator caspases\\n(e.g., caspase-9) are activated by autoproteolysis induced by\\nPro-Apoptotic Regulators Permit Caspase\\nother types of proteins, which help the initiators to aggre-\\nActivation in the Absence of Trophic Factors \\ngate. Activated initiator caspases cleave effector caspases\\n(e.g., caspase-3) and thus quickly amplify the total caspase Having introduced the major participants in the apoptotic\\nactivity level in the dying cell. The various effector caspases pathway, we now take a closer look at how the effector cas-\\nrecognize and cleave short amino acid sequences and differ in pases are activated in mammalian cells. Although the normal\\ntheir preferred target sequences. Their specific intracellular function of CED-9 or Bcl-2 is to suppress the cell-death path-\\ntargets include proteins of the nuclear lamina and cytoskele- way, other intracellular regulatory proteins promote apopto-\\nton whose cleavage leads to the demise of a cell. Seven lines sis. The first pro-apoptotic regulator to be identified, named\\nof knockout mice have been made, each lacking a particular Bax, was found associated with Bcl-2 in extracts of cells ex-\\ncaspase function. Some homozygotes die as embryos; others pressing high levels of Bcl-2. Sequence analysis demonstrated\\nare viable but with subtle defects. Strikingly, some exhibit that Bax is related in sequence to CED-9 and Bcl-2, but over-\\nan excessive number of neurons (i.e., neuronal hyperplasia), production of Bax induces cell death rather than protecting\\nreaffirming the importance of cell death in development of cells from apoptosis, as CED-9 and Bcl-2 do. Thus this family\\nthe nervous system; other mutants have cells that are resis- of regulatory proteins comprises both anti-apoptoticmem-\\ntant to apoptosis. The principal effector protease inC. elegans bers (e.g., CED-9, Bcl-2) and pro-apoptoticmembers (e.g.,\\nis the CED-3 caspase.  Bax). All members of this family, which we refer to as the Bcl-\\nCell-culture studies have yielded important insights about 2 family,are single-pass transmembrane proteins and can par-\\na group of adapter proteinsthat coordinate the action of reg- ticipate in oligomeric interactions. In mammals, six Bcl-2\\nulators and effectors to control apoptosis (see Figure 22-31). family members prevent apoptosis and nine promote it. Thus\\nthe fate of a given cell—survival or death—may reflect the\\nparticular spectrum of Bcl-2 family members made by that\\nRegulator Adapter Effector\\ncell and the intracellular signaling pathways regulating them.\\nFPO\\nSome Bcl-2 family members preserve or disrupt the in-\\nC. elegans CED-9 CED-4 CED-3 Death tegrity of mitochondria, thereby controlling release of mito-\\nchondrial proteins such as cytochrome c. In normal healthy\\nVertebrates Bcl-2 Apaf-1 Casp9 Casp3 Death\\ncells, cytochrome cis localized between the inner and outer\\nmitochondrial membrane, but in cells undergoing apoptosis,\\n▲FIGURE 22-31 Overview of the evolutionarily conserved cytochrome cis released into the cytosol. This release can be\\napoptotis pathway in C.elegansand vertebrates.Three blocked by overproduction of Bcl-2; conversely, overproduc-\\ngeneral types of proteins are critical in this conserved pathway.\\ntion of Bax promotes release of cytochrome cinto the cytosol\\nRegulators either promote or suppress apoptosis; two\\nand apoptosis. Moreover, injection of cytochrome cinto the\\northologous regulators shown here, CED-9 and Bcl-2, suppress\\ncytosol of cells induces apoptosis. A variety of death-inducing\\napoptosis in the presence of trophic factors. Adapters interact\\nstimuli cause Bax monomers to move from the cytosol to the\\nwith both regulators and effectors; in the absence of trophic\\nouter mitochondrial membrane where they oligomerize. Bax\\nfactors, they promote activation of effectors. Proteases called\\nhomodimers, but not Bcl-2 homodimers or Bcl-2/Bax het-\\ncaspases serve as effector proteins; their activation leads to\\ndegradation of various intracellular substrates and eventually  erodimers, permit influx of ions through the mitochondrial\\ncell death. [Adapted from D. L. Vaux and S. J. Korsemeyer, 1999,  membrane. It remains unclear how this ion influx triggers the\\nCell96:245.] release of cytochrome c.Once cytochrome cis released into\\n22.5 • Cell Death and Its Regulation 31\\nthe cytosol, it binds to the adapter protein Apaf-1 (the mam- Some Trophic Factors Induce Inactivation \\nmalian homolog of CED-4) and promotes activation of a cas- of a Pro-Apoptotic Regulator\\npase cascade leading to cell death (Figure 22-32a). Bax can\\ncause cell death by a second pathway that triggers mitochon- We saw earlier that neurotrophins such as nerve growth fac-\\ndrial dysfunction independently of any caspase action, at least tor (NGF) protect neurons from cell death. The intracellular\\nin part by stimulating mitochondrial depolarization. signaling pathways linking such survival factors to inactiva-\\nGene knockout experiments have dramatically confirmed tion of the cell-death machinery are quite elaborate. The\\nthe importance of both pro-apoptotic and anti-apoptotic  finding that trophic factors appear to work largely inde-\\nBcl-2 family members in neuronal development. Mice lacking pendent of protein synthesis suggested that these external\\nthe bcl-xlgene, which encodes an anti-apoptotic protein, have signals lead to changes in the activities of preexisting proteins\\nmassive defects in nervous system development with wide- rather than to activation of gene expression. Scientists\\nspread cell death in the spinal cord, dorsal root ganglion, and demonstrated that in the absence of trophic factors, the non-\\nbrain of developing embryos. In contrast, baxknockouts ex- phosphorylated form of Bad is associated with Bcl-2/Bcl-xl at\\nhibit a marked increase in neurons in some regions of the the mitochondrial membrane (see Figure 22-32a). Binding of\\nnervous system. These knockouts also demonstrate that the Bad inhibits the anti-apoptotic function of Bcl-2/Bcl-xl,\\neffects of Bcl-2 family members generally are tissue specific. thereby promoting cell death. Phosphorylated Bad, however,\\n(a)  Absence of trophic factor: Caspase activation (b)  Presence of trophic factor: Inhibition of caspase activation\\nTrophic factor\\nTrophic factor receptor\\nF M\\n1 o\\nc E\\nu\\nPlasma membrane s D\\n A I\\nDeath n A\\nim  \\nCleavage of substrates a C\\ntio O\\nn N\\nPI-3 kinase Procaspase 3 Caspase 3 PI-3 kinase Procaspase 3 : A N\\np\\no E\\np C\\n14-3-3 Caspase 9 to T\\n14-3-3 ATP s\\nAkt FPO P ADP Akt is IO\\n4\\nN\\nBad\\nProcaspase 9 2 Procaspase 9 S\\nApaf 1 Apaf 1\\nCyt c\\n3\\nBad\\n1 Bcl-2 Bcl-xl Bax Bax mOuetmerb mrainteochondrial Bcl-2Bcl-xl BaxBax\\n2 Cyt c 3\\nCyt c Ions\\n▲FIGURE 22-32 Proposed intracellular pathways leading adapter protein Apaf-1 (3), promoting a caspase cascade that\\nto cell death by apoptosis or to trophic factor–mediated cell leads to cell death (4). (b) In some cells, binding of a trophic\\nsurvival in mammalian cells.(a) In the absence of a trophic factor (e.g., NGF) stimulates PI-3 kinase activity, leading to\\nfactor, Bad, a soluble pro-apoptotic protein, binds to the  activation of the downstream kinase Akt, which phosphorylates\\nanti-apoptotic proteins Bcl-2 and Bcl-xl, which are inserted into Bad (1). Phosphorylated Bad then forms a complex with the \\nthe mitochondrial membrane (1). Bad binding prevents the  14-3-3 protein (2). With Bad sequestered in the cytosol, the \\nanti-apoptotic proteins from interacting with Bax, a membrane- anti-apoptotic Bcl-2/Bcl-xl proteins can inhibit the activity of\\nbound pro-apoptotic protein. As a consequence, Bax forms Bax (3), thereby preventing the release of cytochrome cand\\nhomo-oligomeric channels in the membrane that mediate ion flux activation of the caspase cascade. [Adapted from B. Pettman \\n(2). Through an as-yet-unknown mechanism, this leads to the and C. E. Henderson, 1998, Neuron20:633.]\\nrelease of cytochrome cinto the cytosol, where it binds to the\\n32 CHAPTER 22 • Cell Birth, Lineage, And Death\\ncannot bind to Bcl-2/Bcl-xl and is found in the cytosol com- ■ Genetic studies in C. elegansdefined an evolutionarily\\nplexed to the phosphoserine-binding protein 14-3-3. Hence, conserved apoptotic pathway with three major compo-\\nsignaling pathways leading to Bad phosphorylation would be nents: regulatory proteins, adapter proteins, and effector\\nparticularly attractive candidates for transmitting survival proteases called caspases in vertebrates (see Figure 22-31). \\nsignals.\\n■ Once activated, apoptotic proteases cleave specific intra-\\nA number of trophic factors including NGF have been\\ncellular substrates leading to the demise of a cell. Adapter\\nshown to trigger the PI-3 kinase signaling pathway, leading\\nproteins (e.g., Apaf-1), which bind both regulatory proteins\\nto activation of a downstream kinase called PKB (see Figure\\nand caspases, are required for caspase activation. \\n14-27). Activated PKB  phosphorylates Bad at sites known to\\ninhibit its pro-apoptotic activity. Moreover, a constitutively ■ Pro-apoptotic regulator proteins (e.g., Bax, Bad) pro-\\nactive form of PKB can rescue cultured neurotrophin- mote  caspase  activation,  and  anti-apoptotic  regulators\\ndeprived neurons, which otherwise would undergo apoptosis (e.g., Bcl-2) suppress activation. Direct interactions be-\\nand die. These findings support the mechanism for the sur- tween pro-apoptotic and anti-apoptotic proteins lead to\\nvival action of trophic factors depicted in Figure 22-32b. In cell death in the absence of trophic factors. Binding of ex-\\nother cell types, different trophic factors may promote cell tracellular trophic factors can trigger changes in these in-\\nsurvival through post-translational modification of other teractions, resulting in cell survival (see Figure 22-32).\\ncomponents of the cell-death machinery. \\n■ The Bcl2 family contains both pro-apoptotic and anti-\\napoptotic proteins; all are single-pass transmembrane pro-\\nTumor Necrosis Factor and Related Death  teins and engage in protein-protein interactions.\\nSignals Promote Cell Murder by Caspases  ■ Binding of extracellular death signals, such as tumor\\nnecrosis factor and Fas ligand, to their receptors activates\\nAlthough cell death can arise as a default in the absence of\\nan associated protein (FADD) that in turn triggers the cas-\\nsurvival factors, apoptosis can also be stimulated by posi-\\npase cascade leading to cell murder. \\ntively acting “death” signals.For instance, tumor necrosis\\nfactor (TNF),which is released by macrophages, triggers the\\ncell death and tissue destruction seen in certain chronic in-\\nflammatory diseases. Another important death-inducing sig- PERSPECTIVES FOR THE FUTURE\\nnal, the Fas ligand,is a cell-surface protein expressed by\\nactivated natural killer cells and cytotoxic T lymphocytes. Cell birth, lineage, and death lie at the heart of the growth\\nThis signal can trigger death of virus-infected cells, some of an organism and are also central to disease processes,\\ntumor cells, and foreign graft cells.  most notably cancer. Few transformations seem more \\nBoth  TNF  and  Fas  ligand  act  through  cell-surface remarkable than the blooming of cell types during develop-\\n“death” receptors that have a single transmembrane domain ment. A lineage beginning with a fertilized egg, a “plain\\nand are activated when ligand binding brings three receptor vanilla” 200-$m sphere, produces neurons a yard long,\\nmolecules into close proximity. The trimeric receptor com- pulsating multinucleate muscle cells, exquisitely light-\\nplex attracts a protein called FADD (Fas-associated death sensitive retina cells, ravenous macrophages that recognize\\ndomain), which serves as an adapter to recruit and in some and engulf germs, and all the hundreds of other cell types.\\nway activate caspase-8, an initiator caspase, in cells receiving Regulators of cell lineage produce this rich variety by \\na death signal. The death domain found in FADD is a se- controlling two critical decisions: (1) when and where to\\nquence that is present in a number of proteins involved in activate the cell division cycle (Chapter 21) and (2) whether\\napoptosis. Once activated, caspase-8 activates other caspases the two daughter cells will be the same or different. A \\nand the amplification cascade begins. To test the ability of cell may be just like its parent, or it may embark on a new\\nthe Fas receptor to induce cell death, researchers incubated path. \\ncells with antibodies against the receptor. These antibodies, Cell birth is normally carefully restricted to specific lo-\\nwhich bind and cross-link Fas receptors, were found to stim- cales and times, such as the basal layer of the skin or the root\\nulate cell death, indicating that activation of the Fas recep- meristem. Liver regenerates when there is injury, but liver\\ntor is sufficient to trigger apoptosis.  cancer is prevented by restricting unnecessary growth at\\nother times. Cell lineage is patterned by the asymmetric dis-\\ntribution of key regulators to the daughter cells of a division.\\nKEY CONCEPTS OF SECTION 22.5 Some of these regulators are intrinsic to the parent cell, be-\\ncoming asymmetrically distributed during polarization of the\\nCell Death and Its Regulation\\ncell; other regulators are external signals that differentially\\n■ All cells require trophic factors to prevent apoptosis and reach the daughter cells. Asymmetry of cells becomes asym-\\nthus survive. In the absence of these factors, cells commit metry of tissues and whole organisms. Our left and right\\nsuicide.  hands differ only as a result of cell asymmetry. \\nReview the Concepts 33\\nSome cells persist for the life of the organism, but  embryonic stem (ES)  mesoderm 000\\nothers such as blood and intestinal cells turn over rapidly. cells 000 micro RNAs \\nMany cells live for awhile and are then programmed to die endoderm 000 (miRNAs) 000\\nand be replaced by others arising from a stem-cell popula-\\nganglion mother  muscle regulatory \\ntion. Programmed cell death is also the basis for the metic-\\ncell (GMC) 000 factors (MRFs) 000\\nulous elimination of potentially harmful cells, such as\\ngerm line 000 neurotrophins 000\\nautoreactive immune cells, which attack the body’s own\\nheterochronic mutants 000 pluripotent 000\\ncells, or neurons that have failed to properly connect. Cell-\\ndeath programs have also evolved as a defense against in- MATlocus 000 precursor/progenitor \\nfection, and virus-infected cells are selectively murdered in mating factor 000 cells 000\\nresponse to death signals. Viruses, in turn, devote much of mating-type  somatic cells 000\\ntheir effort to evading host defenses. For example, p53, a switching 000 stem cells 000\\ntranscription factor that senses cell stresses and damage\\nmeristems 000 trophic factors 000\\nand activates transcription of pro-apoptotic members of the\\nbcl-2gene family, is inhibited by the adenovirus E1B pro-\\ntein. It has been estimated that about a third of the aden-\\nREVIEW THE CONCEPTS\\novirus genome is directed at evading host defenses. Cell\\ndeath is relevant to toxic chemicals as well as viral infec-\\n1. What two properties define a stem cell? Distinguish be-\\ntions; malformations due to poisons often originate from\\ntween a totipotential stem cell, a pluripotent stem cell, and\\nexcess apoptosis.\\na progenitor cell.\\nFailures of programmed cell death can lead to uncon-\\ntrolled cancerous growth (Chapter 23). The proteins that re- 2. Where are stem cells located in plants? Where are stem\\nstrain the death of cancer cells therefore become possible cells located in adult animals? How does the concept of stem\\ntargets for drugs. A tumor may contain a mixture of cells, cell differ between animal and plant systems?\\nsome capable of seeding new tumors or continued uncon-\\n3. In 1997, Dolly the sheep was cloned by a technique\\ntrolled growth, and some capable only of growing in place or\\ncalled somatic cell nuclear transfer. A nucleus from an\\nfor a limited time. In this sense the tumor has its own stem\\nadult mammary cell was transferred into an egg from\\ncells, and they must be found and studied, so they become\\nwhich the nucleus had been removed. The egg was allowed\\nvulnerable to our medicine. One option is to manipulate the\\nto divide several times in culture, then the embryo was\\ncell-death pathway to our own advantage, to send the signals\\ntransferred to a surrogate mother who gave birth to \\nthat will make cancer cells destroy themselves.\\nDolly. Dolly died in 2003 after mating and giving birth\\nMuch attention is now being given to the regulation of\\nherself to viable offspring. What does the creation of Dolly\\nstem cells in an effort to understand how dividing popula-\\ntell us about the potential of nuclear material derived from\\ntions of cells are created and maintained. This has clear im-\\na fully differentiated adult cell? Does the creation of Dolly\\nplications for repair of tissue, for example, to help damaged\\ntell us anything about the potential of an intact, fully dif-\\neyes, torn cartilage, degenerating brain tissue, or failing or-\\nferentiated adult cell? Name three types of information\\ngans. One interesting possibility is that some populations of\\nthat function to preserve cell type. Which of these types\\nstem cells with the potential to generate or regenerate tissue\\nof information was shown to be reversible by the Dolly \\nare normally eliminated by cell death during later develop-\\nexperiment?\\nment. If so, finding ways to selectively block the death of the\\ncells could make regeneration more likely. Could the elimi-\\n4. The roundworm C. eleganshas proven to be a valuable\\nnation of such cells during mammalian development be the\\nmodel organism for studies of cell birth, cell lineage, and cell\\ndifference between an amphibian capable of limb regenera-\\ndeath. What properties of C. elegansrender it so well suited\\ntion and a mammal that is not?\\nfor these studies? Why is so much information from C. elegans\\nexperiments of use to investigators interested in mammalian\\ndevelopment?\\nKEY TERMS\\n5. In the budding yeast S. cerevisiae,what is the role of the\\nMCM1 protein in the following:\\napical complex 000 caspases 000\\napoptosis 000 cell lineage 000 a. transcription of a-specific genes in acells\\nasymmetric cell  death signals 000 b. blocking transcription of \"-specific genes in acells\\ndivision 000 determination 000\\nc. transcription of \"-specific genes in \"cells\\n!-catenin 000 differentiation 000\\nBcl-2 family 000 ectoderm 000 d. blocking transcription of a-specific genes in \"cells\\n34 CHAPTER 22 • Cell Birth, Lineage, And Death\\n6. In S. cerevisiae, what ensures that a and \" cells mate One common characteristic of cancer cells is a loss of func-\\nwith one another rather than with cells of the same mating tion in the apoptotic pathway. Which of the mutations listed\\ntype (i.e., awith aor \"with \")? might you expect to find in some cancer cells?\\n7. Exposure of C3H 10T1/2 cells to 5-azacytidine, a nu-\\ncleotide analog, is a model system for muscle differentiation.\\nANALYZE THE DATA\\nHow was 5-azacytidine treatment used to isolate the genes\\ninvolved in muscle differentiation? \\nTo better understand the potential of adult stem cells to dif-\\n8. Through the experiments on C3H 10T1/2 cells treated ferentiate into various cell types, the following studies were\\nwith 5-azacytidine, MyoD was identified as a key transcrip- performed.\\ntion factor in regulating the differentiation of muscle. To Bone marrow was harvested from adult male mice and\\nwhat general class of DNA-binding proteins does MyoD be- then transplanted into isogeneic female recipient mice that\\nlong? How do the interactions of MyoD with the following had been treated with a level of irradiation sufficient to\\nproteins affect its function? (a) E2A, (b) MEFs, (c) Id destroy their own hematopoetic stem cells (see the figure).\\nAlthough the dose of irradiation given was lethal to mice\\n9. The mechanisms that regulate muscle differentiation in\\nthat did not receive a transplant, the majority of the re-\\nmammals and neural differentiation in Drosophila (and\\ncipient mice receiving the transplant survived. After 4\\nprobably mammals as well) bear remarkable similarities.\\nweeks, peripheral blood from the recipient mice was ana-\\nWhat proteins function analogous to MyoD, myogenin, Id,\\nlyzed. The composition of the blood was normal with re-\\nand E2A in neural cell differentiation in Drosophila? Based\\nspect to all blood cell types. Every blood cell examined\\non these analogies, predict the effect of microinjection of\\nwas determined to be positive for the presence of the Y\\nMyoD mRNA on the development of Xenopusembryos.\\nchromosome.\\n10. Predict the effect of the following mutations on the abil-\\nity of mother and daughter cells of S. cerevisaeto undergo male mouse\\nmating type switching following cell division: \\na. loss-of-function mutation in the HO endonuclease Harvest marrow\\nb. gain-of-function  mutation  that  renders  HO  endo-\\nnuclease gene constitutively expressed independent of Isogeneic female mouse Bone\\nSWI/SNF marrow\\ntransplant\\nc. gain-of-function mutation in SWI/SNF that renders it in-\\nLethal irradiation\\nsensitive to Ash1\\nFPO\\n11. Asymmetric cell division often relies on cytoskeletal el-\\nements to generate or maintain asymmetric distribution of\\ncellular factors. In S. cerevisiae,what factor is localized to\\n\"labeled\" mouse\\nthe bud by myosin motors? In Drosophilaneuroblasts, what\\nfactors are localized apically by microtubules?\\n12. How do studies of brain development in knockout mice\\nsupport the statement that apoptosis is a default pathway in\\nneuronal cells?\\na. What was the purpose of using male mice as the bone\\n13. What morphologic features distinguish programmed cell marrow donors and female mice as the recipients? Would the\\ndeath and necrotic cell death? TNF and Fas ligand bind cell converse experiment (female donors, male recipients) have\\nsurface receptors to trigger cell death. Although the death worked? \\nsignal is generated external to the cell, why do we consider\\nb. What was the purpose of irradiating the recipient mice\\nthe death induced by these molecules to be apoptotic rather\\nprior to transplantation? What outcome might you expect if\\nthan necrotic?\\nbone marrow cells were transplanted to nonirradiated mice? \\n14. Predict the effects of the following mutations on the\\nc. What method could have been used to purify hematopo-\\nability of a cell to undergo apoptosis:\\netic stem cells from bone marrow before transplanting these\\na. mutation in Bad such that it cannot be phosphorylated by cells into the recipient mouse? How might a purification step\\nAkt have affected the outcome of the experiment?\\nb. overexpression of Bcl-2\\nAfter 8 weeks, some of the recipient mice were sacrificed and\\nc. mutation in Bax such that it cannot form homodimers histochemical analysis was performed on tissue sections of'"
            ]
          },
          "metadata": {},
          "execution_count": 55
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 171
        },
        "id": "hdWIS5Fgn_Wv",
        "outputId": "5f8eb835-f0ab-4d2c-e275-5d31f87bb83a"
      },
      "source": [
        "all_text"
      ],
      "execution_count": 54,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'\\n1\\nLIFE BEGINS \\nWITH CELLS\\nA single ~200 micrometer (!m) cell,the human egg,with\\nsperm,which are also single cells.From the union of an egg\\nand sperm will arise the 10 trillion cells of a human body.\\n[Photo Researchers, Inc.]\\nLike ourselves, the individual cells that form our bodies 1.1 The Diversity and Commonality \\ncan grow, reproduce, process information, respond to\\nof Cells\\nstimuli, and carry out an amazing array of chemical re-\\nactions. These abilities define life. We and other multicellular\\nCells come in an amazing variety of sizes and shapes (Figure\\norganisms contain billions or trillions of cells organized into\\n1-1). Some move rapidly and have fast-changing structures, as\\ncomplex structures, but many organisms consist of a single\\nwe can see in movies of amoebae and rotifers. Others are\\ncell. Even simple unicellular organisms exhibit all the hall-\\nlargely stationary and structurally stable. Oxygen kills some\\nmark properties of life, indicating that the cell is the funda-\\ncells but is an absolute requirement for others. Most cells in\\nmental unit of life. As the twenty-first century opens, we face\\nmulticellular organisms are intimately involved with other\\nan explosion of new data about the components of cells,\\ncells. Although some unicellular organisms live in isolation,\\nwhat structures they contain, how they touch and influence\\nothers form colonies or live in close association with other\\neach other. Still, an immense amount remains to be learned,\\ntypes of organisms, such as the bacteria that help plants to ex-\\nparticularly about how information flows through cells and\\ntract nitrogen from the air or the bacteria that live in our in-\\nhow they decide on the most appropriate ways to respond. \\ntestines and help us digest food. Despite these and numerous\\nMolecular cell biology is a rich, integrative science that\\nbrings together biochemistry, biophysics, molecular biology,\\nmicroscopy, genetics, physiology, computer science, and de-\\nO UTLINE\\nvelopmental biology. Each of these fields has its own em-\\nphasis  and  style  of  experimentation.  In  the  following 1.1  The Diversity and Commonality of Cells\\nchapters, we will describe insights and experimental ap-\\n1.2  The Molecules of a Cell\\nproaches drawn from all of these fields, gradually weaving\\nthe multifaceted story of the birth, life, and death of cells. We 1.3  The Work of Cells\\nstart in this prologue chapter by introducing the diversity of\\n1.4  Investigating Cells and Their Parts\\ncells, their basic constituents and critical functions, and what\\nwe can learn from the various ways of studying cells.  1.5  A Genome Perspective on Evolution\\n1\\n21.6 • Cell-Cycle Control in Mammalian Cells 33\\nsee Table 21-1.) Disruption of cyclin A function inhibits\\nCyclin E\\nDNA synthesis in mammalian cells, suggesting that cyclin\\nCyclin D\\nCDK2 A-CDK2 complexes may function like S. cerevisiaeS-phase\\nCDK4/6 cyclin-CDK complexes to trigger initiation of DNA synthe-\\nFPOP P P\\nsis. There is also evidence that cyclin E-CDK2 may con-\\nRb E2F Rb + E2F tribute to activation of pre-replication complexes. \\nThree related CDK inhibitory proteins, or CIPs(p27KIP1,\\nMid G1 Late G1 p57KIP2, and p21CIP), appear to share the function of the \\n▲ FIGURE 21-30 Regulation of Rb and E2F activities in S. cerevisiaeS-phase inhibitor Sic1 (see Figure 21-24). Phos-\\nphorylation of p27KIP1by cyclin E-CDK2 targets it for poly-\\nmid-late G . Stimulation of G cells with mitogens induces\\n1 0\\nexpression of CDK4, CDK6, D-type cyclins, and the E2F ubiquitination by the mammalian SCF complex (see Figure\\ntranscription factors, all encoded by delayed-response genes. Rb 21-2, step 5). The SCF subunit that targets p27KIP1is syn-\\nprotein initially inhibits E2F activity. When signaling from thesized as cells approach the G → S transition. The mecha-\\n1\\nmitogens is sustained, the resulting cyclin D-CDK4/6 complexes nism for degrading p21CIP and p57KIP2 is less well under-\\nbegin phosphorylating Rb, releasing some E2F, which stimulates stood. The activity of mammalian cyclin-CDK2 complexes is\\ntranscription of the genes encoding cyclin E, CDK2, and E2F also regulated by phosphorylation and dephosphorylation\\n(autostimulation). The cyclin E-CDK4 complexes further mechanisms similarly to those controlling the S. pombe\\nphosphorylate Rb, resulting in positive feedback loops (blue mitosis-promoting factor, MPF (see Figure 21-14). The\\narrows) that lead to a rapid rise in the expression and activity of\\nCdc25A phosphatase,which removes the inhibitory phos-\\nboth E2F and cyclin E-CDK2 as the cell approaches the G → S\\n1 phate from CDK2, is a mammalian equivalent of S. pombe\\ntransition.\\nCdc25 except that it functions at the G → S transition rather\\n1\\nthan the G → M transition. The mammalian phosphatase\\n2\\nnormally is activated late in G , but is degraded in the re-\\n1\\nRb protein is one of the most significant substrates of sponse of mammalian cells to DNA damage to prevent the\\nmammalian G1cyclin-CDK complexes. Phosphorylation of cells from entering S phase (see Section 21.7).\\nRb protein at multiple sites prevents its association with Once cyclin A-CDK2 is activated by Cdc25A and the \\nE2Fs, thereby permitting E2Fs to activate transcription of S-phase inhibitors have been degraded, DNA replication \\ngenes required for entry into S phase. As shown in Figure  is initiated at pre-replication complexes. The general mech-\\n21-30, phosphorylation of Rb protein is initiated by cyclin anism is thought to parallel that in S. cerevisiae(see Figure\\nD-CDK4 and cyclin D-CDK6 in mid G1. Once cyclin E and 21-26), although small differences are found in vertebrates.\\nCDK2 are induced by phosphorylation of some Rb, the re- As in yeast, phosphorylation of certain initiation factors by\\nsulting cyclin E-CDK2 further phosphorylates Rb in late G1. cyclin A-CDK2 most likely promotes initiation of DNA\\nWhen cyclin E-CDK2 accumulates to a critical threshold replication and prevents reassembly of pre-replication com-\\nlevel, further phosphorylation of Rb by cyclin E-CDK2 con- plexes until the cell passes through mitosis, thereby assuring\\ntinues even when cyclin D-CDK4/6 activity is removed. This that replication from each origin occurs only once during\\nis one of the principle biochemical events responsible for pas- each cell cycle. In metazoans, a second small protein, gemi-\\nsage through the restriction point. At this point, further nin, contributes to the inhibition of re-initiation at origins\\nphosphorylation of Rb by cyclin E-CDK2 occurs even when until cells complete a full cell cycle.\\nmitogens are withdrawn and cyclin D and CDK4/6 levels The principle mammalian CDK in G and mitosis is\\n2\\nfall. Since E2F stimulates its own expression and that of cy- CDK1 (see Figure 21-28). This CDK, which is highly ho-\\nclin E and CDK2, positive cross-regulation of E2F and cyclin mologous with S. pombeCdc2, associates with cyclins A and\\nE-CDK2 produces a rapid rise of both activities in late G1.  B. The mRNAs encoding either of these mammalian cyclins\\nAs they accumulate, S-phase cyclin-CDK and mitotic  can promote meiotic maturation when injected into Xenopus\\ncyclin-CDK complexes maintain Rb protein in the phospho- oocytes arrested in G (see Figure 21-6), demonstrating that\\n2\\nrylated state throughout the S, G2, and early M phases. After they function as mitotic cyclins. Thus mammalian cyclin \\ncells complete anaphase and enter early G1or G0, the fall in A-CDK1 and cyclin B-CDK1 are functionally equivalent to\\ncyclin-CDK levels leads to dephosphorylation of Rb by un- the S. pombeMPF (mitotic cyclin-CDK). The kinase activ-\\nopposed phosphatases. As a consequence, hypophosphory- ity of these mammalian complexes also appears to be regu-\\nlated Rb is available to inhibit E2F activity during early G1of lated by proteins analogous to those that control the activity\\nthe next cycle and in G0-arrested cells. of the S. pombeMPF (see Figure 21-14). The inhibitory\\nphosphate on CDK1 is removed by Cdc25C phosphatase,\\nwhich is analogous to S. pombeCdc25 phosphatase. \\nCyclin A Is Required for DNA Synthesis \\nIn cycling mammalian cells, cyclin B is first synthesized\\nand CDK1 for Entry into Mitosis\\nlate in the S phase and increases in concentration as cells pro-\\nHigh levels of E2Fs activate transcription of the cyclin A ceed through G , peaking during metaphase and dropping\\n2\\ngene as mammalian cells approach the G → S transition. after late anaphase. This parallels the time course of cyclin B\\n1 \\n(Despite its name, cyclin A is a B-type cyclin, not a G cyclin; expression in Xenopuscycling egg extracts (see Figure 21-9).\\n1\\n34 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nIn human cells, cyclin B first accumulates in the cytosol and ■ Mammalian cells use several CDKs and cyclins to regu-\\nthen enters the nucleus just before the nuclear envelope late passage through the cell cycle. Cyclin D-CDK4/6 func-\\nbreaks down early in mitosis. Thus MPF activity is con- tion in mid to late G ; cyclin E-CDK2 in late G and early\\n1 1\\ntrolled not only by phosphorylation and dephosphorylation S; cyclin A-CDK2 in S; and cyclin A/B-CDK1 in G and\\n2\\nbut also by regulation of the nuclear transport of cyclin B. M through anaphase (see Figure 21-28).\\nIn fact, cyclin B shuttles between the nucleus and cytosol,\\n■ Unphosphorylated Rb protein binds to E2Fs, converting\\nand the change in its localization during the cell cycle results\\nthem into transcriptional repressors. Phosphorylation of\\nfrom a change in the relative rates of import and export. As\\nRb by cyclin D-CDK4/6 in mid G liberates E2Fs to acti-\\nin Xenopuseggs and S. cerevisiae,cyclins A and B are poly- 1\\nvate transcription of genes encoding cyclin E, CDK2, and\\nubiquitinated by the anaphase-promoting complex (APC)\\nother proteins required for the S phase. E2Fs also auto-\\nduring late anaphase and then are degraded by proteasomes\\nstimulate transcription of their own genes. \\n(see Figure 21-2, step 9).\\n■ Cyclin E-CDK2 further phosphorylates Rb, further ac-\\ntivating E2Fs. Once a critical level of cyclin E-CDK2 has\\nTwo Types of Cyclin-CDK Inhibitors Contribute \\nbeen expressed, a positive feedback loop with E2F results\\nto Cell-Cycle Control in Mammals \\nin a rapid rise of both activities that drives passage through\\nAs noted above, three related CIPs—p21CIP, p27KIP2, and the restriction print (see Figure 21-30). \\np57KIP2—inhibit cyclin A-CDK2 activity and must be de-\\n■ The activity of cyclin A-CDK2, induced by high E2F ac-\\ngraded before DNA replication can begin. These same CDK\\ntivity, initially is held in check by CIPs, which function like\\ninhibitory proteins also can bind to and inhibit the other\\nan S-phase inhibitor, and by the presence of an inhibitory\\nmammalian cyclin-CDK complexes involved in cell-cycle\\nphosphate on CDK2. Proteasomal degradation of the in-\\ncontrol. As we discuss later, p21CIP plays a role in the re-\\nhibitors and activation of the Cdc25A phosphatase, as cells\\nsponse of mammalian cells to DNA damage. Experiments\\napproach the G → S transition, generate active cyclin \\nwith knockout mice lacking p27KIP2 have shown that this 1\\nA-CDK2.  This  complex  activates  pre-replication  com-\\nCIP is particularly important in controlling generalized cell\\nplexes to initiate DNA synthesis by a mechanism similar\\nproliferation soon after birth. Although p27KIP2 knockouts\\nto that in S. cerevisiae(see Figure 21-26). \\nare larger than normal, most develop normally otherwise.\\nIn contrast, p57KIP2 knockouts exhibit defects in cell differ- ■ Cyclin A/B-CDK1 induce the events of mitosis through\\nentiation and most die shortly after birth due to defective de- early anaphase. Cyclins A and B are polyubiquitinated by\\nvelopment of various organs.  the  anaphase-promoting  complex  (APC)  during  late\\nA second class of cyclin-CDK inhibitors called INK4s anaphase and then are degraded by proteasomes.\\n(inhibitors of kinase 4) includes several small, closely related\\n■ The activity of mammalian mitotic cyclin-CDK com-\\nproteins that interact only with CDK4 and CDK6 and thus\\nplexes also are regulated by phosphorylation and dephos-\\nfunction specifically in controlling the mid-G phase. Binding\\n1 phorylation similar to the mechanism in S. pombe, with\\nof INK4s to CDK4/6 blocks their interaction with cyclin D\\nthe Cdc25C phosphatase removing inhibitory phosphates\\nand hence their protein kinase activity. The resulting de-\\n(see Figure 21-14). \\ncreased phosphorylation of Rb protein prevents transcrip-\\ntional activation by E2Fs and entry into the S phase. One ■ The activities of mammalian cyclin-CDK complexes also\\nINK4 called p16 is a tumor suppressor, like Rb protein dis- are regulated by CDK inhibitors (CIPs), which bind to and\\ncussed earlier. The presence of two mutant p16 alleles in a inhibit each of the mammalian cyclin-CDK complexes, and\\nlarge fraction of human cancers is evidence for the important INK4 proteins, which block passage through G1by specif-\\nrole of p16 in controlling the cell cycle (Chapter 23).  ically inhibiting CDK4 and CDK6. \\nKEY CONCEPTS OF SECTION 21.6 21.7 Checkpoints in Cell-Cycle\\nCell-Cycle Control in Mammalian Cells Regulation\\n■ Various  polypeptide  growth  factors  called  mitogens Catastrophic genetic damage can occur if cells progress to\\nstimulate cultured mammalian cells to proliferate by in-\\nthe next phase of the cell cycle before the previous phase\\nducing expression of early-response genes. Many of these\\nis properly completed. For example, when S-phase cells\\nencode transcription factors that stimulate expression of\\nare induced to enter mitosis by fusion to a cell in mitosis,\\ndelayed-response genes encoding the G CDKs, G cyclins,\\n1 1 the MPF present in the mitotic cell forces the chromo-\\nand E2F transcription factors.\\nsomes of the S-phase cell to condense. However, since the\\n■ Once cells pass the restriction point, they can enter the replicating chromosomes are fragmented by the conden-\\nS phase and complete S, G , and mitosis in the absence of sation process, such premature entry into mitosis is dis-\\n2\\ngrowth factors. astrous for a cell. \\n21.7 • Checkpoints in Cell-Cycle Regulation 35\\nAnother example concerns attachment of kineto- !FIGURE 21-31\\nchores to microtubules of the mitotic spindle during Abnormal Nondisjunction.This\\nspindle abnormality occurs when\\nmetaphase. If anaphase is initiated before both kine-\\nchromosomes segregate \\ntochores of a replicated chromosome become attached to micro-\\nin anaphase before the\\ntubules from opposite spindle poles, daughter cells are produced\\nAnaphase kinetochore of each sister\\nthat have a missing or extra chromosome (Figure 21-31). When\\nchromatid has attached \\nthis process, called nondisjunction,occurs during the meiotic di- FPO\\nto microtubules (red lines)\\nvision that generates a human egg, Down syndrome can occur\\nfrom the opposite spindle\\nfrom trisomy of chromosome 21, resulting in developmental ab-\\npoles. As a result, one\\nnormalities and mental retardation. ❚ daughter cell contains two\\ncopies of one chromosome,\\nTo minimize the occurrence of such mistakes in cell-cycle Cytokinesis\\nwhile the other daughter\\nevents, a cell’s progress through the cycle is monitored at sev-\\ncell lacks that chromosome.\\neral checkpoints (Figure 21-32). Control mechanisms that\\n[Adapted from A. Murray and \\noperate at these checkpoints ensure that chromosomes are\\nT. Hunt, 1993, The Cell Cycle:\\nintact and that each stage of the cell cycle is completed before\\nAn Introduction,W. H. Freeman\\nthe following stage is initiated. Our understanding of these\\nand Company.]\\ncontrol mechanisms at the molecular level has advanced con- Missing Extra\\nsiderably in recent years. chromosome chromosome\\n3\\n2 Chromosome-segregation\\nSpindle-assembly checkpoint\\ncheckpoint\\nAPC-Cdh1\\nMad2\\npolyubiquitination Cdc14 Sic1\\nof B-type cyclins\\nAPC-Cdc20\\nATM/R ATM/R\\npolyubiquitination\\nTelophase\\nof securin \\n4a\\n4d\\np53 p53 DNA-damage\\nDNA-damage\\nAnaphase checkpoint\\ncheckpoint\\np21CIP M G1 Cyclin D−CDK4/6 p21CIP\\nFPO\\nCyclin A/B−CDK1 M-phase\\nentry\\nG\\nCdc25C 2\\nS-phase Cyclin E/A−CDK2\\nS\\nentry\\n1\\nChk1\\nUnreplicated-DNA p21CIP Cdc25A\\ncheckpoint Cyclin A−CDK2\\nATR 4b\\np53\\nDNA-damage\\np21CIP Cdc25A\\ncheckpoint\\n4c ATM/R Chk1/2\\np53\\nDNA-damage\\ncheckpoint\\nATM/R Chk1/2\\n▲ FIGURE 21-32 Overview of checkpoint controls in the Cdc14 phosphatase from nucleoli, thereby blocking activation of\\ncell cycle.The unreplicated-DNA checkpoint (1) prevents the APC specificity factor (Cdh1) required for polyubiquitination of\\nactivation of cyclin A-CDK1 and cyclin B-CDK1 (i.e., mitosis- B-type cyclins as well as induction of Sic1. As a result, the\\npromoting factor, MPF) by activation of an ATR-Chk1 protein decrease in MPF activity required for the events of telophase\\nkinase cascade that phosphorylates and inactivates Cdc25C, does not occur. In the initial phase of the DNA-damage\\nthereby inhibiting entry into mitosis. In the spindle-assembly checkpoint (4), the ATM or ATR protein kinase (ATM/R) is\\ncheckpoint (2), Mad2 and other proteins inhibit activation of the activated. The active kinases then trigger two pathways: the Chk-\\nAPC specificity factor (Cdc20) required for polyubiquitination of Cdc25A pathway (4a and 4c), blocking entry into or through the\\nsecurin, thereby preventing entry into anaphase. The S phase, and the p53-p21CIP pathway, leading to arrest in G , S,\\n1\\nchromosome-segregation checkpoint (3) prevents release of the and G (4a–4b). See text for further discussion.\\n2\\n36 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nThe Presence of Unreplicated DNA Prevents lived activated form that can interact with and inhibit\\nEntry into Mitosis Cdc20. Microtubule attachment prevents this activation of\\nMad2. Consequently, once all kinetochore complexes bind\\nCells that fail to replicate all their chromosomes do not enter\\na microtubule, generation of the activated form of Mad2\\nmitosis. Operation of the unreplicated-DNA checkpointcon-\\nceases, the inhibition of Cdc20 is relieved, and Cdc20 is free\\ntrol involves the recognition of unreplicated DNA and inhi-\\nto direct the APC to polyubiquitinate securin, thereby initi-\\nbition of MPF activation (see Figure 21-32,  1). Recent\\nating the onset of anaphase.\\ngenetic studies in S. pombeand biochemical studies with\\nXenopusegg extracts suggest that the ATR and Chk1 protein\\nkinases, which also function in the DNA-damage check- Proper Segregation of Daughter Chromosomes Is\\npoint, inhibit entry into mitosis by cells that have not com- Monitored by the Mitotic Exit Network \\npleted DNA synthesis. \\nOnce chromosomes have segregated properly, telophase\\nThe association of ATR with replication forks is thought\\ncommences. The various events of telophase and subsequent\\nto activate its protein kinase activity, leading to the phos-\\ncytokinesis, collectively referred to as the exit from mitosis,\\nphorylation and activation of the Chk1 kinase. Active Chk1\\nrequire inactivation of MPF. As discussed earlier, dephos-\\nthen phosphorylates and inactivates the Cdc25 phosphatase\\nphorylation of the APC specificity factor Cdh1 by the Cdc14\\n(Cdc25C in vertebrates), which normally removes the in-\\nphosphatase leads to degradation of mitotic cyclins and loss\\nhibitory phosphate from CDKs that function during mito-\\nof MPF activity late in anaphase (see Figure 21-10). During\\nsis. As a result, the cyclin A/B-CDK1 complexes remain\\ninterphase and early mitosis, Cdc14 is sequestered in the nu-\\ninhibited and cannot phosphorylate targets required to ini-\\ncleolus and inactivated. The chromosome-segregation check-\\ntiate mitosis. ATR continues to initiate this protein kinase\\npoint, which  monitors  the  location  of  the  segregating\\ncascade until all replication forks complete DNA replication\\ndaughter chromosomes at the end of anaphase, determines\\nand disassemble.\\nwhether active Cdc14 is available to promote exit from mi-\\ntosis (see Figure 21-32, 3). \\nOperation of this checkpoint in S. cerevisiaedepends on a\\nImproper Assembly of the Mitotic Spindle\\nset of proteins referred to as the mitotic exit network.A key\\nPrevents the Initiation of Anaphase \\ncomponent is a small (monomeric) GTPase, called Tem1. This\\nThe  spindle-assembly  checkpoint prevents  entry  into member of the GTPase superfamilyof switch proteins controls\\nanaphase when just a single kinetochore of one chromatid the activity of a protein kinase cascade similarly to the way\\nfails to associate properly with spindle microtubules. Clues Ras controls MAP kinase pathways (Chapter 14). During\\nabout how this checkpoint operates has come from isolation anaphase, Tem1 becomes associated with the spindle pole\\nof yeast mutants in the presence of benomyl, a microtubule- body (SPB) closest to the daughter cell bud. (The SPB, from\\ndepolymerizing  drug.  Low  concentrations  of  benomyl  which spindle microtubules originate, is equivalent to the cen-\\nincrease the time required for yeast cells to assemble the  trosome in higher eukaryotes.) At the SPB, Tem1 is maintained\\nmitotic spindle and attach kinetochores to microtubules. in the inactive GDP-bound state by a specific GAP (GTPase-\\nWild-type cells exposed to benomyl do not begin anaphase accelerating  protein).  The  GEF  (guanosine  nucleotide–\\nuntil these processes are completed and then proceed on exchange factor) that activates Tem1 is localized to the cortex\\nthrough mitosis, producing normal daughter cells. In con- of the bud and is absent from the mother cell. When spindle\\ntrast, mutants defective in the spindle-assembly checkpoint microtubule elongation at the end of anaphase has correctly\\nproceed through anaphase before assembly of the spindle positioned segregating daughter chromosomes into the bud,\\nand attachment of kinetochores is complete; consequently, Tem1 comes into contact with its GEF and is converted into\\nthey mis-segregate their chromosomes, producing abnormal the active GTP-bound state. The terminal kinase in the cas-\\ndaughter cells that die.  cade triggered by Tem1 · GTP then phosphorylates the nu-\\nAnalysis of these mutants identified a protein called Mad2 cleolar anchor that binds and inhibits Cdc14, releasing it into\\nand other proteins that regulate Cdc20, the specificity factor the cytoplasm and nucleoplasm in both the bud and mother\\nrequired to target the APC to securin (see Figure 21-32, 2). cell (Figure 21-33, 1). Once active Cdc14 is available, a cell\\nRecall that APC-mediated polyubiquitination of securin and can proceed through telophase and cytokinesis. If daughter\\nits  subsequent  degradation  is  required  for  entry  into chromosomes fail to segregate into the bud, Tem1 remains in\\nanaphase (see Figure 21-19). Mad2 has been shown to asso- its inactive state, Cdc14 is not released from the nucleolus,\\nciate with kinetochores that are unattached to microtubules. and mitotic exit is blocked (Figure 21-33, 2).\\nExperiments with Mad2 fused to green fluorescent protein In the fission yeast S. pombe,formation of the septum\\n(GFP) indicate that kinetochore-bound Mad2 rapidly ex- that divides daughter cells is regulated by proteins homolo-\\nchanges with a soluble form of Mad2. Current models pro- gous to those that constitute the mitotic exit network in \\npose that when Mad2 associates with a kinetochore complex S. cerevisiae. Genes encoding similar proteins also have been\\nthat is not bound by a microtubule, it is converted to a short- found in higher organisms where the homologs probably\\n21.7 • Checkpoints in Cell-Cycle Regulation 37\\nTem1-GEF Arrest in G and S prevents copying of damaged bases, which\\n1\\nBud would fix mutations in the genome. Replication of damaged\\nTem1 . GDP\\nDNA also promotes chromosomal rearrangements that can\\n(inactive)\\nTem1-GAP GDP Linker protein contribute to the onset of cancer. Arrest in G2allows DNA\\ndouble-stranded breaks to be repaired before mitosis. If a\\nNucleus\\nNucleolus double-stranded break is not repaired, the broken distal por-\\nSpindle\\nCdc14 microtubule tion of the damaged chromosome is not properly segregated\\n(inactive) because it is not physically linked to a centromere, which is\\nSPB\\nMother pulled toward a spindle pole during anaphase. \\nAs we discuss in detail in Chapter 23, inactivation of\\nProper chromosome Improper chromosome\\nsegregation segregation tumor-suppressor genes contributes to the development of\\n1 2\\ncancer. The proteins encoded by several tumor-suppressor\\ngenes, including ATMand Chk2,normally function in the\\nGTP Tem1 . GTP DNA-damage checkpoint. Patients with mutations in both\\n(active) copies of ATMor Chk2develop cancers far more frequently\\nFPO\\nthan normal. Both of these genes encode protein kinases. \\nDNA damage due to UV light somehow activates the\\nATM kinase, which phosphorylates Chk2, thereby activating\\nG\\nCdc14 PD its kinase activity. Activated Chk2 then phosphorylates the\\n(active) Cdc25A phosphatase, marking it for polyubiquitination by\\nan as-yet undetermined ubiquitin ligase and subsequent pro-\\nteasomal degradation. Recall that removal of the inhibitory\\nphosphate from mammalian CDK2 by Cdc25A is required\\nExit from mitosis Late mitotic arrest for onset of and passage through the S phase mediated by cy-\\nclin E-CDK2 and cyclin A-CDK2. Degradation of Cdc25A\\n▲ FIGURE 21-33 Operation of the chromosome-segregation\\nresulting from activation of the ATM-Chk2 pathway in G\\ncheckpoint.In S. cerevisiae,Cdc14 phosphatase activity is 1\\nor S-phase cells thus leads to G or S arrest (see Figure 21-32, \\nrequired for the exit from mitosis. (Top) During interphase and 1\\n4b and 4c). A similar pathway consisting of the protein \\nearly mitosis, Cdc14 is sequestered and inactivated in the\\nnucleolus.Inactive Tem1 · GDP (purple) associates with the kinases ATR and Chk1 leads to phosphorylation and poly-\\nspindle pole body (SPB) nearest to the bud early in anaphase ubiquitination of Cdc25A in response to !-radiation. As dis-\\nwith the aid of a linker protein (green) and is maintained in the cussed earlier for the unreplicated-DNA checkpoint, Chk1\\ninactive state by a specific GAP (GTPase-accelerating protein). If also inactivates Cdc25C, preventing the activation of CDK1\\nchromosome segregation occurs properly (1), extension of the and entry into mitosis.\\nspindle microtubules inserts the daughter SPB into the bud, Another tumor-suppressor protein, p53, contributes to\\ncausing Tem1 to come in contact with a specific GEF (guanine arrest of cells with damaged DNA. Cells with functional p53\\nnucleotide–exchange factor) localized to the cortex of the bud arrest in G and G when exposed to !-irradiation, whereas\\n1 2\\n(brown). This GEF converts inactive Tim1 · GDP to active \\ncells lacking functional p53 do not arrest in G . Although the\\nTem1 · GTP, which triggers a protein kinase cascade leading to 1\\np53 protein is a transcription factor, under normal condi-\\nrelease of active Cdc14 and exit from mitosis. If the spindle\\ntions it is extremely unstable and generally does not accu-\\napparatus fails to place the daughter SPB in the bud (2), Tem1\\nmulate to high enough levels to stimulate transcription. The\\nremains in the inactive GDP-bound state and Cdc14 remains\\ninstability of p53 results from its polyubiquitination by a\\nassociated with nucleoli. Arrest in late mitosis results. [Adapted\\nubiquitin ligase called Mdm2 and subsequent proteasomal\\nfrom G. Pereira and E. Schiebel, 2001, Curr. Opin. Cell Biol.13:762.]\\ndegradation. The rapid degradation of p53 is inhibited by\\nATM and probably ATR, which phosphorylate p53 at a site\\nfunction in an analogous checkpoint that leads to arrest in that interferes with binding by Mdm2. This and other mod-\\nlate mitosis when daughter chromosomes do not segregate ifications of p53 in response to DNA damage greatly increase\\nproperly.  its ability to activate transcription of specific genes that help\\nthe cell cope with DNA damage. One of these genes encodes\\np21CIP, a generalized CIP that binds and inhibits all mam-\\nCell-Cycle Arrest of Cells with Damaged DNA\\nmalian cyclin-CDK complexes. As a result, cells are arrested\\nDepends on Tumor Suppressors\\nin G and G until the DNA damage is repaired and p53 and\\n1 2\\nThe DNA-damage checkpointblocks progression subsequently p21CIPlevels fall (see Figure 21-32, 4a–4d). \\nthrough the cell cycle until the damage is repaired. Under some circumstances, such as when DNA damage\\nDamage to DNA can result from chemical agents is extensive, p53 also activates expression of genes that lead\\nand from irradiation with ultraviolet (UV) light or !-rays. to apoptosis, the process of programmed cell death that \\n38 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nnormally occurs in specific cells during the development of \"FIGURE 21-34 Meiosis. Premeiotic cells have two copies\\nmulticellular animals. In vertebrates, the p53 response of each chromosome (2n), one derived from the paternal parent\\nevolved to induce apoptosis in the face of extensive DNA and one from the maternal parent. For simplicity, the paternal\\ndamage, presumably to prevent the accumulation of multiple and maternal homologs of only one chromosome are\\nmutations that might convert a normal cell into a cancer cell. diagrammed. Step  1: All chromosomes are replicated during the\\nS phase before the first meiotic division, giving a 4n\\nThe dual role of p53 in both cell-cycle arrest and the induc-\\nchromosomal complement. Cohesin complexes (not shown) link\\ntion of apoptosis may account for the observation that nearly\\nthe sister chromatids composing each replicated chromosome\\nall cancer cells have mutations in both alleles of the p53gene\\nalong their full lengths. Step  2: After chromosomes condense\\nor in the pathways that stabilize p53 in response to DNA\\nduring the first meiotic prophase, replicated homologs become\\ndamage (Chapter 23). The consequences of mutations in\\npaired as the result of at least one crossover event between a\\np53, ATM,and Chk2provide dramatic examples of the sig-\\npaternal and a maternal chromatid. This pairing of replicated\\nnificance of cell-cycle checkpoints to the health of a multi-\\nhomologous chromosomes is called synapsis. At metaphase,\\ncellular organism. ❚ shown here, both chromatids of one chromosome associate with\\nmicrotubules emanating from one spindle pole, but each member\\nof a homologous chromosome pair associates with microtubules\\nKEY CONCEPTS OF SECTION 21.7 emanating from opposite poles. Step  3: During anaphase of\\nmeiosis I, the homologous chromosomes, each consisting of two\\nCheckpoints in Cell-Cycle Regulation\\nchromatids, are pulled to opposite spindle poles. Step  4:\\n■ Checkpoint controls function to ensure that chromo- Cytokinesis yields the two daughter cells (now 2n), which enter\\nsomes are intact and that critical stages of the cell cycle meiosis II without undergoing DNA replication. At metaphase of\\nare completed before the following stage is initiated.  meiosis II, shown here, the chromatids composing each\\nreplicated chromosome associate with spindle microtubules from\\n■ The unreplicated-DNA checkpoint operates during S and opposite spindle poles, as they do in mitosis. Steps  5 and  6:\\nG to prevent the activation of MPF before DNA synthe-\\n2 Segregation of chromatids to opposite spindle poles during the\\nsis is complete by inhibiting the activation of CDK1 by second meiotic anaphase followed by cytokinesis generates\\nCdc25C (see Figure 21-32, 1). haploid germ cells (1n) containing one copy of each chromosome\\n■ The spindle-assembly checkpoint, which prevents pre- (referred to as chromatids earlier).\\nmature initiation of anaphase, utilizes Mad2 and other pro-\\nteins to regulate the APC specificity factor Cdc20 that tar-\\ngets securin for polyubiquitination (see Figures 21-32, 2, 21.8 Meiosis: A Special Type \\nand 21-19).\\nof Cell Division \\n■ The chromosome-segregation checkpoint prevents telophase\\nand cytokinesis until daughter chromosomes have been prop- In nearly all eukaryotes, meiosisgenerates haploid germ cells\\nerly segregated, so that the daughter cell has a full set of chro- (eggs and sperm), which can then fuse to generate a diploid\\nmosomes (see Figure 21-32, 3). zygote (Figure 21-34). During meiosis, a single round of\\n■ In the chromosome-segregation checkpoint, the small DNA replication is followed by two cycles of cell division,\\nGTPase Tem1 controls the availability of Cdc14 phos- termed meiosis I and meiosis II. Crossing over of chro-\\nphatase, which in turn activates the APC specificity factor matids, visible in the first meiotic metaphase, produces \\nCdh1 that targets B-type cyclins for degradation, causing recombination between parental chromosomes. This in-\\ninactivation of MPF (see Figure 21-10). creases the genetic diversity among the individuals of a\\nspecies. During meiosis I, both chromatids of each homolo-\\n■ The DNA-damage checkpoint arrests the cell cycle in re-\\ngous chromosome segregate together to opposite spindle\\nsponse to DNA damage until the damage is repaired. Three\\npoles, so that each of the resulting daughter cells contains\\ntumor-suppressor proteins (ATM/ATR, Chk1, and p53) are\\none homologous chromosome consisting of two chromatids.\\ncritical to this checkpoint.\\nDuring meiosis II, which resembles mitosis, the chromatids\\n■ Activation of the ATM or ATR protein kinases in re- of one chromosome segregate to opposite spindle poles, gen-\\nsponse to DNA damage due to UV light or !-irradiation erating haploid germ cells. Meiosis generates four haploid\\nleads to arrest in G1 and the S phase via a pathway that germ cells from one diploid premeiotic cell.\\nleads to loss of Cdc25A phosphatase activity. A second\\npathway from activated ATM/R stabilizes p53, which stim-\\nulates expression of p21CIP. Subsequent inhibition of mul- Repression of G1 Cyclins and Meiosis-Specific\\ntiple CDK-cyclin complexes by p21CIP causes prolonged Ime2 Prevents DNA Replication in Meiosis II \\narrest in G and G (see Figure 21-32, 4a–4d). \\n1 2 In S. cerevisiaeand S. pombe, depletion of nitrogen and car-\\n■ In response to extensive DNA damage, p53 also acti- bon sources induces diploid cells to undergo meiosis, yielding\\nvates genes that induce apoptosis. haploid spores (see Figure 1-5). This process is analogous to\\n21.8 • Meiosis: A Special Type of Cell Division 39\\nPaternal the formation of germ cells in higher eukaryotes. Multiple\\nhomolog yeast mutants that cannot form spores have been isolated,\\nPremeiotic\\ncell (2n) Maternal and the wild-type proteins encoded by these genes have been\\nhomolog analyzed. These studies have identified specialized cell-cycle\\nproteins that are required for meiosis. \\nUnder starvation conditions, expression of G cyclins\\n1 1\\n(Cln1/2/3) in S. cerevisiaeis repressed, blocking the normal\\nprogression of G in cells as they complete mitosis. Instead, a\\n1\\nset of early meiotic proteins are induced. Among these is\\nReplicated\\nIme2,a protein kinase that performs the essential G cyclin-\\nchromosomes 1\\n(4n) CDK function of phosphorylating the S-phase inhibitor Sic1,\\nleading to release of active S-phase cyclin-CDK complexes\\nand the onset of DNA replication in meiosis I (see Figure \\n2 21-34, step 1). The absence of Ime2 expression during meio-\\nsis II and the continued repression of Cln expression account\\nfor the failure to replicate DNA during the second meiotic di-\\nCrossing over vision (steps 5 and 6).\\nand synapsis\\n(metaphase I)\\nCrossing Over and Meiosis-Specific Rec8 \\ns I Are Necessary for Specialized Chromosome\\nsi 3\\no Segregation in Meiosis I \\nei\\nM\\nRecall that in mitosis, sister chromatids replicated during \\nthe S phase are initially linked by cohesin complexes at mul-\\ntiple positions along their full length (see Figure 21-19, left). \\nAnaphase I\\nAs chromosomes condense, cohesin complexes become re-\\nstricted to the region of the centromere, and at metaphase, \\nthe sister chromatids composing each (replicated) chromo-\\nsome associate with microtubules emanating from opposite \\nspindle poles. Although motor proteins pull sister chro-\\n4 matids toward opposite spindle poles, their movement \\ninitially is resisted by the cohesin complexes linking them \\nFPO at the centromere. The subsequent separase-catalyzed cleav-\\nDaughter cells\\nage of the Scc1 cohesin subunit permits movement of \\nin metaphase II\\n(2n) the chromatids toward the spindle poles to begin, herald-\\ning the onset of anaphase (Figure 21-35a; see also Figure \\n21-19, right).\\n5 In metaphase of meiosis I, both sister chromatids in one\\n(replicated) chromosome associate with microtubules ema-\\nnating from the samespindle pole, rather than from opposite\\npoles as they do in mitosis. Two physical links between ho-\\ns II mologous chromosomes are thought to resist the pulling\\nosi Anaphase II force of the spindle until anaphase: (a) crossing over between\\nei chromatids, one from each pair of homologous chromo-\\nM\\nsomes, and (b) cohesin cross-links between chromatids distal\\nto the crossover point. \\nEvidence for the role of crossing over in meiosis in \\n6 S. cerevisiaecomes from the observation that when re-\\ncombination is blocked by mutations in proteins essential\\nfor the process, chromosomes segregate randomly during\\nmeiosis I; that is, homologous chromosomes do not nec-\\nessarily segregate to opposite spindle poles. Such segrega-\\ntion to opposite spindle poles normally occurs because\\nboth chromatids of homologous chromosome pairs associ-\\n1n 1n 1n 1n ate with spindle fibers emanating from opposite spindle\\n40 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\n(a) Mitosis chromatid linking through cohesin complexes contribute\\nto synapsis in meiosis I.\\nMetaphase Anaphase\\nDuring meiosis I, the cohesin cross-links between chro-\\nmosome arms are cleaved by separase, allowing the homol-\\nogous chromosomes to separate, but cohesin complexes at\\nthe centromere remain linked (Figure 21-35b, top). The\\nSeparase\\nmaintenance of centromeric cohesion during meiosis I is\\nnecessary for the proper segregation of chromatids during\\nScc1 meiosis II. Recent studies with a S. pombemutant have\\nshown that a specialized cohesin subunit, Rec8,maintains\\ncentromeric cohesion between sister chromatids during\\n(b) Meiosis meiosis I. Expressed only during meiosis, Rec8 is homolo-\\nMetaphase I Anaphase I gous to Scc1, the cohesin subunit that forms the actual\\nbridge between sister chromatids in mitosis (see Figure 21-\\n19). Immunolocalization experiments in S. pombehave re-\\nvealed that during early anaphase of meiosis I, Rec8 is lost\\nfrom chromosome arms but is retained at centromeres.\\nSeparase\\nFPO\\nHowever, during early anaphase of meiosis II, centromeric\\nRec8 is degraded by separase, so the chromatids can segre-\\ngate, as they do in mitosis (Figure 21-35b, bottom). A spe-\\nMetaphase II Anaphase II cific protein expressed during meiosis I, but not during\\nmeiosis II, protects centromeric Rec8 from separase cleav-\\nage in meiosis I. \\nS. cerevisiaeRec8 has been shown to localize and function\\nsimilarly to S. pombeRec8.Homologs of Rec8 also have been\\nSeparase\\nidentified in higher organisms, and RNA interference (RNAi)\\nexperiments in C. elegans(Chapter 9) indicate that the Rec8\\nhomolog in that organism has a similar function. Recent \\n▲ FIGURE 21-35 Cohesin function during mitosis and micromanipulation experiments during grasshopper sper-\\nmeiosis.(a) During mitosis, sister chromatids generated by DNA matogenesis support the hypothesis that kinetochore-bound\\nreplication in the S phase are initially associated by cohesin\\nproteins protect centromeric Rec8 from cleavage during\\ncomplexes along the full length of the chromatids. During\\nmeiosis I but not during meiosis II and also direct kinetochore\\nchromosome condensation, cohesin complexes (gold) become\\nattachment to microtubules emanating from the correct spin-\\nrestricted to the region of the centromere at metaphase, as\\ndle pole (Figure 21-36). Thus crossing over, Rec8, and spe-\\ndepicted here. Once separase cleaves the Scc1 cohesin subunit,\\ncial kinetochore-associated proteins appear to function in\\nsister chromatids can separate, marking the onset of anaphase\\nmeiosis in all eukaryotes. \\n(see Figure 21-19). (b) In metaphase of meiosis I, crossing over\\nRecent DNA microarrayanalyses in S. cerevisiaehave re-\\nbetween maternal and paternal chromatids produces synapsis of\\nhomologous parental chromosomes. The chromatids of each vealed other proteins that are required for meiosis. As dis-\\nreplicated chromosome are cross-linked by cohesin complexes cussed  in  other  chapters,  researchers  can  monitor\\nalong their full length. Rec8, a meiosis-specific homolog of Scc1, transcription of thousands of genes, indeed entire genomes,\\nis cleaved in chromosome arms but not in the centromere, with DNA microarrays (see Figure 9-35). One of the multi-\\nallowing homologous chromosome pairs to segregate to ple genes found to be expressed in S. cerevisiaecells during\\ndaughter cells. Centromeric Rec8 is cleaved during meiosis II, meiosis but not during mitosis is MAM1,which encodes a\\nallowing individual chromatids to segregate to daughter cells. protein called monopolin.Subsequent gene-specific mutage-\\n[Modified from F. Uhlmann, 2001, Curr. Opin. Cell Biol.13:754.] nesis studies revealed that deletion of MAM1causes sister\\nchromatids in metaphase of meiosis I to associate with the\\nfirst meiotic spindle as though they were mitotic chromatids\\npoles (see Figure 21-34, step 3). This in turn requires that or chromatids in meiosis II. That is, kinetochores of the sis-\\nhomologous chromosomes pair during meiosis I, a process ter chromatids composing a single (replicated) chromosome\\ncalled synapsisthat can be visualized microscopically in attached to microtubules emanating from opposite spindle\\neukaryotes with large chromosomes. Consequently, the poles rather than from the same spindle pole. This result in-\\nfinding that mutations that block recombination also dicates that monopolin is required for formation of a spe-\\nblock proper segregation in meiosis I implies that recom- cialized kinetochore in meiosis I responsible for the unique\\nbination is required for synapsis in S. cerevisiae.In higher co-orientation of sister chromatids of synapsed homologous\\neukaryotes, processes in addition to recombination and chromosomes in the first meiotic division.\\n21.8 • Meiosis: A Special Type of Cell Division 41\\nUnmanipulated Manipulated\\nmeiosis I meiosis I\\nsynapsed synapsed\\nchromosomes chromosomes\\nManipulated Unmanipulated\\nmeiosis II meiosis II\\nchromosomes chromosomes\\nMeiosis I spindle Meiosis II spindle\\n70 min\\nFPO\\n▲EXPERIMENTAL FIGURE 21-36 Anaphase movements pair toward one spindle pole and the other homologous pair\\nand cohesion of meiotic chromosomes are determined by toward the other) whether attached to the meiosis I spindle (left)\\nproteins associated with the chromosomes. Grasshopper or meiosis II spindle (right).Similarly, the meiosis II\\nspermatocytes in meiosis I and II were fused so that both types chromosomes (red) separated normally (i.e., one chromatid\\nof spindles with their associated chromosomes were present in a toward one spindle pole and the other chromatid toward the\\nsingle fused cell. Then a micromanipulation needle was used to opposite pole) independent of which spindle they were attached\\nmove some meiosis I chromosomes (blue) and meiosis II to. These results indicate that the association of kinetochores\\nchromosomes (red) from one spindle to another; other with spindle microtubules and the stability of cohesins linking\\nchromosomes were left attached to their normal spindles. After chromosomes are determined by factors that are associated with\\n70 minutes, both spindles with their attached chromosomes had the chromosomes and not by the spindles or soluble\\ncompleted anaphase movements. The synapsed meiosis I components of cells in meiosis I and II. [See L. V. Paliulis and R. B.\\nchromosomes (blue) separated normally (i.e., one homologous Nicklas, 2000, J. Cell Biol.150:1223.]\\nforms the function of G cyclin-CDK complexes in pro-\\n1\\nKEY CONCEPTS OF SECTION 21.8 moting initiation of DNA replication during meiosis I.\\nDNA replication does not occur during meiosis II because\\nMeiosis: A Special Type of Cell Division \\nneither Ime2 nor G cyclins are expressed. \\n1\\n■ Meiosis involves one cycle of chromosome replication fol-\\nlowed by two cycles of cell division to produce haploid germ ■ In S. cerevisiae,recombination (crossing over) between\\nchromatids of homologous parental chromatids and cohesin\\ncells from a diploid premeiotic cell (see Figure 21-34). Dur-\\ncross-links between chromatids distal to the crossover are\\ning meiosis I, replicated homologous chromosomes pair\\nresponsible for synapsis of homologous chromosomes dur-\\nalong their lengths in a process called synapsis.\\ning prophase and metaphase of meiosis I. A specialized \\n■ At least one recombination between chromatids of ho- cohesin subunit, Rec8, replaces the Scc1 cohesin subunit\\nmologous chromosomes almost invariably occurs. \\nduring meiosis. \\n■ Most of the cell-cycle proteins that function in mitoti- ■ During early anaphase of meiosis I, Rec8 in the chro-\\ncally dividing cells also function in cells undergoing meio-\\nmosome arms is cleaved, but a meiosis-specific protein as-\\nsis, but some proteins are unique to meiosis. \\nsociated with the kinetochore protects Rec8 in the region\\n■ In S. cerevisiae, expression of G1 cyclins (Clns) is re- of the centromere from cleavage. As a result, the chro-\\npressed  throughout  meiosis.  Meiosis-specific  Ime2  per- matids of homologous chromosomes remain associated\\n42 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nduring segregation in meiosis I. Cleavage of centromeric trols and checkpoints, new strategies for treating p53-minus\\nRec8 during anaphase of meiosis II allows individual chro- cancers might be possible. For instance, some chemothera-\\nmatids to segregate into germ cells (see Figure 21-35b). peutic agents inhibit microtubule function, interfering with\\nmitosis. Resistant cells selected during the course of treatment\\n■ Monopolin, another meiosis-specific protein, is required\\nmay be defective in the spindle-assembly checkpoint as the re-\\nfor both chromatids of homologous chromosomes to as-\\nsult of mutations in the genes encoding the proteins involved.\\nsociate with microtubules emanating from the same spin-\\nPerhaps the loss of this checkpoint could be turned to thera-\\ndle poles during meiosis I.\\npeutic advantage. Only better understanding of the molecular\\nprocesses involved will tell. ❚\\nPERSPECTIVES FOR THE FUTURE\\nKEY TERMS\\nThe remarkable pace of cell-cycle research over the last 25\\nyears has led to the model of eukaryotic cell-cycle control anaphase-promoting \\noutlined in Figure 21-2. A beautiful logic underlies these mo- complex (APC) 000 interphase 000\\nlecular controls. Each regulatory event has two important\\nAPC specificity  meiosis 000\\nfunctions: to activate a step of the cell cycle and to prepare\\nfactors 000 mitogens 000\\nthe cell for the next event of the cycle. This strategy ensures\\nCdc25 phosphatase 000 mitosis-promoting \\nthat the phases of the cycle occur in the proper order. \\ncheckpoints 000 factor (MPF) 000\\nAlthough the general logic of cell-cycle regulation now\\nseems well established, many critical details remain to be dis- CIPs 000 mitotic cyclin 000\\ncovered. For instance, although researchers have identified cohesins 000 p53 protein 000\\nsome components of the pre-replication complex that must condensin 000 quiescent cells 000\\nbe phosphorylated by S-phase CDK-cyclin complexes to ini-\\ncrossing over 000 Rb protein 000\\ntiate DNA replication, other components remain to be de-\\ncyclin-dependent  restriction point 000\\ntermined. Also still largely unknown are the substrates\\nkinases (CDKs) 000 S-phase inhibitor 000\\nphosphorylated by mitotic cyclin-CDK complexes, leading to\\nchromosome condensation and the remarkable reorganiza- destruction box 000 S phase–phase promoting \\ntion of microtubules that results in assembly of the beauti- DNA-damage  factor (SPF) 000\\nful mitotic spindle. Relatively little is known about how the checkpoint 000 securin 000\\nactivities of Wee1 kinase and Cdc25 phosphatase are con- E2F proteins 000 synapsis 000\\ntrolled; these proteins in turn regulate the kinase activity of\\nG1 cyclins 000 Wee1 protein-tyrosine \\nthe CDK subunit in most cyclin-CDK complexes. \\nkinase 000\\nMuch has been discovered recently about operation of\\nthe cell-cycle checkpoints, but the mechanisms that activate\\nREVIEW THE CONCEPTS\\nATM and ATR in the DNA-damage checkpoint are poorly\\nunderstood. Likewise, much remains to be learned about the\\n1. What strategy ensures that passage through the cell cycle\\ncontrol and mechanism of Mad2 in the spindle-assembly\\nis unidirectional and irreversible? What is the molecular \\ncheckpoint and of Cdc14 in the chromosome-segregation\\nmachinery that underlies this strategy?\\ncheckpoint in higher cells. As we learn in the next chapter,\\nasymmetric cell division plays a critical role in the normal de- 2. When fused with an S-phase cell, cells in which of the\\nvelopment of multicellular organisms. Many questions re- following phases of the cell cycle will initiate DNA replica-\\nmain about how the plane of cytokinesis and the localization tion prematurely—G1? G2? M? Predict the effect of fusing\\nof daughter chromosomes are determined in cells that divide a cell in G1 and a cell in G2 with respect to the timing of \\nasymmetrically. Similarly, the mechanisms that underlie the S phase in each cell. \\nunique segregation of chromatids during meiosis I have not\\n3. In 2001, the Nobel Prize in Physiology or Medicine was\\nbeen elucidated yet. \\nawarded to three cell cycle scientists. Sir Paul Nurse was rec-\\nognized for his studies with the fission yeast S. pombe,in\\nUnderstanding these detailed aspects of cell-cycle\\nparticular for the discovery and characterization of the wee1\\ncontrol will have significant consequences, partic-\\ngene. What is the wee phenotype? What did the characteri-\\nularly for the treatment of cancers. Radiation and\\nzation of the wee1gene tell us about cell cycle control?\\nmany forms of chemotherapy cause DNA damage and cell-\\ncycle arrest in the target cells, leading to their apoptosis. But 4. Tim Hunt shared the 2001 Nobel Prize for his work in the\\nthis induction of apoptosis depends on p53 function. For this discovery and characterization of cyclin proteins in eggs and\\nreason, human cancers associated with mutations of both p53 embryos. What experimental evidence indicates that cyclin B\\nalleles, which is fairly common, are particularly resistant to is required for a cell to enter mitosis? What evidence indicates\\nthese therapies. If more were understood about cell-cycle con- that cyclin B must be destroyed for a cell to exit mitosis? \\nAnalyze the Data 43\\n5. Leeland Hartwell, the third recipient of the 2001 Nobel how the following mutations, which are found in some can-\\nPrize, was acknowledged for his characterization of cell cycle cer cells, lead to a bypass of restriction point controls: \\ncheckpoints in the budding yeast S. cerevisiae. What is a cell (a) overexpression of cyclin D, (b) loss of Rb function, \\ncycle checkpoint? Where do checkpoints occur in the cell (c) infection by retroviruses encoding v-Fos and v-Jun.\\ncycle? How do cell cycle checkpoints help to preserve the \\n13. Individuals with the hereditary disorder ataxia telan-\\nfidelity of the genome?\\ngiectasia suffer from neurodegeneration, immunodeficiency,\\n6. In Xenopus,one of the substrates of MPF is the Cdc25 and increased incidence of cancer. The genetic basis for\\nphosphatase. When phosphorylated by MPF, Cdc25 is acti- ataxia telangiectasia is a loss-of-function mutation in the\\nvated. What is the substrate of Cdc25? How does this infor- ATM gene (ATM \" ataxia telangiectasia-mutated). Name\\nmation explain the autocatalytic nature of MPF as described two substrates of ATM. How does the phosphorylation of\\nin Figure 21-6?  these substrates lead to inactivation of CDKs to enforce cell\\ncycle arrest at a checkpoint?\\n7. Explain how CDK activity is modulated by the follow-\\ning proteins: (a) cyclin, (b) CAK, (c) Wee1, (d) p21. 14. Meiosis and mitosis are overall analogous processes in-\\nvolving many of the same proteins. However, some proteins\\n8. Three known substrates of MPF or kinases regulated by\\nfunction uniquely in each of these cell division events. Ex-\\nMPF are the nuclear lamins, subunits of condensin, and myosin\\nplain the meiosis-specific function of the following: (a) Ime2,\\nlight chain. Describe how the phosphorylation of each of these\\n(b) Rec8, (c) monopolin.\\nproteins affects its function and progression through mitosis.\\n9. Describe the series of events by which the APC promotes\\nthe separation of sister chromatids at anaphase.\\nANALYZE THE DATA\\n10. A common feature of cell cycle regulation is that the\\nevents of one phase ensure progression into a subsequent As genes encoding cyclin-dependent kinases were cloned\\nphase. In S. cerevisiae, Cdc28-Clns catalyze progression from fission yeast, budding yeast, and animal cells, investi-\\nthrough G1, and Cdc28-Clbs catalyze progression through gators working in plant systems designed experiments to de-\\nS/G2/M. Name three ways in which the activity of Cdc28- termine whether plants also possessed CDKs that functioned\\nClns promotes the activation of Cdc28-Clbs. as the catalysts for cell cycle progression. To isolate a cDNA\\nencoding a CDK in maize, investigators aligned the predicted\\n11. For S phase to be completed in a timely manner, DNA\\namino acid sequences of human cdc2, S. pombe cdc2, and\\nreplication initiates from multiple origins in eukaryotes. In \\nS. cerevisiae cdc28genes (see the figure below). Degenerate\\nS. cerevisiae, what role do S-phase CDK-cyclin complexes\\noligonucleotide PCR primers corresponding to the boxed\\nplay to ensure that the entire genome is replicated once and\\namino acids were generated and a DNA fragment of the ex-\\nonly once per cell cycle?\\npected size was synthesized by PCR using these primers and\\n12. What is the functional definition of the restriction point? maize cDNA library as a template. Why were the boxed re-\\nCancer cells typically lose restriction point controls. Explain gions selected as the basis for primer design?\\n44 CHAPTER 21 • Regulating the Eukaryotic Cell Cycle\\nThe PCR product was then used as a probe to screen the Fang, G., H. Yu, and M. W. Kirschner. 1999. Control of mitotic\\nmaize cDNA library, and a full-length cDNA clone was iso- transitions by the anaphase-promoting complex. Philos. Trans. R.\\nSoc. London Ser. B354:1583–1590.\\nlated. The clone was sequenced, and the predicted amino\\nR. Jessberger. 2002. The many functions of SMC proteins in\\nacid sequence was 64% identical to human cdc2and 63%\\nchromosome dynamics. Nature Rev. Mol. Cell Biol.3:67–78.\\nidentical to S. pombe cdc2and toS. cerevisiae cdc28. What\\nY. Masui. 2001. From oocyte maturation to the in vitro cell cy-\\ndoes the level of sequence similarity suggest about the evo- cle: the history of discoveries of Maturation-Promoting Factor (MPF)\\nlution of cdc2genes? and Cytostatic Factor (CSF). Differentiation69:1–17.\\nA complementation experiment was performed with \\nS. cerevisiaecells possessing a temperature-sensitive cdc28 Genetic Studies with S. pombe\\nmutation. Wild-type cells, cdc28ts cells, and cdc28ts cells\\nNurse, P. 2002. Cyclin dependent kinases and cell cycle control\\ntransformed with the maize cdc2cDNA under the influence (Nobel Lecture). Chembiochem.3:596–603.\\nof a strong promoter were grown at the permissive (25 °C) or\\nrestrictive (37 °C) temperatures. Cell proliferation was mon- Molecular Mechanisms for Regulating Mitotic Events\\nitored by the growth of colonies on the culture plates (shown\\nBurke, B., and J. Ellenberg. 2002. Remodeling the walls of the\\nin the figure below).\\nnucleus. Nature Rev. Mol. Cell Biol.3:487–497.\\nNelson, W. J. 2000. W(h)ither the Golgi during mitosis? J. Cell\\nBiol.149:243–248.\\nNigg, E. A. 2001. Mitotic kinases as regulators of cell division\\nand its checkpoints. Nature. Rev. Mol. Cell Biol.2:21–32.\\nGenetic Studies with S. cerevisiae\\nBell, S. P., and A. Dutta. 2002. DNA replication in eukaryotic\\ncells. Ann. Rev. Biochem.71:333–374.\\nDeshaies, R. J. 1999. SCF and Cullin/Ring H2-based ubiquitin\\nligases. Ann. Rev. Cell Devel. Biol.15:435–467.\\nDiffley, J. F., and K. Labib. 2002. The chromosome replication\\ncycle. J. Cell Sci.115:869–872.\\nHartwell, L. H. 2002. Yeast and cancer (Nobel Lecture). Biosci.\\nRep.22:373–394.\\nKelly, T. J., and G. W. Brown. 2000. Regulation of chromosome\\nreplication. Ann. Rev. Biochem.69:829–880.\\nKitagawa, K., and P. Hieter. 2001. Evolutionary conservation be-\\ntween budding yeast and human kinetochores. Nature Rev. Mol. Cell\\nBiol.2:678–687.\\nLei, M., and B. K. Tye. 2001. Initiating DNA synthesis: from re-\\ncruiting to activating the MCM complex. J. Cell Sci.114:1447–1454.\\nNasmyth, K. 2001. Disseminating the genome: joining, resolv-\\ning, and separating sister chromatids during mitosis and meiosis.\\nAnn. Rev. Genet.35:673–745.\\nNasmyth, K. 2002. Segregating sister genomes: the molecular bi-\\nology of chromosome separation. Science297:559–565.\\nUhlmann, F. 2001. Chromosome cohesion and segregation in mi-\\nWhy do the cdc28tscells form colonies at 25 °C but not tosis and meiosis. Curr. Opin. Cell Biol.13:754–761.\\n37 °C? What is the significance of colony formation of the\\ncdc28ts#maize cdc2cells at 37 °C? What does this experi- Cell-Cycle Control in Mammalian Cells\\nment tell us about the functional homology of cyclin- Ekholm, S. V., and S. I. Reed. 2000. Regulation of G(1) cyclin-\\ndependent kinase genes among eukaryotic species? dependent kinases in the mammalian cell cycle. Curr. Opin. Cell Biol.\\n12:676–684.\\nHarper, J. W., J. L. Burton, and M. J. Solomon. 2002. The\\nanaphase-promoting complex: it’s not just for mitosis any more.\\nREFERENCES\\nGenes Devel.16:2179–2206.\\nSears, R. C., and J. R. Nevins. 2002. Signaling networks that\\nOverview of the Cell Cycle and Its Control link cell proliferation and cell fate. J. Biol. Chem.277:11617–11620.\\nNasmyth, K. 2001. A prize for proliferation. Cell107: 689–701. Sherr, C. J. 2001. The INK4a/ARF network in tumour suppres-\\nsion. Nature Rev. Mol. Cell Biol.2:731–737.\\nBiochemical Studies with Oocytes, Eggs, and Early Embryos\\nCheckpoints in Cell-Cycle Regulation\\nDoree, M., and T. Hunt. 2002. From Cdc2 to Cdk1: when did\\nthe  cell  cycle  kinase  join  its  cyclin  partner? J. Cell Sci. 115: Bardin, A. J., and A. Amon. 2001. Men and sin: what’s the dif-\\n2461–2464. ference? Nature Rev. Mol. Cell Biol.2:815–826.\\nReferences 45\\nOsborn, A. J., S. J. Elledge, and L. Zou. 2002. Checking on the Meiosis: A Special Type of  Cell Division\\nfork: the DNA-replication stress-response pathway. Trends Cell Biol.\\nLee, B., and A. Amon. 2001. Meiosis: how to create a special-\\n12:509–516.\\nized cell cycle. Curr. Opin. Cell Biol.13:770–777. \\nPereira, G., and E. Schiebel. 2001. The role of the yeast spindle\\nPetronczki, M., M. F. Siomos, and K. Nasmyth. 2003. Un menage\\npole body and the mammalian centrosome in regulating late mitotic\\na quatre: the molecular biology of chromosome segregation in meio-\\nevents. Curr. Opin. Cell Biol.13:762–769.\\nsis. Cell112:423–440.\\nShah, J. V., and D. W. Cleveland. 2000. Waiting for anaphase:\\nMad2 and the spindle assembly checkpoint. Cell103:997–1000.\\n22\\nCELL BIRTH, \\nLINEAGE, \\nAND DEATH\\nCells being born in the developing cerebellum.All nuclei are\\nlabeled in red;the green cells are dividing and migrating into\\ninternal layers of the neural tissue.[Tal Raveh, Matthew Scott, and\\nJane Johnson.]\\nDuring the evolution of multicellular organisms, new cell lineage ultimately culminates in formation of terminally\\nmechanisms arose to diversify cell types, to coordi- differentiated cells such as skin cells, neurons, or muscle\\nnate their production, to regulate their size and num- cells. Terminal differentiation generally is irreversible, and\\nber, to organize them into functioning tissues, and to the resulting highly specialized cells often cannot divide; they\\neliminate extraneous or aged cells. Signaling between cells survive, carry out their functions for varying lengths of time,\\nbecame even more important than it is for single-celled  and then die. \\norganisms. The mode of reproduction also changed, with Many cell lineages contain intermediate cells, referred to\\nsome cells becoming specialized as germ cells (e.g., eggs, as precursor cellsor progenitor cells¸whose potential to form\\nsperm), which give rise to new organisms, as distinct from all different kinds of differentiated cells is more limited than\\nother body cells, called somatic cells. Under normal condi- that of the stem cells from which they arise.(Although some\\ntions somatic cells will never be part of a new individual. researchers distinguish between precursor and progenitor\\nThe formation of working tissues and organs during  cells, we will use these terms interchangeably.) Once a new\\ndevelopmentof multicellular organisms depends on specific precursor cell type is created, it often produces transcription\\npatterns of mitotic cell division. A series of such cell divisions factors characteristic of its fate. These transcription factors\\nakin to a family tree is called a cell lineage,which traces the\\nprogressive differentiationof cells into specialized cell types.\\nCell lineages are controlled by intrinsic (internal) factors—cells\\nO UTLINE\\nacting according to their history and internal regulators—\\nas well as by extrinsic (external) factors such as cell-cell sig- 22.1 The Birth of Cells \\nnals and environmental inputs (Figure 22-1). A cell lineage\\n22.2 Cell-Type Specification in Yeast\\nbegins with stem cells, unspecialized cells that can poten-\\ntially reproduce themselves and generate more-specialized 22.3 Specification and Differentiation of Muscle\\ncells indefinitely. Their name comes from the image of a\\n22.4 Regulation of Asymmetric Cell Division\\nplant stem, which grows upward, continuing to form more\\nstem, while sending off leaves and branches to the side. A 22.5 Cell Death and Its Regulation\\n1\\n2 CHAPTER 22 • Cell Birth, Lineage, And Death\\n!FIGURE 22-1 Overview of the birth,lineage,and death\\nof cells. Following growth, cells are “born” as the result of\\nsymmetric or asymmetric cell division. (a) The two daughter cells\\nCell growth\\nresulting from symmetric division are essentially identical to each\\nother and to the parental cell. Such daughter cells subsequently (a) (b)\\ncan have different fates if they are exposed to different signals.\\nSymmetric\\nThe two daughter cells resulting from asymmetric division differ\\ndivision Cell\\nfrom birth and consequently have different fates. Asymmetric birth\\ndivision commonly is preceded by the localization of regulatory\\nmolecules (green) in one part of the parent cell. (b) A series of Cell growthFPO\\nsymmetric and/or asymmetric cell divisions, called a cell lineage,\\ngives birth to each of the specialized cell types found in a\\nAsymmetric\\nmulticellular organism. The pattern of cell lineage can be under Cell\\ndivision \\ndeath\\ntight genetic control. Programmed cell death occurs during\\nnormal development (e.g., in the webbing that initially develops\\nwhen fingers grow) and also in response to infection or poison. \\nA series of specific programmed events, called apoptosis, is\\nactivated is these situations.\\nCell lineage\\ncoordinately activate, or repress, batteries of genes that com- produced by such asymmetric cell division may differ in\\nplete the differentiation process. For instance, a few key reg- size, shape, and/or composition, or their genes may be in\\nulatory transcription factors create the different mating types different states of activity or potential activity. The differ-\\nof budding yeast and coordinate the numerous genes neces- ences in these internal signals confer different fates on the\\nsary to turn a precursor cell into a muscle cell, two exam- two cells. \\nples that we discuss in this chapter.  Here we discuss some general features of how different\\nTypically we think of cell fates in terms of the differenti- cell types are generated, culminating with the best-understood\\nated cell types that are formed. A quite different cell fate, complex cell lineage, that of the nematode Caenorhabditis \\nprogrammed cell death,also is absolutely crucial in the for- elegans. In later sections, we focus on examples of the mo-\\nmation and maintenance of many tissues. A precise genetic lecular mechanisms that determine particular cell types in\\nregulatory system, with checks and balances, controls cell yeast, Drosophila, and mammals. \\ndeath just as other genetic programs control cell differentia-\\ntion. In this chapter, then, we consider the life cycle of cells—\\nStem Cells Give Rise to Stem Cells \\ntheir birth, their patterns of division (lineage), and their\\nand to Differentiating Cells \\ndeath. These aspects of cell biology converge with develop-\\nmental biology and are among the most important processes Stem cells, which give rise to the specialized cells composing\\nregulated by various signaling pathways discussed in earlier the tissues of the body, exhibit several patterns of cell divi-\\nchapters.  sion. A stem cell may divide symmetrically to yield two\\ndaughter stem cells identical to itself (Figure 22-2a). Alter-\\nnatively, a stem cell may divide asymmetrically to generate a\\n22.1 The Birth of Cells copy of itself and a derivative stem cell that has more-\\nrestricted capabilities, such as dividing for a limited period of\\nMany descriptions of cell division imply that the parental time or giving rise to fewer types of progeny compared with\\ncell gives rise to two daughter cells that look and behave the parental stem cell (Figure 22-2b). A pluripotent(or multi-\\nexactly like the parental cell, that is, cell division is sym- potent) stem cell has the capability of generating a number of\\nmetric, and the progeny do not change their properties. But different cell types, but not all. For instance, a pluripotent\\nif this were always the case, none of the hundreds of dif- blood stem cell will form more of itself plus multiple types of\\nferentiated cell types would ever be formed. Differences blood cells, but never a skin cell. In contrast, a unipotent\\namong cells can arise when two initially identical daughter stem cell divides to form a copy of itself plus a cell that can\\ncells diverge by receiving distinct developmental or envi- form only one cell type. In many cases, asymmetric division\\nronmental signals. Alternatively, the two daughter cells of a stem cell generates a progenitor cell, which embarks on\\nmay differ from “birth,” with each inheriting different a path of differentiation, or even a terminally differentiating\\nparts of the parental cell (see Figure 22-1). Daughter cells cell (Figure 22-2c, d). \\n22.1 • The Birth of Cells 3\\nproperties. For example, the early mouse embryo passes\\nStem\\ncell through an eight-cell stage in which each cell can give rise\\nto any cell type of the embryo. If the eight cells are experi-\\n(a) mentally separated and individually implanted into a suit-\\nSymmetric able foster mother, each can form a whole mouse with no\\ndivision\\nparts missing. This experiment shows that the eight cells\\nStem\\ncell are all able to form every tissue; that is, they are totipotent.\\nThus the subdivision of body parts and tissue fates among\\nthe early embryonic cells has not irreversibly occurred at\\n(b)\\nAsymmetric the eight-cell stage. At the 16-cell stage, this is no longer\\ndivision\\nMore-restricted true; some of the cells are committed to particular differ-\\npotential stem  entiation paths. \\nFPO\\ncell\\nQuite different specialized cell types can arise from a\\n(c) common precursor cell. A hematopoietic stem cell can give\\nAsymmetric\\nrise to all the multifarious types of blood cell. However, a\\ndivision\\nwhole series of cell divisions is not required; a single divi-\\nProgenitor cell\\nsion of a pluripotent precursor cell can yield distinct prog-\\neny. For instance, lineage studies in which cells are marked\\nby stable infection with a detectable retrovirus have shown\\n(d)\\nAsymmetric that neurons and glial cells arise from a single division of a\\ndivision\\nTerminally particular precursor cell. These cell types are quite differ-\\ndifferentiating ent: neurons propagating and transmitting electrical sig-\\ncell\\nnals and glial cells providing electrical insulation and\\nsupport. The precursor that generates neurons and glial\\n▲FIGURE 22-2 Patterns of stem-cell division. Symmetric\\ncells is not a stem cell, since it is incapable of self-\\ndivision of a stem cell (a) produces two identical daughter cells\\nrenewal; presumably the neuron–glial cell precursor arises\\nwith the same potential as the parental stem cell. Asymmetric\\nfrom a stem cell further back in the lineage. Recent obser-\\ndivision of a stem cell always generates one daughter that is a\\nfunctional duplicate of the parental cell and one daughter that vations have raised the interesting possibility that stem\\nmay be a stem cell with a more-restricted set of possible cells for one tissue may be induced under certain condi-\\nprogeny cells (b), a progenitor (precursor) cell that undergoes tions to act as stem cells for a rather different tissue. As we\\nmore divisions before the progeny begin differentiating (c), or a discuss below, postnatal animals contain stem cells for\\ncell that begins differentiating immediately with no further cell many tissues including the blood, intestine, skin, ovaries\\ndivisions (d). and testes, muscle, liver, and even the adult brain where lit-\\ntle cell division normally occurs. In muscle and liver, stem\\ncells are most important in healing, as relatively little cell\\ndivision occurs in the adult tissues otherwise. \\nThe two critical properties of stem cells that distin-\\nguish them from all other cells are the ability to reproduce\\nthemselves indefinitely by symmetric division, often called\\nself-renewal,and the ability to divide asymmetrically to Cultured Embryonic Stem Cells Can Differentiate \\nform one daughter stem cell identical to itself and one into Various Cell Types\\ndaughter  cell  that  is  different  and  usually  of  more-\\nrestricted potential. In this way, mitotic division of stem Embryonic stem (ES) cells can be isolated from early em-\\ncells preserves a population of undifferentiated cells while bryos and grown in culture (Figure 22-3a). Cultured ES cells\\nsteadily producing a stream of differentiating cells. Al- can differentiate into a wide range of cell types, either in\\nthough some types of precursor cells can divide symmet- vitro or after reinsertion into a host embryo. When grown\\nrically to form more of themselves, they do so only for in suspension culture, human ES cells first differentiate into\\nlimited periods of time. Moreover, in contrast to stem multicellular aggregates, called embryoid bodies, that resem-\\ncells, the asymmetric division of a precursor cell generates ble early embryos. When these are subsequently transferred\\ntwo distinct daughter cells, neither of which is identical to to a solid medium, they grow into confluent cell sheets con-\\nthe parental precursor cell.  taining a variety of differentiated cell types including neural\\nThe fertilized egg, or zygote, is a single cell with the  cells and pigmented and nonpigmented epithelial cells (Fig-\\ncapability to generate all the cell types of the body. Al- ure 22-3b). Under other conditions, ES cells have been in-\\nthough not technically a stem cell because it is not self- duced to differentiate into precursors for various types of\\nrenewing, the zygote does give rise to cells with stem-cell blood cells.\\n4 CHAPTER 22 • Cell Birth, Lineage, And Death\\n!EXPERIMENTAL FIGURE 22-3 (a) (b)\\nCleavage\\nEmbryonic stem (ES) cells can be stage embryo\\nmaintained in culture and form\\ndifferentiated cell types.(a) Human\\nblastocysts are grown from cleavage-stage\\nInner\\nembryos produced by in vitro fertilization.\\ncell mass FPO\\nThe inner cell mass is separated from the Cultured\\nsurrounding extra-embryonic tissues and Blastocoel blastocyst\\nplated onto a layer of fibroblast cells that\\nhelp to nourish the embryonic cells. Trophoblast\\nIndividual cells are replated and form\\nEmbryoid body\\ncolonies of ES cells, which can be\\nmaintained for many generations and can Isolated inner\\ncell mass\\nbe stored frozen. (b) In suspension culture,\\nhuman ES cells differentiate into\\nmulticellular aggregates (embryoid bodies) Fibroblast\\nfeeder cells\\n(top). After embryoid bodies are\\ntransferred to a gelatinized solid medium, FPO\\nthey differentiate further into confluent cell\\nFPO\\nsheets containing a variety of\\ndifferentiated cell types including neural Dissociate cells\\ncells (middle), and pigmented and\\nNeurons\\nnonpigmented epithelial cells (bottom).\\nNew feeder\\n[Parts (a) and (b) adapted from J. S. Odorico  cells\\nReplate\\net al., 2001, Stem Cells19:193–204.]\\nFPO\\nEstablished\\nES cell cultures\\nEpithelial cells\\nThe possibility of using stem cells therapeutically the feasibility of using stem cells for such purposes can be as-\\nto restore or replace damaged tissue is fueling sessed adequately. ❚\\nmuch research on how to recognize and culture\\nthese remarkable cells from embryos and from various tis- Apart from their possible benefit in treating disease, ES\\nsues in postnatal (adult) animals. For example, if neurons cells have already proven invaluable for producing mouse\\nthat produce the neurotransmitter dopamine could be gen- mutants useful in studying a wide range of diseases, devel-\\nerated from stem cells grown in culture, it might be possible opmental mechanisms, behavior, and physiology. By tech-\\nto treat people with Parkinson’s disease who have lost such niques described in Chapter 9, it is possible to eliminate or\\nneurons. For such an approach to succeed, a way must be modify the function of a specific gene in ES cells (see Figure\\nfound to direct a population of embryonic or other stem cells 9-38). If the disrupted gene is not required for growth of ES\\nto form the right type of dopamine-producing neurons, and cells, then the mutated ES cells can be employed to produce\\nrejection by the immune system nust be prevented. In one on- mice with a gene knockout(see Figure 9-39). Analysis of the\\ngoing study in which embryonic neurons were transplanted effects caused by deleting or modifying a gene in this way\\ninto more than 300 Parkinson’s patients, some of the inserted often provides clues about the normal function of the gene\\ncells have survived for more than 12 years and have provided and its encoded protein. \\nsignificant clinical improvement. However, the fetal tissue\\nused in this study is scarce and its use is controversial. Stem\\nTissues Are Maintained by Associated\\ncells grown from very early embryos are another option for\\nPopulations of Stem Cells \\ntreating Parkinson’s disease and perhaps other neurodegen-\\nerative conditions such as Alzheimer’s disease. Similar pos- Many differentiated cell types are sloughed from the body\\nsibilities exist for generating blood, pancreas, and other cell or have life spans that are shorter than that of the organism.\\ntypes. Many important questions must be answered before Disease and trauma also can lead to loss of differentiated\\n22.1 • The Birth of Cells 5\\ncells. Since these cells generally do not divide, they must be continuously regenerate from a stem-cell population located\\nreplenished from nearby stem-cell populations.  deep in the intestinal wall in pits called crypts(Figure 22-4a).\\nOur skin, for instance, is a multilayered epithelium(the The stem cells produce precursor cells that proliferate and dif-\\nepidermis) underlain by a layer of stem cells that give rise ferentiate as they ascend the sides of crypts to form the sur-\\nboth to more of themselves and to keratinocytes,the major face layer of the finger-like gut projections called villi,across\\ncell type in skin. As keratinocytes move toward the outer sur- which intestinal absorption occurs. Pulse-chase labeling ex-\\nface, they become increasingly flattened and filled with ker- periments have shown that the time from cell birth in the\\natin intermediate filaments. It normally takes about 15–30 crypts to the loss of cells at the tip of the villi is only about 2\\ndays for a newly “born” keratinocyte in the lowest layer to to 3 days (Figure 22-4b). Thus enormous numbers of cells\\ndifferentiate and move to the topmost layer. The “cells” must be produced continually to keep the epithelium intact.\\nforming the topmost layer are actually dead and are contin- The production of new cells is precisely controlled: too little\\nually shed from the surface.  division would eliminate villi and lead to breakdown of the in-\\nIn contrast to epidermis, the epithelium lining the small testinal surface; too much division would create an excessively\\nintestine is a single cell thick (Figure 6-4). This thin layer keeps large epithelium and also might be a step toward cancer.\\ntoxins and pathogens from entering our bodies and also trans- Specific signals are required for creating and maintaining\\nports nutrients essential for survival from the intestinal lumen stem-cell populations. In the skin and the intestinal epithelium,\\ninto the body (Chapter 7). The cells of the intestinal epithelium stem-cell growth is regulated in part by !-catenin,a protein\\nthat helps link certain cell-cell junctions to the cytoskeleton\\n(see Figure 6-7) and also functions as a signal transducer in \\n(a)\\nthe Wnt pathway (see Figure 15-32). Overproduction of active\\nDirection of !-catenin leads to excess proliferation of the intestinal epithe-\\ncell migration lium and of hair follicles, which are specialized structures de-\\nrived from skin stem cells and embedded in the epidermis.\\nVillus\\nBlocking the function of !-catenin by interfering with the TCF\\ntranscription factor that it activates abolishes the stem cells in\\nDifferentiated\\nCrypt-villus region the intestine. In the absence of !-catenin function, skin stem\\njunction cells form epidermis, but the hair follicles and hair do not\\nform. It seems likely that Wnt signaling, or at least compo-\\nFPO nents of the intracellular pathway, is critical for forming, main-\\ntaining, or activating stem cells in a variety of tissues. \\nProliferative\\nregion\\nCrypt\\n\"EXPERIMENTAL FIGURE 22-4 Regeneration of the\\nintestinal epithelium from stem cells can be demonstrated in\\nStem cell pulse-chase experiments.(a) Schematic drawing of the lining of\\nregion the small intestine, which contains numerous villi formed from a\\nStem cells column of cells. These epithelial cells are born near the base of\\nPaneth cells\\npits (crypts) located between the villi. Located at the very bottom\\nof crypts are Paneth cells, a type of support cell; just above these\\n(b) are four to six stem cells, which divide about once a day, forming\\nprecursor cells that also actively divide. As the differentiated cells\\nenter the epithelium of a villus, they stop dividing and begin\\ntaking up nutrients from the gut. (b) Results from a pulse-chase\\nexperiment in which radioactively labeled thymidine (the pulse)\\nwas added to a tissue culture of the intestinal epithelium. \\nVilli\\nDividing cells incorporated the labeled thymidine into their newly\\nFPO synthesized DNA. The labeled thymidine was washed away and\\nreplaced with nonlabeled thymidine (the chase) after a brief\\nperiod; cells that divided after the chase did not become labeled.\\nThese micrographs show that 40 minutes after labeling, all the\\nlabel is in cells near the base of the crypt. At later times, the\\nlabeled cells are seen progressively farther away from their point\\nCrypt of birth in the crypt. Cells at the top are shed. This process\\nensures contant replenishment of the gut epithelium with new\\ncells. [Part (a) adapted from C. S. Potten, 1998, Philos. Trans. R. Soc.\\nLondon, Ser. B353:821. Part (b) from P. Kaur and C. S. Potten, 1986, \\n40 min 1 day 2 days 3 days Cell Tiss. Kinet.19:601.]\\n6 CHAPTER 22 • Cell Birth, Lineage, And Death\\nSkin also contains dendritic epidermal T cells, an factor \"(TGF\"), and TGF!. Discovering how all these sig-\\nimmune-system cell that produces a certain form nals work together to control growth and stimulate healing\\nof the T-cell receptor (see Table 14-1). When den- is a substantial challenge that will advance our understand-\\ndritic epidermal T cells are genetically modified so they do ing of diseases such as psoriasis and skin cancer and perhaps\\nnot produce T-cell receptors, wound healing is slow and less pave the way for effective treatments. ❚\\ncomplete than in normal skin. Normal healing is restored by\\naddition of keratinocyte growth factor. The current hypoth- Another continuously replenished tissue is the blood,\\nesis is that when dendritic epidermal T cells recognize anti- whose stem cells are located in the bone marrow in adult an-\\ngens on cells in damaged tissue, they respond by producing imals. The various types of blood cells all derive from a sin-\\nstimulating proteins, such as keratinocyte growth factor, that gle type of pluripotent hematopoietic stem cell,which gives\\npromote keratinocyte growth and wound healing. Many rise to the more-restricted myeloid and lymphoid stem cells\\nother signals also control the growth of skin cells, including (Figure 22-5). The frequency of hematopoietic stem cells is\\nWnt/!-catenin, Hedgehog, calcium, transforming growth about 1 cell per 104bone marrow cells, even lower than the\\nG-CSF\\nGranulocytes\\nIL-3, GM-CSF, SCF, IL-6\\nGM-CFC M-CSF\\nMonocytes\\nMyeloid\\nstem cell IL-3, GM-CSF\\nFPO\\nEo-CFC Eosinophils\\nEpo, SCF Epo\\nGM-CSF\\nIL-3\\nBFU-E CFU-E Erythrocytes\\nPluripotent\\nhematopoietic SCF, Tpo, IL-3, GM-CSF\\nstem cell\\nMegakaryocytes\\nCFC-mega\\n(platelet-forming cells)\\nLymphoid\\nstem cell\\nIIL-2, IL-7, IL-12, SDF-1,\\nFLT-3 ligand, TNF-α,\\nTGF-β1\\nT-CFC B-CFC T and B cells of the\\nimmune system\\n▲FIGURE 22-5 Formation of differentiated blood cells from eliminated and the precursor cells introduced. Further \\nhematopoietic stem cells in the bone marrow. Pluripotent cytokine-induced proliferation, commitment, and differentiation of\\nstem cells may divide symmetrically to self-renew (curved arrow) the precursor cells give rise to the various types of blood cells.\\nor divide asymmetrically to form myeloid and lymphoid stem Some of the cytokines that support this process are indicated\\ncells (light green). Although these stem cells are capable of  (red labels). GM # granulocyte-macrophage; Eo # eosinophil; \\nself-renewal, they are committed to one of the two major E # erythrocyte; mega # megakaryocyte; T #T-cell; B # B-cell;\\nhematopoietic lineages. Depending on the types and amounts of CFU # colony-forming unit; CSF # colony-stimulating factor; \\ncytokines present, the myeloid and lymphoid stem cells generate IL # interleukin; SCF # stem cell factor; Epo # erythropoietin;\\nvarious precursor cells (dark green), which are incapable of  Tpo # thrombopoietin; TNF # tumor necrosis factor; TGF #\\nself-renewal. These cells are detected by their ability to form transforming growth factor; SDF # stromal cell–derived factor;\\ncolonies containing the differentiated cell types shown at right, FLT-3 ligand # ligand for fms-like tyrosine kinase receptor 3.\\nmeasured as “colony-forming cells (CFCs).” The colonies are [Adapted from M. Socolovsky et al., 1998, Proc. Nat’l. Acad. Sci. USA\\ndetected on the spleen of animals that have had their own cells 95:6573.]\\n22.1 • The Birth of Cells 7\\nfrequency of intestinal stem cells in crypts. Numerous extra- Numerous genes have been found to regulate the forma-\\ncellular growth factors called cytokinesregulate proliferation tion, maintenance, and properties of meristems. Many of\\nand differentiation of the precursor cells for various blood- these genes encode transcription factors. For instance, a hier-\\ncell lineages. For example, erythropoietincan activate several archy of regulators, particularly transcription factors, con-\\ndifferent intracellular signal-transduction pathways, leading trols the separation of differentiating cells from SAMs as\\nto changes in gene expression that promote formation of  leaves form; similarly, three types of regulators control for-\\nerythrocytes (see Figure 14-7).  mation of the floral organs from floral meristems (Chapter\\nThe hematopoietic lineage originally was worked out by 15). In both cases, a cascade of gene interactions occurs, with\\ninjecting the various types of precursor cells into mice whose earlier transcription factors causing production of later ones.\\nprecursor cells had been wiped out by irradiation. By ob- At the same time, cells are dividing and the differentiating\\nserving which blood cells were restored in these transplant ones are spreading away from their original birth sites. ❚\\nexperiments, researchers could infer which precursors or ter-\\nminally differentiated cells (e.g., erythrocytes, monocytes)\\narise from a particular type of precursor. The first step in\\n(a) (d)\\nthese experiments was separation of the different types of\\nhematopoietic precursors. This is possible because each type 1\\n1 3\\nproduces unique combinations of cell-surface proteins that 1 3 3\\n1 6 5 3\\ncan serve as type-specific markers. If bone marrow extracts 6 8 5\\n1 8 7 3\\nare treated with fluorochrome-labeled antibodies for these 10 x 8 picas 6 i 5\\nmarkers, cells with different surface markers can be sepa- 4 i 7 2\\n6 7 5\\nrated in a fluorescence-activated cell sorter (see Figure 5-34).  4 6 5 2\\n4 4 2\\n4 2\\n2\\nFPO\\nTo date, bone marrow transplants represent the\\n(b)\\nmost successful and widespread use of stem cells\\nin medicine. The stem cells in the transplanted\\nmarrow can generate new, functional blood cells in pa-\\ntients with certain hereditary blood diseases and in cancer\\npatients who have received irradiation and/or chemother- 10 x 8 picas\\ni\\napy, both of which destroy the bone marrow cells as well\\nas cancer cells. Recent work is directed at exploring\\nwhether embryonic stem cells can be induced to differen-\\ntiate into cells types that would be useful therapeutically.\\nFor example, mouse stem cells treated with inhibitors of (c)\\nphosphatidylinositol-3 kinase, a regulator in one of the 7 8\\nphosphoinositide signaling pathways (Chapter 14), turn\\ninto cells that resemble pancreatic ! cells in their produc- 1\\n6\\ntion of insulin, their sensitivity to glucose levels, and their\\naggregation into structures reminiscent of pancreas struc- 10 x 8 picas 4\\n3\\ntures. Implantation of these cells into diabetic mice re-\\n2\\nstored their growth, weight, glucose levels, and survival 5\\nrates to normal. ❚\\nStem cells in plants are located in meristems,pop- ▲FIGURE 22-6 Cell fates in meristems of Arabidopsis.In\\nulations of undifferentiated cells found at the tips these longitudinal sections through a shoot apical meristem, cell\\nof growing shoots. Shoot apical meristems (SAMs) nuclei are revealed by staining with propidium iodide, which\\nproduce leaves and shoots, and of course more stem cells binds to DNA. (a) The shoot apical meristem (SAM) produces\\nthat constitute the nearly immortal meristems. Meristems shoots, leaves, and more meristem. Flower production occurs\\nwhen the meristem switches from leaf/shoot production to\\ncan persist for thousands of years in long-lived species such\\nflower production, concomitant with an increase in the number\\nas redwood trees and bristlecone pines. As a plant grows, the\\nof meristem cells to form floral meristems (FMs), as shown\\ncells “left behind” the meristems are encased in rigid walls\\nhere. (b) Cells in a SAM exhibit different fates and behaviors.\\nand can no longer grow. SAMs can split to form branches,\\nCells divide rapidly in the peripheral zone (PZ) to produce leaves\\neach branch with its own SAM, or be converted into floral\\nand in the rib zone (Rib) to produce central shoot structures.\\nmeristems (Figure 22-6). Floral meristems give rise to the\\nCells in the central zone (CZ) divide more slowly, producing an\\nfour floral organs—sepals, stamens, carpels, and petals—that\\nongoing source of meristem and contributing cells to the PZ and\\nform flowers. Unlike SAMs, floral meristems are gradually Rib. Scale bars, 50 $m. [See E. Meyerowitz, 1997, Cell88:299;\\ndepleted as they give rise to the floral organs. micrographs courtesy Elliot Meyerowitz.]\\n8 CHAPTER 22 • Cell Birth, Lineage, And Death\\nCell Fates Are Progressively Restricted  genome comes from the successful production of cloned an-\\nDuring Development imals by nuclear-transfer cloning. In this procedure, the \\nnucleus of an adult (somatic) cell is introduced into an egg\\nThe eight cells resulting from the first three divisions of a that lacks its nucleus; the manipulated egg, which contains\\nmammalian zygote (fertilized egg) all look the same. As the diploid number of chromosomes and is equivalent to a \\ndemonstrated experimentally in mice, each of the cells has zygote, then is implanted into a foster mother. The only\\nthe potential to give rise to a complete animal. Additional di- source of genetic information to guide development of the\\nvisions produce a mass, composed of ≈64 cells, that separates embryo is the nuclear genome of the donor somatic cell. The\\ninto two cell types: trophectoderm, which will form extra- frequent failure of such cloning experiments, however, raises\\nembryonic tissues like the placenta, and the inner cell mass,\\nwhich gives rise to the embryo proper. The inner cell mass (a) Primitive node Primitive streak\\neventually forms three germ layers, each with distinct fates.\\nOne layer, ectoderm,will make neural and epidermal cells; Epiblast Amnioblasts\\nanother,mesoderm,will make muscle and connective tissue;\\nthe third layer, endoderm, will make gut epithelia (Figure \\n22-7). If cells from any of these layers are transplanted into\\nYolk sac\\none of the other layers, they do not give rise to cells appro-\\npriate to their new location. This conclusion is based on ex-\\nperiments with chimeric animals composed of chicken and Invaginating Hypoblast\\nmesoderm cells\\nquail cells. Embryos composed of cells from both bird species\\ndevelop fairly normally, yet the cells derived from each donor (b)\\nare distinguishable under the microscope. Thus the contri-\\nbutions of the different donor cells to the final bird can be \\nascertained. \\nOnce the three germ layers are established, they subse-\\nquently divide into cell populations with different fates. For\\ninstance, the ectoderm becomes divided into those cells that\\nare precursors to the skin epithelium and those that are pre-\\ncursors to the nervous system. There appears to be a pro- FPO\\ngressive restriction in the range of cell types that can be\\nformed from stem cells and precursor cells as development (c)\\nproceeds. An early embryonic stem cell, as we’ve seen, can ECTODERM MESODERM ENDODERM\\nform every type of cell; an ectodermal cell has a choice be- Central nervous system Skull Liver\\ntween neural and epidermal fates; and a keratinocyte pre- Retina and lens  Head, skeletal muscle Pancreas\\ncursor can form skin but not neurons.  Cranial and sensory  Skeleton Urinary bladder\\nGanglia and nerves Dermis of skin Epithelial parts of: \\nAnother restriction that occurs early in animal develop-\\nPigment cells Connective tissue    trachea\\nment is the setting aside of cells that will form the germ line, Head connective tissue Urigential system    lungs\\nthat is, the stem cells and precursor cells that eventually will Epidermis Heart    pharynx\\nHair Blood, lymph cells    thyroid\\ngive rise to eggs in a female and sperm in a male. Only the\\nMammary glands Spleen\\ngenome of the germ line will ever be passed on to progeny.\\nThe setting aside of germ-line cells early in development has ▲FIGURE 22-7 Fates of the germ layers in animals. During\\nbeen hypothesized to protect chromosomes from damage by the early period of mammalian development, cells migrate to\\nreducing the number of rounds of replication they undergo create the three germ layers: ectoderm, mesoderm, and\\nor by allowing special packaging of the cells that are critical endoderm. Cells in the different layers have largely distinct fates\\nto heredity. Whatever the reason, the early segregation of the and therefore represent distinct cell lineages. The detailed cell\\ngerm line is widespread (though not universal) among ani- lineage, however, is not exactly the same in different individuals.\\n(a) Shown here is a sketch of a human embryo about 16 days\\nmals. In contrast, plants do nothing of the sort; most meri-\\nafter fertilization of an egg. The first cells to move from the\\nstems can give rise to germ-line cells. \\nepiblast into the interior form endoderm, and they are followed by\\nOne consequence of the early segregation of germ-line\\ninvaginating cells that become mesoderm. The remaining epiblast\\ncells is that the loss or rearrangement of genes in somatic\\ncells become ectoderm. Hypoblast #covering layer. Amnioblasts #\\ncells would not affect the inherited genome of a future zy-\\ncells that will line amniotic cavity. Yolk sac #reminder of our \\ngote. Although segments of the genome are rearranged and\\negg-bound ancestors. (b) Scanning electron micrograph of a \\nlost during development of lymphocytes from hematopoi-\\ncross section of a similar-stage embryo. (c) Some of the tissue\\netic precursors, most somatic cells seem to have an intact derivatives of the three germ layers are listed. [Part (b) from \\ngenome, equivalent to that in the germ line. Evidence that at T. W. Sadler, 2000, Langman’s Medical Embryology,8th ed. (Baltimore:\\nleast some somatic cells have a complete and functional Lippincott Williams and Wilkins), pp. 64–65, Figure 4.3).]\\n22.1 • The Birth of Cells 9\\nquestions about how many adult somatic cells do in fact individuals. The best-documented example of a reproducible\\nhave a complete functional genome. Even the successes, like pattern of cell divisions comes from the nematode C. elegans.\\nthe famous cloned sheep “Dolly,” appear to have some med- Scientists have traced the lineage of all the somatic cells in\\nical problems. Even if differentiated cells have a physically C. elegansfrom the fertilized egg to the mature worm by fol-\\ncomplete genome, clearly only portions of it are transcrip- lowing the development of live worms using Nomarski in-\\ntionally active (Chapter 10). Whether the genome of a dif- terference microscopy (Figure 22-8).\\nferentiated cell can revert to having the full developmental About 10 rounds of cell division, or fewer, create the \\npotential characteristic of an embryonic cell is a matter of adult worm, which is about 1 mm long and 70 $m in di-\\nconsiderable debate. A cell could, for example, have an intact ameter. The adult worm has 959 somatic cell nuclei (her-\\ngenome, but be unable to properly reactivate it due to inher- maphrodite form) or 1031 (male). The number of somatic \\nited chromatin states.  cells is somewhat fewer than the number of nuclei because \\nThese observations raise two important questions: How some cells contain multiple nuclei (i.e., they are syncytia). \\nare cell fates progressively restricted during development? Remarkably, the pattern of cell divisions starting from a \\nAre these restrictions irreversible? In addressing these ques- C. elegansfertilized egg is nearly always the same. As we \\ntions, it is important to remember that what a cell in its nor- discuss later in the chapter, many cells that are generated \\nmal in vivo location will do may differ from what a cell is during development undergo programmed cell death and \\ncapable of doing if it is manipulated experimentally. Thus the are missing in the adult worm. The consistency of the \\nobserved limits to what a cell can do may result from natural C. eleganscell lineage does not result entirely from each \\nregulatory mechanisms or may reflect a failure to find con- newly born cell inheriting specific information about its \\nditions that reveal the cell’s full potential. destiny. In some cases, various signals direct initially iden-\\nAlthough our focus in this chapter is on how cells be- tical cells to particular fates, and the outcomes of these sig-\\ncome different, their ability to remain the same also is criti- nals are consistent from one animal to the next. That is, \\ncal to the functioning of tissues and the whole organism. at their birth cells are not necessarily “hard wired” by \\nNondividing differentiated cells with particular characteris- their own internal inherited instructions to follow a par-\\ntics must retain them, sometimes for many decades. Stem ticular path of differentiation. \\ncells that divide regularly, such as a skin stem cell, must pro- The first few cell divisions in C. elegansproduce six dif-\\nduce one daughter cell with the properties of the parental ferent founder cells,each with a separate fate as shown in\\ncell, retaining its characteristic composition, shape, behavior, Figure 22-9a, b. The initial division is asymmetric, giving rise\\nand responses to specific external signals. Meanwhile, the to P1 and the AB founder cell. Further divisions in the P lin-\\nother daughter cell with its own distinct inheritance, as the eage form the other five founder cells. Some of the signals\\nresult of asymmetric cell division, embarks on a differentia- controlling the asymmetry are known. For example, Wnt\\ntion pathway, which may be determined by the signals the signals from the P2 precursor control the asymmetric divi-\\ncell receives.  sion of the EMS cell into E and MS founder cells. Wnt sig-\\nnaling (see Figure 15-32) is also used in other asymmetric\\ndivisions in worms. \\nThe Complete Cell Lineage of C. elegans\\nThe bilateral symmetry of the worm implies duplication\\nIs Known \\nof lineages on the two sides; curiously, though, functionally\\nIn the development of some organisms, cell lineages are equivalent cells on each side can arise from a pattern of \\nunder tight genetic control and thus are identical in all indi- division that is different on the two sides. Some of the em-\\nviduals of a species. In other organisms the exact number bryonic cells function as stem cells, dividing repeatedly to\\nand arrangement of cells vary substantially among different form more of themselves or another type of stem cell, while\\nDorsal nerve\\nPharynx cord Intestinal lumen\\nMouth Intestine Intestinal nuclei\\nExcretory Anus\\ncell Gonad\\nVentral nerve cord Rectum\\nVentral\\nnuclei\\nnerve cord 20 $m\\n▲FIGURE 22-8 Newly hatched larva of C.elegans.Many of in this micrograph obtained by Nomarski interference microscopy.\\nthe 959 somatic cell nuclei in this hermaphrodite form are visualized [From J. E. Sulston and H. R. Horvitz, 1977, Devel. Biol.56:110.]\\n10 CHAPTER 22 • Cell Birth, Lineage, And Death\\n(a) (b)\\nP0\\nAB P1\\nneurons\\nhypodermis EMS P2\\npharyngeal muscle\\nbody muscle MS E\\nother body muscle gut P3 C FPO\\npharyngeal muscle hypodermis\\nneurons body muscle\\nglands neurons\\nsomatic gonad P4 D\\nother germ cells body muscle\\n(c) Zygote\\ns\\nsi\\ne\\nn Germ\\nge FPO line\\no Pharynx Intestine\\ny\\nr\\nb\\nm\\nE\\ns\\nn\\na\\ng\\ne\\nel\\nC.  Neurons\\nf \\no\\ng \\nn\\ngi\\nS a\\nm\\nN\\nI\\nO e \\ns Vulva\\nI p Epidermis\\nT a Somatic gonad\\nL\\nC -\\ne\\nE m ▲FIGURE 22-9 C.eleganslineage.(a) Pattern of the aside very early, as in most animals. All the other lineages give\\nN Ti first few divisions starting with P0 (the zygote) and leading rise to somatic cells. (b) Light micrographs of the first few\\nN n:  to formation of the six founder cells (yellow highlights). The divisions of the embryo that generate the founder cells with cells\\nO o\\nC ati first division, into P1 and AB, is asymmetric. Further labeled as in part (a). The texture of the cells shows the presence\\nm divisions in the P lineage generates the other founder of organelles and is revealed by differential interference contrast\\n \\nA ni cells. Note that more than one lineage can lead to the microscopy, sometimes called Nomarski microscopy. (c) Full\\nDI o A same tissue type (e.g., muscle or neurons). The EMS cell lineage of the entire body of the worm, showing some of the\\nE e is so named because it is the precursor to most of the tissues formed. Note that any particular cell undergoes relatively\\nd\\nM Vi endoderm and mesoderm. The lineage beginning with the few divisions, typically fewer than 15. [Einhard Schierenberg\\nP4 cell gives rise to all of the germ-line cells, which are set Zoologisches Institut, Universität Köln.]\\nspinning off differentiating cells that give rise to a particular of development occurs too early (precocious development) or\\ntissue. In one pattern of division, one of the daughter cells too late (retarded development). An example of the former is\\nfrom a division initiates a repeat of the lineage pattern of the a premature cell division to yield a cell that differentiates and\\nprevious few divisions. The complete lineage of C. elegans a cell that dies; as a result, the lineage that should have fol-\\nis shown in Figure 22-9c. This organism has been a powerful lowed from the dead cell never happens. In the latter case,\\nmodel system for genetic studies to identify the regulators the delayed occurrence of a lineage causes juvenile structures\\nthat control cell lineages in time and space. to be produced, incorrectly, in more mature animals. In both\\ncases, the character of a parental cell is, in essence, changed\\nto the character of a cell at a different stage of development. \\nHeterochronic Mutants Provide Clues \\nOne example of precocious development in C. elegans\\nAbout Control of Cell Lineage \\ncomes from loss-of-function mutations in the lin-14gene,\\nIntriguing evidence for the genetic control of cell lineage has which cause premature formation of the PDNB neuroblast\\ncome from isolation and analysis of heterochronicmutants. (Figure 22-10a). The lin-14gene and several others found to\\nIn these mutants, a developmental event typical of one stage be defective in heterochronic worm mutants encode RNA-\\n22.1 • The Birth of Cells 11\\n(b) L1 L2 L3/4 Adult\\nlin-14, lin-28 lin-14, lin-28 lin-14, lin-28 lin-14, lin-28\\nmRNA mRNA mRNA mRNA\\nlin-4 lin-4 lin-4 let-7\\n(a) RNA RNA RNA RNA\\nFPO\\nWild type lin-14 mutant\\nLIN-14, LIN-28\\nV5 V5 proteins lin-41 mRNA\\nL1\\nlin-41 mRNA lin-41 mRNA lin-41 mRNA\\nPDNB let-7\\nRNA\\nX\\nLIN-41 LIN-41 LIN-41\\nprotein protein protein lin-29 mRNA\\nL2 PDNB\\n_ _ _\\nlin-29 mRNA lin-29 mRNA lin-29 mRNA LIN-29\\nprotein\\nn\\nei LIN-14, LIN-28\\not\\nr\\np LIN-41\\nr \\nA oel lin-4 let-7 LIN-29\\nNv\\nRe\\nl\\n▲FIGURE 22-10 Timing of specific types of cell division RNA to the 3% untranslated regions (UTRs) of lin-14and lin-28\\nduring development of C.elegans.(a) The pattern of cell mRNAs prevents translation of these mRNAs into protein. This\\ndivision for the V5 cell of C. elegansis shown for normal worms occurs following the first larval (L1) stage, permitting\\nand for a heterochronic mutant called lin-14.In the lin-14mutant, development to proceed to the later larval stages. Starting in the\\nthe pattern of cell division (red arrows) that normally occurs only fourth larval stage (L4), production of let-7RNA begins. It\\nin the second larval stage (L2) occurs in the first larval stage (L1), hybridizes to lin-14,lin-28, and lin-41mRNAs, preventing their\\ncausing the PDNB neuroblast to be generated prematurely. In  translation. LIN-41 protein is an inhibitor of translation of the \\nthe mutant, the V5 cell behaves during L1 like cell “X” (purple) lin-29mRNA, so the appearance of let-7RNA allows production\\nnormally does in L2. The inference is that the LIN-14 protein of LIN-29 protein, which is needed for generation of adult cell\\nprevents L2-type cell divisions, although precisely how it does so lineages. LIN-4 may also bind to lin-41RNA at later stages. Only\\nis unknown. (b) Two small regulatory RNAs, lin-4and let-7,serve the 3% UTRs of the mRNAs are depicted. [Adapted from B. J.\\nas coordinating timers of gene expression. Binding of the lin-4 Reinhart et al., 2000, Nature403:901.]\\nbinding or DNA-binding proteins, which presumably coordi- other miRNAs during the life cycle of C. elegansserve as a regu-\\nnate expression of other genes. However, two other genes (lin- latory clock for cell lineage. Molecules related to let-7RNA have\\n4and let-7) involved in regulating the timing of cell divisions been identified in many other animals including vertebrates and\\nwere initially extremely puzzling, as they appeared not to en- insects; since their production is temporally regulated in these an-\\ncode any protein. To discover the products of these genes, sci- imals as well, they may serve a similar function in their develop-\\nentists first determined which pieces of genomic DNA could ment as in C. elegans. How production of these regulatory\\nrestore gene function, and therefore proper cell lineage, to mu- miRNAs is temporally controlled is not yet known, but they have\\ntants defective in each gene. They then did the same thing with turned out to play many roles in regulating gene expression\\ngenomic DNA from the corresponding genomic regions of dif- (Chapter 12). \\nferent species of worm. Comparison of the “rescuing” frag-\\nments from the different species revealed that they shared\\ncommon short sequences with little protein-coding potential.  KEY CONCEPTS OF SECTION 22.1\\nThe short RNA molecules encoded by lin-4and let-7were\\nThe Birth of Cells\\nsubsequently shown to inhibit translation of the mRNAs en-\\ncoded by lin-14and other heterochronic genes (Figure 22-10b). ■ In asymmetric cell division, two different types of daugh-\\nThese small RNAs, or micro RNAs (miRNAs),are complemen- ter cells are formed from one mother cell. In contrast, both\\ntary to sequences in the 3%untranslated parts of target mRNAs daughter cells formed in symmetric divisions are identical\\nand are believed to control translation of the mNRAs by hy- but may have different fates if they are exposed to differ-\\nbridizing them. Temporal changes in the production of these and ent external signals (see Figure 22-1). \\n12 CHAPTER 22 • Cell Birth, Lineage, And Death\\n■ Pluripotent stem cells can produce more than one type Our first example of cell-type specification comes from\\nof descendant cell, including in some cases a stem cell with the budding yeast, S. cerevisiae.We introduced this useful\\na more-restricted potential to produce differentiated cell unicellular eukaryote way back in Chapter 1 and have en-\\ntypes (see Figure 22-2).  countered it in several other chapters.S. cerevisiaeforms\\nthree cell types: haploid aand \"cells and diploid a/\"cells.\\n■ Cultured embryonic stem cells (ES cells) are capable of\\nEach type has its own distinctive set of active genes; many\\ngiving rise to many kinds of differentiated cell types. They\\nother genes are active in all three cell types. In a pattern com-\\nare useful in production of genetically altered mice and \\nmon to many organisms and tissues, cell-type specification in\\noffer potential for therapeutic uses. \\nyeast is controlled by a small number of transcription factors\\n■ Populations of stem cells associated with most tissues that coordinate the activities of many other genes. Similar\\n(e.g., skin, intestinal epithelium, blood) regenerate differ-\\nregulatory features are found in the responses of higher eu-\\nentiated tissue cells that are damaged or sloughed or be-\\nkaryotic cells to environmental signals and in the specifica-\\ncome aged (see Figures 22-4 and 22-5). \\ntion and patterning of cells and tissues during development. \\n■ Stem cells are prevented from differentiating by specific Recent DNA microarraystudies of transcription patterns\\ncontrols. A high level of !-catenin, a component of the in S. cerevisiaehave provided a genome-wide picture of the\\nWnt signaling pathway, has been implicated in preserving fluctuations of gene expression in the different cell types and\\nstem cells in the skin and intestine.  different stages of the yeast life cycle (see Figure 9-35 for an ex-\\nplanation of the DNA microarray technique). Among other\\n■ Plant stem cells persist for the life of the plant in the\\nthings, these studies identified 32 genes that are transcribed\\nmeristem. Meristem cells can give rise to a broad spectrum\\nat more than twofold higher levels in \"cells than in acells. An-\\nof cell types and structures. \\nother 50 genes are transcribed at more than twofold higher\\n■ During development, precursor cells generally lose po- levels in acells than in \"cells. The products of these 82 genes,\\ntential; that is, they become progressively restricted in the\\nwhich initially are activated by cell-type specification tran-\\nnumber of different cell types they can form. \\nscription regulators, convey many of the critical differences be-\\n■ Early in animal development, the three germ layers— tween the two cell types. Since transcription of more than\\nectoderm, mesoderm, and endoderm—form. Each gives 6000 genes was assessed in these experiments, the results\\nrise to specific tissues and organs (see Figure 22-7). clearly demonstrate that changes in expression of only a small\\nfraction of the genome can significantly alter the behavior and\\n■ Germ-line cells give rise to eggs or sperm. By definition,\\nproperties of cells. Transcription of a much larger number of\\nall other cells are somatic cells. \\ngenes, about 25 percent of the total assayed, differed substan-\\n■ Embryonic development of C. elegansbegins with asym- tially in diploid cells compared with haploid cells. A powerful\\nmetric division of the fertilized egg (zygote). The lineage\\nadvantage of the microarray approach is that it systematically\\nof all the cells in adult worms is known and is highly re-\\nidentifies a large fraction of the relevant genes controlling var-\\nproducible (see Figure 22-9). \\nious processes, allowing scientists to concentrate on the most\\n■ Short regulatory RNAs control the timing of developmen- important players that regulate differences in cell types. \\ntal cell divisions by preventing translation of mRNAs whose\\nencoded proteins control cell lineages (see Figure 22-10).\\nTranscription Factors Encoded at the MATLocus\\nAct in Concert with MCM1 to Specify Cell Type\\n22.2 Cell-Type Specification in Yeast Each of the three S. cerevisiaecell types expresses a unique set\\nof regulatory genes that is responsible for all the differences\\nIn  the  previous  section,  we  saw  that  stem  cells  and  among the three cell types. All haploid cells express certain\\nprecursor/progenitor cells produce progeny that embark haploid-specific genes; in addition, acells express a-specific\\nupon specific differentiation paths. The elegant regulatory genes, and \"cells express \"-specific genes. In a/\"diploid cells,\\nmechanisms whereby cells become different is referred to as diploid-specific genes are expressed, whereas haploid-specific,\\ncell-type specification.Specification usually involves a com- a-specific, and \"-specific genes are not. As illustrated in Fig-\\nbination of external signals with internal signal-transduction ure 22-11, three cell type–specific transcription factors (\"1,\\nmechanisms like those described in Chapters 14 and 15. The \"2, and a1) encoded at the MAT locusin combination with\\ntransition from an undifferentiated cell to a differentiating a general transcription factor called MCM1,which is ex-\\none often involves the production of one or a small number pressed in all three cell types, mediate cell type–specific gene\\nof transcription factors. The newly produced transcription expression in S. cerevisiae. Thus the actions of just three tran-\\nfactors serve as powerful switches that trigger the activation scription factors can set the yeast cell on a specific differenti-\\n(and sometimes repression) of large batteries of subservient ation pathway culminating in a particular cell type. From the\\ngenes. Thus an initially modest change can cause massive DNA microarray experiments we know one outcome: the ac-\\nchanges in gene expression that confer a new character on tivation or repression of many dozens of genes that control\\nthe cell.  cell characteristics.\\n22.2 • Cell-Type Specification in Yeast 13\\n(a) a cell MCM1 was the first member of the MADS familyof\\na1 αsg transcription factors to be discovered. (MADS is an acronym\\nMCM1 for the initial four factors identified in this family.) The\\na1\\nMATa DNA-binding proteins composing this family dimerize and\\n+ + asg\\ncontain a similar N-terminal MADS domain. In Section 22.3\\nwe will encounter other MADS transcription factors that\\nFPO\\n+ + hsg participate in development of skeletal muscle. MADS tran-\\nscription factors also specify cell types in floral organs (see\\nFigure 15-28). MCM1 exhibits different activity in haploid a\\n(b)  α cell\\n+ and \"cells due to its association with \"1 or \"2 protein in \"\\nα2 α1 α1 + cells. As a result of this combinatorial action, MCM1 pro-\\nαsg\\nmotes transcription of \"-specific genes and represses tran-\\nMATα2 MATα1\\nscription of a-specific genes in \"cells. Acting alone, MCM1\\n+ − asg\\nactivates transcription of a-specific genes in a cells and of\\nα2 haploid-specific genes in both \" and a cells (see Figure \\n22-11a, b). Now let’s take a closer look at how MCM1 and\\n+ + hsg\\nthe MAT-encoded proteins exert their effects. \\n(c) a/α cell\\nMCM1 and !1-MCM1 Complexes Activate \\nα2 α2\\n− αsg Gene Transcription\\nMATα2 MATα1 + In acells, homodimeric MCM1 binds to the so-called P box\\n− asg sequence in the upstream regulatory sequences (URSs) of \\nα2 a-specific  genes,  stimulating  their  transcription  (Figure \\na1 a1\\n22-12a). Transcription of \"-specific genes is controlled by\\n− hsg\\ntwo adjacent sequences—the P box and the Q box—located\\nMATa\\nin the URSs associated with these genes. Although MCM1\\nalone binds to the P box in a-specific URSs, it does not bind\\n▲FIGURE 22-11 Transcriptional control of cell type–specific\\nto the P box in \"-specific URSs. Thus a cells do not tran-\\ngenes in S.cerevisiae.The coding sequences carried at the\\nscribe \"-specific genes.\\nMATlocus differ in haploid \" and a cells and in diploid cells.\\nIn \"cells, which produce the \"1 transcription factor en-\\nThree type-specific transcription factors (\"1, \"2, and a1) encoded\\ncoded by MAT\", the simultaneous binding of MCM1 and\\nat the MATlocus act with MCM1, a constitutive transcription\\n\"1 to PQ sites occurs with high affinity (Figure 22-12b). This\\nfactor produced by all three cell types, to produce a distinctive\\npattern of gene expression in each of the three cell types.  binding turns on transcription of \"-specific genes. Therefore\\nasg# a-specific genes/mRNAs; \"sg#\"-specific genes/mRNAs; a-specific transcription is a simple matter of a single tran-\\nhsg# haploid-specific genes/mRNAs. scription factor binding to its target genes, while \"-specific\\n\"FIGURE 22-12 Activity of MCM1 in a\\nand ! yeast cells. MCM1 binds as a dimer\\n(a)  a cells MCM1 dimer\\nto the P site in \"-specific and a-specific\\nupstream regulatory sequences (URSs),\\nwhich control transcription of \"-specific\\nP Q αsg α2 P α2 asg\\ngenes and a-specific genes, respectively. \\nα-specific URS No transcription of a-specific URS Transcription of\\n(a) In a cells, MCM1 stimulates transcription\\nα-specific genes a-specific genes\\nof a-specific genes. MCM1 does not bind\\nFPO efficiently to the P site in \"-specific URSs in\\n(b)  α cells α2 dimer the absence of \"1 protein. (b) In \" cells, the\\nactivity of MCM1 is modified by its\\nassociation with \"1 or \"2. The \"1-MCM1\\ncomplex stimulates transcription of \"-specific\\nα1 genes, whereas the \"2-MCM1 complex\\nP Q αsg α2 P α2 asg blocks transcription of a-specific genes. The\\nα-specific URS Transcription of a-specific URS No transcription of \"2-MCM1 complex also is produced in\\nα-specific genes a-specific genes diploid cells, where it has the same blocking\\neffect on transcription of a-specific genes\\n(see Figure 22-11c).\\n14 CHAPTER 22 • Cell Birth, Lineage, And Death\\ntranscription requires a combination of two factors—neither\\na factor\\nof which can activate target genes alone. α factor\\na α\\n!2-MCM1 and !2-a1 Complexes \\nRepress Transcription\\nFlanking the P box in each a-specific URS are two \"2- α-factor\\nbinding sites. Both MCM1 and \"2 can bind independently receptor a-factor receptor\\nto an a-specific URS with relatively low affinity. However, in\\n\"cells only the highly cooperative, simultaneous binding of\\nPheremone signaling\\nboth \"2 and MCM1 proteins occurs with high affinity. This\\nhigh-affinity binding represses transcription of a-specific\\ngenes, ensuring that they are not expressed in \" cells and\\ndiploid cells (see Figure 22-12b, right). MCM1 promotes a α\\nbinding of \"2 to an a-specific URS by orienting the two DNA-\\nbinding domains of the \"2 dimer to the \"2-binding sequences\\nin this URS. Since a dimeric \"2 molecule binds to both sites in\\nan \"-specific URS, each DNA site is referred to as a half-site. Cell-cycle arrest\\nThe relative positions of both half-sites and their orientation Morphogenesis\\nare highly conserved among different a-specific URSs.\\nThe presence of numerous \"2-binding sites in the genome\\na α\\nand the “relaxed” specificity of \"2 protein may expand the\\nrange of genes that it can regulate. For instance, in a/\" FPO\\ndiploid cells, \"2 forms a heterodimer with a1 that represses\\nboth haploid-specific genes and the gene encoding \"1 (see\\nCell fusion\\nFigure 22-11c). The example of \"2 suggests that relaxed\\nspecificity may be a general strategy for increasing the regu-\\nlatory range of a single transcription factor. Highly specific\\nbinding, then, occurs as a consequence of the interaction of\\n\"2 with other transcription factors at different sites in DNA.\\nNuclear fusion\\nPheromones Induce Mating of ! and a Cells \\nto Generate a Third Cell Type \\na/α\\nDiploid cell\\nAn important feature of the yeast life cycle is the ability of\\nhaploid aand \"cells to mate, that is, attach and fuse giving\\nrise to a diploid a/\"cell (see Figure 1-5). Each haploid cell + Nutrients − Nutrients\\ntype secretes a different mating factor,a small polypeptide\\npheromone,and expresses a cell-surface G protein–coupled Mitotic growth Meiosis\\nreceptor that recognizes the pheromone secreted by cells of\\nthe other type. Thus aand \"cells both secrete and respond to a/α a/α\\npheromones (Figure 22-13). Binding of the mating factors\\nSporulation\\nto their receptors induces expression of a set of genes encod-\\ning proteins that direct arrest of the cell cycle in G and pro- a α\\n1\\nmote attachment/fusion of haploid cells to form diploid cells.\\nIn the presence of sufficient nutrients, the diploid cells will a α\\ncontinue to grow. Starvation, however, induces diploid cells\\nto progress through meiosis, each yielding four haploid\\n▲FIGURE 22-13 Pheromone-induced mating of haploid\\nspores. If the environmental conditions become conducive to\\nyeast cells.The \" cells produce \" mating factor and a receptor;\\nvegetative growth, the spores will germinate and undergo \\nthe a cells produce a factor and \"receptor. Binding of the\\nmitotic division.\\nmating factors to their cognate receptors on cells of the opposite\\nStudies with yeast mutants have provided insights into\\ntype leads to gene activation, resulting in mating and production\\nhow the a and \"pheromones induce mating. For instance, of diploid cells. In the presence of sufficient nutrients, these cells\\nhaploid yeast cells carrying mutations in the sterile 12 will grow as diploids. Without sufficient nutrients, cells will\\n(STE12)locus cannot respond to pheromones and do not undergo meiosis and form four haploid spores.\\n22.3 • Specification and Differentiation of Muscle 15\\nmate. The STE12gene encodes a transcription factor that biologists do not yet know the complete set of molecules that\\nbinds to a DNA sequence referred to as the pheromone- distinguishes any one cell type (e.g., muscle) from all the\\nresponsive element,which is present in many different a- and other cell types in a multicellular organism. The extensive\\n\"-specific URSs. Binding of mating factors to cell-surface re- cell specification and differentiation that occur during \\nceptors induces a cascade of signaling events, resulting in development of animals and plants depend on both quanti-\\nphosphorylation of various proteins including the Ste12 pro- tative and qualitative differences in gene expression, con-\\ntein (see Figure 14-24). This rapid phosphorylation is corre- trolled at the level of transcription, as well as on cell\\nlated with an increase in the ability of Ste12 to stimulate structures and protein activity states. \\ntranscription. It is not yet known, however, whether Ste12 An impressive array of molecular strategies, some analo-\\nmust be phosphorylated to stimulate transcription in re- gous to those found in yeast cell-type specification, have\\nsponse to pheromone.  evolved to carry out the complex developmental pathways\\nInteraction of Ste12 protein with DNA has been studied that characterize multicellular organisms. Muscle cells have\\nmost extensively at the URS controlling transcription of been the focus of many such studies because their develop-\\nSTE2,an a-specific gene encoding the receptor for the \" ment can be studied in cultured cells as well as in intact ani-\\npheromone. Adjacent to this a-specific URS is a pheromone- mals. Early advances in understanding the formation of\\nresponsive element that binds Ste12. When acells are treated muscle cells (myogenesis) came from discovery of regulatory\\nwith \"pheromone, transcription of the STE2gene increases genes that could convert cultured cells into muscle cells.\\nin  a  process  that  requires  Ste12  protein.  Presumably, Then mouse mutations affecting those genes were created\\npheromone-induced production of the \"receptor encoded by and studied to learn the functions of the proteins encoded\\nSTE2increases the efficiency of the mating process. Ste12 by these genes, following which scientists have investigated\\nprotein has been found to bind most efficiently to the how the muscle regulatory genes control other genes.\\npheromone-responsive element in the STE2 URS when DNA microarray analysis has been applied to under-\\nMCM1 is simultaneously bound to the adjacent P site. We stand the function of twistand other regulatory genes in\\nsaw previously that MCM1 can act as an activator or a  muscle development. For example, the expression pattern of\\nrepressor at different URSs depending on whether it com- normal Drosophilaembryos recently has been compared\\nplexes with \"1 or \"2. In this case, the function of MCM1  with that of mutant embryos in which the twistgene is de-\\nas an activator is stimulated by the binding of yet another fective. Previous research had shown that the Twist protein\\ntranscription factor, Ste12, whose activity is modified by  encoded by this gene is a transcription factor necessary to\\nextracellular signals. create muscle cells in flies and other animals. Twist turns\\non production of other transcription factors that in turn ac-\\ntivate genes encoding myosin, actin, and other muscle-\\nKEY CONCEPTS OF SECTION 22.2 specific proteins. To assess how many genes are needed to\\nspecify muscle, researchers determined the expression of\\nCell-Type Specification in Yeast\\nabout 4000 fly genes (about 30 percent of the total) in nor-\\n■ Specification of each of the three yeast cell types—the a mal fly embryos and in twistmutants. Of the genes included\\nand \"haploid cells and the diploid a/\"cells—is determined in the microarray, about 130 (3.3 percent), including many\\nby a unique set of transcription factors acting in different known muscle differentiation genes, were transcribed at\\ncombinations  at  specific  regulatory  sites  in  the  yeast lower levels (or not at all) in the twistmutants. These results\\ngenome (see Figure 22-11). suggest that transcriptional changes in at least several hun-\\n■ Some transcription factors can act as repressors or acti- dred genes are associated with differentiation of a highly\\nvators depending on the specific regulatory sites they bind specialized cell type such as muscle. \\nand the presence or absence of other transcription factors Other recent microarray studies have looked for genes\\nbound to neighboring sites. whose transcription differs in various subtypes of muscle in\\nmice. These have identified 49 genes out of 3000 genes ex-\\n■ Binding of mating-type pheromones by haploid yeast amined that are transcribed at substantially different levels in\\ncells activates expression of genes encoding proteins that\\nred (endurance) muscle and white (fast response) muscle.\\nmediate mating, thereby generating the third yeast cell type\\nClues to the molecular basis of the functional differences be-\\n(see Figure 22-13).\\ntween red and white muscle are likely to come from studying\\nthose 49 genes and their products. \\nHere we examine the role of certain transcription factors\\n22.3 Specification and Differentiation\\nin creating skeletal muscle in vertebrates. These muscle reg-\\nof Muscle  ulators illustrate how coordinated transcription of sets of\\ntarget genes can produce differentiated cell types and how a\\nAs indicated by global expression patterns, yeast cells of dif- cascade of transcriptional events and signals is necessary to\\nferent mating types are still rather similar. Developmental  coordinate cell behaviors and functions. \\n16 CHAPTER 22 • Cell Birth, Lineage, And Death\\nEmbryonic Somites Give Rise to Myoblasts,  siently. These signals trigger production of intracellular fac-\\nthe Precursors of Skeletal Muscle Cells  tors that maintain the myogenic program after the inducing\\nsignals are gone. We discuss the identification and functions\\nVertebrate  skeletal  myogenesis  proceeds  through  three\\nof these myogenic proteins, and their interactions, in the next\\nstages: determination of the precursor muscle cells, called\\nseveral sections.\\nmyoblasts;proliferation and in some cases migration of \\nmyoblasts; and their terminal differentiation into mature\\nmuscle (Figure 22-14). In the first stage, myoblasts arise from Myogenic Genes Were First Identified in Studies\\nblocks of mesoderm cells, called somites,that are located with Cultured Fibroblasts\\nnext to the neural tube in the embryo. Specific signals from\\nIn vitro studies with the fibroblast cell line designated C3H \\nsurrounding tissue play an important role in determining\\n10T1⁄ have played a central role in dissecting the transcription-\\nwhere myoblasts will form in the developing somite. At the 2\\ncontrol mechanisms regulating skeletal myogenesis. When \\nmolecular level, the decision of a mesoderm cell to adopt a\\nthese cells are incubated in the presence of 5-azacytidine, a \\nmuscle cell fate reflects the activation of genes encoding par-\\ncytidine derivative that cannot be methylated and therefore \\nticular transcription factors.\\nalters transcription, they differentiate into myotubes. Upon\\nAs myoblasts proliferate and migrate, say to a developing\\nentry into cells, 5-azacytidine is converted to 5-azadeoxycytidine\\nlimb bud, they become aligned, stop dividing, and fuse to\\ntriphosphate and then is incorporated into DNA in place of\\nform a syncytium(a cell containing many nuclei but sharing\\ndeoxycytidine. Because methylated deoxycytidine residues\\na common cytoplasm). We refer to this multinucleate cell as\\ncommonly are present in transcriptionally inactive DNA re-\\na myotube.Concomitant with cell fusion is a dramatic rise in\\ngions, replacement of cytidine residues with a derivative that\\nthe expression of genes necessary for further muscle devel-\\ncannot be methylated may permit activation of genes previ-\\nopment and function. \\nously repressed by methylation. \\nThe specific extracellular signals that induce determina-\\nThe high frequency at which azacytidine-treated C3H\\ntion of each group of myoblasts are expressed only tran-\\n10T1⁄ cells are converted into myotubes suggested to early\\n2\\nworkers that reactivation of one or a small number of closely\\nlinked genes is sufficient to drive a myogenic program. To test\\n!EXPERIMENTAL FIGURE 22-15 Myogenic genes\\nisolated from azacytidine-treated cells can drive myogenesis\\nMyotome\\nwhen transfected into other cells.(a) When C3H 10T1⁄2 cells \\n(a fibroblast cell line) are treated with azacytidine, they develop\\nSomite\\ninto myotubes at high frequency. To isolate the genes responsible\\nMyoD/Myf5 1 Determination for converting azacytidine-treated cells into myotubes, all the\\nmRNAs from treated cells first were isolated from cell extracts\\nFPO on an oligo-dT column. Because of their poly(A) tails, mRNAs are\\nselectively retained on this column. Steps  1 and  2: The \\nisolated mRNAs were converted to radiolabeled cDNAs. Step  3:\\nMyoblasts When the cDNAs were mixed with mRNAs from untreatedC3H\\nMyogenin and MEFs 2 Proliferation and/or migration 10T1⁄2 cells, only cDNAs derived from mRNAs (light red)\\nproduced by both azacytidine-treated cells and untreated cells\\nhybridized. The resulting double-stranded DNA was separated\\nPremuscle masses\\nfrom the unhybridized cDNAs (dark blue) produced only by\\nazacytidine-treated cells. Step  4: The cDNAs specific for\\n3 Differentiation and cell fusion\\nazacytidine-treated cells then were used as probes to screen a\\ncDNA library from azacytidine-treated cells (Chapter 9). At least\\nsome of the clones identified with these probes correspond to\\ngenes required for myogenesis. (b) Each of the cDNA clones\\nMyotube (muscle cell)\\nidentified in part (a) was incorporated into a plasmid carrying a\\n▲FIGURE 22-14 Three stages in development of  strong promoter. Steps  1 and  2: C3H 10T1⁄2 cells were\\nvertebrate skeletal muscle. Somites are epithelial spheres of cotransfected with each recombinant plasmid plus a second\\nembryonic mesoderm cells, some of which (the myotome) plasmid carrying a gene conferring resistance to an antibiotic\\nbecome determined as myoblasts after receiving signals from called G418; only cells that have incorporated the plasmids will\\nother tissues (1). After the myoblasts proliferate and migrate  grow on a medium containing G418. One of the selected clones,\\nto the limb buds and elsewhere (2), they undergo terminal designated myoD,was shown to drive conversion of C3H 10T1⁄2\\ndifferentiation into multinucleate skeletal muscle cells, called cells into muscle cells, identified by their binding of labeled\\nmyotubes (3). Key transcription factors that help drive the antibodies against myosin, a muscle-specific protein (step  3).\\nmyogenic program are highlighted in yellow. See also Figure 22-17. [See R. L. Davis et al., 1987, Cell51:987.]\\n22.3 • Specification and Differentiation of Muscle 17\\n(a)  Screen for myogenic genes this hypothesis, researchers isolated DNA from C3H 10T1⁄2\\ncells grown in the presence of 5-azacytidine and transfected it\\ninto untreated cells. The observation that 1 in 104cells trans-\\nfected with this DNA was converted into myotubes is consis-\\ntent with the hypothesis that one or a small set of closely linked\\nTotal mRNA from azacytidine-treated cells genes is responsible for converting fibroblasts into myotubes. \\n1 Incubate with reverse Subsequent studies led to the isolation and characteriza-\\ntranscriptase and tion of four genes that can convert C3H 10T1⁄ cells into\\n2\\n[32P]dNTPs\\nmuscle. Figure 22-15 outlines the experimental protocol \\n2 Remove mRNAs for identifying and assaying one of these genes, called the\\nmyogenic determination (myoD)gene. C3H 10T1⁄ cells\\n2\\ntransfected with myoDcDNA and those treated with 5-\\nazacytidine both formed myotubes. The myoDcDNA also\\nwas able to convert a number of other cultured cell lines into\\nmuscle. Based on these findings, the myoDgene was pro-\\n32P-labeled cDNAs\\nposed to play a key role in muscle development. A similar ap-\\n3 Hybridize with proach identified three other genes—myogenin, myf5,and\\nDiscard\\nexcess of mRNAs mrf4—that also function in muscle development. \\nfrom untreated\\nC3H 10T1/2 cells\\n(                 ) Muscle-Regulatory Factors (MRFs) \\nand Myocyte-Enhancing Factors (MEFs) Act \\nin Concert to Confer Myogenic Specificity \\n32P-labeled azamyoblast- The four myogenic proteins—MyoD, Myf5, myogenin, and\\nspecific cDNAs\\nMRF4—are  all  members  of  the  basic  helix-loop-helix\\n(bHLH) family of DNA-binding transcription factors (see \\nScreen cDNA library Figure 11-22b). Near the center of these proteins is a DNA-\\n4\\nfrom treated cells binding basic (B) region adjacent to the HLH domain, which\\nmediates dimer formation. Flanking this central DNA-\\nClones enriched\\nin myogenic genes binding/dimerization region are two activation domains. We\\nrefer to the four myogenic bHLH proteins collectively as\\nFPO muscle regulatory factors,or MRFs(Figure 22-16a). \\n(a) Structure of muscle-regulatory factors (MRFs)\\nDNA binding/dimerization\\n(b)  Assay for myogenic activity of myoD cDNA\\nH2N Transactivation B HLH Transactivation COOH\\nC3H 10T1/2 cell MEF-interacting domain\\nFPO\\n1 Transfect with a plasmid (b) Structure of myocyte-enhancing factors (MEFs)\\ncarrying myoD cDNA and a\\ngene conferring resistance DNA binding/dimerization\\nto G418\\n2 Select on G418-containing H2N MADS MEF Transactivation COOH\\nmedium\\nMyogenin-interacting domain\\n▲FIGURE 22-16 General structures of two classes of\\n3 Stain with transcription factors that participate in myogenesis. MRFs\\nlabeled antimyosin (muscle regulatory factors) are bHLH (basic helix-loop-helix)\\nantibody (   )  proteins produced only in developing muscle. MEFs (myocyte-\\nenhancing factors), which are produced in several tissues in\\naddition to developing muscle, belong to the MADS family. \\nThe myogenic activity of MRFs is enhanced by their interaction\\nwith MEFs.\\n18 CHAPTER 22 • Cell Birth, Lineage, And Death\\nbHLH proteins form homo- and heterodimers that bind scribed at several levels. First, the muscle regulators are ac-\\nto a 6-bp DNA site with the consensus sequence CANNTG tivated only in mesoderm cells, at the right place and time\\n(N = any nucleotide). Referred to as the E box, this sequence in the embryo, in response to spatial regulators of the sorts\\nis present in many different locations within the genome  that are described in Chapter 15. Other proteins mediate ad-\\n(on a purely random basis the E box will be found every 256 ditional mechanisms for assuring tight control over myoge-\\nnucleotides). Thus some mechanism(s) must ensure that nesis: chromatin-remodeling proteins are needed to make\\nMRFs specifically regulate muscle-specific genes and not target genes accessible to MRFs; inhibitory proteins can re-\\nother genes containing E boxes in their transcription- strict when MRFs act; and antagonistic relations between\\ncontrol regions. One clue to how this myogenic specificity is cell-cycle regulators and differentiation factors like MRFs en-\\nachieved was the finding that the DNA-binding affinity of sure that differentiating cells will not divide. All these factors\\nMyoD is tenfold greater when it binds as a heterodimer com- control when and where muscles form.\\nplexed with E2A, another bHLH protein, than when it binds\\nas a homodimer. Moreover, in azacytidine-treated C3H Activating Chromatin-Remodeling Proteins MRF proteins\\n10T1⁄2 cells, MyoD is found as a heterodimer complexed control batteries of muscle-specific genes, but can do so only\\nwith E2A, and both proteins are required for myogenesis in if chromatin factors allow access. Remodeling of chromatin,\\nthese cells. The DNA-binding domains of E2A and MyoD which usually is necessary for gene activation, is carried out\\nhave similar but not identical amino acid sequences, and by large protein complexes (e.g., Swi/Snf complex) that have\\nboth proteins recognize E box sequences. The other MRFs ATPase and perhaps helicase activity. These complexes are\\nalso form heterodimers with E2A that have properties simi- thought to recruit histone acetylases that modify chromatin\\nlar to MyoD-E2A complexes.  to make genes accessible to transcription factors (Chapter\\nSince E2A is expressed in many tissues, the requirement 11). The need for remodeling complexes to help myogenic\\nfor E2A is not sufficient to confer myogenic specificity. Sub- factors was tested using dominant-negative versions of the\\nsequent studies suggested that specific amino acids in the ATPase proteins that form the cores of these complexes. (Re-\\nbHLH domain of all the MRFs may confer myogenic speci- call from Chapter 9 that a dominant-negative mutation pro-\\nficity by allowing MRF-E2A complexes to bind specifically duces a mutant phenotype even when a normal allele of the\\nto another family of DNA-binding proteins called myocyte- gene also is present.) When genes carrying these dominant-\\nenhancing factors,or MEFs.MEFs were considered excellent negative mutations were transfected into C3H 10T1⁄ cells,\\n2\\ncandidates for interaction with MRFs for two reasons. First, the subsequent introduction of myogenic genes no longer\\nmany muscle-specific genes contain recognition sites for both converted the cells into myotubes. In addition, a muscle-\\nMEFs and MRFs. Second, although MEFs cannot induce specific gene that is normally activated did not exhibit its\\nmyogenic conversion of azacytidine-treated C3H 10T1⁄2 usual pattern of chromatin changes in the doubly transfected\\ncells, they enhance the ability of MRFs to do so. This en- C3H 10T1⁄ cells. These results indicate that transcription ac-\\n2\\nhancement requires physical interaction between a MEF and tivation by myogenic proteins depends on a suitable chro-\\nMRF-E2A heterodimer. Later, MEFs were found to belong to matin structure in the regions of muscle-specific genes.\\nthe MADS family of transcription factors and to contain a\\nMEF domain, adjacent to the MADS domain, that mediates Inhibitory Proteins Screens for genes related to myoDled to\\ninteraction with myogenin (Figure 22-16b). The synergistic identification of a related protein that retains the dimeriza-\\naction of the MEF homodimer and MRF-E2A heterodimer is tion region but lacks the basic DNA-binding region and\\nthought to drive high-level expression of muscle-specific hence is unable to bind to E box sequences in DNA. By bind-\\ngenes.  ing to MyoD or E2A, this protein inhibits formation of\\nKnockout mice and Drosophilamutants have been used MyoD-E2A heterodimers and hence their high-affinity bind-\\nto explore the roles of MRF and MEF proteins in conferring ing to DNA. Accordingly, this protein is referred to as Id, for\\nmyogenic specificity in intact animals rather than cell cul- inhibitor of DNA binding. Id prevents cells that produce\\nture. These experiments demonstrated the importance of MyoD and E2A from activating transcription of the muscle-\\nthree of the MRF proteins and of MEF proteins for distinct specific gene encoding creatine kinase. This finding pre-\\nsteps in muscle development (see Figure 22-14). The function sumably reflects the formation of MyoD-Id or E2A-Id \\nof the fourth myogenic protein, Mrf4, is not entirely clear; complexes, which cannot bind to and activate transcription\\nit may be expressed later and help maintain differentiated of muscle-specific genes. As a result, the cells remain in a\\nmuscle cells.  proliferative growth state. When these cells are induced to\\ndifferentiate into muscle (for instance, by the removal of\\nserum, which contains the growth factors required for pro-\\nTerminal Differentiation of Myoblasts Is Under\\nliferative growth), the Id concentration falls. MyoD-E2A\\nPositive and Negative Control \\ndimers now can form and bind to the regulatory regions of\\nPowerful developmental regulators like the MRFs cannot be target genes, driving differentiation of C3H 10T1⁄2cells into\\nallowed to run rampant. In fact, their actions are circum- myoblast-like cells. \\n22.3 • Specification and Differentiation of Muscle 19\\nRecent work shows that histone acetylases and deacety- Signals Are Crucial for Muscle Cell-Fate\\nlases are crucial for regulating muscle-specific genes. As ex- Determination and Myoblast Migration \\nplained in Chapter 11, acetylation of histones in chromatin \\nAs noted already, after myoblasts arise from somites, they\\nis necessary to activate many genes; in contrast, histone\\nmust not only proliferate but also move to their proper lo-\\ndeacetylases  cause  transcriptional  repression  (see  Figure \\ncations and form the correct attachments as they differ-\\n11-32). MEF2 recruits histone acetylases such as p300/CBP,\\nentiate into muscle cells (Figure 22-17). Myogenic gene \\nthrough another protein that serves as a mediator, to activate\\nexpression often follows elaborate organizational events that\\ntranscription of target genes. Chromatin immunoprecipitation\\ntell certain somite cells to delaminate from the somite ep-\\nexperiments with antibodies against acetylated histone H4\\nithelium and which way to move. A transcription factor,\\nshow that the acetylated histone level associated with MEF2-\\nPax3, is produced in the subset of somite cells that will form\\nregulated genes is higher in differentiated myotubes than in\\nmuscle and appears to be at the top of the regulatory hierar-\\nmyoblasts (see Figure 11-31). The role of histone deacetylases\\nchy controlling muscle formation in the body wall and limbs.\\nin muscle development was revealed in experiments in which\\nMyoblasts that will migrate, but not cells that remain be-\\nscientists first introduced extra myoDgenes into cultured C3H\\nhind, also produce a transcription factor called Lbx1. If Pax3\\n10T1⁄ cells to raise the level of MyoD. This resulted in in-\\n2\\nis not functional, Lbx1 transcripts are not seen and myo-\\ncreased activation of target genes and more rapid differentia-\\nblasts do not migrate. Both Pax3 and Lbx1 can affect ex-\\ntion of the cells into myotubes. However, when genes encoding\\npression of myoD.\\nhistone deacetylases also were introduced into the C3H 10T1⁄\\n2\\nDeparture of myoblasts from somites depends upon a \\ncells, the muscle-inducing effect of MyoD was blocked and the\\nsecreted protein signal appropriately called scatter factor,\\ncells did not differentiate into myotubes. \\nor hepatocyte growth factor (SF/HGF).This signal is pro-\\nThe explanation for how histone deacetylases inhibit\\nduced by embryonic connective tissue cells (mesenchyme) in\\nMyoD-induced muscle differentiation came from the sur-\\nthe limb buds to which myoblasts migrate. Production of\\nprising finding that MEF2 can bind, through its MADS do-\\nSF/HGF is previously induced by other external signals such\\nmain, to a histone deacetylase. This interaction, which can\\nas fibroblast growth factor and Sonic hedgehog, which are\\nprevent MEF2 function and muscle differentiation, is nor-\\ncritical to limb development (Chapter 15). The cell-surface\\nmally blocked during differentiation because the histone\\nreceptor for SF/HGF, which is expressed by myoblasts, be-\\ndeacetylase is phosphorylated by a calcium/calmodulin-\\nlongs to the receptor tyrosine kinase (RTK) class of receptors\\ndependent protein kinase; the phosphorylated deacetylase\\n(Chapter 14). Cells migrate from the somites at the head-to-\\nthen is moved from the nucleus to the cytoplasm. Taken to-\\ntail body levels where limbs will form, and not elsewhere,\\ngether, these results indicate that activation of muscle genes\\nby MyoD and MEF2 is in competition with inactivation of\\nmuscle genes by repressive chromatin structures and that nu-\\nclear versus cytoplasmic localization of chromatin factors is Neural tube\\na key regulatory step. Dermomyotome\\n(gives rise to\\ndermis of skin)\\nCell-Cycle Proteins The onset of terminal differentiation in Epidermis\\nmany cell types is associated with arrest of the cell cycle, Myoblasts\\nmost commonly in G , suggesting that the transition from (migrate to FPO\\n1\\nlimb bud to\\nthe determined to differentiated state may be influenced by\\nform skeletal\\ncell-cycle proteins including cyclins and cyclin-dependent  muscle)\\nkinases(Chapter 21). For instance, certain inhibitors of cyclin-\\ndependent kinases can induce muscle differentiation in cell\\nculture, and the amounts of these inhibitors are markedly\\nNotochord\\nhigher in differentiating muscle cells than in nondifferentiat-\\nSclerotome \\ning ones in vivo. Conversely, differentiation of cultured  (gives rise to skeletal structures \\nmyoblasts can be inhibited by transfecting the cells with such as vertebrae)\\nDNA encoding cyclin D1 under the control of a constitutively\\n▲FIGURE 22-17 Embryonic determination and migration\\nactive promoter. Expression of cyclin D1, which normally\\nof myoblasts in mammals. After formation of the neural tube,\\noccurs only during G , is induced by mitogenic factors in\\n1 each somite forms sclerotome, which develops into skeletal\\nmany cell types and drives the cell cycle (see Figure 21-28).\\nstructures, and dermomyotome. The dermomyotome gives rise to\\nThe ability of cyclin D1 to prevent myoblast differentiation\\nthe dermis of the skin and to the muscles. Lateral myoblasts\\nin vitro may mimic aspects of the in vivo signals that antag-\\nmigrate to the limb bud; medial myoblasts develop into the trunk\\nonize the differentiation pathway. The antagonism between muscles. The remainder of a dermomyotome gives rise to the\\nnegative and positive regulators of G1progression is likely to connective tissue of the skin. [Adapted from M. Buckingham, 1992,\\nplay an important role in controlling myogenesis in vivo. Trends Genet.8:144.]\\n20 CHAPTER 22 • Cell Birth, Lineage, And Death\\ndue to the presence of SF/HGF at limb locations and not teins, designated Da and Emc, are analogous in structure and\\nelsewhere. If the SF/HGF signal or its receptor is not func- function to vertebrate E2A and Id, respectively. For example,\\ntional, somite cells will produce Lbx1 but not go on to mi- heterodimeric complexes of Da with Ac or Sc bind to DNA\\ngrate; thus no muscles will form in the limbs. Expression of better than the homodimeric forms of Ac and Sc. Emc, like\\nthe myogeningene, which is necessary for myotube forma- Id, lacks a DNA-binding basic domain; it binds to Ac and\\ntion, and of mrf4, which is necessary for muscle fiber differ- Sc proteins, thus inhibiting their association with Da and\\nentiation, does not begin until migrating myoblasts approach binding to DNA. The similar functions of these myogenic\\ntheir limb-bud destination; these steps in terminal muscle dif- and neurogenic proteins are depicted in Figure 22-18. \\nferentiation are presumably not compatible with migration.  A family of bHLH proteins related to the Drosophila\\nWe have touched on just a few of the many external sig- Achaete and Scute proteins has been identified in vertebrates.\\nnals and transcription factors that participate in develop- One of these, called neurogenin,which has been identified in\\nment of a properly patterned muscle. The function of all the rat, mouse, and frog, appears to control the formation of\\nthese regulatory molecules must be coordinated both in neuroblasts. In situ hybridization experiments showed that\\nspace and in time during myogenesis.  neurogenin is produced at an early stage in the developing nerv-\\nous system and may induce production of NeuroD, another\\nbHLH protein that acts later (Figure 22-19a). Injection of large\\nbHLH Regulatory Proteins Function in Creation \\namounts of neurogeninmRNA into Xenopusembryos further\\nof Other Tissues\\ndemonstrated the ability of neurogenin to induce neurogenesis\\nFour bHLH transcription factors that are remarkably similar (Figure 22-19b). These studies suggest that the function of neu-\\nto the myogenic bHLH proteins control neurogenesis in rogenin is analogous to that of the Achaete and Scute in\\nDrosophila. Similar proteins appear to function in neurogen- Drosophila;likewise, NeuroD and Asense may have analogous\\nesis in vertebrates and perhaps in the determination and dif- functions in vertebrates and Drosophila,respectively.\\nferentiation of hematopoietic cells.  In addition to neurons, the nervous system contains a\\nThe neurogenic DrosophilabHLH proteins are encoded large number of glial cells, which also arise from the neuro-\\nby an ≈100-kb stretch of genomic DNA, termed the achaete- ectoderm. Glial cells support and insulate neurons; they also\\nscute complex (AS-C), containing four genes designated provide guidance and contact surfaces for migrating neurons\\nachaete (ac),scute (sc),lethal of scute (l’sc),and asense (a). during development and send a signal to neurons that pro-\\nAnalysis of the effects of loss-of-function mutations indicate motes formation of synapses. Neurogenins control the fates\\nthat the Achaete (Ac) and Scute (Sc) proteins participate in of precursor cells that are capable of making either neurons\\ndetermination of neuronal precursor cells, called neuroblasts, or glial cells by promoting neural development and repress-\\nwhile the Asense (As) protein is required for differentiation ing glial development. Neurogenins are therefore switches\\nof these cells into neurons. These functions are analogous to that control the decision between two alternative cell fates,\\nthe roles of MyoD and Myf5 in muscle determination and just as the yeast mating-type regulators select among three\\nof myogenin in differentiation. Two other Drosophilapro- cell types. \\nFly neurogenesis\\nac, sc mRNA Da(general) as mRNA\\n+\\nachaete asense\\nAc, Sc Asense Neurons\\nscute (others)\\n−\\nProneural Neural\\ngenes Emc precursor\\nFPgenesO\\n!FIGURE 22-18 Comparison of genes\\nthat regulate Drosophilaneurogenesis\\nVertebrate myogenesis\\nand mammalian myogenesis. bHLH\\nmyf5, myoD \\ntranscription factors have analogous mRNA myogenin mRNA\\nE2A(general)\\nfunctions in determination of precursor\\n+\\ncells (i.e., neuroblasts and myoblasts) and myf5 myogenin\\ntheir subsequent differentiation into mature myoD Myf5, MyoD mrf4 (?) Myogenin Muscle\\nneurons and muscle cells. In both cases, (others) (others)\\n−\\nthe proteins encoded by the earliest-acting Promuscular Muscle precursor\\nId\\ngenes (left) are under both positive and genes genes\\nnegative control by other related proteins\\n(blue type). [Adapted from Y. N. Jan and L. Y. Precursor determination Differentiation\\nJan, 1993, Cell75:827.]\\n22.4 • Regulation of Asymmetric Cell Division 21\\n(b)\\n(a)\\nFPO\\nneurogenin mRNA  neuroD mRNA  β-tubulin mRNA  neurogenin mRNA \\n▲EXPERIMENTAL FIGURE 22-19 In situ hybridization  ventricular zone (other arrow). (b) One of the two cells in early\\nand injection experiments demonstrate that neurogenin acts Xenopusembryos was injected with neurogeninmRNA (inj) and\\nbefore NeuroD in vertebrate neurogenesis.(a) Sections of rat then stained with a probe specific for neuron-specific mRNAs\\nneural tube were treated with a probe specific for neurogenin encoding !-tubulin (left) or NeuroD (right).The region of the\\nmRNA (left) or neuroDmRNA (right).The open space in the embryo derived from the uninjected cell served as a control\\ncenter is the ventricle, and the cells lining this cavity are in the (con). The neurogeninmRNA induced a massive increase in the\\nventricular layer. All the neural cells are born in the ventricular number of neuroblasts expressing neuroDmRNA and neurons\\nlayer and then migrate outward. As illustrated in these expressing !-tubulinmRNA in the region of the neural tube\\nmicrographs, neurogeninmRNA is produced in proliferating derived from the injected cell. [From Q. Ma et al., 1996, Cell87:43;\\nneuroblasts in the ventricular layer (arrow), whereas neuroD courtesy of D. J. Anderson.]\\nmRNA is present in migrating neuroblasts that have left the\\n■ Migration of myoblasts to the limb buds is induced by\\nKEY CONCEPTS OF SECTION 22.3 scatter factor/hepatocyte growth factor (SF/HGF), a protein\\nsignal secreted by mesenchymal cells (see Figure 22-17).\\nSpecification and Differentiation of Muscle\\nMyoblasts must express both the Pax3 and Lbx1 tran-\\n■ Development of skeletal muscle begins with the signal- scription factors to migrate. \\ninduced determination of certain mesoderm cells in somites\\nas myoblasts. Following their proliferation and migration, ■ Terminal differentiation of myoblasts and induction of\\nmuscle-specific proteins do not occur until myoblasts stop\\nmyoblasts stop dividing and differentiate into multinucle-\\ndividing and begin migrating. \\nate muscle cells (myotubes) that express muscle-specific\\nproteins (see Figure 22-14). ■ Neurogenesis in Drosophiladepends upon a set of four neu-\\nrogenic bHLH proteins that are conceptually and structurally\\n■ Four  myogenic  bHLH  transcription  factors—MyoD, similar to the vertebrate myogenic proteins (see Figure 22-18). \\nmyogenin, Myf5, and MRF4, called muscle-regulatory fac-\\ntors (MRFs)—associate with E2A and MEFs to form large ■ A related vertebrate protein, neurogenin, is required for\\nformation of neural precursors and also controls their di-\\ntranscriptional complexes that drive myogenesis and ex-\\nvision into neurons or glial cells. \\npression of muscle-specific genes. \\n■ Dimerization of bHLH transcription factors with dif- 22.4 Regulation of Asymmetric \\nferent partners modulates the specificity or affinity of their\\nbinding to specific DNA regulatory sites, and also may pre- Cell Division \\nvent their binding entirely.\\nDuring embryogenesis,the earliest stage in animal develop-\\n■ The myogenic program driven by MRFs depends on the ment, asymmetric cell division often creates the initial diver-\\nSwi/Snf chromatin-remodeling complex, which makes tar-\\nsity that ultimately culminates in formation of specific\\nget genes accessible. \\ndifferentiated cell types. Even in bacteria, cell division may\\n■ The myogenic program is inhibited by binding of Id pro- yield unequal daughter cells, for example, one that remains\\ntein to MyoD, thereby blocking binding of MyoD to DNA, attached to a stalk and one that develops flagella used for\\nand by histone deacetylases, which repress activation of swimming. Essential to asymmetric cell division is polariza-\\ntarget genes by MRFs. tionof the parental cell and then differential incorporation \\n22 CHAPTER 22 • Cell Birth, Lineage, And Death\\nCytoplasm Interestingly, some haploid yeast cells can switch repeat-\\nProtein\\nor mRNA edly between the \" and a types. Mating-type switching\\noccurs when the \"allele occupying the MATlocus is replaced\\nby the aallele, or vice versa. The first step in this process is\\ncatalyzed by HO endonuclease, which is expressed in mother\\nPolarized parental cell cells but not in daughter cells. Thus mating-type switch-\\ning occurs only in mother cells (Figure 22-21). Transcription\\nFPO of the HOgene is dependent on the Swi/Snf chromatin-\\nremodeling complex(see Figure 11-37), the same complex\\nthat we encountered earlier in our discussion of myogenesis.\\nDaughter yeast cells arising by budding from mother cells\\ncontain a protein called Ash1 that prevents recruitment of\\nMitotic the Swi/Snf complex to the HOgene, thereby preventing its\\nspindle\\ntranscription. The absence of Ash1 from mother cells allows\\nthem to transcribe the HOgene. \\nRecent experiments have revealed how the asymmetry in\\nthe distribution of Ash1 between mother and daughter cells\\nis established. ASH1mRNA accumulates in the growing bud\\nAsymmetric daughter cells that will form a daughter cell due to the action of a myosin\\nmotor protein (Chapter 19). This motor protein, called\\n▲FIGURE 22-20 General features of asymmetric cell Myo4p, moves the ASH1mRNA, as a ribonucleoprotein\\ndivision.Various mechanisms can lead to asymmetric distribution\\ncomplex, along actin filaments in one direction only, toward\\nof cytoplasmic components, such as particular proteins or\\nthe bud (Figure 22-22). Two connector proteins tether ASH1\\nmRNAs (red dots) to form a polarized parental cell. Division of a\\nmRNA to the motor protein. By the time the bud separates\\npolarized cell will be asymmetric if the mitotic spindle is oriented\\nfrom the mother cell, the mother cell is largely depleted of\\nso that the localized cytoplasmic components are distributed\\nASH1mRNA and thus can switch mating type in the fol-\\nunequally to the two daughter cells, as shown here. However, if\\nlowing G before additional ASH1mRNA is produced and\\nthe spindle is positioned differently relative to the localized 1\\nbefore DNA replication in the S phase.\\ncytoplasmic components, division of a polarized cell may produce\\nequivalent daughter cells. \\nBud\\n\"\\nof parts of the parental cell into the two daughters (Figure\\n22-20). A variety of molecular mechanisms are employed to Division\\ncreate and propagate the initial asymmetry that polarizes the\\nparental cell. In addition to being different, the daughter cells \" \"\\nmust often be placed in a specific orientation with respect to\\n− Ash1 M D + Ash1\\nsurrounding structures. HO transcriptiFon      PO     No HO transcription\\nWe begin with an especially well-understood example of Switching                   No switching\\nasymmetric cell division, the budding of yeast cells, and\\nmove on to recently discovered protein complexes important a a \" \"\\nfor asymmetric cell divisions in multicellular organisms. We\\nM D M D\\nsee in this example an elegant system that links asymmetric\\ndivision to the process of controlling cell type. \\n\" \" a a\\nM D M D\\nYeast Mating-Type Switching Depends \\nupon Asymmetric Cell Division ▲FIGURE 22-21 Specificity of mating-type switching in\\nhaploid yeast cells. Division by budding forms a larger mother\\nWhether a haploid yeast cell exhibits the \"or amating type\\ncell (M) and smaller daughter cell (D), both of which have the\\nis determined by which genes are present at the MATlocus\\nsame mating type as the original cell (\" in this example). The\\n(see Figure 22-11). As described in Chapter 11, the MAT\\nmother can switch mating type during G of the next cell cycle\\nlocus in the S. cerevisiaegenome is flanked by two “silent,” 1\\nand then divide again, producing two cells of the opposite a\\ntranscriptionally inactive loci containing the alternative \"or\\ntype. Switching depends on transcription of the HOgene, which\\na sequences (see Figure 11-28). A specific DNA rearrange-\\noccurs only in the absence of Ash1 protein. The smaller daughter\\nment brings the genes that encode the \"-specific or a-specific cells, which produce Ash1 protein, cannot switch; after growing\\ntranscription factors from these silent loci to the active MAT in size through interphase, they divide to form a mother cell and\\nlocus where they can be transcribed. daughter cell. Orange cells and arrows indicate switch events.\\n22.4 • Regulation of Asymmetric Cell Division 23\\n!FIGURE 22-22 Model for restriction of mating-\\ntype switching to mother cells in S.cerevisiae. Ash1\\nprotein prevents a cell from transcribing the HOgene AV M\\nwhose encoded protein initiates the DNA rearrangement ASH1 SHide E\\nttsohe apata .r rSeatsweuisltt csfhr oiinnmg m  oaa ctncineugwrs-tl yyo pnbeluy ds idwne ittdhc hedi anmuggo fhtrhtoeemrr   ccaeelltllo,,   ba\"fetceoarr  ui\"tse mRNA Bud 1 mRNo Anim DIA \\nAsh1 protein is present in the daughter cell but not in She3p Aa C\\nltohcea lmizaottihoenr  ocfe lAl.s Thh1e i sm tohlee counlaer- wbaasyi str afonrs pthoirst  doiff fAerSeHn1tial  Location:  ON\\nmRNA into the bud. A linking protein, She2p, binds  She2Fp POMyo4p Actin liz N\\na E\\nto specific 3% untranslated sequences in the ASH1 t\\nio C\\nmRNA and also binds to She3p protein. This protein in n T\\nturn binds to a myosin motor, Myo4p, which moves I\\nO\\nalong actin filaments into the bud. [See S. Koon and \\nN\\nB. J. Schnapp, 2001, Curr. Biology11:R166–R168.]\\nS\\nBudding yeasts use a relatively simple mechanism to  dermal fate (see Figures 14-29 and 15-36). The delaminat-\\ncreate molecular differences between the two cells formed by ing cells move inside and become spherical neuroblasts,\\ndivision. In higher organisms, polarization of the parental while the prospective epidermal cells remain behind and\\ncell involves many more participants, and, in addition, the close up to form a tight sheet.\\nmitotic spindle must be oriented in such a way that each Once formed, the neuroblasts undergo asymmetric divi-\\ndaughter cell receives its own set of cytoplasmic components. sions, at each division recreating themselves and producing a\\nTo illustrate these complexities, we focus on asymmetric  ganglion mother cell (GMC)at the basal side of the neuro-\\ndivision of neuroblasts in Drosophila. Genetic studies in  blast (Figure 22-23). A single neuroblast will produce several\\nC. elegansand Drosophilahave revealed the key participants, GMCs; each GMC in turn forms two neurons. Depending \\na first step in understanding at the molecular level how asym- on where they form in the embryo and consequent regula-\\nmetric cell division is regulated in multicellular organisms. tory events, neuroblasts may form more or fewer GMCs.\\nLarge Protein Complexes Are Localized \\nat Opposite Ends of Dividing Neuroblasts  1 Ectodermal\\ncells\\nin Drosophila\\nDV M\\nFly neuroblasts arise from a sheet of ectoderm cells that is 2 roside E\\noo D\\none cell thick. As in vertebrates, the Drosophilaectoderm ph A I\\nfeoctromdse rbmo tche llesp hidaevrem thise  panodte nthtiea ln teor avsosuusm sey esittehmer,  aa nndeu mraal noyr  3 ilanim A \\nEa C\\neopnildye rinm acel rfatatein.  Ucneldlse,r  sthoem ceo notfr tohl eo fc eglelns eisn tchraeta sbee cionm siez aec atinvde Neuroblast mbrtion ON\\nbegin to loosen from the ectodermal layer. At this point, the 4 yo: E N\\nFPO c\\ndelaminating cells use Notch signaling to mediate lateral  to E\\ninhibitionof their neighbors, causing them to retain the epi- GMC de C\\nr T\\nm\\nI\\n5 al C ON\\n!FIGURE 22-23 Asymmetric cell division during Drosophila GMC ell  S\\nneurogenesis.The ectodermal sheet (1) of the early embryo gives rise to both D\\nepidermal cells and neural cells. Neuroblasts, the stem cells for the nervous iv\\nis\\nsystem, are formed when ectoderm cells enlarge, separate from the ectodermal 6 io\\nepithelium, and move into the interior of the embryo (2–4). Each neuroblast n\\ns\\nthat arises divides asymmetrically to recreate itself and produce a ganglion Neurons  in\\nmother cell (GMC) (5). Subsequent divisions of a neuroblast produce more  t\\nGMC h\\nGMCs, creating a stack of these precursor cells (6). Each GMC divides once  e\\n \\nto give rise to two neurons (7). Neuroblasts and their GMC descendants can\\n7\\nhave different fates depending on their location.\\n24 CHAPTER 22 • Cell Birth, Lineage, And Death\\nNeuroblasts and GMCs in different locations exhibit differ- in the process of division. This complex is already localized\\nent patterns of gene expression, an indicator of their fates. in ectoderm cells and stays at the apical end as a prospective\\nAnalysis of fly mutants led to the discovery of key proteins neuroblast delaminates from the ectodermal sheet (Figure\\nthat participate in creating specific neuroblasts, inhibiting 22-24a, left). Three proteins compose the apical complex:\\nepidermal cells from becoming neuroblasts, and directing Baz, Par6, and PKC3, the last being an isoform of protein \\nneuroblasts to divide asymmetrically. kinase C. The Baz/Par6/PKC3 complex persists at the apical\\nend from late interphase (G stage) until late anaphase in mi-\\n2\\nApical Baz/Par6/PKC3 Complex A ternary complex of pro- tosis and then disperses or is destroyed (see Figure 20-29 for\\nteins, referred to as the apical complex,is located at the end mitotic stages). After each neuroblast division, the complex\\nof a fly neuroblast that will become the “new” neuroblast re-forms at the apical end of the progeny neuroblast in the\\nnext interphase. It remains unclear how polarity information\\nis preserved through telophase, since no known proteins are\\nlocalized during that time. As noted previously, the very first\\nApical (Baz, \\nProtein complexes: Miranda Numb\\nPar6, PKC3) C. eleganscell division after fertilization is asymmetric, form-\\ning an AB cell and P1 cell, which have quite different fates\\n(see Figure 22-9). Remarkably, a protein complex like the\\nfly Baz/Par6/PKC3 complex controls the asymmetry of the\\n(a) Neuroblast P1-AB division; similar protein complexes exist in mammals\\nas well.\\nFPO Two mechanisms control the apical localization of the\\nDrosophilaBaz/Par6/PKC3 complex: the first operates in the\\nGMC\\nectoderm prior to neuroblast delamination; the second is ac-\\ntive during the repeating divisions of delaminated neuro-\\nINTERPHASE ANAPHASE blasts. The first mechanism involves another multiprotein\\nNEUROBLAST NEUROBLAST complex composed of at least three proteins, Scribble (Scrib),\\nCentromere Discs-large (Dlg), and Lethal giant larvae (Lgl). This complex\\nis located in the cortical region of ectoderm cells (i.e., the re-\\ngion just below the plasma membrane), but it is not polar-\\n(b) ized along the apical-basal axis of these cells. Thus, though\\nNeuroblast\\nnecessary for the apical localization of the Baz/Par6/PKC3\\ncomplex, the Scrib/Dlg/Lgl complex probably is not suffi-\\ncient. Once a neuroblast has separated from the ectoderm,\\nGMC two other proteins called Inscuteable and Partner of Inscute-\\nCytoskeleton: able join the Baz/Par6/PKC3 complex, forming a five-protein\\nActin microfilaments complex. These two additional proteins probably stabilize\\nChromosomes\\nMicrotubules the Baz/Par6/PKC3 complex. Reassembly of the entire com-\\nplex at the apical end following each neuroblast division re-\\n▲FIGURE 22-24 Localized protein complexes that control quires microtubules and perhaps microfilaments (Figure\\nasymmetric cell division in the Drosophilaneuroblast.(a) The 22-24b). This finding suggests that motor protein–mediated\\napical complex (blue), which consists of three proteins (Baz, Par6, transport of these protein complexes along cytoskeletal ele-\\nand PKC3), is localized in ectoderm cells and in delaminating ments plays a role in their apical localization. \\nneuroblasts. As a neuroblast begins dividing, the Miranda\\ncomplex (orange) assembles at the apical side and then moves Basal Complexes In a dividing neuroblast two basal com-\\nto the basal side, where it will be incorporated into the ganglion plexesare found at the end that will form a ganglion mother\\nmother cell (GMC). The second basal complex (green), containing cell (GMC) at division (see Figure 22-24a, right). Localiza-\\nthe Numb protein, is detected only basally. Mutations in genes tion of these complexes, called the Miranda complex and the\\nthat encode polarized proteins disrupt asymmetric cell division.\\nNumb complex, at the basal end of neuroblasts requires the\\n(b) Motor protein–mediated transport along cytoskeletal filaments\\napical Baz/Par6/PKC3 complex. The Numb complex is lo-\\nis thought to localize the apical and basal complexes. Actin\\ncated throughout the cell during interphase; at prophase it\\nmicrofilaments (red) lie just under the cell surface at all times.\\nbecomes basally located. In contrast, the Miranda complex\\nMicrotubules (blue) radiate from the centrosome during\\ninitially assembles at the apical end; it then moves to the\\ninterphase and then assemble into the mitotic spindle, attached\\nbasal end at prophase. The apical Baz/Par6/PKC3 complex\\nto the duplicated centrosomes, during cell division. Note that in\\nthe neuroblast-GMC division the centrosome is skewed to the forms even when either of the basal complexes cannot as-\\nneuroblast end of the cell, and the chromosomes (light gray) are semble due to mutations. Mutations in any of the compo-\\ndiagrammed lined up on the spindle. [Adapted from C. Q. Doe and nents of the apical complex, however, prevent the Miranda\\nB. Bowerman, 2001, Curr. Opin. Cell Biol.13:68.] and Numb complexes from moving to the basal region. \\n22.4 • Regulation of Asymmetric Cell Division 25\\nInstead they accumulate in the cortical region or in clumps blasts, the mitotic spindle first aligns perpendicular to the\\nthroughout the cell. The simplest idea, that the basal com- apical-basal axis and then turns 90 degrees to align with it\\nplexes form wherever the apical complex is absent, is prob- at the same time that the basal complexes become localized\\nably wrong, since parts of the cell contain neither of these to the basal side (Figure 22-25). The apical complex, which\\ncomplexes.  is already in place before spindle rotation, controls the final\\nThe basal Miranda complex has four components: Pros- orientation of the spindle. This is supported by the finding\\npero protein, a presumed transcription factor that contains that mutations in any of the components of the apical com-\\na DNA-binding homeodomain; Miranda protein, which has plex eliminate the coordination of the spindle with apical-\\na coiled-coil structure; Staufen, an RNA-binding protein; basal polarity, causing the spindle orientation to become\\nand prosperomRNA. During each round of neuroblast di- random. \\nvision, new Prospero protein is made during interphase and Spindle orientation is linked to actin and myosin-related\\nincorporated into the Miranda basal complex, which is dis- proteins. Mammalian Par6, a component of the apical com-\\ntributed to the GMC at the time of cleavage. In the absence plex, can bind two small GTPase proteins, Cdc42 and\\nof Prospero, GMCs do not transcribe the appropriate genes Rac1, that control the arrangement of actin microfilaments\\nand do not develop into normal neurons. The participation in the cytoskeleton (see Figure 19-29). The protein Lethal\\nof a transcription factor in a localized cytoplasmic complex giant larvae (Lgl), a component of the Scrib/Dlg/Lgl asym-\\nprovides a link between the asymmetry of the cytoplasm and metry complex, helps to localize the Miranda complex\\nsubsequent cell-fate determination. Since the apical and basal basally in fly neuroblasts, although Lgl itself is localized\\ncomplexes are presumably localized by the cytoskeleton in apically. Lgl binds to myosin II, which functions in cyto-\\nconcert with motor proteins, the involvement of a transcrip- kinesis (see Figure 19-20), suggesting that the location of\\ntion factor in one of the complexes potentially explains how the cleavage furrow may be coordinated with localization\\na transcription factor is preferentially distributed to one of of the apical complex. \\ntwo daughter cells. The involvement of an mRNA and a Two yeast proteins related to Lgl also bind to myosin II;\\nRNA-binding protein in the Miranda complex suggests that they have been implicated in exocytosis and secretion, specif-\\nit also mediates translational control in GMCs.  ically in the docking of post-Golgi vesicles with the plasma\\nmembrane (Chapter 17). Mutations in the two yeast proteins\\nsuppress mutations in the myosin IIgene. This observation is\\nOrientation of the Mitotic Spindle Is Linked \\ninterpreted to mean that the function of myosin II in spindle\\nto Cytoplasmic Cell-Asymmetry Factors\\norientation is opposite to that of the Lgl-like proteins: re-\\nFor localized protein complexes to be differentially incorpo- duction of one protein’s function is ameliorated by reduction\\nrated into two daughter cells requires that the plane of cell of the other, restoring a semblance of the normal balance. \\ndivision be appropriately oriented. In dividing fly neuro- In this case, therefore, myosin and Lgl probably act in \\nFPO inV M\\n Dide E\\nroo D\\nso A I\\npn A\\n0 s 32 s 64 s 80 s hilaima  C\\nt\\nNio O\\nen N\\nu: \\nrM N\\no\\nbit E\\n▲imEagXiPngE RreIMveEaNlsT rAoLta tFioIGn UoRf Eth 2e 2m-2it5otiTci mspei-nladples ein fluorescence fauren cvtiisoinb laes o tnh eo psppoinsditlee  spiodleess ;o af st hme itcoeslils.  Tphreoscee eds the last Cell otic Spin CTIO\\nDd N\\nasymmetrically dividing neuroblasts. Early Drosophilaembryos microtubules forming the mitotic spindle are assembled ivle S\\nwere injected with a hybrid gene composed of the gene from the poles (see Figure 20-29). In successive  is A\\nencoding green fluorescent protein (GFP) fused to the gene images (at 32, 64, and 80 seconds), the bipolar spindle ionssy\\nencoding Tau, a protein that binds to microtubules. At the top  can be seem to form and rotate 90 degrees to align  m\\ne\\nare time-lapse images of a single dividing neuroblast in a live with the apical-basal axis, as schematically depicted at t\\nr\\nembryo. The basal side is at the top, and the apical side at the the bottom. [From J. A. Kaltschmidt et al, 2000, Nature Cell  y \\nbottom. At time 0, equivalent to prophase, the two centrosomes Biol.2:7–12.]\\n26 CHAPTER 22 • Cell Birth, Lineage, And Death\\nopposite directions: myosin II moving Miranda or other ma- organisms require signals to stay alive. In the absence of such\\nterials to control spindle orientation and Lgl restraining it.  survival signals, frequently referred to as trophic factors,cells\\nFurther evidence for the importance of myosin in spin- activate a “suicide” program. Second, in some developmental\\ndle orientation comes the finding that myosin VI binds di- contexts, including the immune system, specific signals induce\\nrectly to Miranda. In addition, mutations in myosin VI in a “murder” program that kills cells. Whether cells commit sui-\\nflies prevent basal targeting of Miranda and simultaneously cide for lack of survival signals or are murdered by killing sig-\\nblock proper spindle orientation. These various findings give nals from other cells, recent studies suggest that death is\\nus the beginning of a picture of how localized protein com- mediated by a common molecular pathway. In this section, we\\nplexes are coordinated with cell division, so that each daugh- first distinguish programmed cell death from death due to tis-\\nter cell receives the appropriate amounts of the various sue injury, then consider the role of trophic factors in neuronal\\ncomplexes.  development, and finally describe the evolutionarily conserved\\neffector pathway that leads to cell suicide or murder.\\nKEY CONCEPTS OF SECTION 22.4\\nProgrammed Cell Death Occurs \\nRegulation of Asymmetric Cell Division Through Apoptosis\\n■ Asymmetric cell division requires polarization of the di- The demise of cells by programmed cell death is marked by\\nviding cell, which usually entails localization of some cyto- a well-defined sequence of morphological changes, collec-\\nplasmic components, and then the unequal distribution of tively referred to as apoptosis, a Greek word that means\\nthese components to the daughter cells (see Figure 22-20).  “dropping off” or “falling off,” as in leaves from a tree.\\nDying cells shrink and condense and then fragment, releas-\\n■ In the asymmetric division of budding yeasts, a myosin-\\ning small membrane-bound apoptotic bodies, which gen-\\ndependent transport system carries ASH1mRNA into the\\nerally are engulfed by other cells (Figure 22-26; see also\\nbud (see Figure 22-22). \\nFigure 1-19). The nuclei condense and the DNA is frag-\\n■ Ash1 protein is produced in the daughter cell soon after mented. Importantly, the intracellular constituents are not\\ndivision  and  prevents  expression  of  HO  endonuclease,\\nreleased into the extracellular milieu where they might have\\nwhich is necessary for mating-type switching. Thus a daugh-\\ndeleterious effects on neighboring cells. The highly stereo-\\nter cell cannot switch mating type, whereas the mother cell\\ntyped changes accompanying apoptosis suggested to early\\nfrom which it arises can (see Figure 22-21). \\nworkers that this type of cell death was under the control of\\n■ Asymmetric cell division in Drosophilaneuroblasts depends a strict program. This program is critical during both em-\\non  localization  of  the  apical  protein  complex  (Baz/Par6/ bryonic and adult life to maintain normal cell number and\\nPKC3) and two basal complexes. The basal complexes are in- composition.\\ncorporated into the ganglion mother cell (GMC) and contain The genes involved in controlling cell death encode pro-\\nproteins that determine cell fate (see Figure 22-24).  teins with three distinct functions: \\n■ The apical Baz/Par6/PKC3 complex also controls asym- ■ “Killer” proteins are required for a cell to begin the\\nmetric cell division in C. elegansand perhaps mammals.  apoptotic process. \\n■ Asymmetry factors exert their influence at least in part ■ “Destruction” proteins do things like digest DNA in a\\nby controlling the orientation of the mitotic spindle, so that dying cell. \\nasymmetrically localized proteins and structures are differ-\\nentially incorporated into the two daughter cells. Myosin ■ “Engulfment” proteins are required for phagocytosis of\\nthe dying cell by another cell. \\nproteins bind to proteins that control asymmetry factors of\\ncells to control spindle orientation. At first glance, engulfment seems to be simply an after-death\\ncleanup process, but some evidence suggests that it is part\\nof the final death decision. For example, mutations in killer\\n22.5 Cell Death and Its Regulation  genes always prevent cells from initiating apoptosis, whereas\\nmutations that block engulfment sometimes allow cells to sur-\\nProgrammed cell death is a cell fate, an odd sort of cell fate vive that would normally die. That is, cells with engulfment-\\nbut nonetheless one that is essential. Cell death keeps our gene mutations can initiate apoptosis but then sometimes\\nhands from being webbed, our embryonic tails from persist- recover. \\ning, our immune system from responding to our own pro- In contrast to apoptosis, cells that die in response to tis-\\nteins, and our brain from being filled with useless electrical sue damage exhibit very different morphological changes,\\nconnections. In fact, the majority of cells generated during referred to as necrosis. Typically, cells that undergo this\\nbrain development also die during development. process swell and burst, releasing their intracellular con-\\nCellular interactions regulate cell death in two fundamen- tents that can damage surrounding cells and frequently\\ntally different ways. First, most cells, if not all, in multicellular cause inflammation.\\n22.5 • Cell Death and Its Regulation 27\\n!FIGURE 22-26 Ultrastructural\\nfeatures of cell death by apoptosis.\\n(a) Schematic drawings illustrating  F M\\no\\nthe progression of morphologic c E\\nu\\nchanges observed in apoptotic  s D\\ncells. Early in apoptosis, dense  A I\\nn A\\nchromosome condensation occurs im  \\nalong the nuclear periphery. The cell a C\\nbody also shrinks, although most tio O\\nn N\\norganelles remain intact. Later both : A N\\nthe nucleus and cytoplasm fragment, p\\no E\\nforming apoptotic bodies, which are p C\\nt\\nphagocytosed by surrounding cells. o T\\ns\\n(b) Photomicrographs comparing  is I\\nO\\na normal cell (top) and apoptotic cell\\nN\\n(bottom).Clearly visible in the latter\\nS\\nare dense spheres of compacted\\nchromatin as the nucleus begins to\\nfragment. [Part (a) adapted from J. Kuby,\\n28p (w) X 34p (d)\\n1997, Immunology,3d ed., W. H. Freeman\\n& Co., p. 53. Part (b) from M. J. Arends\\nand A. H. Wyllie, 1991, Int’l. Rev. Exp.\\nPathol.32:223.]\\nNeurotrophins Promote Survival of Neurons cremental increases in the target-field size are accompanied\\nby commensurate incremental increases in the number of\\nThe earliest studies demonstrating the importance of trophic neurons innervating the target field. This relation was found\\nfactors in cellular development came from analyses of the de- to result from the selective survival of neurons rather than\\nveloping nervous system. When neurons grow to make con- changes in their differentiation or proliferation. The obser-\\nnections to other neurons or to muscles, sometimes over vation that many sensory and motor neurons die after reach-\\nconsiderable distances, more cells grow than will eventually ing their peripheral target field suggested that these neurons\\nsurvive. Those that make connections prevail and survive; compete for survival factors produced by the target tissue. \\nthose that fail to connect die.  Subsequent to these early observations, scientists discov-\\nIn the early 1900s the number of neurons innervating the ered that transplantation of a mouse sarcoma tumor into a\\nperiphery was shown to depend upon the size of the tissue to chick led to a marked increase in the numbers of certain\\nwhich they would connect, the so-called “target field.” For in- types of neurons. This finding implicated the tumor as a rich\\nstance, removal of limb buds from the developing chick em- source of the presumed trophic factor. To isolate and purify\\nbryo leads to a reduction in the number of sensory neurons this factor, known simply as nerve growth factor (NGF), sci-\\nand motoneurons innervating the bud (Figure 22-27). Con- entists used an in vitro assay in which outgrowth of neurites\\nversely, grafting additional limb tissue to a limb bud leads  from sensory ganglia (nerves) was measured. Neurites are ex-\\nto an increase in the number of neurons in corresponding  tensions of the cell cytoplasm that can grow to become the\\nregions of the spinal cord and sensory ganglia. Indeed, in- long wires of the nervous system, the axons and dendrites\\n28 CHAPTER 22 • Cell Birth, Lineage, And Death\\nBDNF\\nNGF NT-3 NT-4/5 NT-3 NT-3\\nFPO\\nExterior\\nATP ATP ATP ATP ATP ATP\\nP P P P P P\\n21p (w) X 28p (d) ADP ADP ADP ADP ADP ADP\\nCytosol Tyrosine\\nkinase\\nTrkA TrkB TrkC\\n▲FIGURE 22-28 Specificity of neurotrophins for Trks,a\\nfamily of receptor tyrosine kinases. Each neurotrophin binds\\nwith high affinity to one Trk receptor indicated by the solid arrow\\nfrom the ligand to the receptor. NT-3 also can bind with lower\\naffinity to both TrkA and TrkB as indicated by the dashed arrow. \\nIn addition, neurotrophins bind to a distinct receptor called\\np75NTR either alone or in combination with Trks. NGF # nerve\\ngrowth factor; BDNF # brain-derived neurotrophic factor; \\nNT-3 # neurotrophin-3. [Adapted from W. D. Snider, 1994, Cell77:627.]\\nNeurotrophin Receptors Neurotrophins bind to and activate\\na family of receptor tyrosine kinases called Trks(pronounced\\n“tracks”). (The general structure of receptor tyrosine kinases\\n▲EXPERIMENTAL FIGURE 22-27 The survival of motor\\nand the intracellular signaling pathways they activate are cov-\\nneurons depends on the size of the muscle target field they\\nered in Chapter 14.) As shown in Figure 22-28, NGF binds to\\ninnervate.(a) Removal of a limb bud from one side of a chick\\nTrkA; BDNF, to TrkB; and NT-3, to TrkC. Binding of these fac-\\nembryo at about 2.5 days results in a marked decrease in the\\ntors to their receptors provides a survival signal for different\\nnumber of motor neurons on the affected side. In an amputated\\nembryo, normal numbers of motor neurons are generated on classes of neurons. A second type of receptor called p75NTR\\nboth sides (middle).Later in development, many fewer motor (NTR #neurotrophin receptor) also binds to neurotrophins,\\nneurons remain on the side of the spinal cord with the missing but with lower affinity. However, p75NTRforms heteromulti-\\nlimb than on the normal side (bottom).Note that only about 50 meric complexes with the different Trk receptors; this associa-\\npercent of the motor neurons that originally are generated tion increases the affinity of Trks for their ligands. Some studies\\nnormally survive. (b) Transplantation of an extra limb bud into an indicate that the binding of NGF to p75NTRin the absence of\\nearly chick embryo produces the opposite effect, more motor TrkA may promote cell death rather than prevent it.\\nneurons on the side with additional target tissue than on the\\nnormal side. [Adapted from D. Purves, 1988, Body and Brain: A Trophic Knockouts of Neurotrophins and Their Receptors To criti-\\nTheory of Neural Connections(Cambridge, MA: Harvard University Press), cally address the role of the neurotrophins in development, sci-\\nand E. R. Kandel, J. H. Schwartz, and T. M. Jessell, 2000, Principles of entists produced mice with knockout mutations in each of the\\nNeural Science,4th ed. (New York: McGraw-Hill), p. 1054, Figure 53-11.] neurotrophins and their receptors. These studies revealed that\\ndifferent neurotrophins and their corresponding receptors are\\nrequired for the survival of different classes of sensory neurons,\\n(see Figure 7-29). The later discovery that the submaxillary which carry signals from peripheral sensory systems to the\\ngland in the mouse also produces large quantities of NGF en- brain (Figure 22-29). For instance, pain-sensitive (nociceptive)\\nabled biochemists to purify it to homogeneity and to se- neurons, which express TrkA, are selectively lost from the dor-\\nquence it. A homodimer of two 118-residue polypeptides, sal root ganglion of knockout mice lacking NGF or TrkA,\\nNGF belongs to a family of structurally and functionally re- whereas TrkB- and TrkC-expressing neurons are unaffected in\\nlated trophic factors collectively referred to as neurotrophins. such knockouts. In contrast, TrkC-expressing proprioceptive\\nBrain-derived neurotrophic factor (BDNF) and neurotro- neurons, which detect the position of the limbs, are missing\\nphin-3 (NT-3) also are members of this protein family. from the dorsal root ganglion in TrkCand NT-3mutants.\\n22.5 • Cell Death and Its Regulation 29\\nWild Type Mutant Mutant \"EXPERIMENTAL FIGURE 22-29\\nDifferent classes of sensory neurons are\\nNGF −/− NT-3 −/− lost in knockout mice lacking different\\nSpinal Mechanoreceptors or or\\ntrophic factors or their receptors.In\\ncord Dorsal root TrkA −/− TrkC −/− animals lacking nerve growth factor (NGF)\\nganglion or its receptor TrkA, small nociceptive \\nPropioceptive (pain-sensing) neurons (blue) that innervate\\nneurons the skin are missing. These neurons\\nNociceptive express TrkA receptor and innervate \\nneurons NGF-producing targets. In animals lacking\\neither neurotrophin-3 (NT-3) or its receptor\\nTrkC, large propioceptive neurons (red)\\ninnervating muscle spindles are missing.\\nMuscle produces NT-3 and the\\nSkin\\npropioceptive neurons express TrkC.\\nMechanoreceptors (brown), another class\\nof sensory neurons in the dorsal root\\nganglion, are unaffected in these mutants.\\nMuscle Motoneurons\\n[Adapted from W. D. Snider, 1994, Cell77:627.]\\nA Cascade of Caspase Proteins Functions  (a)\\nin the Apoptotic Pathway \\nV M\\nid\\nKey insights into the molecular mechanisms regulating cell e E\\no D\\ndeath came from genetic studies in C. elegans.As we noted   A I\\nin Section 22.1, scientists have traced the lineage of all the nn A\\nsomatic cells in C. elegansfrom the fertilized egg to the  im  C\\nmature worm (see Figure 22-9). Of the 947 nongonadal cells at O\\ngenerated during development of the adult hermaphrodite ion N\\nform, 131 cells undergo programmed cell death.  : P N\\nr E\\nSpecific mutations have identified a variety of genes o\\ng C\\nwhose encoded proteins play an essential role in controlling ra T\\nprogrammed cell death during C. elegansdevelopment. For (b) m I\\nm O\\ninstance, programmed cell death does not occur in worms FPO e N\\nd\\ncarrying loss-of-function mutations in the ced-3gene or the  C S\\nced-4gene; as a result, the 131 “doomed” cells survive (Fig- e\\nure 22-30). CED-4 is a protease-activating factor that ll D\\ne\\ncleaves the CED-3 precursor protein, creating an active a\\nt\\nCED-3 protease that initiates cell death. In contrast, in ced- h\\n9mutants, all cells die during embryonic life, so the adult  in\\n C\\nform never develops. These genetic studies indicate that the . e\\nCED-3 and CED-4 proteins are required for cell death, that le\\ng\\nCED-9 suppresses apoptosis, and that the apoptotic path- a\\nn\\nway can be activated in all cells. Moreover, the finding that s\\ncell death does not occur in ced-9/ced-3double mutants sug- ▲EXPERIMENTAL FIGURE 22-30 Mutations in the Em\\ngests that CED-9 acts “upstream” of CED-3 to suppress the ced-3gene block programmed cell death in C.elegans. b\\nr\\napoptotic pathway. (a) Newly hatched larva carrying a mutation in the ced-1 yo\\nThe confluence of genetic studies in worms and studies gene. Because mutations in this gene prevent engulfment nic\\non human cancer cells first suggested that an evolutionarily of dead cells, highly refractile dead cells accumulate  D\\nconserved pathway mediates apoptosis. The first apoptotic (arrows), facilitating their visualization. (b) Newly hatched ev\\nlarva with mutations in both the ced-1and ced-3genes. e\\ngene to be cloned, bcl-2,was isolated from human follicular lo\\nlymphomas. A mutant form of this gene, created in lym- The absence of refractile dead cells in these double p\\nm\\nmutants indicates that no cell deaths occurred. Thus CED-3\\nphoma cells by a chromosomal rearrangement, was shown to e\\nprotein is required for programmed cell death. [From H. M. n\\nact as an oncogenethat promoted cell survival rather than t\\nEllis and H. R. Horvitz, 1986, Cell91:818; courtesy of Hilary Ellis.]\\ncell death (Chapter 23). The human Bcl-2 protein and worm\\nCED-9 protein are homologous, and a bcl-2transgene can\\n30 CHAPTER 22 • Cell Birth, Lineage, And Death\\nblock the extensive cell death found in ced-9mutant worms. For instance, expression of C. elegansCED-4 in a human\\nThus both proteins act as regulators that suppress the apop- kidney cell line induces rapid apoptosis. This can be blocked\\ntotic pathway (Figure 22-31). In addition, both proteins con- by co-expression of the negative regulator CED-9 (or mam-\\ntain a single transmembrane domain and are localized to the malian Bcl-2), indicating that CED-9 opposes CED-4 action.\\nouter mitochondrial, nuclear, and endoplasmic reticulum In addition, CED-9 has been shown to directly bind to \\nmembranes, where they serve as sensors that control the CED-4 and move it from the cytosol to intracellular mem-\\napoptotic pathway in response to external stimuli. As we dis- branes. Thus the pro-apoptotic function of CED-4 is directly\\ncuss below, other regulators promote apoptosis. suppressed by the anti-apoptotic function of CED-9. CED-4\\nThe effector proteinsin the apoptotic pathway are en- also binds directly to the CED-3 caspase (and related mam-\\nzymes called caspasesin vertebrates,so named because they malian caspases) and promotes activation of its protease ac-\\ncontain a key cysteine residue in the catalytic site and selec- tivity, as noted previously. Biochemical studies have shown\\ntively cleave proteins at sites just C-terminal to aspartate that CED-4 can simultaneously bind to both CED-9 and\\nresidues. Humans have 15 different caspases, all of which are CED-3. These results fit with the genetics, which shows that\\ninitially made as procaspases that must be cleaved to become the absence of CED-9 has no effect if CED-3 is also missing\\nactive. Such proteolytic processing of proproteins is used re- (ced-3/ced-9double mutants have no cell death, like ced-3\\npeatedly in blood clotting, generation of digestive enzymes, mutants).\\nand generation of hormones (Chapter 17). Initiator caspases\\n(e.g., caspase-9) are activated by autoproteolysis induced by\\nPro-Apoptotic Regulators Permit Caspase\\nother types of proteins, which help the initiators to aggre-\\nActivation in the Absence of Trophic Factors \\ngate. Activated initiator caspases cleave effector caspases\\n(e.g., caspase-3) and thus quickly amplify the total caspase Having introduced the major participants in the apoptotic\\nactivity level in the dying cell. The various effector caspases pathway, we now take a closer look at how the effector cas-\\nrecognize and cleave short amino acid sequences and differ in pases are activated in mammalian cells. Although the normal\\ntheir preferred target sequences. Their specific intracellular function of CED-9 or Bcl-2 is to suppress the cell-death path-\\ntargets include proteins of the nuclear lamina and cytoskele- way, other intracellular regulatory proteins promote apopto-\\nton whose cleavage leads to the demise of a cell. Seven lines sis. The first pro-apoptotic regulator to be identified, named\\nof knockout mice have been made, each lacking a particular Bax, was found associated with Bcl-2 in extracts of cells ex-\\ncaspase function. Some homozygotes die as embryos; others pressing high levels of Bcl-2. Sequence analysis demonstrated\\nare viable but with subtle defects. Strikingly, some exhibit that Bax is related in sequence to CED-9 and Bcl-2, but over-\\nan excessive number of neurons (i.e., neuronal hyperplasia), production of Bax induces cell death rather than protecting\\nreaffirming the importance of cell death in development of cells from apoptosis, as CED-9 and Bcl-2 do. Thus this family\\nthe nervous system; other mutants have cells that are resis- of regulatory proteins comprises both anti-apoptoticmem-\\ntant to apoptosis. The principal effector protease inC. elegans bers (e.g., CED-9, Bcl-2) and pro-apoptoticmembers (e.g.,\\nis the CED-3 caspase.  Bax). All members of this family, which we refer to as the Bcl-\\nCell-culture studies have yielded important insights about 2 family,are single-pass transmembrane proteins and can par-\\na group of adapter proteinsthat coordinate the action of reg- ticipate in oligomeric interactions. In mammals, six Bcl-2\\nulators and effectors to control apoptosis (see Figure 22-31). family members prevent apoptosis and nine promote it. Thus\\nthe fate of a given cell—survival or death—may reflect the\\nparticular spectrum of Bcl-2 family members made by that\\nRegulator Adapter Effector\\ncell and the intracellular signaling pathways regulating them.\\nFPO\\nSome Bcl-2 family members preserve or disrupt the in-\\nC. elegans CED-9 CED-4 CED-3 Death tegrity of mitochondria, thereby controlling release of mito-\\nchondrial proteins such as cytochrome c. In normal healthy\\nVertebrates Bcl-2 Apaf-1 Casp9 Casp3 Death\\ncells, cytochrome cis localized between the inner and outer\\nmitochondrial membrane, but in cells undergoing apoptosis,\\n▲FIGURE 22-31 Overview of the evolutionarily conserved cytochrome cis released into the cytosol. This release can be\\napoptotis pathway in C.elegansand vertebrates.Three blocked by overproduction of Bcl-2; conversely, overproduc-\\ngeneral types of proteins are critical in this conserved pathway.\\ntion of Bax promotes release of cytochrome cinto the cytosol\\nRegulators either promote or suppress apoptosis; two\\nand apoptosis. Moreover, injection of cytochrome cinto the\\northologous regulators shown here, CED-9 and Bcl-2, suppress\\ncytosol of cells induces apoptosis. A variety of death-inducing\\napoptosis in the presence of trophic factors. Adapters interact\\nstimuli cause Bax monomers to move from the cytosol to the\\nwith both regulators and effectors; in the absence of trophic\\nouter mitochondrial membrane where they oligomerize. Bax\\nfactors, they promote activation of effectors. Proteases called\\nhomodimers, but not Bcl-2 homodimers or Bcl-2/Bax het-\\ncaspases serve as effector proteins; their activation leads to\\ndegradation of various intracellular substrates and eventually  erodimers, permit influx of ions through the mitochondrial\\ncell death. [Adapted from D. L. Vaux and S. J. Korsemeyer, 1999,  membrane. It remains unclear how this ion influx triggers the\\nCell96:245.] release of cytochrome c.Once cytochrome cis released into\\n22.5 • Cell Death and Its Regulation 31\\nthe cytosol, it binds to the adapter protein Apaf-1 (the mam- Some Trophic Factors Induce Inactivation \\nmalian homolog of CED-4) and promotes activation of a cas- of a Pro-Apoptotic Regulator\\npase cascade leading to cell death (Figure 22-32a). Bax can\\ncause cell death by a second pathway that triggers mitochon- We saw earlier that neurotrophins such as nerve growth fac-\\ndrial dysfunction independently of any caspase action, at least tor (NGF) protect neurons from cell death. The intracellular\\nin part by stimulating mitochondrial depolarization. signaling pathways linking such survival factors to inactiva-\\nGene knockout experiments have dramatically confirmed tion of the cell-death machinery are quite elaborate. The\\nthe importance of both pro-apoptotic and anti-apoptotic  finding that trophic factors appear to work largely inde-\\nBcl-2 family members in neuronal development. Mice lacking pendent of protein synthesis suggested that these external\\nthe bcl-xlgene, which encodes an anti-apoptotic protein, have signals lead to changes in the activities of preexisting proteins\\nmassive defects in nervous system development with wide- rather than to activation of gene expression. Scientists\\nspread cell death in the spinal cord, dorsal root ganglion, and demonstrated that in the absence of trophic factors, the non-\\nbrain of developing embryos. In contrast, baxknockouts ex- phosphorylated form of Bad is associated with Bcl-2/Bcl-xl at\\nhibit a marked increase in neurons in some regions of the the mitochondrial membrane (see Figure 22-32a). Binding of\\nnervous system. These knockouts also demonstrate that the Bad inhibits the anti-apoptotic function of Bcl-2/Bcl-xl,\\neffects of Bcl-2 family members generally are tissue specific. thereby promoting cell death. Phosphorylated Bad, however,\\n(a)  Absence of trophic factor: Caspase activation (b)  Presence of trophic factor: Inhibition of caspase activation\\nTrophic factor\\nTrophic factor receptor\\nF M\\n1 o\\nc E\\nu\\nPlasma membrane s D\\n A I\\nDeath n A\\nim  \\nCleavage of substrates a C\\ntio O\\nn N\\nPI-3 kinase Procaspase 3 Caspase 3 PI-3 kinase Procaspase 3 : A N\\np\\no E\\np C\\n14-3-3 Caspase 9 to T\\n14-3-3 ATP s\\nAkt FPO P ADP Akt is IO\\n4\\nN\\nBad\\nProcaspase 9 2 Procaspase 9 S\\nApaf 1 Apaf 1\\nCyt c\\n3\\nBad\\n1 Bcl-2 Bcl-xl Bax Bax mOuetmerb mrainteochondrial Bcl-2Bcl-xl BaxBax\\n2 Cyt c 3\\nCyt c Ions\\n▲FIGURE 22-32 Proposed intracellular pathways leading adapter protein Apaf-1 (3), promoting a caspase cascade that\\nto cell death by apoptosis or to trophic factor–mediated cell leads to cell death (4). (b) In some cells, binding of a trophic\\nsurvival in mammalian cells.(a) In the absence of a trophic factor (e.g., NGF) stimulates PI-3 kinase activity, leading to\\nfactor, Bad, a soluble pro-apoptotic protein, binds to the  activation of the downstream kinase Akt, which phosphorylates\\nanti-apoptotic proteins Bcl-2 and Bcl-xl, which are inserted into Bad (1). Phosphorylated Bad then forms a complex with the \\nthe mitochondrial membrane (1). Bad binding prevents the  14-3-3 protein (2). With Bad sequestered in the cytosol, the \\nanti-apoptotic proteins from interacting with Bax, a membrane- anti-apoptotic Bcl-2/Bcl-xl proteins can inhibit the activity of\\nbound pro-apoptotic protein. As a consequence, Bax forms Bax (3), thereby preventing the release of cytochrome cand\\nhomo-oligomeric channels in the membrane that mediate ion flux activation of the caspase cascade. [Adapted from B. Pettman \\n(2). Through an as-yet-unknown mechanism, this leads to the and C. E. Henderson, 1998, Neuron20:633.]\\nrelease of cytochrome cinto the cytosol, where it binds to the\\n32 CHAPTER 22 • Cell Birth, Lineage, And Death\\ncannot bind to Bcl-2/Bcl-xl and is found in the cytosol com- ■ Genetic studies in C. elegansdefined an evolutionarily\\nplexed to the phosphoserine-binding protein 14-3-3. Hence, conserved apoptotic pathway with three major compo-\\nsignaling pathways leading to Bad phosphorylation would be nents: regulatory proteins, adapter proteins, and effector\\nparticularly attractive candidates for transmitting survival proteases called caspases in vertebrates (see Figure 22-31). \\nsignals.\\n■ Once activated, apoptotic proteases cleave specific intra-\\nA number of trophic factors including NGF have been\\ncellular substrates leading to the demise of a cell. Adapter\\nshown to trigger the PI-3 kinase signaling pathway, leading\\nproteins (e.g., Apaf-1), which bind both regulatory proteins\\nto activation of a downstream kinase called PKB (see Figure\\nand caspases, are required for caspase activation. \\n14-27). Activated PKB  phosphorylates Bad at sites known to\\ninhibit its pro-apoptotic activity. Moreover, a constitutively ■ Pro-apoptotic regulator proteins (e.g., Bax, Bad) pro-\\nactive form of PKB can rescue cultured neurotrophin- mote  caspase  activation,  and  anti-apoptotic  regulators\\ndeprived neurons, which otherwise would undergo apoptosis (e.g., Bcl-2) suppress activation. Direct interactions be-\\nand die. These findings support the mechanism for the sur- tween pro-apoptotic and anti-apoptotic proteins lead to\\nvival action of trophic factors depicted in Figure 22-32b. In cell death in the absence of trophic factors. Binding of ex-\\nother cell types, different trophic factors may promote cell tracellular trophic factors can trigger changes in these in-\\nsurvival through post-translational modification of other teractions, resulting in cell survival (see Figure 22-32).\\ncomponents of the cell-death machinery. \\n■ The Bcl2 family contains both pro-apoptotic and anti-\\napoptotic proteins; all are single-pass transmembrane pro-\\nTumor Necrosis Factor and Related Death  teins and engage in protein-protein interactions.\\nSignals Promote Cell Murder by Caspases  ■ Binding of extracellular death signals, such as tumor\\nnecrosis factor and Fas ligand, to their receptors activates\\nAlthough cell death can arise as a default in the absence of\\nan associated protein (FADD) that in turn triggers the cas-\\nsurvival factors, apoptosis can also be stimulated by posi-\\npase cascade leading to cell murder. \\ntively acting “death” signals.For instance, tumor necrosis\\nfactor (TNF),which is released by macrophages, triggers the\\ncell death and tissue destruction seen in certain chronic in-\\nflammatory diseases. Another important death-inducing sig- PERSPECTIVES FOR THE FUTURE\\nnal, the Fas ligand,is a cell-surface protein expressed by\\nactivated natural killer cells and cytotoxic T lymphocytes. Cell birth, lineage, and death lie at the heart of the growth\\nThis signal can trigger death of virus-infected cells, some of an organism and are also central to disease processes,\\ntumor cells, and foreign graft cells.  most notably cancer. Few transformations seem more \\nBoth  TNF  and  Fas  ligand  act  through  cell-surface remarkable than the blooming of cell types during develop-\\n“death” receptors that have a single transmembrane domain ment. A lineage beginning with a fertilized egg, a “plain\\nand are activated when ligand binding brings three receptor vanilla” 200-$m sphere, produces neurons a yard long,\\nmolecules into close proximity. The trimeric receptor com- pulsating multinucleate muscle cells, exquisitely light-\\nplex attracts a protein called FADD (Fas-associated death sensitive retina cells, ravenous macrophages that recognize\\ndomain), which serves as an adapter to recruit and in some and engulf germs, and all the hundreds of other cell types.\\nway activate caspase-8, an initiator caspase, in cells receiving Regulators of cell lineage produce this rich variety by \\na death signal. The death domain found in FADD is a se- controlling two critical decisions: (1) when and where to\\nquence that is present in a number of proteins involved in activate the cell division cycle (Chapter 21) and (2) whether\\napoptosis. Once activated, caspase-8 activates other caspases the two daughter cells will be the same or different. A \\nand the amplification cascade begins. To test the ability of cell may be just like its parent, or it may embark on a new\\nthe Fas receptor to induce cell death, researchers incubated path. \\ncells with antibodies against the receptor. These antibodies, Cell birth is normally carefully restricted to specific lo-\\nwhich bind and cross-link Fas receptors, were found to stim- cales and times, such as the basal layer of the skin or the root\\nulate cell death, indicating that activation of the Fas recep- meristem. Liver regenerates when there is injury, but liver\\ntor is sufficient to trigger apoptosis.  cancer is prevented by restricting unnecessary growth at\\nother times. Cell lineage is patterned by the asymmetric dis-\\ntribution of key regulators to the daughter cells of a division.\\nKEY CONCEPTS OF SECTION 22.5 Some of these regulators are intrinsic to the parent cell, be-\\ncoming asymmetrically distributed during polarization of the\\nCell Death and Its Regulation\\ncell; other regulators are external signals that differentially\\n■ All cells require trophic factors to prevent apoptosis and reach the daughter cells. Asymmetry of cells becomes asym-\\nthus survive. In the absence of these factors, cells commit metry of tissues and whole organisms. Our left and right\\nsuicide.  hands differ only as a result of cell asymmetry. \\nReview the Concepts 33\\nSome cells persist for the life of the organism, but  embryonic stem (ES)  mesoderm 000\\nothers such as blood and intestinal cells turn over rapidly. cells 000 micro RNAs \\nMany cells live for awhile and are then programmed to die endoderm 000 (miRNAs) 000\\nand be replaced by others arising from a stem-cell popula-\\nganglion mother  muscle regulatory \\ntion. Programmed cell death is also the basis for the metic-\\ncell (GMC) 000 factors (MRFs) 000\\nulous elimination of potentially harmful cells, such as\\ngerm line 000 neurotrophins 000\\nautoreactive immune cells, which attack the body’s own\\nheterochronic mutants 000 pluripotent 000\\ncells, or neurons that have failed to properly connect. Cell-\\ndeath programs have also evolved as a defense against in- MATlocus 000 precursor/progenitor \\nfection, and virus-infected cells are selectively murdered in mating factor 000 cells 000\\nresponse to death signals. Viruses, in turn, devote much of mating-type  somatic cells 000\\ntheir effort to evading host defenses. For example, p53, a switching 000 stem cells 000\\ntranscription factor that senses cell stresses and damage\\nmeristems 000 trophic factors 000\\nand activates transcription of pro-apoptotic members of the\\nbcl-2gene family, is inhibited by the adenovirus E1B pro-\\ntein. It has been estimated that about a third of the aden-\\nREVIEW THE CONCEPTS\\novirus genome is directed at evading host defenses. Cell\\ndeath is relevant to toxic chemicals as well as viral infec-\\n1. What two properties define a stem cell? Distinguish be-\\ntions; malformations due to poisons often originate from\\ntween a totipotential stem cell, a pluripotent stem cell, and\\nexcess apoptosis.\\na progenitor cell.\\nFailures of programmed cell death can lead to uncon-\\ntrolled cancerous growth (Chapter 23). The proteins that re- 2. Where are stem cells located in plants? Where are stem\\nstrain the death of cancer cells therefore become possible cells located in adult animals? How does the concept of stem\\ntargets for drugs. A tumor may contain a mixture of cells, cell differ between animal and plant systems?\\nsome capable of seeding new tumors or continued uncon-\\n3. In 1997, Dolly the sheep was cloned by a technique\\ntrolled growth, and some capable only of growing in place or\\ncalled somatic cell nuclear transfer. A nucleus from an\\nfor a limited time. In this sense the tumor has its own stem\\nadult mammary cell was transferred into an egg from\\ncells, and they must be found and studied, so they become\\nwhich the nucleus had been removed. The egg was allowed\\nvulnerable to our medicine. One option is to manipulate the\\nto divide several times in culture, then the embryo was\\ncell-death pathway to our own advantage, to send the signals\\ntransferred to a surrogate mother who gave birth to \\nthat will make cancer cells destroy themselves.\\nDolly. Dolly died in 2003 after mating and giving birth\\nMuch attention is now being given to the regulation of\\nherself to viable offspring. What does the creation of Dolly\\nstem cells in an effort to understand how dividing popula-\\ntell us about the potential of nuclear material derived from\\ntions of cells are created and maintained. This has clear im-\\na fully differentiated adult cell? Does the creation of Dolly\\nplications for repair of tissue, for example, to help damaged\\ntell us anything about the potential of an intact, fully dif-\\neyes, torn cartilage, degenerating brain tissue, or failing or-\\nferentiated adult cell? Name three types of information\\ngans. One interesting possibility is that some populations of\\nthat function to preserve cell type. Which of these types\\nstem cells with the potential to generate or regenerate tissue\\nof information was shown to be reversible by the Dolly \\nare normally eliminated by cell death during later develop-\\nexperiment?\\nment. If so, finding ways to selectively block the death of the\\ncells could make regeneration more likely. Could the elimi-\\n4. The roundworm C. eleganshas proven to be a valuable\\nnation of such cells during mammalian development be the\\nmodel organism for studies of cell birth, cell lineage, and cell\\ndifference between an amphibian capable of limb regenera-\\ndeath. What properties of C. elegansrender it so well suited\\ntion and a mammal that is not?\\nfor these studies? Why is so much information from C. elegans\\nexperiments of use to investigators interested in mammalian\\ndevelopment?\\nKEY TERMS\\n5. In the budding yeast S. cerevisiae,what is the role of the\\nMCM1 protein in the following:\\napical complex 000 caspases 000\\napoptosis 000 cell lineage 000 a. transcription of a-specific genes in acells\\nasymmetric cell  death signals 000 b. blocking transcription of \"-specific genes in acells\\ndivision 000 determination 000\\nc. transcription of \"-specific genes in \"cells\\n!-catenin 000 differentiation 000\\nBcl-2 family 000 ectoderm 000 d. blocking transcription of a-specific genes in \"cells\\n34 CHAPTER 22 • Cell Birth, Lineage, And Death\\n6. In S. cerevisiae, what ensures that a and \" cells mate One common characteristic of cancer cells is a loss of func-\\nwith one another rather than with cells of the same mating tion in the apoptotic pathway. Which of the mutations listed\\ntype (i.e., awith aor \"with \")? might you expect to find in some cancer cells?\\n7. Exposure of C3H 10T1/2 cells to 5-azacytidine, a nu-\\ncleotide analog, is a model system for muscle differentiation.\\nANALYZE THE DATA\\nHow was 5-azacytidine treatment used to isolate the genes\\ninvolved in muscle differentiation? \\nTo better understand the potential of adult stem cells to dif-\\n8. Through the experiments on C3H 10T1/2 cells treated ferentiate into various cell types, the following studies were\\nwith 5-azacytidine, MyoD was identified as a key transcrip- performed.\\ntion factor in regulating the differentiation of muscle. To Bone marrow was harvested from adult male mice and\\nwhat general class of DNA-binding proteins does MyoD be- then transplanted into isogeneic female recipient mice that\\nlong? How do the interactions of MyoD with the following had been treated with a level of irradiation sufficient to\\nproteins affect its function? (a) E2A, (b) MEFs, (c) Id destroy their own hematopoetic stem cells (see the figure).\\nAlthough the dose of irradiation given was lethal to mice\\n9. The mechanisms that regulate muscle differentiation in\\nthat did not receive a transplant, the majority of the re-\\nmammals and neural differentiation in Drosophila (and\\ncipient mice receiving the transplant survived. After 4\\nprobably mammals as well) bear remarkable similarities.\\nweeks, peripheral blood from the recipient mice was ana-\\nWhat proteins function analogous to MyoD, myogenin, Id,\\nlyzed. The composition of the blood was normal with re-\\nand E2A in neural cell differentiation in Drosophila? Based\\nspect to all blood cell types. Every blood cell examined\\non these analogies, predict the effect of microinjection of\\nwas determined to be positive for the presence of the Y\\nMyoD mRNA on the development of Xenopusembryos.\\nchromosome.\\n10. Predict the effect of the following mutations on the abil-\\nity of mother and daughter cells of S. cerevisaeto undergo male mouse\\nmating type switching following cell division: \\na. loss-of-function mutation in the HO endonuclease Harvest marrow\\nb. gain-of-function  mutation  that  renders  HO  endo-\\nnuclease gene constitutively expressed independent of Isogeneic female mouse Bone\\nSWI/SNF marrow\\ntransplant\\nc. gain-of-function mutation in SWI/SNF that renders it in-\\nLethal irradiation\\nsensitive to Ash1\\nFPO\\n11. Asymmetric cell division often relies on cytoskeletal el-\\nements to generate or maintain asymmetric distribution of\\ncellular factors. In S. cerevisiae,what factor is localized to\\n\"labeled\" mouse\\nthe bud by myosin motors? In Drosophilaneuroblasts, what\\nfactors are localized apically by microtubules?\\n12. How do studies of brain development in knockout mice\\nsupport the statement that apoptosis is a default pathway in\\nneuronal cells?\\na. What was the purpose of using male mice as the bone\\n13. What morphologic features distinguish programmed cell marrow donors and female mice as the recipients? Would the\\ndeath and necrotic cell death? TNF and Fas ligand bind cell converse experiment (female donors, male recipients) have\\nsurface receptors to trigger cell death. Although the death worked? \\nsignal is generated external to the cell, why do we consider\\nb. What was the purpose of irradiating the recipient mice\\nthe death induced by these molecules to be apoptotic rather\\nprior to transplantation? What outcome might you expect if\\nthan necrotic?\\nbone marrow cells were transplanted to nonirradiated mice? \\n14. Predict the effects of the following mutations on the\\nc. What method could have been used to purify hematopo-\\nability of a cell to undergo apoptosis:\\netic stem cells from bone marrow before transplanting these\\na. mutation in Bad such that it cannot be phosphorylated by cells into the recipient mouse? How might a purification step\\nAkt have affected the outcome of the experiment?\\nb. overexpression of Bcl-2\\nAfter 8 weeks, some of the recipient mice were sacrificed and\\nc. mutation in Bax such that it cannot form homodimers histochemical analysis was performed on tissue sections of'"
            ]
          },
          "metadata": {},
          "execution_count": 54
        }
      ]
    }
  ]
}